<div>
<svg id="cell" preserveaspectratio="xMidYMin slice" style="width:100%; padding-bottom:80%; height:1px; overflow:visible" version="1.1" viewbox="0 0 613 490" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g class="col cytosol cytoplasmic_bodies" descendants="level 0: ['cytosol, pval = 5.4e-05']
level 1: ['ER membrane insertion complex, pval = --', 'FMN reductase complex, pval = --', 'Yae1-Lto1 complex, pval = --', 'taurine dioxygenase complex, pval = --', 'SsuD-SsuE complex, pval = --', 'alkanesulfonate monooxygenase complex, pval = --', 'intramolecular phosphotransferase complex, pval = --', 'diamine N-acetyltransferase complex, pval = --', 'iron-sulfur cluster transfer complex, pval = --', 'IkappaB kinase complex, pval = 0.40992', 'L-cysteine desulfurase complex, pval = --', 'iron-sulfur cluster assembly complex, pval = --', 'sulfurtransferase complex, pval = --', 'H-NS-Hha complex, pval = --', '3-isopropylmalate dehydratase complex, pval = --', 'molybdopterin cofactor (Moco) biosynthesis adenylyltransferase complex, pval = --', 'cytosolic valyl-tRNA aminoacylation, pval = --', 'ripoptosome, pval = 0.68643', 'H-NS complex, pval = --', 'apoptosome, pval = --', 'BORC complex, pval = --', 'CIA complex, pval = --', 'H-NS-Cnu complex, pval = --', 'bacterial degradosome, pval = --', 'ModE complex, pval = --', 'cytosolic tRNA wobble base thiouridylase complex, pval = --', '6-phosphofructokinase complex, pval = --', 'hemoglobin complex, pval = --', 'fatty-acyl-CoA synthase complex, pval = --', 'fatty acid synthase complex, pval = --', 'ribonucleoside-diphosphate reductase complex, pval = --', 'chaperonin-containing T-complex, pval = --', 'steroid hormone aporeceptor complex, pval = --', 'guanylate cyclase complex, soluble, pval = --', 'BLOC complex, pval = --', 'cytosolic creatine kinase complex, pval = --', 'cytosolic aryl hydrocarbon receptor complex, pval = --', 'region of cytosol, pval = 0.82446', 'protein glycosylation in cytosol, pval = --', 'calcium ion binding involved in regulation of cytosolic calcium ion concentration, pval = --', 'cytosolic ribosome, pval = 0.68643', 'TSC1-TSC2 complex, pval = --', 'cytoplasmic translational termination, pval = --', 'Las1 complex, pval = --', 'cytosolic DNA-directed RNA polymerase complex, pval = --', 'cytoplasmic RNA cap binding complex, pval = --', 'GLI-SUFU complex, pval = --', 'inflammasome complex, pval = --', 'mCRD-mediated mRNA stability complex, pval = --', 'chaperonin ATPase complex, pval = --', 'cytosolic lipolysis, pval = --', 'cytosolic pyruvate dehydrogenase complex, pval = --', '6-phosphofructo-2-kinase/fructose-2,6-biphosphatase complex, pval = --', 'cytosolic oxoglutarate dehydrogenase complex, pval = --', 'cytosol pyruvate dehydrogenase (lipoamide) phosphatase complex, pval = --', 'cytosolic tricarboxylic acid cycle enzyme complex, pval = --', 'cytosolic electron transfer flavoprotein complex, pval = --', 'phosphopyruvate hydratase complex, pval = --', 'cytosolic transport, pval = --', 'FtsBL complex, pval = --', 'cytosolic proteasome complex, pval = --']
level 2: ['BAT3 complex, pval = --', 'TRC complex, pval = --', 'phosphomannomutase complex, pval = --', 'IscS-TusA complex, pval = --', 'IscS-IscU complex, pval = --', 'Nbp35-Cfd1 ATPase complex, pval = --', 'MPT synthase complex, pval = --', 'thiazole synthase complex, pval = --', 'zona pellucida receptor complex, pval = --', 'BLOC-2 complex, pval = --', 'BLOC-1 complex, pval = --', 'BLOC-3 complex, pval = --', 'postsynaptic cytosol, pval = 0.90208', 'presynaptic cytosol, pval = 0.82468', 'protein galactosylation in cytosol, pval = --', 'calcium ion binding involved in regulation of presynaptic cytosolic calcium ion concentration, pval = --', 'calcium ion binding involved in regulation of postsynaptic cytosolic calcium ion concentration, pval = --', 'cytosolic small ribosomal subunit, pval = 0.43974', 'cytosolic large ribosomal subunit, pval = --', 'regulation of cytoplasmic translational termination, pval = --', 'AIM2 inflammasome complex, pval = --', 'IPAF inflammasome complex, pval = --', 'NLRP1 inflammasome complex, pval = --', 'NLRP3 inflammasome complex, pval = --', 'GroEL-GroES complex, pval = --', 'cytosolic alpha-ketoglutarate dehydrogenase complex, pval = --', 'cytosolic isocitrate dehydrogenase complex (NAD+), pval = --', 'cytoplasmic transport, nurse cell to oocyte, pval = --', 'positive regulation of cytoplasmic transport, pval = --', 'negative regulation of cytoplasmic transport, pval = --', 'regulation of cytoplasmic transport, pval = --', 'early endosome to recycling endosome transport, pval = --', 'early endosome to late endosome transport, pval = --', 'Golgi to endosome transport, pval = --', 'retrograde transport, endosome to Golgi, pval = --', 'plastid to vacuole vesicle-mediated transport, pval = --', 'dense core granule transport, pval = --', 'vesicle-mediated cholesterol transport, pval = --', 'Golgi to lysosome transport, pval = --', 'cytoplasmic streaming, pval = --', 'HslUV protease complex, pval = --', 'proteasome storage granule, pval = --', 'cytosolic proteasome regulatory particle, pval = --', 'cytosolic proteasome core complex, pval = --']
level 3: ['eukaryotic 48S preinitiation complex, pval = --', 'eukaryotic 43S preinitiation complex, pval = --', 'cytoskeleton-dependent cytoplasmic transport, nurse cell to oocyte, pval = --', 'positive regulation of dense core granule transport, pval = --', 'positive regulation of retrograde transport, endosome to Golgi, pval = --', 'positive regulation of early endosome to recycling endosome transport, pval = --', 'positive regulation of early endosome to late endosome transport, pval = --', 'negative regulation of dense core granule transport, pval = --', 'negative regulation of retrograde transport, endosome to Golgi, pval = --', 'negative regulation of early endosome to late endosome transport, pval = --', 'regulation of dense core granule transport, pval = --', 'modulation by virus of host cytoplasmic transport, pval = --', 'regulation of retrograde transport, endosome to Golgi, pval = --', 'regulation of early endosome to recycling endosome transport, pval = --', 'regulation of early endosome to late endosome transport, pval = --', 'vesicle targeting, trans-Golgi to endosome, pval = --', 'recycling endosome to Golgi transport, pval = --', 'late endosome to Golgi transport, pval = --', 'early endosome to Golgi transport, pval = --', 'dense core granule cytoskeletal transport, pval = --', 'cholesterol ester hydrolysis involved in cholesterol transport, pval = --', 'endosome to lysosome transport of low-density lipoprotein particle, pval = --', 'lysosome to ER cholesterol transport, pval = --', 'sequestration of proteasome core complex in proteasome storage granule, pval = --', 'cytosolic proteasome regulatory particle, base subcomplex, pval = --', 'cytosolic proteasome regulatory particle, lid subcomplex, pval = --', 'cytosolic proteasome core complex, alpha-subunit complex, pval = --', 'cytosolic proteasome core complex, beta-subunit complex, pval = --']
level 4: ['positive regulation of anterograde dense core granule transport, pval = --', 'positive regulation of retrograde dense core granule transport, pval = --', 'negative regulation of anterograde dense core granule transport, pval = --', 'negative regulation of retrograde dense core granule transport, pval = --', 'regulation of anterograde dense core granule transport, pval = --', 'regulation of retrograde dense core granule transport, pval = --', 'clathrin coating of Golgi vesicle, trans-Golgi to endosome targeting, pval = --', 'retrograde neuronal dense core vesicle transport, pval = --', 'anterograde neuronal dense core vesicle transport, pval = --']" id="cytosol" init-pval="5.4e-05" min-pval="5.4e-05" min-pval-children="0.40992" title="Cytosol">
<g>
<path d="M48.7,227.2c0,0-0.1,0.1-0.2,0.3c-4.6,7.6-9,15.5-13.4,24.3c-18.9,37.4-27.1,55.5-26,92.3
			c1,32.4,8.7,41.2,23.9,58.7c1.9,2.2,3.9,4.5,6,7c12.6,14.9,40.4,21.9,69.8,29.3c11.5,2.9,23.3,5.9,35,9.5
			c32.6,10,64.6,14.1,98.6,18.3c9.9,1.2,20.2,2.5,30.6,4c6.7,0.9,13.5,2,20.6,3.2c21.3,3.4,43.3,6.9,66.4,6.9
			c23.9,0,45.3-3.9,65.6-11.8c34.9-13.6,60.8-20.9,84.3-27.2c0.2-0.1,0.8-0.3,0.9-0.6c0.2-0.4-0.1-1.2-0.7-2c-0.8-1.2-1-2.7-0.4-4.2
			c0.7-1.9,2.5-3.4,4.8-4c0.9-0.2,1.8-0.4,2.7-0.4c3.3,0,5.6,1.8,5.6,4.5c0,1.4,0.2,2.1,0.4,2.5c0.2,0.4,0.4,0.5,0.6,0.5
			c0.1,0,0.3,0,0.4-0.1c9.6-2.6,20-5.4,31-8.7c38.7-11.6,40.6-32.2,44.8-77.5c0.3-3.6,0.7-7.5,1.1-11.4c3.2-32.9-2.7-53.7-8.5-73.8
			c-3.6-12.6-7-24.6-7.8-38.6c-0.9-15.9-1.6-25.4-2.7-33.7c-0.1-0.9-0.4-1.8-1-1.8c-0.2,0-0.7,0.1-1.4,0.7c-1.6,1.3-3.1,2-4.5,2
			c-3.3,0-5.2-3.4-5.6-6.5c-0.4-2.8,0.1-5.1,1.5-6.6c1.5-1.7,3.8-2.2,6.3-1.4c0.8,0.3,1.6,0.2,1.9-0.2c0.2-0.2,0.2-0.7,0.1-1.3
			c-1.3-5.3-3.2-11.3-6.1-20c-8.7-25.5-10.3-35-15.6-77.1c-2.6-20.9-11.9-34.1-30.3-42.9c-18.5-8.9-46.8-13.8-91.9-15.8
			c-23.8-1.1-41-2.7-56.1-4.1c-4.1-0.4-8.1-0.8-12-1.1c-0.5,0-0.6,0.1-0.7,0.1c-0.1,0.1-0.4,0.6-0.3,1.9c0.1,1.3-0.4,2.5-1.2,3.4
			c-0.7,0.8-2.1,1.7-4.6,1.7c-0.7,0-1.5-0.1-2.4-0.2c-3.4-0.6-5-2.7-4.6-6.1c0-0.3,0.1-0.5,0.1-0.8c0.1-0.3,0.1-0.7,0.1-0.9
			c0,0-0.3-0.2-1.6-0.2c-12.7-0.7-23.8-0.7-36.5,0c-29,1.6-54.3,8.7-76.6,14.9c-9.7,2.7-18.8,5.3-27.4,7.1
			c-28,6.1-46.9,15.4-72.7,49.5c-17.2,22.7-24.1,34.6-32.8,49.7c-4.5,7.9-9.7,16.8-17.1,28.6c-8.3,13.2-15.2,23-21.9,32.5
			c-4.3,6.1-8.4,11.9-12.6,18.2c-0.3,0.4-0.3,0.5-0.3,0.5c0.1,0,0.5,0.5,3,1.1c1.8,0.5,3.2,1.7,3.7,3.3c0.7,1.9,0.2,4.2-1.3,6.2
			c-1.3,1.7-3.1,2.7-5,2.7c-1.8,0-3.3-0.8-4.3-2.3C49.6,227.6,48.9,227.2,48.7,227.2z" style="fill:#ff00ff;opacity:0.1;"></path>
</g>
</g>
<g class="ton cytosol cytoplasmic_bodies" descendants="level 0: ['cytosol, pval = 5.4e-05']
level 1: ['ER membrane insertion complex, pval = --', 'FMN reductase complex, pval = --', 'Yae1-Lto1 complex, pval = --', 'taurine dioxygenase complex, pval = --', 'SsuD-SsuE complex, pval = --', 'alkanesulfonate monooxygenase complex, pval = --', 'intramolecular phosphotransferase complex, pval = --', 'diamine N-acetyltransferase complex, pval = --', 'iron-sulfur cluster transfer complex, pval = --', 'IkappaB kinase complex, pval = 0.40992', 'L-cysteine desulfurase complex, pval = --', 'iron-sulfur cluster assembly complex, pval = --', 'sulfurtransferase complex, pval = --', 'H-NS-Hha complex, pval = --', '3-isopropylmalate dehydratase complex, pval = --', 'molybdopterin cofactor (Moco) biosynthesis adenylyltransferase complex, pval = --', 'cytosolic valyl-tRNA aminoacylation, pval = --', 'ripoptosome, pval = 0.68643', 'H-NS complex, pval = --', 'apoptosome, pval = --', 'BORC complex, pval = --', 'CIA complex, pval = --', 'H-NS-Cnu complex, pval = --', 'bacterial degradosome, pval = --', 'ModE complex, pval = --', 'cytosolic tRNA wobble base thiouridylase complex, pval = --', '6-phosphofructokinase complex, pval = --', 'hemoglobin complex, pval = --', 'fatty-acyl-CoA synthase complex, pval = --', 'fatty acid synthase complex, pval = --', 'ribonucleoside-diphosphate reductase complex, pval = --', 'chaperonin-containing T-complex, pval = --', 'steroid hormone aporeceptor complex, pval = --', 'guanylate cyclase complex, soluble, pval = --', 'BLOC complex, pval = --', 'cytosolic creatine kinase complex, pval = --', 'cytosolic aryl hydrocarbon receptor complex, pval = --', 'region of cytosol, pval = 0.82446', 'protein glycosylation in cytosol, pval = --', 'calcium ion binding involved in regulation of cytosolic calcium ion concentration, pval = --', 'cytosolic ribosome, pval = 0.68643', 'TSC1-TSC2 complex, pval = --', 'cytoplasmic translational termination, pval = --', 'Las1 complex, pval = --', 'cytosolic DNA-directed RNA polymerase complex, pval = --', 'cytoplasmic RNA cap binding complex, pval = --', 'GLI-SUFU complex, pval = --', 'inflammasome complex, pval = --', 'mCRD-mediated mRNA stability complex, pval = --', 'chaperonin ATPase complex, pval = --', 'cytosolic lipolysis, pval = --', 'cytosolic pyruvate dehydrogenase complex, pval = --', '6-phosphofructo-2-kinase/fructose-2,6-biphosphatase complex, pval = --', 'cytosolic oxoglutarate dehydrogenase complex, pval = --', 'cytosol pyruvate dehydrogenase (lipoamide) phosphatase complex, pval = --', 'cytosolic tricarboxylic acid cycle enzyme complex, pval = --', 'cytosolic electron transfer flavoprotein complex, pval = --', 'phosphopyruvate hydratase complex, pval = --', 'cytosolic transport, pval = --', 'FtsBL complex, pval = --', 'cytosolic proteasome complex, pval = --']
level 2: ['BAT3 complex, pval = --', 'TRC complex, pval = --', 'phosphomannomutase complex, pval = --', 'IscS-TusA complex, pval = --', 'IscS-IscU complex, pval = --', 'Nbp35-Cfd1 ATPase complex, pval = --', 'MPT synthase complex, pval = --', 'thiazole synthase complex, pval = --', 'zona pellucida receptor complex, pval = --', 'BLOC-2 complex, pval = --', 'BLOC-1 complex, pval = --', 'BLOC-3 complex, pval = --', 'postsynaptic cytosol, pval = 0.90208', 'presynaptic cytosol, pval = 0.82468', 'protein galactosylation in cytosol, pval = --', 'calcium ion binding involved in regulation of presynaptic cytosolic calcium ion concentration, pval = --', 'calcium ion binding involved in regulation of postsynaptic cytosolic calcium ion concentration, pval = --', 'cytosolic small ribosomal subunit, pval = 0.43974', 'cytosolic large ribosomal subunit, pval = --', 'regulation of cytoplasmic translational termination, pval = --', 'AIM2 inflammasome complex, pval = --', 'IPAF inflammasome complex, pval = --', 'NLRP1 inflammasome complex, pval = --', 'NLRP3 inflammasome complex, pval = --', 'GroEL-GroES complex, pval = --', 'cytosolic alpha-ketoglutarate dehydrogenase complex, pval = --', 'cytosolic isocitrate dehydrogenase complex (NAD+), pval = --', 'cytoplasmic transport, nurse cell to oocyte, pval = --', 'positive regulation of cytoplasmic transport, pval = --', 'negative regulation of cytoplasmic transport, pval = --', 'regulation of cytoplasmic transport, pval = --', 'early endosome to recycling endosome transport, pval = --', 'early endosome to late endosome transport, pval = --', 'Golgi to endosome transport, pval = --', 'retrograde transport, endosome to Golgi, pval = --', 'plastid to vacuole vesicle-mediated transport, pval = --', 'dense core granule transport, pval = --', 'vesicle-mediated cholesterol transport, pval = --', 'Golgi to lysosome transport, pval = --', 'cytoplasmic streaming, pval = --', 'HslUV protease complex, pval = --', 'proteasome storage granule, pval = --', 'cytosolic proteasome regulatory particle, pval = --', 'cytosolic proteasome core complex, pval = --']
level 3: ['eukaryotic 48S preinitiation complex, pval = --', 'eukaryotic 43S preinitiation complex, pval = --', 'cytoskeleton-dependent cytoplasmic transport, nurse cell to oocyte, pval = --', 'positive regulation of dense core granule transport, pval = --', 'positive regulation of retrograde transport, endosome to Golgi, pval = --', 'positive regulation of early endosome to recycling endosome transport, pval = --', 'positive regulation of early endosome to late endosome transport, pval = --', 'negative regulation of dense core granule transport, pval = --', 'negative regulation of retrograde transport, endosome to Golgi, pval = --', 'negative regulation of early endosome to late endosome transport, pval = --', 'regulation of dense core granule transport, pval = --', 'modulation by virus of host cytoplasmic transport, pval = --', 'regulation of retrograde transport, endosome to Golgi, pval = --', 'regulation of early endosome to recycling endosome transport, pval = --', 'regulation of early endosome to late endosome transport, pval = --', 'vesicle targeting, trans-Golgi to endosome, pval = --', 'recycling endosome to Golgi transport, pval = --', 'late endosome to Golgi transport, pval = --', 'early endosome to Golgi transport, pval = --', 'dense core granule cytoskeletal transport, pval = --', 'cholesterol ester hydrolysis involved in cholesterol transport, pval = --', 'endosome to lysosome transport of low-density lipoprotein particle, pval = --', 'lysosome to ER cholesterol transport, pval = --', 'sequestration of proteasome core complex in proteasome storage granule, pval = --', 'cytosolic proteasome regulatory particle, base subcomplex, pval = --', 'cytosolic proteasome regulatory particle, lid subcomplex, pval = --', 'cytosolic proteasome core complex, alpha-subunit complex, pval = --', 'cytosolic proteasome core complex, beta-subunit complex, pval = --']
level 4: ['positive regulation of anterograde dense core granule transport, pval = --', 'positive regulation of retrograde dense core granule transport, pval = --', 'negative regulation of anterograde dense core granule transport, pval = --', 'negative regulation of retrograde dense core granule transport, pval = --', 'regulation of anterograde dense core granule transport, pval = --', 'regulation of retrograde dense core granule transport, pval = --', 'clathrin coating of Golgi vesicle, trans-Golgi to endosome targeting, pval = --', 'retrograde neuronal dense core vesicle transport, pval = --', 'anterograde neuronal dense core vesicle transport, pval = --']" id="cytosol_ton" init-pval="5.4e-05" min-pval="5.4e-05" min-pval-children="0.40992" title="Cytosol">
<g>
<path d="M48.7,227.2c0,0-0.1,0.1-0.2,0.3c-4.6,7.6-9,15.5-13.4,24.3c-18.9,37.4-27.1,55.5-26,92.3
			c1,32.4,8.7,41.2,23.9,58.7c1.9,2.2,3.9,4.5,6,7c12.6,14.9,40.4,21.9,69.8,29.3c11.5,2.9,23.3,5.9,35,9.5
			c32.6,10,64.6,14.1,98.6,18.3c9.9,1.2,20.2,2.5,30.6,4c6.7,0.9,13.5,2,20.6,3.2c21.3,3.4,43.3,6.9,66.4,6.9
			c23.9,0,45.3-3.9,65.6-11.8c34.9-13.6,60.8-20.9,84.3-27.2c0.2-0.1,0.8-0.3,0.9-0.6c0.2-0.4-0.1-1.2-0.7-2c-0.8-1.2-1-2.7-0.4-4.2
			c0.7-1.9,2.5-3.4,4.8-4c0.9-0.2,1.8-0.4,2.7-0.4c3.3,0,5.6,1.8,5.6,4.5c0,1.4,0.2,2.1,0.4,2.5c0.2,0.4,0.4,0.5,0.6,0.5
			c0.1,0,0.3,0,0.4-0.1c9.6-2.6,20-5.4,31-8.7c38.7-11.6,40.6-32.2,44.8-77.5c0.3-3.6,0.7-7.5,1.1-11.4c3.2-32.9-2.7-53.7-8.5-73.8
			c-3.6-12.6-7-24.6-7.8-38.6c-0.9-15.9-1.6-25.4-2.7-33.7c-0.1-0.9-0.4-1.8-1-1.8c-0.2,0-0.7,0.1-1.4,0.7c-1.6,1.3-3.1,2-4.5,2
			c-3.3,0-5.2-3.4-5.6-6.5c-0.4-2.8,0.1-5.1,1.5-6.6c1.5-1.7,3.8-2.2,6.3-1.4c0.8,0.3,1.6,0.2,1.9-0.2c0.2-0.2,0.2-0.7,0.1-1.3
			c-1.3-5.3-3.2-11.3-6.1-20c-8.7-25.5-10.3-35-15.6-77.1c-2.6-20.9-11.9-34.1-30.3-42.9c-18.5-8.9-46.8-13.8-91.9-15.8
			c-23.8-1.1-41-2.7-56.1-4.1c-4.1-0.4-8.1-0.8-12-1.1c-0.5,0-0.6,0.1-0.7,0.1c-0.1,0.1-0.4,0.6-0.3,1.9c0.1,1.3-0.4,2.5-1.2,3.4
			c-0.7,0.8-2.1,1.7-4.6,1.7c-0.7,0-1.5-0.1-2.4-0.2c-3.4-0.6-5-2.7-4.6-6.1c0-0.3,0.1-0.5,0.1-0.8c0.1-0.3,0.1-0.7,0.1-0.9
			c0,0-0.3-0.2-1.6-0.2c-12.7-0.7-23.8-0.7-36.5,0c-29,1.6-54.3,8.7-76.6,14.9c-9.7,2.7-18.8,5.3-27.4,7.1
			c-28,6.1-46.9,15.4-72.7,49.5c-17.2,22.7-24.1,34.6-32.8,49.7c-4.5,7.9-9.7,16.8-17.1,28.6c-8.3,13.2-15.2,23-21.9,32.5
			c-4.3,6.1-8.4,11.9-12.6,18.2c-0.3,0.4-0.3,0.5-0.3,0.5c0.1,0,0.5,0.5,3,1.1c1.8,0.5,3.2,1.7,3.7,3.3c0.7,1.9,0.2,4.2-1.3,6.2
			c-1.3,1.7-3.1,2.7-5,2.7c-1.8,0-3.3-0.8-4.3-2.3C49.6,227.6,48.9,227.2,48.7,227.2z" style="fill:#ff00ff;"></path>
</g>
</g>
<g class="ton intermediate_filaments" descendants="level 0: ['intermediate filament, pval = --']
level 1: ['lamin filament, pval = --', 'glial filament, pval = --', 'type III intermediate filament, pval = --', 'keratin filament, pval = --', 'neurofilament, pval = --']
level 2: ['neurofilament cytoskeleton, pval = --']" id="intermediate_filaments_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Intermediate filaments">
<path d="M234.6,259.6c-0.1,1.7-0.7,8.3-0.9,10.1
		c-0.2,1.8-5.8,2-5.8,0c0.1-2.9,0.4-8.5,0.6-10.5C228.8,257.2,234.7,257.5,234.6,259.6z" style="fill:#15eaff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col intermediate_filaments" descendants="level 0: ['intermediate filament, pval = --']
level 1: ['lamin filament, pval = --', 'glial filament, pval = --', 'type III intermediate filament, pval = --', 'keratin filament, pval = --', 'neurofilament, pval = --']
level 2: ['neurofilament cytoskeleton, pval = --']" id="intermediate_filaments" init-pval="--" min-pval="--" min-pval-children="--" title="Intermediate filaments">
<path d="M226.8,262.6c-1.7,0-6.9,0-8.4,0
		c-2.7,0-3,6-0.2,6c2,0,7.2,0,8.8,0C229.1,268.7,229.5,262.6,226.8,262.6z" style="fill:#15eaff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton actin_filaments" descendants="level 0: ['actin filament, pval = 0.2377']
level 1: ['actomyosin contractile ring actin filament, pval = --', 'actin filament of cell cortex of cell tip, pval = --', 'actin filament bundle, pval = --', 'filamentous actin, pval = 0.19304', 'actin fusion focus, pval = --', 'actin wave, pval = --', 'actin filament branch point, pval = --', 'actomyosin, actin portion, pval = --', 'actin-based cell projection, pval = 0.1586']
level 2: ['parallel actin filament bundle, pval = --', 'contractile actin filament bundle, pval = --', 'microvillus, pval = 0.36956', 'actin tubule, pval = --', 'actin rod, pval = --', 'actin filament bundle of actin-based cell projection, pval = --', 'cluster of actin-based cell projections, pval = 0.68643', 'filopodium, pval = 0.23999', 'stereocilium, pval = 0.43974']
level 3: ['formin-nucleated actin cable, pval = --', 'stress fiber, pval = --', 'rhabdomere microvillus, pval = --', 'brush border, pval = --', 'Schwann cell microvillus, pval = --', 'microvillus membrane, pval = 0.40992', 'microvillar actin bundle, pval = --', 'cofilin-actin rod, pval = --', 'actin filament bundle of filopodium, pval = --', 'actin filament bundle of stereocilium, pval = --', 'stereocilium bundle, pval = 0.43974', 'growth cone filopodium, pval = 0.43974', 'filopodium membrane, pval = 0.03701', 'dendritic filopodium, pval = --', 'filopodium tip, pval = --', 'stereocilium tip, pval = 0.43974', 'stereocilium membrane, pval = --', 'stereocilium shaft, pval = --', 'stereocilium base, pval = --']
level 4: ['rhabdomere microvillus membrane, pval = --', 'brush border membrane, pval = --', 'stereocilia coupling link, pval = --', 'kinocilium, pval = 0.43974']
level 5: ['stereocilia tip link, pval = --', 'stereocilia ankle link, pval = --', 'kinociliary basal body, pval = 0.43974']
level 6: ['stereocilia tip-link density, pval = --', 'stereocilia ankle link complex, pval = --']
level 7: ['upper tip-link density, pval = --', 'lower tip-link density, pval = --']" id="actin_filaments_ton" init-pval="0.2377" min-pval="0.03701" min-pval-children="0.03701" title="Actin filaments">
<path d="M234.6,259.6c-0.1,1.7-0.7,8.3-0.9,10.1
		c-0.2,1.8-5.8,2-5.8,0c0.1-2.9,0.4-8.5,0.6-10.5C228.8,257.2,234.7,257.5,234.6,259.6z" style="fill:#15eaff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col actin_filaments" descendants="level 0: ['actin filament, pval = 0.2377']
level 1: ['actomyosin contractile ring actin filament, pval = --', 'actin filament of cell cortex of cell tip, pval = --', 'actin filament bundle, pval = --', 'filamentous actin, pval = 0.19304', 'actin fusion focus, pval = --', 'actin wave, pval = --', 'actin filament branch point, pval = --', 'actomyosin, actin portion, pval = --', 'actin-based cell projection, pval = 0.1586']
level 2: ['parallel actin filament bundle, pval = --', 'contractile actin filament bundle, pval = --', 'microvillus, pval = 0.36956', 'actin tubule, pval = --', 'actin rod, pval = --', 'actin filament bundle of actin-based cell projection, pval = --', 'cluster of actin-based cell projections, pval = 0.68643', 'filopodium, pval = 0.23999', 'stereocilium, pval = 0.43974']
level 3: ['formin-nucleated actin cable, pval = --', 'stress fiber, pval = --', 'rhabdomere microvillus, pval = --', 'brush border, pval = --', 'Schwann cell microvillus, pval = --', 'microvillus membrane, pval = 0.40992', 'microvillar actin bundle, pval = --', 'cofilin-actin rod, pval = --', 'actin filament bundle of filopodium, pval = --', 'actin filament bundle of stereocilium, pval = --', 'stereocilium bundle, pval = 0.43974', 'growth cone filopodium, pval = 0.43974', 'filopodium membrane, pval = 0.03701', 'dendritic filopodium, pval = --', 'filopodium tip, pval = --', 'stereocilium tip, pval = 0.43974', 'stereocilium membrane, pval = --', 'stereocilium shaft, pval = --', 'stereocilium base, pval = --']
level 4: ['rhabdomere microvillus membrane, pval = --', 'brush border membrane, pval = --', 'stereocilia coupling link, pval = --', 'kinocilium, pval = 0.43974']
level 5: ['stereocilia tip link, pval = --', 'stereocilia ankle link, pval = --', 'kinociliary basal body, pval = 0.43974']
level 6: ['stereocilia tip-link density, pval = --', 'stereocilia ankle link complex, pval = --']
level 7: ['upper tip-link density, pval = --', 'lower tip-link density, pval = --']" id="actin_filaments" init-pval="0.2377" min-pval="0.03701" min-pval-children="0.03701" title="Actin filaments">
<path d="M226.8,262.6c-1.7,0-6.9,0-8.4,0
		c-2.7,0-3,6-0.2,6c2,0,7.2,0,8.8,0C229.1,268.7,229.5,262.6,226.8,262.6z" style="fill:#15eaff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton focal_adhesion_sites" descendants="level 0: [None]" id="focal_adhesion_sites_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Focal adhesion sites">
<path d="M128,182.5c2.4,7.4,7.3,14.9,14.7,22.2
			c4.8,4.7,10.2,9.2,16.8,13.6c10.5,7.1,22,13,33.2,18.7l0.5,0.2c1.5,0.7,2.9,1.5,4.4,2.2c1.8,0.9,3.6,1.8,5.3,2.7
			c1.8,0.9,3.6,1.8,5.4,2.8c1.8,1,3.5,2,5.3,3.1c0.9,0.5,1.8,1.2,2.6,1.8c0.8,0.6,3.3,1.5,2.6,2.9c-0.6,1.3-2,0-2.7-0.5
			c-1.1-0.9-2.3-1.6-3.5-2.3c-1.6-1-3.3-2-5-3c-1.7-1-3.4-1.9-5.1-2.7c-1.7-0.9-3.4-1.8-5.2-2.6c-0.2-0.1-0.4-0.2-0.6-0.3
			c-1.5-0.7-3-1.5-4.4-2.2l-0.5-0.2c-11.2-5.7-22.8-11.6-33.4-18.8c-6.6-4.5-12.2-9-17.1-13.9c-7.5-7.5-12.6-15.3-15.1-23
			c-0.2-0.5,0.1-1.1,0.6-1.3C127.3,181.7,127.9,182,128,182.5z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M229.8,250.2c1.4-5.4,1.9-11.1,0.1-17.2
			c-3.2-10.9-10.7-20.4-18-29.6c-1.1-1.4-2.3-2.9-3.4-4.3c-21.9-28.1-41.4-56.2-40.8-90.2c0.3-17.7,6.1-44.2,21.3-60.5
			c0.4-0.4,1-0.4,1.4-0.1c0.4,0.4,0.4,1,0,1.4c-14.8,15.9-20.5,41.9-20.8,59.2c-0.6,33.4,18.8,61.1,40.4,89c1.1,1.4,2.2,2.8,3.4,4.3
			c7.4,9.3,15.1,19,18.3,30.3c1.9,6.6,1.4,12.7-0.1,18.4c-0.2,0.6-0.2,1.5-0.9,1.8c-0.5,0.2-1.1-0.1-1.3-0.6
			C229.3,251.5,229.7,250.7,229.8,250.2z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M225.5,250.7c-4.9-21-28.8-37.8-47.3-49.1
			c-13.3-8.1-27.2-17.3-37.6-32.1c-7.9-11.2-13.7-23.7-18.9-35.5c-0.2-0.5,0-1.1,0.5-1.3c0.5-0.2,1.1,0,1.3,0.5
			c5.2,11.7,11,24.1,18.7,35.2c10.2,14.5,23.9,23.5,37,31.6c18.9,11.5,43.3,28.7,48.3,50.5c0.2,0.8,0.8,2.5-0.6,2.6
			C225.8,253,225.7,251.4,225.5,250.7z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M243.7,257c0.4-0.4,0.8-0.7,1.3-1.1
			c11.1-9.7,19.1-22.2,26.7-34.6c-2.6,8.4-6.2,16.3-11.6,23.8c-2.7,3.7-7.8,11-13.9,14.3c-0.7,0.4-2.1,0.9-1.9,1.9
			c0.1,0.9,1,0.7,1.6,0.5c7-3.2,12.9-11.4,15.8-15.6c6.9-9.7,11.1-20,13.7-31c7.6-12.3,15.7-24.7,26.9-33.8
			c6.6-5.4,14.2-9.3,22.2-12.4c-3.3,3.9-6.5,8.2-9.2,12.8c-5.6,9.3-9.3,19.6-12.9,29.5c-0.5,1.5-1.1,3-1.6,4.4
			c-7.8,21.3-18,34.5-32.9,42.7c-6.7,3.7-13.2,5.2-20.6,5.7c-0.8,0-2.3,0-2.4,1.1c-0.1,1.4,1.8,1,2.6,0.9c7.6-0.5,14.4-2,21.4-5.9
			c15.3-8.5,25.8-22,33.8-43.8c0.5-1.5,1.1-3,1.6-4.4c3.6-9.8,7.3-20,12.7-29.1c3.4-5.6,7.3-10.7,11.4-15.3c6.9-2.4,14-4.4,21-6.3
			l0.3-0.1c7.6-2.1,15.4-4.2,22.9-6.1c20.8-5.5,42.3-11.2,63-18.3c27-9.2,55.3-20.1,79.2-37.8c13.7-10.1,23.9-22.9,32.1-40.1
			c0.2-0.5,0-1.1-0.5-1.3c-0.5-0.2-1.1,0-1.3,0.5c-8,16.9-18.1,29.5-31.5,39.4c-23.7,17.5-51.8,28.4-78.7,37.5
			c-20.7,7-42.2,12.7-62.9,18.2c-7.5,2-15.3,4.1-22.9,6.2l-0.3,0.1c-5.9,1.6-11.8,3.2-17.6,5.2c3.4-3.5,6.8-6.8,10-9.7l1-0.9
			c8.8-8.1,18-16.4,26-25.8c3.9-4.6,7.7-9.3,11.4-14c5.9-7.3,11.9-14.9,18.5-21.8c30.6-32.2,69-43.1,105.9-50.1
			c0.5-0.1,0.9-0.6,0.8-1.2c-0.1-0.5-0.6-0.9-1.2-0.8c-37.2,7-75.9,18.1-106.9,50.7c-6.6,7-12.7,14.6-18.6,21.9
			c-3.7,4.6-7.5,9.4-11.4,13.9c-7.9,9.2-17,17.5-25.8,25.6l-1,0.9c-4.1,3.7-8.5,7.9-12.7,12.6c-9.4,3.4-18.4,7.7-26.2,14
			c-10,8.2-17.5,18.8-24.4,29.7c2-10.7,2.9-22.2,3.5-34.6c0.7-12.6,2.2-26,8.7-38.3c5-9.4,12.2-18.3,22.8-27.8
			c4.4-4,9.2-8.1,15.6-13.4c3.6-3,7.5-6.1,11.2-9.1c14.4-11.5,29.3-23.4,39.9-38.4c0.3-0.5,0.2-1.1-0.2-1.4
			c-0.5-0.3-1.1-0.2-1.4,0.2c-10.5,14.8-25.2,26.6-39.5,38c-3.7,3-7.6,6-11.2,9.1c-6.4,5.3-11.2,9.5-15.6,13.5
			c-10.8,9.8-18.1,18.8-23.2,28.4c-6.7,12.6-8.2,26.3-8.9,39.2c-0.8,14.5-1.8,27.6-4.6,39.7c-0.7,1.2-1.4,2.3-2.2,3.5
			c-0.2,0.3-0.4,0.7-0.6,1c-1.8,2.8-3.5,5.7-5.3,8.5c-1.8,2.8-3.7,5.6-5.6,8.4c-1,1.4-1.9,2.7-2.9,4.1c-1,1.3-2.1,2.6-3.2,3.9
			c-2,2.6-4.4,4.9-6.6,7.4c-0.7,0.7-1.4,1.4-2.2,2c-0.8,0.5-1.6,1-2.2,1.6c-0.4,0.4-0.7,0.8-0.8,1.3c0,0,0,0,0,0.1
			C241.3,258,242.8,257.8,243.7,257z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M236.3,313.9c3.6,7,8.6,15.9,15.9,23.4
			c2.2,2.3,4.5,4.5,6.8,6.6c5.1,4.8,9.9,9.4,14,15c16.8,23.1,18.4,48.7,20,75.7l0.2,2.7c0.4,6.8,0.8,13.9,2.7,20.4
			c0.1,0.4,0.5,0.7,1,0.7c0.1,0,0.2,0,0.3,0c0.5-0.2,0.8-0.7,0.7-1.2c-1.9-6.3-2.3-13.2-2.6-19.9l-0.2-2.7
			c-1.6-27.4-3.2-53.2-20.4-76.8c-4.2-5.8-9.3-10.6-14.2-15.3c-2.2-2.1-4.5-4.3-6.7-6.6c-7.1-7.3-12-16.1-15.6-22.9
			c-6-11.7-8.7-21.8-8.4-31.9c0-0.7,0.4-2.9-0.8-2.8c-0.5,0-1,0.5-1.1,0.9c-0.2,0.6-0.1,1.3-0.1,1.9
			C227.3,291.6,230.1,301.9,236.3,313.9z M238.2,285.8c1.4,3,2.8,6.1,4.4,9c4.5,8.1,10.6,15.1,17.1,22.2
			c5.6,6.2,12.1,12.7,20.4,20.5l1.9,1.8c5.3,5,10.7,10.1,15.9,15.2c14.1,13.8,25.8,30.7,37.1,53.3l0.9,1.8c2.5,5.1,5.2,10.4,8,15.5
			c7.3,13.2,17.1,21.4,30.9,25.7c13.2,4.1,27.1,6.5,40.6,8.7c3.3,0.5,6.5,1.1,9.7,1.7c0.1,0,0.1,0,0.2,0c0.5,0,0.9-0.3,1-0.8
			c0.1-0.5-0.3-1.1-0.8-1.2c-3.2-0.6-6.5-1.1-9.8-1.7c-13.4-2.2-27.2-4.6-40.3-8.6c-13.2-4.1-22.7-12-29.7-24.7
			c-2.8-5-5.4-10.3-7.9-15.4l-0.9-1.8c-11.3-22.8-23.2-39.9-37.4-53.8c-5.3-5.2-10.7-10.3-16-15.3l-1.9-1.8
			c-8.3-7.8-14.7-14.3-20.3-20.4c-6.4-7-12.5-13.9-16.8-21.8c-1.6-2.9-3-5.9-4.3-8.9c-0.9-2-1.8-4.1-2.9-6.1
			c-0.4-0.9-0.8-1.8-1.5-2.5c-1.1-1.4-1.8,0.8-1.2,1.7c0.3,0.5,0.6,1,0.8,1.6C236.4,281.6,237.3,283.7,238.2,285.8z M239.5,272.6
			c-0.5,0.3-0.3,1.1,0,1.4c0.4,0.5,1.1,0.7,1.7,0.9c3.4,1.3,6.9,2.7,10.3,4c2.4,0.9,4.9,1.9,7.3,2.8c1.2,0.5,2.3,1,3.5,1.3
			c8.2,3.2,16.3,6.6,24.2,10.5c4.3,7.1,8,14.9,11.5,22.4c3.1,6.7,6.1,13,9.3,18.5c19.6,33,60.9,74.7,146.1,86.6
			c15.3,2.1,30,3.2,44.1,3.2c31.2,0,59.1-5.3,83.2-15.8c0.5-0.2,0.7-0.8,0.5-1.3c-0.2-0.5-0.8-0.7-1.3-0.5
			c-34.7,15.1-77.1,19.3-126.2,12.5c-84.4-11.8-125.2-53.1-144.6-85.7c-3.2-5.4-6.1-11.7-9.2-18.3c-3.1-6.6-6.3-13.5-10-19.9
			c1.6,0.8,3.3,1.7,4.9,2.6c6.3,3.5,12,8.1,17.6,12.5l1.8,1.4c16,12.6,29.3,22.5,41.8,31.2c25.7,17.8,60.5,39.7,101.5,50.7
			c0.1,0,0.2,0,0.3,0c0.4,0,0.8-0.3,1-0.7c0.1-0.5-0.2-1.1-0.7-1.2c-40.7-10.9-75.3-32.7-100.9-50.4c-12.5-8.6-25.7-18.5-41.7-31.1
			l-1.8-1.4c-5.6-4.5-11.5-9.1-17.9-12.7c-2.5-1.4-5.1-2.8-7.7-4.1c-1.5-2.4-3-4.7-4.7-6.9c2.6-0.1,5.1-0.3,7.7-0.6
			c5.6-0.5,11.4-1,17.2-0.6c23.8,1.4,47.2,8.6,68.9,16.2c24.4,8.5,48.5,18.9,71.9,28.9c7.8,3.3,15.8,6.8,23.8,10.1
			c2.4,1,4.9,2.1,7.4,3.1c22,9.4,44.7,19.1,68.1,19.8c0.5,0,1.1,0,1.6,0c16.8,0,28.7-8,38.2-15.9c0.4-0.4,0.5-1,0.1-1.4
			c-0.4-0.4-1-0.5-1.4-0.1c-17,14.1-29,15.7-38.5,15.4c-23-0.7-45.6-10.3-67.4-19.6c-2.5-1.1-5-2.1-7.4-3.2
			c-7.9-3.3-16-6.8-23.7-10.1c-23.4-10-47.6-20.4-72-28.9c-10.4-3.6-21.2-7.2-32.2-10c2.1,0.1,4.1,0.2,6.1,0.2
			c12.2,0,23.7-2.1,34.4-6.4c9.6-3.9,18.8-8.9,27.7-13.7c9.7-5.3,19.7-10.8,30.2-14.7c41.6-15.6,93.3-12.6,138.1,8.2
			c0.5,0.2,1.1,0,1.3-0.5c0.2-0.5,0-1.1-0.5-1.3c-45.3-21-97.5-24-139.6-8.2c-10.7,4-20.8,9.5-30.5,14.8c-8.8,4.8-18,9.8-27.5,13.6
			c-16.6,6.7-35.5,8-56.1,4c-13.7-2.7-26.2-7.4-39.6-12.4L288,271c-5.6-2.1-10.6-2.9-17-2.8c-0.6,0-1,0.5-1,1c0,0.6,0.5,1,1,1
			c6.2-0.1,10.9,0.6,16.2,2.7l1.7,0.6c8.3,3.1,16.8,6.3,25.5,8.9c-2.1-0.2-4.1-0.4-6.2-0.5c-5.9-0.3-11.8,0.2-17.5,0.6
			c-3.1,0.3-6.1,0.5-9,0.6c-6.3-7.7-13.9-13.6-24-15.7c-1.4-0.3-4.9-0.3-8.9-0.3c-0.8,0-1.7-0.1-2.5,0.2c-0.9,0.4-0.4,1.5,0.3,1.8
			c0.9,0.4,2.2,0.1,3.1,0.1c1.2,0,2.3-0.1,3.5-0.1c1.3,0,2.8-0.1,4.1,0.2c9,1.9,16,7.1,21.8,13.8c-5.5,0.1-10.8-0.4-16.1-2.1
			c-1.2-0.5-2.3-0.9-3.5-1.3c-0.2-0.1-0.4-0.1-0.5-0.2c-4.8-2.2-9.4-4.8-14-7.5c-0.7-0.4-1.4-1-2.2-1.2c-0.6-0.2-1.8-0.5-2.2,0.1
			c-0.3,0.3-0.1,0.6-0.1,0.9c0,0.2,0,0.3-0.1,0.4C240.3,272.4,239.9,272.4,239.5,272.6z M272.7,285c2.7,0.2,5.4,0.3,8.1,0.2
			c1.3,1.6,2.5,3.3,3.7,5C280.6,288.4,276.7,286.7,272.7,285z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M239.9,253c6.7-6.9,9.2-15.6,7.6-26.4
			c-0.6-3.8-1.5-7.6-2.4-11.2c-0.8-3.4-1.7-6.9-2.3-10.4c-1.9-11.4-2.2-22.9-0.8-34.4c1.4-12.4,5.1-23.1,11.1-32.6
			c2.2-3.5,4.5-6.9,6.8-10.2c4.9-7.1,9.9-14.5,13.8-22.6c10-21.1,19.9-50.9,9.9-72.7c-0.2-0.5,0-1.1,0.5-1.3c0.5-0.2,1.1,0,1.3,0.5
			c10.3,22.6,0.3,52.9-9.9,74.4c-3.9,8.2-9,15.7-13.9,22.9c-2.2,3.3-4.6,6.7-6.7,10.1c-5.9,9.3-9.5,19.7-10.8,31.8
			c-1.3,11.2-1,22.6,0.8,33.8c0.6,3.4,1.4,6.9,2.2,10.2c0.9,3.7,1.9,7.6,2.4,11.4c1.7,11.4-1.1,20.8-8.3,28.2
			c-0.4,0.4-1.1,1.3-1.8,1.1c-0.5-0.1-0.7-0.8-0.6-1.2C238.9,253.9,239.5,253.5,239.9,253z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M198.8,264.8c-0.7,0.2-1.5,0.4-2.2,0.6
			c-2.7,0.8-5.3,1.7-7.9,2.7c-23.9,9.5-40.7,25.9-57.2,43.2c-18.6,19.6-37.7,37.8-65.6,45.8c-5.9,1.7-11.5,2.8-17.1,3.6
			c-4.4-2.2-8.8-4.5-13.5-6.9c-4.4-2.4-8.9-4.9-13.2-7.3c-2.6-1.5-5.2-3-7.8-4.4c-0.5-0.3-1.1-0.1-1.4,0.4c-0.3,0.5-0.1,1.1,0.4,1.4
			c2.6,1.5,5.2,2.9,7.8,4.4c4.3,2.4,8.8,5,13.2,7.3c3.7,2,7.3,3.9,10.8,5.6c-9.1,1.1-17.9,1.3-26.8,1.4c-0.6,0-1,0.5-1,1
			c0,0.5,0.5,1,1,1c9.9-0.1,19.7-0.4,30-1.8c5.9,2.9,11.8,5.4,17.9,7.5c7.4,2.5,16.6,4.7,26.5,4.7c8.8,0,18-1.7,26.8-6.5
			c14.1-7.6,21.5-20,29.3-33.2l1.7-2.9c11.4-19.1,25.3-33.2,43.7-44.5c1.9-1.2,3.9-2.3,6-3.5c3.8-2.1,7.6-4.3,11-6.8
			c-1.1,1.7-2.2,3.4-3.3,5.2l-0.1,0.2c-4,6.4-7.1,11.7-9.8,16.7c-2.9,5.3-5.6,10.7-8.2,16.1c-4.3,9.1-5.9,18.6-7.4,27.7
			c-0.8,4.8-1.6,9.8-2.8,14.8c-5.8,23.7-25.3,39.3-44.1,54.4l-2.7,2.2c-0.4,0.3-0.5,1-0.2,1.4c0.2,0.2,0.5,0.4,0.8,0.4
			c0.2,0,0.4-0.1,0.6-0.2l2.7-2.2c18.2-14.6,38.8-31.2,44.8-55.5c1.2-5.1,2.1-10.1,2.9-15c1.5-9.4,3-18.3,7.2-27.2
			c2.5-5.3,5.2-10.6,8.1-16c2.7-4.9,5.8-10.2,9.8-16.6l0.1-0.2c1.2-1.9,2.3-3.7,3.6-5.5c1.2-1.8,2.6-3.5,4-5.1
			c0.4-0.5,0.9-0.8,1.3-1.3c0.2-0.2,0.3-0.4,0.4-0.7c0-0.2,0.1-0.7,0-0.9c-0.3-0.5-1-0.7-1.5-0.5c-0.3,0.1-0.6,0.4-0.8,0.6
			c-1.9,1.4-3.5,3.3-5.4,4.7c-2.4,1.9-5,3.5-7.6,5c-1.4,0.8-2.9,1.7-4.3,2.5c-2,1.1-4.1,2.3-6.1,3.5c-18.7,11.4-32.8,25.8-44.4,45.1
			l-1.7,2.9c-7.7,12.9-14.9,25.1-28.5,32.4c-18.1,9.7-38.1,6.3-51.7,1.6c-5.1-1.7-10-3.8-15-6.2c4.7-0.8,9.5-1.8,14.5-3.2
			c28.4-8.1,47.7-26.5,66.5-46.3c17-17.8,32.9-33.4,56.5-42.7c1.8-0.7,3.7-1.4,5.6-2c1.1-0.3,2.1-0.6,3.2-0.9
			c1.9-0.5,3.8-0.9,5.7-1.3c1.8-0.4,3.7-0.4,5.5-0.6c0.6-0.1,1.9-0.3,1.9-1c0-0.7-0.8-1.2-1.4-1.4c-0.4-0.1-0.8-0.1-1.3,0
			c-0.7,0.1-1.4,0.3-2,0.4c-0.1,0-0.2,0-0.3,0.1C203.8,263.9,201.3,264.3,198.8,264.8z M199.6,256.9c-1.6-0.5-3.2-1-4.7-1.5
			c-1.3-0.4-2.5-0.8-3.9-1.1c1.9-0.2,3.8-0.4,5.7-0.5c1.7,0,3.4,0.1,5.1,0.2c1.8,0.1,3.7,0.3,5.5,0.8c1.5,0.4,2.9,1,4.4,1.2
			c0.6,0.1,1.6-0.1,1.4-0.9c-0.1-0.5-0.6-1-1.1-1.2c-0.8-0.4-1.7-0.4-2.5-0.6c-1.2-0.3-2.5-0.5-3.7-0.7c-2.8-0.5-5.7-0.8-8.6-0.7
			c-1.8,0-3.5,0.2-5.3,0.4c-0.9,0.1-1.8,0.2-2.7,0.4c-0.4,0.1-0.9,0.1-1.3,0.2c-0.2,0-0.5,0.1-0.7,0.1c-0.2,0-0.5,0.1-0.6,0.1
			c-12-3.3-25.6-6.1-39-3.6c-10,1.9-18.9,6.6-27.6,11.3c-3.5,1.9-7,3.7-10.6,5.4c-21.1-2.4-39.6-8.5-60.6-15.3
			c-2.1-0.7-4.3-1.4-6.4-2.1c-0.5-0.2-1.1,0.1-1.3,0.6c-0.2,0.5,0.1,1.1,0.6,1.3c2.2,0.7,4.3,1.4,6.4,2.1
			c19.9,6.5,37.7,12.3,57.7,15c-1.1,0.5-2.2,0.9-3.3,1.3c-19.7,7.4-40.6,7.3-60.3,6c-0.5,0-1,0.4-1.1,0.9c0,0.6,0.4,1,0.9,1.1
			c6,0.4,12.1,0.7,18.2,0.7c14.3,0,28.9-1.5,42.9-6.8c2.2-0.8,4.3-1.7,6.4-2.7c4.4,0.5,8.8,0.8,13.4,1c1.1,0,2.2,0.1,3.3,0.1
			c14.9,0,29.7-3.3,48.6-10.6c3.9-1.5,7.7-2.7,11.4-3.5c2.7,0.8,5.3,1.5,7.8,2.3c1.5,0.4,3,0.9,4.6,1.5c1.4,0.4,2.7,0.9,4.1,1.3
			c1.1,0.3,2.2,0.8,3.4,1.2c0.7,0.2,2.2,0.5,2.5-0.5c0.4-1.3-1.5-1.5-2.3-1.7c-0.6-0.2-1.2-0.4-1.8-0.5
			C203.1,258,201.3,257.5,199.6,256.9z M182.5,254c-2.6,0.7-5.3,1.6-8.1,2.7c-20.1,7.8-35.4,10.9-51.2,10.5
			c-3.4-0.1-6.8-0.3-10.1-0.6c2.6-1.3,5.3-2.7,7.8-4.1c8.5-4.6,17.3-9.3,27-11.1C159.7,249.2,171.7,251.2,182.5,254z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M515.8,168.5c-22.9,7-47.1,10.2-70.6,13.3
			c-3.6,0.5-7.2,0.9-10.8,1.4c-5.4,0.7-10.9,1.4-16.7,2.1c-37.6,4.5-80.1,9.6-107.5,36.9c-5.3,5.2-10.2,11.4-15,17.4
			c-8.8,11.1-18,22.5-30.1,29.5c-0.2,0.1-0.3,0.1-0.5,0.1c-0.3,0-0.7-0.2-0.9-0.5c-0.3-0.5-0.1-1.1,0.4-1.4
			c11.8-6.8,20.8-18.1,29.5-29c4.8-6,9.8-12.3,15.2-17.6c27.9-27.7,70.8-32.9,108.7-37.4c5.8-0.7,11.3-1.3,16.7-2.1
			c3.6-0.5,7.2-1,10.8-1.4c23.4-3.1,47.6-6.2,70.3-13.2c14.9-4.6,25.6-10.9,34.6-20.6c0.4-0.4,1-0.4,1.4,0c0.4,0.4,0.4,1,0,1.4
			C542,157.3,531.1,163.8,515.8,168.5z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M233.5,249.5c0.7-2.6,1.6-5.1,2.5-7.5
			c1.8-5.3,3.7-10.8,4.3-16.6c1.1-10.7-2.9-21.1-6.7-31.2c-1.9-5-3.9-10.1-5.2-15.2c-6.2-24-9.3-45-9.5-64.3
			c-0.1-6.5,0.2-12.9,0.8-18.9c1.5-14.7,5.3-29.2,9-43.1l0.2-0.7c0.1-0.5,0.7-0.9,1.2-0.7c0.5,0.1,0.9,0.7,0.7,1.2l-0.2,0.7
			c-3.7,13.9-7.4,28.3-8.9,42.8c-0.6,6-0.9,12.3-0.8,18.7c0.2,19.1,3.3,40,9.4,63.8c1.3,5,3.2,10,5.1,14.9c3.9,10.3,8,21,6.9,32.1
			c-0.6,5.9-2.5,11.6-4.4,17c-0.9,2.6-1.8,5.2-2.6,7.9c-0.2,0.7-0.4,2.6-1.6,2.1C232.7,252.2,233.3,250.2,233.5,249.5z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M198.3,410c5.6-10.4,11.2-21,16.1-31.9
			c-0.2-2-0.3-4-0.5-5.9l-0.3-3.2c-0.2-2.6-0.5-5.2-0.7-7.9c-2.6-27.1-5.4-57.4,5.6-82.8c0.3-0.8,0.5-1.6,1-2.2
			c0.5-0.5,1.4-0.9,1.7-0.1c0.4,1-0.6,2.3-1,3.2c-10.8,25-8,55-5.5,81.8c0.3,2.7,0.5,5.3,0.7,7.9l0.3,3.2c0.1,0.8,0.1,1.5,0.2,2.3
			c3.4-7.8,6.3-15.8,8.4-24c3.2-12.2,2.1-23.6,0.4-36.1c-0.7-5.5-1.4-10.9-1.7-16.4c-0.3-5.5-0.4-11,0.2-16.5
			c0.1-0.9,0.2-1.7,0.3-2.6c0.1-0.6,0.1-1.5,0.6-2c0.7-0.7,1.5,0.2,1.6,0.9c0.1,0.9-0.2,1.9-0.3,2.8c-0.2,0.9-0.3,1.8-0.3,2.7
			c-0.1,1.2-0.2,2.5-0.2,3.7c-0.2,5.8,0.1,11.7,0.7,17.4c0.3,3.2,0.7,6.4,1.2,9.6c1.7,12.7,2.8,24.3-0.5,36.9
			c-2.4,9.4-5.9,18.6-9.9,27.6c1.1,13.1,1.8,27.3-1.6,39.8c-1.8,6.9-5.9,13-9.8,19c-1.7,2.7-3.5,5.4-5.1,8.1
			c-0.2,0.3-0.5,0.5-0.9,0.5c-0.2,0-0.3,0-0.5-0.1c-0.5-0.3-0.7-0.9-0.4-1.4c1.6-2.8,3.4-5.5,5.1-8.2c3.8-5.9,7.8-11.9,9.6-18.4
			c3-11.1,2.7-23.6,1.8-35.5c-4.5,9.8-9.6,19.3-14.7,28.8c-2.1,4-4.4,8.2-6.5,12.2c-0.2,0.3-0.5,0.5-0.9,0.5c-0.2,0-0.3,0-0.5-0.1
			c-0.5-0.3-0.7-0.9-0.4-1.3C193.9,418.2,196.1,414,198.3,410z" style="fill:#3fc0ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="
			M128,182.5c2.4,7.4,7.3,14.9,14.7,22.2c4.8,4.7,10.2,9.2,16.8,13.6c10.5,7.1,22,13,33.2,18.7l0.5,0.2c1.5,0.7,2.9,1.5,4.4,2.2
			c1.8,0.9,3.6,1.8,5.3,2.7c1.8,0.9,3.6,1.8,5.4,2.8c1.8,1,3.5,2,5.3,3.1c0.9,0.5,1.8,1.2,2.6,1.8c0.8,0.6,3.3,1.5,2.6,2.9
			c-0.6,1.3-2,0-2.7-0.5c-1.1-0.9-2.3-1.6-3.5-2.3c-1.6-1-3.3-2-5-3c-1.7-1-3.4-1.9-5.1-2.7c-1.7-0.9-3.4-1.8-5.2-2.6
			c-0.2-0.1-0.4-0.2-0.6-0.3c-1.5-0.7-3-1.5-4.4-2.2l-0.5-0.2c-11.2-5.7-22.8-11.6-33.4-18.8c-6.6-4.5-12.2-9-17.1-13.9
			c-7.5-7.5-12.6-15.3-15.1-23c-0.2-0.5,0.1-1.1,0.6-1.3C127.3,181.7,127.9,182,128,182.5z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M229.8,250.2c1.4-5.4,1.9-11.1,0.1-17.2c-3.2-10.9-10.7-20.4-18-29.6c-1.1-1.4-2.3-2.9-3.4-4.3c-21.9-28.1-41.4-56.2-40.8-90.2
			c0.3-17.7,6.1-44.2,21.3-60.5c0.4-0.4,1-0.4,1.4-0.1c0.4,0.4,0.4,1,0,1.4c-14.8,15.9-20.5,41.9-20.8,59.2
			c-0.6,33.4,18.8,61.1,40.4,89c1.1,1.4,2.2,2.8,3.4,4.3c7.4,9.3,15.1,19,18.3,30.3c1.9,6.6,1.4,12.7-0.1,18.4
			c-0.2,0.6-0.2,1.5-0.9,1.8c-0.5,0.2-1.1-0.1-1.3-0.6C229.3,251.5,229.7,250.7,229.8,250.2z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M225.5,250.7c-4.9-21-28.8-37.8-47.3-49.1c-13.3-8.1-27.2-17.3-37.6-32.1c-7.9-11.2-13.7-23.7-18.9-35.5c-0.2-0.5,0-1.1,0.5-1.3
			c0.5-0.2,1.1,0,1.3,0.5c5.2,11.7,11,24.1,18.7,35.2c10.2,14.5,23.9,23.5,37,31.6c18.9,11.5,43.3,28.7,48.3,50.5
			c0.2,0.8,0.8,2.5-0.6,2.6C225.8,253,225.7,251.4,225.5,250.7z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M243.7,257c0.4-0.4,0.8-0.7,1.3-1.1c11.1-9.7,19.1-22.2,26.7-34.6c-2.6,8.4-6.2,16.3-11.6,23.8c-2.7,3.7-7.8,11-13.9,14.3
			c-0.7,0.4-2.1,0.9-1.9,1.9c0.1,0.9,1,0.7,1.6,0.5c7-3.2,12.9-11.4,15.8-15.6c6.9-9.7,11.1-20,13.7-31
			c7.6-12.3,15.7-24.7,26.9-33.8c6.6-5.4,14.2-9.3,22.2-12.4c-3.3,3.9-6.5,8.2-9.2,12.8c-5.6,9.3-9.3,19.6-12.9,29.5
			c-0.5,1.5-1.1,3-1.6,4.4c-7.8,21.3-18,34.5-32.9,42.7c-6.7,3.7-13.2,5.2-20.6,5.7c-0.8,0-2.3,0-2.4,1.1c-0.1,1.4,1.8,1,2.6,0.9
			c7.6-0.5,14.4-2,21.4-5.9c15.3-8.5,25.8-22,33.8-43.8c0.5-1.5,1.1-3,1.6-4.4c3.6-9.8,7.3-20,12.7-29.1c3.4-5.6,7.3-10.7,11.4-15.3
			c6.9-2.4,14-4.4,21-6.3l0.3-0.1c7.6-2.1,15.4-4.2,22.9-6.1c20.8-5.5,42.3-11.2,63-18.3c27-9.2,55.3-20.1,79.2-37.8
			c13.7-10.1,23.9-22.9,32.1-40.1c0.2-0.5,0-1.1-0.5-1.3c-0.5-0.2-1.1,0-1.3,0.5c-8,16.9-18.1,29.5-31.5,39.4
			c-23.7,17.5-51.8,28.4-78.7,37.5c-20.7,7-42.2,12.7-62.9,18.2c-7.5,2-15.3,4.1-22.9,6.2l-0.3,0.1c-5.9,1.6-11.8,3.2-17.6,5.2
			c3.4-3.5,6.8-6.8,10-9.7l1-0.9c8.8-8.1,18-16.4,26-25.8c3.9-4.6,7.7-9.3,11.4-14c5.9-7.3,11.9-14.9,18.5-21.8
			c30.6-32.2,69-43.1,105.9-50.1c0.5-0.1,0.9-0.6,0.8-1.2c-0.1-0.5-0.6-0.9-1.2-0.8c-37.2,7-75.9,18.1-106.9,50.7
			c-6.6,7-12.7,14.6-18.6,21.9c-3.7,4.6-7.5,9.4-11.4,13.9c-7.9,9.2-17,17.5-25.8,25.6l-1,0.9c-4.1,3.7-8.5,7.9-12.7,12.6
			c-9.4,3.4-18.4,7.7-26.2,14c-10,8.2-17.5,18.8-24.4,29.7c2-10.7,2.9-22.2,3.5-34.6c0.7-12.6,2.2-26,8.7-38.3
			c5-9.4,12.2-18.3,22.8-27.8c4.4-4,9.2-8.1,15.6-13.4c3.6-3,7.5-6.1,11.2-9.1c14.4-11.5,29.3-23.4,39.9-38.4
			c0.3-0.5,0.2-1.1-0.2-1.4c-0.5-0.3-1.1-0.2-1.4,0.2c-10.5,14.8-25.2,26.6-39.5,38c-3.7,3-7.6,6-11.2,9.1
			c-6.4,5.3-11.2,9.5-15.6,13.5c-10.8,9.8-18.1,18.8-23.2,28.4c-6.7,12.6-8.2,26.3-8.9,39.2c-0.8,14.5-1.8,27.6-4.6,39.7
			c-0.7,1.2-1.4,2.3-2.2,3.5c-0.2,0.3-0.4,0.7-0.6,1c-1.8,2.8-3.5,5.7-5.3,8.5c-1.8,2.8-3.7,5.6-5.6,8.4c-1,1.4-1.9,2.7-2.9,4.1
			c-1,1.3-2.1,2.6-3.2,3.9c-2,2.6-4.4,4.9-6.6,7.4c-0.7,0.7-1.4,1.4-2.2,2c-0.8,0.5-1.6,1-2.2,1.6c-0.4,0.4-0.7,0.8-0.8,1.3
			c0,0,0,0,0,0.1C241.3,258,242.8,257.8,243.7,257z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M236.3,313.9c3.6,7,8.6,15.9,15.9,23.4c2.2,2.3,4.5,4.5,6.8,6.6c5.1,4.8,9.9,9.4,14,15c16.8,23.1,18.4,48.7,20,75.7l0.2,2.7
			c0.4,6.8,0.8,13.9,2.7,20.4c0.1,0.4,0.5,0.7,1,0.7c0.1,0,0.2,0,0.3,0c0.5-0.2,0.8-0.7,0.7-1.2c-1.9-6.3-2.3-13.2-2.6-19.9
			l-0.2-2.7c-1.6-27.4-3.2-53.2-20.4-76.8c-4.2-5.8-9.3-10.6-14.2-15.3c-2.2-2.1-4.5-4.3-6.7-6.6c-7.1-7.3-12-16.1-15.6-22.9
			c-6-11.7-8.7-21.8-8.4-31.9c0-0.7,0.4-2.9-0.8-2.8c-0.5,0-1,0.5-1.1,0.9c-0.2,0.6-0.1,1.3-0.1,1.9
			C227.3,291.6,230.1,301.9,236.3,313.9z M238.2,285.8c1.4,3,2.8,6.1,4.4,9c4.5,8.1,10.6,15.1,17.1,22.2
			c5.6,6.2,12.1,12.7,20.4,20.5l1.9,1.8c5.3,5,10.7,10.1,15.9,15.2c14.1,13.8,25.8,30.7,37.1,53.3l0.9,1.8c2.5,5.1,5.2,10.4,8,15.5
			c7.3,13.2,17.1,21.4,30.9,25.7c13.2,4.1,27.1,6.5,40.6,8.7c3.3,0.5,6.5,1.1,9.7,1.7c0.1,0,0.1,0,0.2,0c0.5,0,0.9-0.3,1-0.8
			c0.1-0.5-0.3-1.1-0.8-1.2c-3.2-0.6-6.5-1.1-9.8-1.7c-13.4-2.2-27.2-4.6-40.3-8.6c-13.2-4.1-22.7-12-29.7-24.7
			c-2.8-5-5.4-10.3-7.9-15.4l-0.9-1.8c-11.3-22.8-23.2-39.9-37.4-53.8c-5.3-5.2-10.7-10.3-16-15.3l-1.9-1.8
			c-8.3-7.8-14.7-14.3-20.3-20.4c-6.4-7-12.5-13.9-16.8-21.8c-1.6-2.9-3-5.9-4.3-8.9c-0.9-2-1.8-4.1-2.9-6.1
			c-0.4-0.9-0.8-1.8-1.5-2.5c-1.1-1.4-1.8,0.8-1.2,1.7c0.3,0.5,0.6,1,0.8,1.6C236.4,281.6,237.3,283.7,238.2,285.8z M239.5,272.6
			c-0.5,0.3-0.3,1.1,0,1.4c0.4,0.5,1.1,0.7,1.7,0.9c3.4,1.3,6.9,2.7,10.3,4c2.4,0.9,4.9,1.9,7.3,2.8c1.2,0.5,2.3,1,3.5,1.3
			c8.2,3.2,16.3,6.6,24.2,10.5c4.3,7.1,8,14.9,11.5,22.4c3.1,6.7,6.1,13,9.3,18.5c19.6,33,60.9,74.7,146.1,86.6
			c15.3,2.1,30,3.2,44.1,3.2c31.2,0,59.1-5.3,83.2-15.8c0.5-0.2,0.7-0.8,0.5-1.3c-0.2-0.5-0.8-0.7-1.3-0.5
			c-34.7,15.1-77.1,19.3-126.2,12.5c-84.4-11.8-125.2-53.1-144.6-85.7c-3.2-5.4-6.1-11.7-9.2-18.3c-3.1-6.6-6.3-13.5-10-19.9
			c1.6,0.8,3.3,1.7,4.9,2.6c6.3,3.5,12,8.1,17.6,12.5l1.8,1.4c16,12.6,29.3,22.5,41.8,31.2c25.7,17.8,60.5,39.7,101.5,50.7
			c0.1,0,0.2,0,0.3,0c0.4,0,0.8-0.3,1-0.7c0.1-0.5-0.2-1.1-0.7-1.2c-40.7-10.9-75.3-32.7-100.9-50.4c-12.5-8.6-25.7-18.5-41.7-31.1
			l-1.8-1.4c-5.6-4.5-11.5-9.1-17.9-12.7c-2.5-1.4-5.1-2.8-7.7-4.1c-1.5-2.4-3-4.7-4.7-6.9c2.6-0.1,5.1-0.3,7.7-0.6
			c5.6-0.5,11.4-1,17.2-0.6c23.8,1.4,47.2,8.6,68.9,16.2c24.4,8.5,48.5,18.9,71.9,28.9c7.8,3.3,15.8,6.8,23.8,10.1
			c2.4,1,4.9,2.1,7.4,3.1c22,9.4,44.7,19.1,68.1,19.8c0.5,0,1.1,0,1.6,0c16.8,0,28.7-8,38.2-15.9c0.4-0.4,0.5-1,0.1-1.4
			c-0.4-0.4-1-0.5-1.4-0.1c-17,14.1-29,15.7-38.5,15.4c-23-0.7-45.6-10.3-67.4-19.6c-2.5-1.1-5-2.1-7.4-3.2
			c-7.9-3.3-16-6.8-23.7-10.1c-23.4-10-47.6-20.4-72-28.9c-10.4-3.6-21.2-7.2-32.2-10c2.1,0.1,4.1,0.2,6.1,0.2
			c12.2,0,23.7-2.1,34.4-6.4c9.6-3.9,18.8-8.9,27.7-13.7c9.7-5.3,19.7-10.8,30.2-14.7c41.6-15.6,93.3-12.6,138.1,8.2
			c0.5,0.2,1.1,0,1.3-0.5c0.2-0.5,0-1.1-0.5-1.3c-45.3-21-97.5-24-139.6-8.2c-10.7,4-20.8,9.5-30.5,14.8c-8.8,4.8-18,9.8-27.5,13.6
			c-16.6,6.7-35.5,8-56.1,4c-13.7-2.7-26.2-7.4-39.6-12.4L288,271c-5.6-2.1-10.6-2.9-17-2.8c-0.6,0-1,0.5-1,1c0,0.6,0.5,1,1,1
			c6.2-0.1,10.9,0.6,16.2,2.7l1.7,0.6c8.3,3.1,16.8,6.3,25.5,8.9c-2.1-0.2-4.1-0.4-6.2-0.5c-5.9-0.3-11.8,0.2-17.5,0.6
			c-3.1,0.3-6.1,0.5-9,0.6c-6.3-7.7-13.9-13.6-24-15.7c-1.4-0.3-4.9-0.3-8.9-0.3c-0.8,0-1.7-0.1-2.5,0.2c-0.9,0.4-0.4,1.5,0.3,1.8
			c0.9,0.4,2.2,0.1,3.1,0.1c1.2,0,2.3-0.1,3.5-0.1c1.3,0,2.8-0.1,4.1,0.2c9,1.9,16,7.1,21.8,13.8c-5.5,0.1-10.8-0.4-16.1-2.1
			c-1.2-0.5-2.3-0.9-3.5-1.3c-0.2-0.1-0.4-0.1-0.5-0.2c-4.8-2.2-9.4-4.8-14-7.5c-0.7-0.4-1.4-1-2.2-1.2c-0.6-0.2-1.8-0.5-2.2,0.1
			c-0.3,0.3-0.1,0.6-0.1,0.9c0,0.2,0,0.3-0.1,0.4C240.3,272.4,239.9,272.4,239.5,272.6z M272.7,285c2.7,0.2,5.4,0.3,8.1,0.2
			c1.3,1.6,2.5,3.3,3.7,5C280.6,288.4,276.7,286.7,272.7,285z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M239.9,253c6.7-6.9,9.2-15.6,7.6-26.4c-0.6-3.8-1.5-7.6-2.4-11.2c-0.8-3.4-1.7-6.9-2.3-10.4c-1.9-11.4-2.2-22.9-0.8-34.4
			c1.4-12.4,5.1-23.1,11.1-32.6c2.2-3.5,4.5-6.9,6.8-10.2c4.9-7.1,9.9-14.5,13.8-22.6c10-21.1,19.9-50.9,9.9-72.7
			c-0.2-0.5,0-1.1,0.5-1.3c0.5-0.2,1.1,0,1.3,0.5c10.3,22.6,0.3,52.9-9.9,74.4c-3.9,8.2-9,15.7-13.9,22.9c-2.2,3.3-4.6,6.7-6.7,10.1
			c-5.9,9.3-9.5,19.7-10.8,31.8c-1.3,11.2-1,22.6,0.8,33.8c0.6,3.4,1.4,6.9,2.2,10.2c0.9,3.7,1.9,7.6,2.4,11.4
			c1.7,11.4-1.1,20.8-8.3,28.2c-0.4,0.4-1.1,1.3-1.8,1.1c-0.5-0.1-0.7-0.8-0.6-1.2C238.9,253.9,239.5,253.5,239.9,253z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M198.8,264.8c-0.7,0.2-1.5,0.4-2.2,0.6c-2.7,0.8-5.3,1.7-7.9,2.7c-23.9,9.5-40.7,25.9-57.2,43.2c-18.6,19.6-37.7,37.8-65.6,45.8
			c-5.9,1.7-11.5,2.8-17.1,3.6c-4.4-2.2-8.8-4.5-13.5-6.9c-4.4-2.4-8.9-4.9-13.2-7.3c-2.6-1.5-5.2-3-7.8-4.4
			c-0.5-0.3-1.1-0.1-1.4,0.4c-0.3,0.5-0.1,1.1,0.4,1.4c2.6,1.5,5.2,2.9,7.8,4.4c4.3,2.4,8.8,5,13.2,7.3c3.7,2,7.3,3.9,10.8,5.6
			c-9.1,1.1-17.9,1.3-26.8,1.4c-0.6,0-1,0.5-1,1c0,0.5,0.5,1,1,1c9.9-0.1,19.7-0.4,30-1.8c5.9,2.9,11.8,5.4,17.9,7.5
			c7.4,2.5,16.6,4.7,26.5,4.7c8.8,0,18-1.7,26.8-6.5c14.1-7.6,21.5-20,29.3-33.2l1.7-2.9c11.4-19.1,25.3-33.2,43.7-44.5
			c1.9-1.2,3.9-2.3,6-3.5c3.8-2.1,7.6-4.3,11-6.8c-1.1,1.7-2.2,3.4-3.3,5.2l-0.1,0.2c-4,6.4-7.1,11.7-9.8,16.7
			c-2.9,5.3-5.6,10.7-8.2,16.1c-4.3,9.1-5.9,18.6-7.4,27.7c-0.8,4.8-1.6,9.8-2.8,14.8c-5.8,23.7-25.3,39.3-44.1,54.4l-2.7,2.2
			c-0.4,0.3-0.5,1-0.2,1.4c0.2,0.2,0.5,0.4,0.8,0.4c0.2,0,0.4-0.1,0.6-0.2l2.7-2.2c18.2-14.6,38.8-31.2,44.8-55.5
			c1.2-5.1,2.1-10.1,2.9-15c1.5-9.4,3-18.3,7.2-27.2c2.5-5.3,5.2-10.6,8.1-16c2.7-4.9,5.8-10.2,9.8-16.6l0.1-0.2
			c1.2-1.9,2.3-3.7,3.6-5.5c1.2-1.8,2.6-3.5,4-5.1c0.4-0.5,0.9-0.8,1.3-1.3c0.2-0.2,0.3-0.4,0.4-0.7c0-0.2,0.1-0.7,0-0.9
			c-0.3-0.5-1-0.7-1.5-0.5c-0.3,0.1-0.6,0.4-0.8,0.6c-1.9,1.4-3.5,3.3-5.4,4.7c-2.4,1.9-5,3.5-7.6,5c-1.4,0.8-2.9,1.7-4.3,2.5
			c-2,1.1-4.1,2.3-6.1,3.5c-18.7,11.4-32.8,25.8-44.4,45.1l-1.7,2.9c-7.7,12.9-14.9,25.1-28.5,32.4c-18.1,9.7-38.1,6.3-51.7,1.6
			c-5.1-1.7-10-3.8-15-6.2c4.7-0.8,9.5-1.8,14.5-3.2c28.4-8.1,47.7-26.5,66.5-46.3c17-17.8,32.9-33.4,56.5-42.7
			c1.8-0.7,3.7-1.4,5.6-2c1.1-0.3,2.1-0.6,3.2-0.9c1.9-0.5,3.8-0.9,5.7-1.3c1.8-0.4,3.7-0.4,5.5-0.6c0.6-0.1,1.9-0.3,1.9-1
			c0-0.7-0.8-1.2-1.4-1.4c-0.4-0.1-0.8-0.1-1.3,0c-0.7,0.1-1.4,0.3-2,0.4c-0.1,0-0.2,0-0.3,0.1C203.8,263.9,201.3,264.3,198.8,264.8
			z M199.6,256.9c-1.6-0.5-3.2-1-4.7-1.5c-1.3-0.4-2.5-0.8-3.9-1.1c1.9-0.2,3.8-0.4,5.7-0.5c1.7,0,3.4,0.1,5.1,0.2
			c1.8,0.1,3.7,0.3,5.5,0.8c1.5,0.4,2.9,1,4.4,1.2c0.6,0.1,1.6-0.1,1.4-0.9c-0.1-0.5-0.6-1-1.1-1.2c-0.8-0.4-1.7-0.4-2.5-0.6
			c-1.2-0.3-2.5-0.5-3.7-0.7c-2.8-0.5-5.7-0.8-8.6-0.7c-1.8,0-3.5,0.2-5.3,0.4c-0.9,0.1-1.8,0.2-2.7,0.4c-0.4,0.1-0.9,0.1-1.3,0.2
			c-0.2,0-0.5,0.1-0.7,0.1c-0.2,0-0.5,0.1-0.6,0.1c-12-3.3-25.6-6.1-39-3.6c-10,1.9-18.9,6.6-27.6,11.3c-3.5,1.9-7,3.7-10.6,5.4
			c-21.1-2.4-39.6-8.5-60.6-15.3c-2.1-0.7-4.3-1.4-6.4-2.1c-0.5-0.2-1.1,0.1-1.3,0.6c-0.2,0.5,0.1,1.1,0.6,1.3
			c2.2,0.7,4.3,1.4,6.4,2.1c19.9,6.5,37.7,12.3,57.7,15c-1.1,0.5-2.2,0.9-3.3,1.3c-19.7,7.4-40.6,7.3-60.3,6c-0.5,0-1,0.4-1.1,0.9
			c0,0.6,0.4,1,0.9,1.1c6,0.4,12.1,0.7,18.2,0.7c14.3,0,28.9-1.5,42.9-6.8c2.2-0.8,4.3-1.7,6.4-2.7c4.4,0.5,8.8,0.8,13.4,1
			c1.1,0,2.2,0.1,3.3,0.1c14.9,0,29.7-3.3,48.6-10.6c3.9-1.5,7.7-2.7,11.4-3.5c2.7,0.8,5.3,1.5,7.8,2.3c1.5,0.4,3,0.9,4.6,1.5
			c1.4,0.4,2.7,0.9,4.1,1.3c1.1,0.3,2.2,0.8,3.4,1.2c0.7,0.2,2.2,0.5,2.5-0.5c0.4-1.3-1.5-1.5-2.3-1.7c-0.6-0.2-1.2-0.4-1.8-0.5
			C203.1,258,201.3,257.5,199.6,256.9z M182.5,254c-2.6,0.7-5.3,1.6-8.1,2.7c-20.1,7.8-35.4,10.9-51.2,10.5
			c-3.4-0.1-6.8-0.3-10.1-0.6c2.6-1.3,5.3-2.7,7.8-4.1c8.5-4.6,17.3-9.3,27-11.1C159.7,249.2,171.7,251.2,182.5,254z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M515.8,168.5c-22.9,7-47.1,10.2-70.6,13.3c-3.6,0.5-7.2,0.9-10.8,1.4c-5.4,0.7-10.9,1.4-16.7,2.1c-37.6,4.5-80.1,9.6-107.5,36.9
			c-5.3,5.2-10.2,11.4-15,17.4c-8.8,11.1-18,22.5-30.1,29.5c-0.2,0.1-0.3,0.1-0.5,0.1c-0.3,0-0.7-0.2-0.9-0.5
			c-0.3-0.5-0.1-1.1,0.4-1.4c11.8-6.8,20.8-18.1,29.5-29c4.8-6,9.8-12.3,15.2-17.6c27.9-27.7,70.8-32.9,108.7-37.4
			c5.8-0.7,11.3-1.3,16.7-2.1c3.6-0.5,7.2-1,10.8-1.4c23.4-3.1,47.6-6.2,70.3-13.2c14.9-4.6,25.6-10.9,34.6-20.6
			c0.4-0.4,1-0.4,1.4,0c0.4,0.4,0.4,1,0,1.4C542,157.3,531.1,163.8,515.8,168.5z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M233.5,249.5c0.7-2.6,1.6-5.1,2.5-7.5c1.8-5.3,3.7-10.8,4.3-16.6c1.1-10.7-2.9-21.1-6.7-31.2c-1.9-5-3.9-10.1-5.2-15.2
			c-6.2-24-9.3-45-9.5-64.3c-0.1-6.5,0.2-12.9,0.8-18.9c1.5-14.7,5.3-29.2,9-43.1l0.2-0.7c0.1-0.5,0.7-0.9,1.2-0.7
			c0.5,0.1,0.9,0.7,0.7,1.2l-0.2,0.7c-3.7,13.9-7.4,28.3-8.9,42.8c-0.6,6-0.9,12.3-0.8,18.7c0.2,19.1,3.3,40,9.4,63.8
			c1.3,5,3.2,10,5.1,14.9c3.9,10.3,8,21,6.9,32.1c-0.6,5.9-2.5,11.6-4.4,17c-0.9,2.6-1.8,5.2-2.6,7.9c-0.2,0.7-0.4,2.6-1.6,2.1
			C232.7,252.2,233.3,250.2,233.5,249.5z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="
			M198.3,410c5.6-10.4,11.2-21,16.1-31.9c-0.2-2-0.3-4-0.5-5.9l-0.3-3.2c-0.2-2.6-0.5-5.2-0.7-7.9c-2.6-27.1-5.4-57.4,5.6-82.8
			c0.3-0.8,0.5-1.6,1-2.2c0.5-0.5,1.4-0.9,1.7-0.1c0.4,1-0.6,2.3-1,3.2c-10.8,25-8,55-5.5,81.8c0.3,2.7,0.5,5.3,0.7,7.9l0.3,3.2
			c0.1,0.8,0.1,1.5,0.2,2.3c3.4-7.8,6.3-15.8,8.4-24c3.2-12.2,2.1-23.6,0.4-36.1c-0.7-5.5-1.4-10.9-1.7-16.4
			c-0.3-5.5-0.4-11,0.2-16.5c0.1-0.9,0.2-1.7,0.3-2.6c0.1-0.6,0.1-1.5,0.6-2c0.7-0.7,1.5,0.2,1.6,0.9c0.1,0.9-0.2,1.9-0.3,2.8
			c-0.2,0.9-0.3,1.8-0.3,2.7c-0.1,1.2-0.2,2.5-0.2,3.7c-0.2,5.8,0.1,11.7,0.7,17.4c0.3,3.2,0.7,6.4,1.2,9.6
			c1.7,12.7,2.8,24.3-0.5,36.9c-2.4,9.4-5.9,18.6-9.9,27.6c1.1,13.1,1.8,27.3-1.6,39.8c-1.8,6.9-5.9,13-9.8,19
			c-1.7,2.7-3.5,5.4-5.1,8.1c-0.2,0.3-0.5,0.5-0.9,0.5c-0.2,0-0.3,0-0.5-0.1c-0.5-0.3-0.7-0.9-0.4-1.4c1.6-2.8,3.4-5.5,5.1-8.2
			c3.8-5.9,7.8-11.9,9.6-18.4c3-11.1,2.7-23.6,1.8-35.5c-4.5,9.8-9.6,19.3-14.7,28.8c-2.1,4-4.4,8.2-6.5,12.2
			c-0.2,0.3-0.5,0.5-0.9,0.5c-0.2,0-0.3,0-0.5-0.1c-0.5-0.3-0.7-0.9-0.4-1.3C193.9,418.2,196.1,414,198.3,410z" style="fill:#3fc0ff;stroke:#000000;stroke-width:0.15;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;"></path>
<path d="M501.2,380.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C502.8,387.1,500.4,383.1,501.2,380.7z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M187.7,141.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C189.3,148.1,186.9,144.1,187.7,141.7z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M175,382.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C176.7,386.9,174.8,384.6,175,382.2z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,226.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C190.2,230.9,188.3,228.6,188.5,226.2z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M426,364.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C427.7,368.9,425.8,366.6,426,364.2z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M370.6,108.7c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C372.3,113.4,370.4,111.1,370.6,108.7z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M235.5,107.7c-2-0.4-2.7-2.8-2.2-4.8
		c0.4-1.7,2.1-2.8,3.8-2.3c3.2,0.9,3.1,6.4-0.2,7.1C236.4,107.7,235.9,107.8,235.5,107.7z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M180.1,315.4c-2.4-0.3-4.5-3.8-1.7-5.2
		c2.4-1.2,5.9,2.4,3.8,4.6C181.6,315.3,180.8,315.5,180.1,315.4z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M110.6,298.9c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C115.1,300.2,112.1,300.7,110.6,298.9z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M77.4,374.2c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C81.8,375.6,78.9,376.1,77.4,374.2z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M151.6,195.1c0.2-2.2,2.1-4.1,4.6-2.1
		c2.3,1.8,2,7-1.8,6.2C152.2,198.7,151.4,196.8,151.6,195.1z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M537.2,295.8c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C537.4,301.4,535.8,297.6,537.2,295.8z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M388.5,55.1c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C388.7,60.7,387.1,56.8,388.5,55.1z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M473.6,318c1.8-3.7,6.5-1.1,4.7,3
		C476.3,325.4,471.9,321.4,473.6,318z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M440.7,97.8c-0.9-3.1,3.5-7,4.8-2.1
		C446.6,99.6,442.1,102.3,440.7,97.8z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M418,393c1.3-3.1,7.5-2.5,5.2,2
		C421.4,398.4,416.5,396.4,418,393z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M482.3,140.9c3.6,0.7,4,5.1,0.4,6.2
		C477.8,148.6,477.4,139.9,482.3,140.9z" style="fill:#1be4ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M501.2,380.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C502.8,387.1,500.4,383.1,501.2,380.7z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M187.7,141.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C189.3,148.1,186.9,144.1,187.7,141.7z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M175,382.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C176.7,386.9,174.8,384.6,175,382.2z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,226.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C190.2,230.9,188.3,228.6,188.5,226.2z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M426,364.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C427.7,368.9,425.8,366.6,426,364.2z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M370.6,108.7c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C372.3,113.4,370.4,111.1,370.6,108.7z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M235.5,107.7c-2-0.4-2.7-2.8-2.2-4.8
		c0.4-1.7,2.1-2.8,3.8-2.3c3.2,0.9,3.1,6.4-0.2,7.1C236.4,107.7,235.9,107.8,235.5,107.7z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M180.1,315.4c-2.4-0.3-4.5-3.8-1.7-5.2
		c2.4-1.2,5.9,2.4,3.8,4.6C181.6,315.3,180.8,315.5,180.1,315.4z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M110.6,298.9c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C115.1,300.2,112.1,300.7,110.6,298.9z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col focal_adhesion_sites" descendants="level 0: [None]" id="focal_adhesion_sites" init-pval="--" min-pval="--" min-pval-children="--" title="Focal adhesion sites">
<path d="M77.4,374.2c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C81.8,375.6,78.9,376.1,77.4,374.2z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M151.6,195.1c0.2-2.2,2.1-4.1,4.6-2.1
		c2.3,1.8,2,7-1.8,6.2C152.2,198.7,151.4,196.8,151.6,195.1z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M537.2,295.8c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C537.4,301.4,535.8,297.6,537.2,295.8z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M388.5,55.1c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C388.7,60.7,387.1,56.8,388.5,55.1z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M473.6,318c1.8-3.7,6.5-1.1,4.7,3
		C476.3,325.4,471.9,321.4,473.6,318z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M440.7,97.8c-0.9-3.1,3.5-7,4.8-2.1
		C446.6,99.6,442.1,102.3,440.7,97.8z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M418,393c1.3-3.1,7.5-2.5,5.2,2
		C421.4,398.4,416.5,396.4,418,393z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M482.3,140.9c3.6,0.7,4,5.1,0.4,6.2
		C477.8,148.6,477.4,139.9,482.3,140.9z" style="fill:#1be4ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M558.5,272.6c0.7-0.4,3.9-5,2.8-5.7
		c-0.7-0.4-2.2,2.6-2.6,3.2C558.3,270.8,556.3,274,558.5,272.6z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M525.6,202.2c-0.1-2.3-7.6-0.5-5.7,1.5
		c0.8,0.9,2.6,0.8,3.6,0.4C524.1,203.8,525.7,203,525.6,202.2z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M566,160c0.5-2.1-7.1,1.7-3.7,2.8
		C563.9,163.3,565.7,161.4,566,160z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M494,264.8c0.3-1,0.4-5.2-1.4-3.3
		C491.4,262.7,493,267.9,494,264.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M539.1,244.8c0,0.2,0.2,0.4,0.6,0.7
		c1,0.6,2.6,0.4,3.6,0.1c0.8-0.3,2.8-1.4,1.4-2.3C543.6,242.6,539.1,243.7,539.1,244.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M517.7,129.2c0.1-1.9-0.8-3.2-2.1-1.2
		C514.2,130.3,517.5,132.2,517.7,129.2z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M478.8,200.8c1,1.1,2.4,1.6,3.2,0.4
		c1-1.7-3.7-5.4-4.3-3.1C477.5,198.9,478,200,478.8,200.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.2,305.5c1.4,0.6,3.2-3.8,1.5-4.3
		c-1.6-0.4-2.8,2.7-1.9,3.9C517.9,305.3,518.1,305.4,518.2,305.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M469.3,147.7c0.2,0.1,0.5,0.1,1-0.1
		c2.5-1.3,0.3-7.7-0.9-3.4C469.2,144.8,468.6,147.2,469.3,147.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M427.9,76.5c0,0.2,0.2,0.5,0.6,0.7
		c2.5,1.4,6.6-3.9,2.4-2.6C430.3,74.9,427.9,75.7,427.9,76.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M70.3,341.6c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C72.8,343,71.1,341.2,70.3,341.6z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M352.8,409.1c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C355.3,410.5,353.6,408.7,352.8,409.1z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M543.1,184.3c0.1-0.5,0-1-0.3-1.5
		c-0.6-1-4.2-4.4-3.6-1.5C539.4,182.1,542.6,187,543.1,184.3z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M563.8,194c0.1,0.2,0.5,0.5,1.2,0.6
		c0.7,0.2,4.6,0.1,4.1-1.4c-0.3-1.1-2.5-0.8-3.3-0.7C564.7,192.7,563.5,193.4,563.8,194z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M412.8,449c0.7,0.5,6.2,1.5,6.3,0.3
		c0.1-0.8-3.3-1-4-1.1C414.4,448.1,410.8,447.5,412.8,449z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M464.1,390.6c2.1-1-2.6-7.2-3.7-4.6
		c-0.5,1.1,0.3,2.7,1.1,3.5C462,389.8,463.4,390.9,464.1,390.6z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.9,410.6c2.1-0.4-4.4-5.8-4-2.2
		C515.1,409.9,517.6,410.8,518.9,410.6z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M394.1,386.8c1.1-0.1,5-1.7,2.5-2.6
		C394.9,383.5,390.8,387.1,394.1,386.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M430.5,420.1c-0.2,0.1-0.3,0.3-0.4,0.8
		c-0.1,1.2,0.6,2.5,1.4,3.4c0.6,0.6,2.4,2,2.7,0.3C434.4,423.3,431.5,419.6,430.5,420.1z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M527.7,353.9c1.8-0.6,2.6-2,0.3-2.4
		C525.3,351.1,524.9,354.9,527.7,353.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M407.2,354.4c1.8-0.6,2.6-2,0.3-2.4
		C404.8,351.6,404.4,355.4,407.2,354.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M165.7,328.9c1.8-0.6,2.6-2,0.3-2.4
		C163.3,326.1,162.9,329.9,165.7,328.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M446.5,347.1c-0.6,1.3-0.5,2.9,0.9,3.1
		c1.9,0.3,3.5-5.6,1.1-5.2C447.7,345.2,446.9,346.1,446.5,347.1z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366.6,425.4c0,1.5,4.8,1.4,4.5-0.4
		c-0.2-1.7-3.6-1.5-4.4-0.2C366.6,425,366.6,425.2,366.6,425.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M491.3,317c0,0.2,0.1,0.5,0.5,0.8
		c2.2,1.8,7.1-2.8,2.8-2.2C493.9,315.8,491.5,316.2,491.3,317z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M487.5,399.3c0.5-0.1,0.9-0.5,1.2-0.9
		c0.7-0.9,2.3-5.6-0.1-3.9C488,395,484.9,400,487.5,399.3z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.9,422.2c-0.2,0.2-0.2,0.6-0.1,1.3
		c0.1,0.7,1.7,4.2,2.9,3.2c0.9-0.8-0.3-2.6-0.7-3.3C488.5,422.5,487.3,421.7,486.9,422.2z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M152.6,427c0.5,0.7,5.4,3.4,5.9,2.3
		c0.3-0.7-2.8-1.9-3.4-2.3C154.4,426.6,151.1,424.9,152.6,427z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M219.7,387.7c2.3-0.3-0.2-7.6-2-5.6
		c-0.8,0.9-0.5,2.7-0.1,3.6C217.9,386.3,218.9,387.8,219.7,387.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M265.4,424c2.1,0.3-2.3-6.9-3.1-3.4
		C261.9,422.2,264,423.9,265.4,424z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.5,362c1.1,0.2,5.2-0.1,3.2-1.7
		C156.3,359.2,151.2,361.2,154.5,362z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,405.1c-0.2,0-0.4,0.2-0.6,0.6
		c-0.5,1.1-0.2,2.6,0.3,3.6c0.3,0.8,1.6,2.6,2.4,1.2C181.2,409.4,179.6,405,178.5,405.1z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M301.2,384.9c1.9,0,3.1-1.1,1-2.2
		C299.8,381.5,298.2,385,301.2,384.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,244.2c1.9,0,3.1-1.1,1-2.2
		C177.1,240.8,175.5,244.2,178.5,244.2z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M181,176c1.9,0,3.1-1.1,1-2.2
		C179.6,172.6,178,176.1,181,176z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M356.6,27.9c1.9,0,3.1-1.1,1-2.2
		C355.2,24.4,353.6,27.9,356.6,27.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.2,164.9c1.9,0,3.1-1.1,1-2.2
		C484.8,161.5,483.2,165,486.2,164.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M470.2,85.4c1.9,0,3.1-1.1,1-2.2
		C468.8,82,467.2,85.5,470.2,85.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton microtubule_organizing_center" descendants="level 0: ['microtubule organizing center, pval = 0.38096']
level 1: ['katanin complex, pval = --', 'equatorial microtubule organizing center, pval = --', 'ciliary basal body, pval = 0.90208', 'centriole, pval = 0.59248', 'interphase microtubule organizing center, pval = --', 'centrosome, pval = 0.62089', 'spindle pole body, pval = --', 'gamma-tubulin complex, pval = --']
level 2: ['post-anaphase microtubule array, pval = --', 'gamma-tubulin large complex, equatorial microtubule organizing center, pval = --', 'kinociliary basal body, pval = 0.43974', 'right posteriolateral basal body, pval = --', 'right anterior basal body, pval = --', 'left posteriolateral basal body, pval = --', 'right ventral basal body, pval = --', 'left ventral basal body, pval = --', 'right caudal basal body, pval = --', 'left caudal basal body, pval = --', 'left anterior basal body, pval = --', 'gamma-tubulin large complex, interphase microtubule organizing center, pval = --', 'pericentriolar material, pval = --', 'centriolar satellite, pval = --', 'centrosomal corona, pval = --', 'spindle pole centrosome, pval = 0.43974', 'maintenance of protein location to spindle pole body, pval = --', 'mitotic spindle pole body, pval = --', 'gamma-tubulin large complex, spindle pole body, pval = --', 'inner plaque of spindle pole body, pval = --', 'intermediate layer of spindle pole body, pval = --', 'outer plaque of spindle pole body, pval = --', 'central plaque of spindle pole body, pval = --', 'half bridge of spindle pole body, pval = --', 'gamma-tubulin small complex, spindle pole body, pval = --', 'meiotic spindle pole body, pval = --', 'Bfa1-Bub2 complex, pval = --', 'gamma-tubulin small complex, pval = --', 'gamma-tubulin large complex, pval = --']
level 3: ['post-anaphase array microtubule, pval = --', 'right middle basal body pair, pval = --', 'right lateral basal body pair, pval = --', 'left middle basal body pair, pval = --', 'left lateral basal body pair, pval = --', 'gamma-tubulin ring complex, centrosomal, pval = --', 'central plaque of mitotic spindle pole body, pval = --', 'gamma-tubulin large complex, mitotic spindle pole body, pval = --', 'half bridge of mitotic spindle pole body, pval = --', 'gamma-tubulin small complex, mitotic spindle pole body, pval = --', 'inner plaque of mitotic spindle pole body, pval = --', 'intermediate layer of mitotic spindle pole body, pval = --', 'outer plaque of mitotic spindle pole body, pval = --', 'new mitotic spindle pole body, pval = --', 'old mitotic spindle pole body, pval = --', 'gamma-tubulin small complex, centrosomal, pval = --', 'gamma-tubulin ring complex, pval = --']
level 4: ['post-anaphase array microtubule end, pval = --', 'right tetrad, pval = --', 'left tetrad, pval = --']" id="microtubule_organizing_center_ton" init-pval="0.38096" min-pval="0.38096" min-pval-children="0.43974" title="Microtubule organizing center">
<path d="M509.5,281.9c1.9,0,3.1-1.1,1-2.2
		C508.1,278.5,506.5,281.9,509.5,281.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col microtubule_organizing_center" descendants="level 0: ['microtubule organizing center, pval = 0.38096']
level 1: ['katanin complex, pval = --', 'equatorial microtubule organizing center, pval = --', 'ciliary basal body, pval = 0.90208', 'centriole, pval = 0.59248', 'interphase microtubule organizing center, pval = --', 'centrosome, pval = 0.62089', 'spindle pole body, pval = --', 'gamma-tubulin complex, pval = --']
level 2: ['post-anaphase microtubule array, pval = --', 'gamma-tubulin large complex, equatorial microtubule organizing center, pval = --', 'kinociliary basal body, pval = 0.43974', 'right posteriolateral basal body, pval = --', 'right anterior basal body, pval = --', 'left posteriolateral basal body, pval = --', 'right ventral basal body, pval = --', 'left ventral basal body, pval = --', 'right caudal basal body, pval = --', 'left caudal basal body, pval = --', 'left anterior basal body, pval = --', 'gamma-tubulin large complex, interphase microtubule organizing center, pval = --', 'pericentriolar material, pval = --', 'centriolar satellite, pval = --', 'centrosomal corona, pval = --', 'spindle pole centrosome, pval = 0.43974', 'maintenance of protein location to spindle pole body, pval = --', 'mitotic spindle pole body, pval = --', 'gamma-tubulin large complex, spindle pole body, pval = --', 'inner plaque of spindle pole body, pval = --', 'intermediate layer of spindle pole body, pval = --', 'outer plaque of spindle pole body, pval = --', 'central plaque of spindle pole body, pval = --', 'half bridge of spindle pole body, pval = --', 'gamma-tubulin small complex, spindle pole body, pval = --', 'meiotic spindle pole body, pval = --', 'Bfa1-Bub2 complex, pval = --', 'gamma-tubulin small complex, pval = --', 'gamma-tubulin large complex, pval = --']
level 3: ['post-anaphase array microtubule, pval = --', 'right middle basal body pair, pval = --', 'right lateral basal body pair, pval = --', 'left middle basal body pair, pval = --', 'left lateral basal body pair, pval = --', 'gamma-tubulin ring complex, centrosomal, pval = --', 'central plaque of mitotic spindle pole body, pval = --', 'gamma-tubulin large complex, mitotic spindle pole body, pval = --', 'half bridge of mitotic spindle pole body, pval = --', 'gamma-tubulin small complex, mitotic spindle pole body, pval = --', 'inner plaque of mitotic spindle pole body, pval = --', 'intermediate layer of mitotic spindle pole body, pval = --', 'outer plaque of mitotic spindle pole body, pval = --', 'new mitotic spindle pole body, pval = --', 'old mitotic spindle pole body, pval = --', 'gamma-tubulin small complex, centrosomal, pval = --', 'gamma-tubulin ring complex, pval = --']
level 4: ['post-anaphase array microtubule end, pval = --', 'right tetrad, pval = --', 'left tetrad, pval = --']" id="microtubule_organizing_center" init-pval="0.38096" min-pval="0.38096" min-pval-children="0.43974" title="Microtubule organizing center">
<path d="M216.7,340.9c-1,1.1-1.4,2.6-0.1,3.2
		c1.7,0.9,5.1-4.2,2.7-4.6C218.5,339.5,217.5,340.1,216.7,340.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton centrosome" descendants="level 0: ['centrosome, pval = 0.62089']
level 1: ['pericentriolar material, pval = --', 'centriolar satellite, pval = --', 'centrosomal corona, pval = --', 'spindle pole centrosome, pval = 0.43974']
level 2: ['gamma-tubulin ring complex, centrosomal, pval = --']" id="centrosome_ton" init-pval="0.62089" min-pval="0.43974" min-pval-children="0.43974" title="Centrosome">
<path d="M100.6,356.7c-0.4,1.5,4.1,2.8,4.4,1.1
		c0.3-1.7-3-2.6-4.1-1.5C100.7,356.4,100.6,356.6,100.6,356.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M268.8,326.6c-0.1,0.2-0.1,0.5,0.2,0.9
		c1.6,2.4,7.6-0.4,3.3-1.2C271.6,326.2,269.2,325.8,268.8,326.6z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col centrosome" descendants="level 0: ['centrosome, pval = 0.62089']
level 1: ['pericentriolar material, pval = --', 'centriolar satellite, pval = --', 'centrosomal corona, pval = --', 'spindle pole centrosome, pval = 0.43974']
level 2: ['gamma-tubulin ring complex, centrosomal, pval = --']" id="centrosome" init-pval="0.62089" min-pval="0.43974" min-pval-children="0.43974" title="Centrosome">
<path d="M239.1,403.4c0.5,0,1-0.1,1.5-0.5
		c0.9-0.6,4-4.5,1.1-3.7C241,399.5,236.4,403.2,239.1,403.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M231.4,424.9c-0.2,0.1-0.4,0.5-0.5,1.2
		c-0.1,0.7,0.3,4.6,1.8,4c1.1-0.4,0.6-2.5,0.4-3.3C232.7,425.7,231.9,424.6,231.4,424.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton microtubules midbody midbody_ring mitotic_spindle cytokinetic_bridge" descendants="level 0: ['microtubule, pval = 0.08911']
level 1: ['median body, pval = --', 'subpellicular microtubule, pval = --', 'microtubule bundle, pval = --', 'dendritic microtubule, pval = --', 'ventral disc microtubule array, pval = --', 'ventral disc supernumerary microtubule array, pval = --', 'intraconoid microtubule, pval = --', 'axoneme, pval = 0.72894', 'microtubule end, pval = --', 'spindle microtubule, pval = 0.43974', 'post-anaphase array microtubule, pval = --', 'nuclear microtubule, pval = --', 'cytoplasmic microtubule, pval = 0.68643']
level 2: ['cytoplasmic microtubule bundle, pval = --', 'axon microtubule bundle, pval = --', 'dynamic microtubule bundle, pval = --', 'static microtubule bundle, pval = --', 'ventral disc overlap zone, pval = --', 'axonemal heterotrimeric kinesin-II complex, pval = --', 'axonemal nexin link, pval = --', 'axonemal central apparatus, pval = --', 'axonemal dynein complex, pval = --', 'axonemal microtubule, pval = --', 'distal portion of axoneme, pval = --', 'proximal portion of axoneme, pval = --', 'radial spoke, pval = --', 'axonemal microtubule depolymerization, pval = --', 'satellite fibril, pval = --', 'axonemal basal plate, pval = --', 'ciliary shaft, pval = --', 'axonemal outer doublet, pval = --', 'microtubule plus-end, pval = --', 'mitotic spindle astral microtubule end, pval = --', 'microtubule minus-end, pval = --', 'post-anaphase array microtubule end, pval = --', 'astral microtubule, pval = --', 'mitotic spindle microtubule, pval = --', 'polar microtubule, pval = --', 'kinetochore microtubule, pval = --', 'cytoplasmic microtubule plus-end, pval = --', 'cortical microtubule, pval = --', 'cytoplasmic microtubule minus-end, pval = --']
level 3: ['ventral disc, pval = --', 'axonemal central pair projection, pval = --', 'axonemal central bridge, pval = --', 'axonemal central pair, pval = --', 'inner dynein arm, pval = --', 'outer dynein arm, pval = --', 'A axonemal microtubule, pval = --', 'B axonemal microtubule, pval = --', 'C1 axonemal microtubule, pval = --', 'C2 axonemal microtubule, pval = --', 'radial spoke stalk, pval = --', 'radial spoke head, pval = --', 'negative regulation of axonemal microtubule depolymerization, pval = --', '9+0 motile cilium, pval = --', '9+2 motile cilium, pval = 0.43974', 'meiotic spindle astral microtubule, pval = --', 'mitotic spindle astral microtubule, pval = --', 'mitotic spindle kinetochore microtubule, pval = --', 'mitotic spindle polar microtubule, pval = --', 'cortical microtubule plus-end, pval = --', 'cortical microtubule, transverse to long axis, pval = --']
level 4: ['ventral disc lateral crest, pval = --', 'ventral disc dorsal microribbon, pval = --', 'ventral disc crossbridge, pval = --', '9+2 non-motile cilium, pval = 0.43974', 'paraflagellar rod, pval = --', 'transverse flagellum, pval = --', 'longitudinal flagellum, pval = --', 'sperm flagellum, pval = 0.43974', 'right ventral flagellum, pval = --', 'right posteriolateral flagellum, pval = --', 'left ventral flagellum, pval = --', 'left anterior flagellum, pval = --', 'right anterior flagellum, pval = --', 'left posteriolateral flagellum, pval = --', 'right caudal flagellum, pval = --', 'left caudal flagellum, pval = --']
level 5: ['kinocilium, pval = 0.43974', 'outer dense fiber, pval = --', 'sperm fibrous sheath, pval = --', 'sperm connecting piece, pval = --', 'sperm annulus, pval = --', 'sperm principal piece, pval = --', 'sperm end piece, pval = --', 'sperm midpiece, pval = --', 'sperm mitochondrial sheath, pval = --', 'right ventral basal body, pval = --', 'right posteriolateral basal body, pval = --', 'left ventral basal body, pval = --', 'left anterior basal body, pval = --', 'right anterior basal body, pval = --', 'left posteriolateral basal body, pval = --', 'right caudal basal body, pval = --', 'left caudal basal body, pval = --']
level 6: ['kinociliary basal body, pval = 0.43974', 'right lateral basal body pair, pval = --', 'right middle basal body pair, pval = --', 'left lateral basal body pair, pval = --', 'left middle basal body pair, pval = --']
level 7: ['right tetrad, pval = --', 'left tetrad, pval = --']" id="microtubules_ton" init-pval="0.08911" min-pval="0.08911" min-pval-children="0.43974" title="Microtubules">
<g>
<path d="M112.5,138.4c-0.8,0.4-4.1,4.8-3.1,5.5
		c0.7,0.4,2.3-2.5,2.7-3.1C112.6,140.3,114.8,137.2,112.5,138.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M141.8,210.5c0,2.3,7.6,0.9,5.8-1.2
		c-0.8-0.9-2.6-0.9-3.6-0.6C143.4,208.9,141.8,209.6,141.8,210.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M99.4,250.5c-0.6,2,7.2-1.3,3.8-2.6
		C101.7,247.4,99.7,249.2,99.4,250.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M176.6,149.5c-0.4,1-0.7,5.2,1.2,3.4
		C179.1,151.8,177.8,146.5,176.6,149.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M130.5,167.2c0-0.2-0.1-0.5-0.5-0.7
		c-1-0.6-2.5-0.6-3.6-0.3c-0.8,0.2-2.8,1.2-1.5,2.2C125.9,169.2,130.4,168.3,130.5,167.2z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M146.1,283.8c-0.2,1.9,0.7,3.2,2,1.3
		C149.7,282.9,146.5,280.8,146.1,283.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,214.2c-0.9-1.1-2.3-1.8-3.2-0.6
		c-1.1,1.6,3.5,5.6,4.1,3.3C189.7,216.1,189.3,215,188.5,214.2z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.4,107.7c-1.4-0.6-3.4,3.7-1.7,4.2
		c1.6,0.5,3-2.6,2.1-3.8C154.7,107.9,154.6,107.7,154.4,107.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M195.3,267.7c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C195.3,270.6,196.1,268.2,195.3,267.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M63.6,300.5c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C63.6,303.3,64.3,301,63.6,300.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
</g>
<g class="col microtubules midbody midbody_ring mitotic_spindle cytokinetic_bridge" descendants="level 0: ['microtubule, pval = 0.08911']
level 1: ['median body, pval = --', 'subpellicular microtubule, pval = --', 'microtubule bundle, pval = --', 'dendritic microtubule, pval = --', 'ventral disc microtubule array, pval = --', 'ventral disc supernumerary microtubule array, pval = --', 'intraconoid microtubule, pval = --', 'axoneme, pval = 0.72894', 'microtubule end, pval = --', 'spindle microtubule, pval = 0.43974', 'post-anaphase array microtubule, pval = --', 'nuclear microtubule, pval = --', 'cytoplasmic microtubule, pval = 0.68643']
level 2: ['cytoplasmic microtubule bundle, pval = --', 'axon microtubule bundle, pval = --', 'dynamic microtubule bundle, pval = --', 'static microtubule bundle, pval = --', 'ventral disc overlap zone, pval = --', 'axonemal heterotrimeric kinesin-II complex, pval = --', 'axonemal nexin link, pval = --', 'axonemal central apparatus, pval = --', 'axonemal dynein complex, pval = --', 'axonemal microtubule, pval = --', 'distal portion of axoneme, pval = --', 'proximal portion of axoneme, pval = --', 'radial spoke, pval = --', 'axonemal microtubule depolymerization, pval = --', 'satellite fibril, pval = --', 'axonemal basal plate, pval = --', 'ciliary shaft, pval = --', 'axonemal outer doublet, pval = --', 'microtubule plus-end, pval = --', 'mitotic spindle astral microtubule end, pval = --', 'microtubule minus-end, pval = --', 'post-anaphase array microtubule end, pval = --', 'astral microtubule, pval = --', 'mitotic spindle microtubule, pval = --', 'polar microtubule, pval = --', 'kinetochore microtubule, pval = --', 'cytoplasmic microtubule plus-end, pval = --', 'cortical microtubule, pval = --', 'cytoplasmic microtubule minus-end, pval = --']
level 3: ['ventral disc, pval = --', 'axonemal central pair projection, pval = --', 'axonemal central bridge, pval = --', 'axonemal central pair, pval = --', 'inner dynein arm, pval = --', 'outer dynein arm, pval = --', 'A axonemal microtubule, pval = --', 'B axonemal microtubule, pval = --', 'C1 axonemal microtubule, pval = --', 'C2 axonemal microtubule, pval = --', 'radial spoke stalk, pval = --', 'radial spoke head, pval = --', 'negative regulation of axonemal microtubule depolymerization, pval = --', '9+0 motile cilium, pval = --', '9+2 motile cilium, pval = 0.43974', 'meiotic spindle astral microtubule, pval = --', 'mitotic spindle astral microtubule, pval = --', 'mitotic spindle kinetochore microtubule, pval = --', 'mitotic spindle polar microtubule, pval = --', 'cortical microtubule plus-end, pval = --', 'cortical microtubule, transverse to long axis, pval = --']
level 4: ['ventral disc lateral crest, pval = --', 'ventral disc dorsal microribbon, pval = --', 'ventral disc crossbridge, pval = --', '9+2 non-motile cilium, pval = 0.43974', 'paraflagellar rod, pval = --', 'transverse flagellum, pval = --', 'longitudinal flagellum, pval = --', 'sperm flagellum, pval = 0.43974', 'right ventral flagellum, pval = --', 'right posteriolateral flagellum, pval = --', 'left ventral flagellum, pval = --', 'left anterior flagellum, pval = --', 'right anterior flagellum, pval = --', 'left posteriolateral flagellum, pval = --', 'right caudal flagellum, pval = --', 'left caudal flagellum, pval = --']
level 5: ['kinocilium, pval = 0.43974', 'outer dense fiber, pval = --', 'sperm fibrous sheath, pval = --', 'sperm connecting piece, pval = --', 'sperm annulus, pval = --', 'sperm principal piece, pval = --', 'sperm end piece, pval = --', 'sperm midpiece, pval = --', 'sperm mitochondrial sheath, pval = --', 'right ventral basal body, pval = --', 'right posteriolateral basal body, pval = --', 'left ventral basal body, pval = --', 'left anterior basal body, pval = --', 'right anterior basal body, pval = --', 'left posteriolateral basal body, pval = --', 'right caudal basal body, pval = --', 'left caudal basal body, pval = --']
level 6: ['kinociliary basal body, pval = 0.43974', 'right lateral basal body pair, pval = --', 'right middle basal body pair, pval = --', 'left lateral basal body pair, pval = --', 'left middle basal body pair, pval = --']
level 7: ['right tetrad, pval = --', 'left tetrad, pval = --']" id="microtubules" init-pval="0.08911" min-pval="0.08911" min-pval-children="0.43974" title="Microtubules">
<g>
<path d="M26.1,331c-0.1-0.2-0.3-0.4-0.8-0.6
		c-2.8-0.7-5.4,5.4-1.7,3.1C24.2,333.2,26.3,331.8,26.1,331z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M523.7,61.8c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C523.6,64.7,524.4,62.3,523.7,61.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M451.6,313.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C451.6,316.8,452.3,314.4,451.6,313.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M169.5,362.4c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C169.4,365.3,170.2,362.9,169.5,362.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M562.5,372.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C562.4,375.8,563.2,373.4,562.5,372.9z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M123.5,227.4c-0.1,0.5,0,1,0.2,1.5
		c0.5,1,3.9,4.6,3.5,1.6C127.1,229.8,124.1,224.8,123.5,227.4z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M103.3,216.7c-0.1-0.2-0.5-0.5-1.2-0.7
		c-0.7-0.2-4.6-0.3-4.2,1.2c0.3,1.1,2.4,0.9,3.2,0.8C102.3,217.9,103.6,217.3,103.3,216.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M350.9,43.1c-0.6-0.6-5.7-2.9-6.1-1.7
		c-0.3,0.7,3,1.7,3.6,1.9C349.2,43.7,352.6,45,350.9,43.1z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M287.9,88.7c-2.3,0.5,0.9,7.6,2.5,5.3
		c0.7-1,0.3-2.7-0.3-3.6C289.9,89.9,288.7,88.5,287.9,88.7z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.9,57c-2.1-0.1,3,6.6,3.4,3.1
		C242.6,58.5,240.3,57,238.9,57z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
</g>
<g class="ton microtubule_ends" descendants="level 0: ['microtubule end, pval = --']
level 1: ['microtubule plus-end, pval = --', 'mitotic spindle astral microtubule end, pval = --', 'microtubule minus-end, pval = --', 'post-anaphase array microtubule end, pval = --']
level 2: ['cytoplasmic microtubule plus-end, pval = --', 'cytoplasmic microtubule minus-end, pval = --']
level 3: ['cortical microtubule plus-end, pval = --']" id="microtubule_ends_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Microtubule ends">
<path d="M355.4,108c-1.1-0.1-5.2,0.6-3,2
		C353.8,110.9,358.7,108.4,355.4,108z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M320.1,54.3c0.2,0,0.4-0.2,0.5-0.7
		c0.3-1.1-0.1-2.6-0.6-3.6c-0.4-0.7-1.9-2.5-2.5-0.9C317,50.2,318.9,54.5,320.1,54.3z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M217.8,110.3c-1.9,0.2-3,1.4-0.8,2.3
		C219.5,113.6,220.7,110,217.8,110.3z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M295.5,135c0.8-1.2,1.2-2.7-0.2-3.2
		c-1.8-0.7-4.6,4.7-2.2,4.8C293.8,136.6,294.8,135.9,295.5,135z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M390.8,76.5c0.3-1.5-4.4-2.4-4.5-0.6
		c-0.1,1.7,3.2,2.3,4.2,1.1C390.7,76.8,390.7,76.7,390.8,76.5z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M245,154.3c0.1-0.2,0-0.5-0.3-0.9
		c-1.8-2.2-7.6,1.1-3.2,1.5C242.2,154.9,244.7,155.1,245,154.3z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M267.1,75c-0.5,0-1,0.2-1.4,0.6
		c-0.9,0.7-3.5,4.9-0.8,3.8C265.6,79,269.8,74.9,267.1,75z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M272.7,52.8c0.2-0.2,0.4-0.6,0.4-1.3
		c0-0.7-0.8-4.5-2.1-3.8c-1,0.5-0.3,2.6-0.1,3.3C271.3,52.1,272.2,53.2,272.7,52.8z" style="fill:#17e8ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M558.5,272.6c0.7-0.4,3.9-5,2.8-5.7
		c-0.7-0.4-2.2,2.6-2.6,3.2C558.3,270.8,556.3,274,558.5,272.6z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M525.6,202.2c-0.1-2.3-7.6-0.5-5.7,1.5
		c0.8,0.9,2.6,0.8,3.6,0.4C524.1,203.8,525.7,203,525.6,202.2z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M566,160c0.5-2.1-7.1,1.7-3.7,2.8
		C563.9,163.3,565.7,161.4,566,160z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M494,264.8c0.3-1,0.4-5.2-1.4-3.3
		C491.4,262.7,493,267.9,494,264.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M539.1,244.8c0,0.2,0.2,0.4,0.6,0.7
		c1,0.6,2.6,0.4,3.6,0.1c0.8-0.3,2.8-1.4,1.4-2.3C543.6,242.6,539.1,243.7,539.1,244.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M517.7,129.2c0.1-1.9-0.8-3.2-2.1-1.2
		C514.2,130.3,517.5,132.2,517.7,129.2z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M478.8,200.8c1,1.1,2.4,1.6,3.2,0.4
		c1-1.7-3.7-5.4-4.3-3.1C477.5,198.9,478,200,478.8,200.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.2,305.5c1.4,0.6,3.2-3.8,1.5-4.3
		c-1.6-0.4-2.8,2.7-1.9,3.9C517.9,305.3,518.1,305.4,518.2,305.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M469.3,147.7c0.2,0.1,0.5,0.1,1-0.1
		c2.5-1.3,0.3-7.7-0.9-3.4C469.2,144.8,468.6,147.2,469.3,147.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M427.9,76.5c0,0.2,0.2,0.5,0.6,0.7
		c2.5,1.4,6.6-3.9,2.4-2.6C430.3,74.9,427.9,75.7,427.9,76.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M70.3,341.6c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C72.8,343,71.1,341.2,70.3,341.6z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M352.8,409.1c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C355.3,410.5,353.6,408.7,352.8,409.1z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M543.1,184.3c0.1-0.5,0-1-0.3-1.5
		c-0.6-1-4.2-4.4-3.6-1.5C539.4,182.1,542.6,187,543.1,184.3z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M563.8,194c0.1,0.2,0.5,0.5,1.2,0.6
		c0.7,0.2,4.6,0.1,4.1-1.4c-0.3-1.1-2.5-0.8-3.3-0.7C564.7,192.7,563.5,193.4,563.8,194z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col microtubule_ends" descendants="level 0: ['microtubule end, pval = --']
level 1: ['microtubule plus-end, pval = --', 'mitotic spindle astral microtubule end, pval = --', 'microtubule minus-end, pval = --', 'post-anaphase array microtubule end, pval = --']
level 2: ['cytoplasmic microtubule plus-end, pval = --', 'cytoplasmic microtubule minus-end, pval = --']
level 3: ['cortical microtubule plus-end, pval = --']" id="microtubule_ends" init-pval="--" min-pval="--" min-pval-children="--" title="Microtubule ends">
<path d="M412.8,449c0.7,0.5,6.2,1.5,6.3,0.3
		c0.1-0.8-3.3-1-4-1.1C414.4,448.1,410.8,447.5,412.8,449z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M464.1,390.6c2.1-1-2.6-7.2-3.7-4.6
		c-0.5,1.1,0.3,2.7,1.1,3.5C462,389.8,463.4,390.9,464.1,390.6z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.9,410.6c2.1-0.4-4.4-5.8-4-2.2
		C515.1,409.9,517.6,410.8,518.9,410.6z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M394.1,386.8c1.1-0.1,5-1.7,2.5-2.6
		C394.9,383.5,390.8,387.1,394.1,386.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M430.5,420.1c-0.2,0.1-0.3,0.3-0.4,0.8
		c-0.1,1.2,0.6,2.5,1.4,3.4c0.6,0.6,2.4,2,2.7,0.3C434.4,423.3,431.5,419.6,430.5,420.1z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M527.7,353.9c1.8-0.6,2.6-2,0.3-2.4
		C525.3,351.1,524.9,354.9,527.7,353.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M407.2,354.4c1.8-0.6,2.6-2,0.3-2.4
		C404.8,351.6,404.4,355.4,407.2,354.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M165.7,328.9c1.8-0.6,2.6-2,0.3-2.4
		C163.3,326.1,162.9,329.9,165.7,328.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M446.5,347.1c-0.6,1.3-0.5,2.9,0.9,3.1
		c1.9,0.3,3.5-5.6,1.1-5.2C447.7,345.2,446.9,346.1,446.5,347.1z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366.6,425.4c0,1.5,4.8,1.4,4.5-0.4
		c-0.2-1.7-3.6-1.5-4.4-0.2C366.6,425,366.6,425.2,366.6,425.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M491.3,317c0,0.2,0.1,0.5,0.5,0.8
		c2.2,1.8,7.1-2.8,2.8-2.2C493.9,315.8,491.5,316.2,491.3,317z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M487.5,399.3c0.5-0.1,0.9-0.5,1.2-0.9
		c0.7-0.9,2.3-5.6-0.1-3.9C488,395,484.9,400,487.5,399.3z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.9,422.2c-0.2,0.2-0.2,0.6-0.1,1.3
		c0.1,0.7,1.7,4.2,2.9,3.2c0.9-0.8-0.3-2.6-0.7-3.3C488.5,422.5,487.3,421.7,486.9,422.2z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M152.6,427c0.5,0.7,5.4,3.4,5.9,2.3
		c0.3-0.7-2.8-1.9-3.4-2.3C154.4,426.6,151.1,424.9,152.6,427z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M219.7,387.7c2.3-0.3-0.2-7.6-2-5.6
		c-0.8,0.9-0.5,2.7-0.1,3.6C217.9,386.3,218.9,387.8,219.7,387.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M265.4,424c2.1,0.3-2.3-6.9-3.1-3.4
		C261.9,422.2,264,423.9,265.4,424z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.5,362c1.1,0.2,5.2-0.1,3.2-1.7
		C156.3,359.2,151.2,361.2,154.5,362z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,405.1c-0.2,0-0.4,0.2-0.6,0.6
		c-0.5,1.1-0.2,2.6,0.3,3.6c0.3,0.8,1.6,2.6,2.4,1.2C181.2,409.4,179.6,405,178.5,405.1z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M301.2,384.9c1.9,0,3.1-1.1,1-2.2
		C299.8,381.5,298.2,385,301.2,384.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,244.2c1.9,0,3.1-1.1,1-2.2
		C177.1,240.8,175.5,244.2,178.5,244.2z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M181,176c1.9,0,3.1-1.1,1-2.2
		C179.6,172.6,178,176.1,181,176z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M356.6,27.9c1.9,0,3.1-1.1,1-2.2
		C355.2,24.4,353.6,27.9,356.6,27.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton secreted_proteins" descendants="level 0: [None]" id="secreted_proteins_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Secreted proteins">
<path d="M486.2,164.9c1.9,0,3.1-1.1,1-2.2
		C484.8,161.5,483.2,165,486.2,164.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M470.2,85.4c1.9,0,3.1-1.1,1-2.2
		C468.8,82,467.2,85.5,470.2,85.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M509.5,281.9c1.9,0,3.1-1.1,1-2.2
		C508.1,278.5,506.5,281.9,509.5,281.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M216.7,340.9c-1,1.1-1.4,2.6-0.1,3.2
		c1.7,0.9,5.1-4.2,2.7-4.6C218.5,339.5,217.5,340.1,216.7,340.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M100.6,356.7c-0.4,1.5,4.1,2.8,4.4,1.1
		c0.3-1.7-3-2.6-4.1-1.5C100.7,356.4,100.6,356.6,100.6,356.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M268.8,326.6c-0.1,0.2-0.1,0.5,0.2,0.9
		c1.6,2.4,7.6-0.4,3.3-1.2C271.6,326.2,269.2,325.8,268.8,326.6z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M239.1,403.4c0.5,0,1-0.1,1.5-0.5
		c0.9-0.6,4-4.5,1.1-3.7C241,399.5,236.4,403.2,239.1,403.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M231.4,424.9c-0.2,0.1-0.4,0.5-0.5,1.2
		c-0.1,0.7,0.3,4.6,1.8,4c1.1-0.4,0.6-2.5,0.4-3.3C232.7,425.7,231.9,424.6,231.4,424.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M112.5,138.4c-0.8,0.4-4.1,4.8-3.1,5.5
		c0.7,0.4,2.3-2.5,2.7-3.1C112.6,140.3,114.8,137.2,112.5,138.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M141.8,210.5c0,2.3,7.6,0.9,5.8-1.2
		c-0.8-0.9-2.6-0.9-3.6-0.6C143.4,208.9,141.8,209.6,141.8,210.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M99.4,250.5c-0.6,2,7.2-1.3,3.8-2.6
		C101.7,247.4,99.7,249.2,99.4,250.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M176.6,149.5c-0.4,1-0.7,5.2,1.2,3.4
		C179.1,151.8,177.8,146.5,176.6,149.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M130.5,167.2c0-0.2-0.1-0.5-0.5-0.7
		c-1-0.6-2.5-0.6-3.6-0.3c-0.8,0.2-2.8,1.2-1.5,2.2C125.9,169.2,130.4,168.3,130.5,167.2z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M146.1,283.8c-0.2,1.9,0.7,3.2,2,1.3
		C149.7,282.9,146.5,280.8,146.1,283.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,214.2c-0.9-1.1-2.3-1.8-3.2-0.6
		c-1.1,1.6,3.5,5.6,4.1,3.3C189.7,216.1,189.3,215,188.5,214.2z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.4,107.7c-1.4-0.6-3.4,3.7-1.7,4.2
		c1.6,0.5,3-2.6,2.1-3.8C154.7,107.9,154.6,107.7,154.4,107.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M195.3,267.7c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C195.3,270.6,196.1,268.2,195.3,267.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M63.6,300.5c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C63.6,303.3,64.3,301,63.6,300.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M26.1,331c-0.1-0.2-0.3-0.4-0.8-0.6
		c-2.8-0.7-5.4,5.4-1.7,3.1C24.2,333.2,26.3,331.8,26.1,331z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M523.7,61.8c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C523.6,64.7,524.4,62.3,523.7,61.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M451.6,313.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C451.6,316.8,452.3,314.4,451.6,313.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M169.5,362.4c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C169.4,365.3,170.2,362.9,169.5,362.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M562.5,372.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C562.4,375.8,563.2,373.4,562.5,372.9z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M123.5,227.4c-0.1,0.5,0,1,0.2,1.5
		c0.5,1,3.9,4.6,3.5,1.6C127.1,229.8,124.1,224.8,123.5,227.4z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M103.3,216.7c-0.1-0.2-0.5-0.5-1.2-0.7
		c-0.7-0.2-4.6-0.3-4.2,1.2c0.3,1.1,2.4,0.9,3.2,0.8C102.3,217.9,103.6,217.3,103.3,216.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M350.9,43.1c-0.6-0.6-5.7-2.9-6.1-1.7
		c-0.3,0.7,3,1.7,3.6,1.9C349.2,43.7,352.6,45,350.9,43.1z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M287.9,88.7c-2.3,0.5,0.9,7.6,2.5,5.3
		c0.7-1,0.3-2.7-0.3-3.6C289.9,89.9,288.7,88.5,287.9,88.7z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.9,57c-2.1-0.1,3,6.6,3.4,3.1
		C242.6,58.5,240.3,57,238.9,57z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M355.4,108c-1.1-0.1-5.2,0.6-3,2
		C353.8,110.9,358.7,108.4,355.4,108z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M320.1,54.3c0.2,0,0.4-0.2,0.5-0.7
		c0.3-1.1-0.1-2.6-0.6-3.6c-0.4-0.7-1.9-2.5-2.5-0.9C317,50.2,318.9,54.5,320.1,54.3z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M217.8,110.3c-1.9,0.2-3,1.4-0.8,2.3
		C219.5,113.6,220.7,110,217.8,110.3z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M295.5,135c0.8-1.2,1.2-2.7-0.2-3.2
		c-1.8-0.7-4.6,4.7-2.2,4.8C293.8,136.6,294.8,135.9,295.5,135z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M390.8,76.5c0.3-1.5-4.4-2.4-4.5-0.6
		c-0.1,1.7,3.2,2.3,4.2,1.1C390.7,76.8,390.7,76.7,390.8,76.5z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M245,154.3c0.1-0.2,0-0.5-0.3-0.9
		c-1.8-2.2-7.6,1.1-3.2,1.5C242.2,154.9,244.7,155.1,245,154.3z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M267.1,75c-0.5,0-1,0.2-1.4,0.6
		c-0.9,0.7-3.5,4.9-0.8,3.8C265.6,79,269.8,74.9,267.1,75z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M272.7,52.8c0.2-0.2,0.4-0.6,0.4-1.3
		c0-0.7-0.8-4.5-2.1-3.8c-1,0.5-0.3,2.6-0.1,3.3C271.3,52.1,272.2,53.2,272.7,52.8z" style="fill:#17e8ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M433.2,118.5c0.4-1.4,1.6-2.4,3.3-1.4
		c2.7,1.6,0.7,6.8-2.2,4.7C433.2,121,432.9,119.6,433.2,118.5z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M126.4,341.3c0.1-0.8,0.7-1.6,1.6-1.7
		c1.7-0.3,3,1.5,2.5,3.1c-0.5,1.8-2.6,2.5-3.7,0.9C126.4,342.9,126.2,342.1,126.4,341.3z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M351.6,60.2c-0.1-2.7,5.3-4.1,5.3,0.4
		C356.9,65.2,351.7,63.1,351.6,60.2z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M453.6,74.5c0.7-3.3,6.3-3.6,4.8,1.6
		C456.9,81,452.8,78.5,453.6,74.5z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518,196.7c1.5-3.1,5.7-1.8,5.2,1.5
		c-0.7,3.8-5.6,3.3-5.4-0.5C517.8,197.3,517.9,196.9,518,196.7z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M313.2,421.4c0.2-2.8,3.5-3.6,4.4-0.7
		C318.7,424.1,312.9,426,313.2,421.4z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M391.5,442.8c0.9-2.8,4.4-1.7,3.7,1.2
		C394.2,447.7,390.7,445.4,391.5,442.8z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M215.9,456.1c-1.9-1.4,0-5.5,2.6-3.6
		C221,454.4,218.3,457.8,215.9,456.1z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col secreted_proteins" descendants="level 0: [None]" id="secreted_proteins" init-pval="--" min-pval="--" min-pval-children="--" title="Secreted proteins">
<path d="M56.3,242.3c-1.8-2.7,1.7-6.2,4-2.7
		C62.7,243.1,58.2,245.3,56.3,242.3z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.8,82.3c-1.8-2.7,1.7-6.2,4-2.7
		C245.2,83.1,240.7,85.3,238.8,82.3z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M568,295.7c-1.8-2.7,1.7-6.2,4-2.7
		C574.4,296.5,569.9,298.7,568,295.7z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M186.1,403.1c0.1,0,0.1,0,0.2-0.1
		c2.2-0.7,4.8,3,2,4C186,408,183.5,404.1,186.1,403.1z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M124.8,207.8c0.3-0.4,0.7-0.8,1.2-1
		c3.2-1.1,4.3,4,1.2,5C124.7,212.7,123.7,209.6,124.8,207.8z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M190.7,113.3c0.6-1,2.1-1.8,3.5-1
		c2.4,1.2,1,4.9-1.7,4.3C190.2,116,190,114.4,190.7,113.3z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M405,50.6c-0.3-3.2,4.2-4.7,4.9-1
		C410.5,53.4,405.3,54,405,50.6z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M590.3,366.4c-1.7-1.8-0.1-6.4,2.7-3.4
		C595.2,365.4,592.8,369,590.3,366.4z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M545.9,231.4c-0.9-2.6,3.8-6,4.8-2
		c0.7,2.6-2.5,5-4.5,2.7C546.1,231.8,546,231.6,545.9,231.4z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M514.4,318c-0.3-0.7-0.2-1.6,0.7-2.4
		c2.2-2.2,5.8,1.1,3.6,3.3C517.3,320.2,515,319.4,514.4,318z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M542.9,277.5c-0.7-2.5,3-3.5,3.7-0.8
		c0.7,2.5-2.8,3.4-3.7,1C543,277.6,543,277.6,542.9,277.5z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M560.5,343.6c-0.9-2.5,2.8-4.8,3.9-1.4
		C565.6,345.6,561.5,346.5,560.5,343.6z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M534.9,400.4c-1.9-2.4,0.9-6.1,3.4-3.5
		C540.9,399.7,537,403.1,534.9,400.4z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M465.3,410.1c-0.4-0.8-0.3-1.8,0.5-2.8
		c1.9-2.1,5.7,0.1,4.2,2.8C468.7,412.6,466.1,411.8,465.3,410.1z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M460.1,437.9c1.4-2.7,5.9-0.8,4,2.5
		C462.1,443.9,458.5,440.8,460.1,437.9z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M278.2,443.2c0.1-3.1,4.9-3.5,4.6,0.8
		C282.4,448.7,278.1,446.2,278.2,443.2z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M221.7,404.5c0.4-3.3,4.4-2.3,4.3,0.6
		C225.8,408.2,221.4,407.8,221.7,404.5C221.7,404.5,221.7,404.5,221.7,404.5z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M127,387.8c1.7,0.7,2.1,4,0.3,4.5
		c-1.9,0.5-4.5-3.9-1.3-4.6C126.4,387.6,126.7,387.7,127,387.8z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M51.3,383.2c0.1-0.1,0.2-0.2,0.3-0.3
		c2.4-1.9,5.1,2.2,3.2,3.9C53,388.5,49.4,385.3,51.3,383.2z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M45.5,327.8c-0.7-1.2,0.5-3.1,2.3-3
		c0.1,0,0.3,0,0.4,0c2.5,0.4,2.4,4.4-0.9,4C46.4,328.7,45.8,328.3,45.5,327.8z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M148.6,139.1c0.1-0.4,0.5-0.7,1-0.9
		c1.6-0.5,3.4,2,2.4,3.3C150.7,143.6,148,140.8,148.6,139.1z" style="fill:#7a85ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M433.2,118.5c0.4-1.4,1.6-2.4,3.3-1.4
		c2.7,1.6,0.7,6.8-2.2,4.7C433.2,121,432.9,119.6,433.2,118.5z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M126.4,341.3c0.1-0.8,0.7-1.6,1.6-1.7
		c1.7-0.3,3,1.5,2.5,3.1c-0.5,1.8-2.6,2.5-3.7,0.9C126.4,342.9,126.2,342.1,126.4,341.3z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M351.6,60.2c-0.1-2.7,5.3-4.1,5.3,0.4
		C356.9,65.2,351.7,63.1,351.6,60.2z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M453.6,74.5c0.7-3.3,6.3-3.6,4.8,1.6
		C456.9,81,452.8,78.5,453.6,74.5z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518,196.7c1.5-3.1,5.7-1.8,5.2,1.5
		c-0.7,3.8-5.6,3.3-5.4-0.5C517.8,197.3,517.9,196.9,518,196.7z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M313.2,421.4c0.2-2.8,3.5-3.6,4.4-0.7
		C318.7,424.1,312.9,426,313.2,421.4z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M391.5,442.8c0.9-2.8,4.4-1.7,3.7,1.2
		C394.2,447.7,390.7,445.4,391.5,442.8z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M215.9,456.1c-1.9-1.4,0-5.5,2.6-3.6
		C221,454.4,218.3,457.8,215.9,456.1z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M56.3,242.3c-1.8-2.7,1.7-6.2,4-2.7
		C62.7,243.1,58.2,245.3,56.3,242.3z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.8,82.3c-1.8-2.7,1.7-6.2,4-2.7
		C245.2,83.1,240.7,85.3,238.8,82.3z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M568,295.7c-1.8-2.7,1.7-6.2,4-2.7
		C574.4,296.5,569.9,298.7,568,295.7z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M186.1,403.1c0.1,0,0.1,0,0.2-0.1
		c2.2-0.7,4.8,3,2,4C186,408,183.5,404.1,186.1,403.1z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M124.8,207.8c0.3-0.4,0.7-0.8,1.2-1
		c3.2-1.1,4.3,4,1.2,5C124.7,212.7,123.7,209.6,124.8,207.8z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M190.7,113.3c0.6-1,2.1-1.8,3.5-1
		c2.4,1.2,1,4.9-1.7,4.3C190.2,116,190,114.4,190.7,113.3z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M405,50.6c-0.3-3.2,4.2-4.7,4.9-1
		C410.5,53.4,405.3,54,405,50.6z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M590.3,366.4c-1.7-1.8-0.1-6.4,2.7-3.4
		C595.2,365.4,592.8,369,590.3,366.4z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M545.9,231.4c-0.9-2.6,3.8-6,4.8-2
		c0.7,2.6-2.5,5-4.5,2.7C546.1,231.8,546,231.6,545.9,231.4z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M514.4,318c-0.3-0.7-0.2-1.6,0.7-2.4
		c2.2-2.2,5.8,1.1,3.6,3.3C517.3,320.2,515,319.4,514.4,318z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M542.9,277.5c-0.7-2.5,3-3.5,3.7-0.8
		c0.7,2.5-2.8,3.4-3.7,1C543,277.6,543,277.6,542.9,277.5z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M560.5,343.6c-0.9-2.5,2.8-4.8,3.9-1.4
		C565.6,345.6,561.5,346.5,560.5,343.6z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M534.9,400.4c-1.9-2.4,0.9-6.1,3.4-3.5
		C540.9,399.7,537,403.1,534.9,400.4z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M465.3,410.1c-0.4-0.8-0.3-1.8,0.5-2.8
		c1.9-2.1,5.7,0.1,4.2,2.8C468.7,412.6,466.1,411.8,465.3,410.1z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M460.1,437.9c1.4-2.7,5.9-0.8,4,2.5
		C462.1,443.9,458.5,440.8,460.1,437.9z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton lipid_droplets vesicles" descendants="level 0: ['lipid droplet, pval = 0.68643']
level 1: ['ascus lipid droplet, pval = --', 'monolayer-surrounded lipid storage body, pval = --']
level 2: ['monolayer-surrounded lipid storage body outer lipid monolayer, pval = --']" id="lipid_droplets_ton" init-pval="0.68643" min-pval="0.68643" min-pval-children="--" title="Lipid droplets">
<path d="M278.2,443.2c0.1-3.1,4.9-3.5,4.6,0.8
		C282.4,448.7,278.1,446.2,278.2,443.2z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M221.7,404.5c0.4-3.3,4.4-2.3,4.3,0.6
		C225.8,408.2,221.4,407.8,221.7,404.5C221.7,404.5,221.7,404.5,221.7,404.5z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M127,387.8c1.7,0.7,2.1,4,0.3,4.5
		c-1.9,0.5-4.5-3.9-1.3-4.6C126.4,387.6,126.7,387.7,127,387.8z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M51.3,383.2c0.1-0.1,0.2-0.2,0.3-0.3
		c2.4-1.9,5.1,2.2,3.2,3.9C53,388.5,49.4,385.3,51.3,383.2z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M45.5,327.8c-0.7-1.2,0.5-3.1,2.3-3
		c0.1,0,0.3,0,0.4,0c2.5,0.4,2.4,4.4-0.9,4C46.4,328.7,45.8,328.3,45.5,327.8z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M148.6,139.1c0.1-0.4,0.5-0.7,1-0.9
		c1.6-0.5,3.4,2,2.4,3.3C150.7,143.6,148,140.8,148.6,139.1z" style="fill:#7a85ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M409.3,174.2c20.1,21.7,35.6,44.5,27.5,92.6
		c-8,48.1-45.2,57.7-60.9,60.6c-15.6,2.8-71.8,11.1-96.7-14.1c-25-25.2-32.7-45.4-22.9-88c9.8-42.6,42.6-64.8,71-71.2
		C355.7,147.7,389,152.3,409.3,174.2z" style="fill:#7986ff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M409.3,174.2c20.1,21.7,35.6,44.5,27.5,92.6
		c-8,48.1-45.2,57.7-60.9,60.6c-15.6,2.8-71.8,11.1-96.7-14.1c-25-25.2-32.7-45.4-22.9-88c9.8-42.6,42.6-64.8,71-71.2
		C355.7,147.7,389,152.3,409.3,174.2z" style="fill:#7986ff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M281.3,410.3c-2.6,5.5-13.8,2.8-23.1-3
		c-12.2-7.7,0.2-26.3,11-15.6C277.5,399.9,283.6,405.3,281.3,410.3z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M463.6,106c13.9-0.7,23.1,0.5,32.1,17.7
		c5.9,11.2-7,14.4-10.6,7.1c-5.2-10.5-9.4-13.4-21.9-12.6C454.7,118.8,452.5,106.5,463.6,106z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495,340.9c-11,17,0.4,13.3-11.7,30.9
		c-6.5,9.5-16.4,0.6-12.6-6.1c10.6-18.6,0.8-16.5,12.6-32.5C488.4,326.1,500,333.3,495,340.9z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M132.5,351.2c11.1,0.8,14.3,1,22.2-5.3
		c7.1-5.7,12.3,3.3,6.4,9.4c-6.8,7.1-16,11.7-27.4,8.8C123,361.3,122.8,350.5,132.5,351.2z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M324,73.1c5.8-12,16.3-4.1,19.1,2.7
		c2.8,6.8,0.4,12.7-7.8,13C326,89,318.8,83.8,324,73.1z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M483.2,333.2c-11.8,16.1-2,13.9-12.6,32.5
		c-3.8,6.6,6.1,15.6,12.6,6.1c12.1-17.6,0.7-13.9,11.7-30.9C500,333.3,488.4,326.1,483.2,333.2z M478,353.2
		c-0.1-0.7,0.1-1.4,0.3-2.1c0.2-0.8,0.3-1.7,0.5-2.5c0.2-0.8,0.7-1.4,1.7-1.3c0.4,0.1,0.7,0.2,1.1,0.3c0.9,0.2,1.8,0.5,2.8,0.7
		c0.9,0.2,2.8,0.4,2.8-1c0-0.8-0.8-1.2-1.4-1.4c-0.8-0.3-1.5-0.6-2.3-0.9c-1.3-0.5-3.2-1.2-3-2.9c0-0.5,0.3-1.1,0.6-1.5
		c0.5-0.7,1.3-1.2,2.1-1c0.7,0.2,1.3,0.7,2,1c0.5,0.3,1.1,0.1,1.4-0.4c0.3-0.5,0.1-1.2-0.4-1.4c-0.1-0.1-0.3-0.2-0.4-0.2
		c-0.8-0.4-2.4-1-1.9-2.2c0.3-0.7,1-1.1,1.7-0.9c0.7,0.2,1.2,0.8,1.9,1.2c0.5,0.3,1.1,0.1,1.4-0.3c0.3-0.5,0.1-1.1-0.3-1.4
		c-0.4-0.3-0.9-0.6-1.3-0.9c-0.6-0.4-0.6-0.9,0-1.4c0.3-0.2,0.7-0.3,1.1-0.3c1.5-0.2,3.1,0.3,4.3,1.3c0.5,0.4,0.9,0.9,1.2,1.5
		c0.2,0.3,0.3,0.7,0.4,1c0.2,0.8,0.3,1.7-0.3,2.4c-0.6,0.6-1.5,0-2.1-0.2c-0.3-0.1-0.9-0.1-1.1,0.2c-0.5,0.4-0.5,1.1,0,1.5
		c0.4,0.3,1,0.6,1.2,1c0.2,0.4,0.2,0.9-0.1,1.3c-0.2,0.4-0.5,0.8-0.9,1c-0.9,0.5-2-0.3-2.8-0.6c-0.5-0.2-1.2,0-1.4,0.5
		c-0.2,0.4-0.1,0.9,0.3,1.2c0.1,0.1,0.2,0.1,0.2,0.1c0.7,0.3,1.5,0.7,1.8,1.4c0.3,0.7,0.1,1.5-0.1,2.2c-0.2,0.7-0.4,1.5-1.1,1.9
		c-0.5,0.3-1.2,0.4-1.9,0.3c-0.6-0.1-1.2-0.2-1.7-0.3c-0.6-0.1-1.1,0.3-1.2,0.9c-0.1,0.4,0.1,0.8,0.4,1c0.3,0.2,0.7,0.3,1.1,0.3
		c0.5,0.1,0.9,0.1,1.4,0.2c0.5,0,1.1,0,1.5,0.4c0.4,0.4,0.5,1.1,0.3,1.6c-0.6,1.3-2.4,0.1-2.9,1.3c-0.2,0.4-0.1,0.8,0.1,1.2
		c0.2,0.3,0.6,0.4,1,0.5c0.4,0,0.8,0.1,1.2,0.3c1,0.6,0.5,2.3-0.5,2.6c-0.7,0.2-1.4-0.2-2-0.4c-0.5-0.2-1.1,0.1-1.4,0.6
		c-0.2,0.4,0,0.9,0.3,1.2c0.6,0.5,1.7,0.3,2.1,1c0.3,0.5,0.1,1.1-0.3,1.5c-0.6,0.4-1.3,0.3-1.9,0.1c-0.6-0.2-1.4-0.6-1.9,0.1
		c-0.7,1,0.4,1.5,1.1,1.9c0.3,0.2,0.7,0.5,0.8,0.9c0.2,0.6-0.2,1.3-0.5,1.8c-0.4,0.7-1,1.3-1.7,1.7c-0.7,0.4-1.6,0-2.2-0.4
		c-0.3-0.2-0.7-0.5-1-0.7c-0.5-0.3-1.1-0.2-1.5,0.3c-0.2,0.2-0.2,0.6-0.1,0.9c0.3,0.9,2.6,1.1,2.1,2.4c-0.5,1.3-2.5,0.9-3.4,0.4
		c-0.4-0.2-0.8-0.4-1.2-0.7c-0.4-0.4-0.8-0.8-1.1-1.3c-0.3-0.4-0.5-0.9-0.6-1.3c-0.5-1.4-0.3-3.1,1.2-3.9c0.7-0.4,1.4,0.1,2,0.4
		c0.9,0.4,1.7,0.9,2.5,1.5c0.5,0.3,1.1,0.2,1.5-0.3c0.3-0.5,0.2-1.1-0.3-1.5c-0.3-0.2-0.6-0.4-0.9-0.5c-0.7-0.4-1.3-0.7-2-1.1
		c-0.6-0.3-1.2-0.6-1.5-1.2c-0.3-0.5-0.1-1.2,0.3-1.6c0.5-0.6,1.2-0.7,1.9-0.4c0.6,0.2,1.1,0.4,1.7,0.7c0.5,0.3,1.1,0,1.4-0.5
		c0.3-0.5,0-1.2-0.5-1.4c-0.3-0.2-0.7-0.3-1-0.4c-0.6-0.3-1.5-0.5-1.7-1.3c-0.1-0.4,0.2-0.9,0.6-1.1c0.4-0.3,1-0.2,1.4-0.1
		c0.5,0.1,1.1,0.3,1.6,0.5c1.3,0.5,2-1.5,0.8-2c-1-0.4-1.9-0.9-2.5-1.7C478.2,354.1,478.1,353.7,478,353.2z" style="fill:#40bfff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.7,345.9c-7.9,6.3-11.1,6.1-22.2,5.3
		c-9.6-0.7-9.4,10.1,1.2,12.9c11.3,2.9,20.6-1.7,27.4-8.8C167,349.1,161.7,340.2,154.7,345.9z M161.3,349.1c-0.4,0.1-1-0.5-1.2-0.8
		c-0.3-0.3-0.7-0.4-1.1-0.2c-0.2,0.1-0.3,0.2-0.4,0.3c-0.3,0.4-0.3,0.9-0.1,1.3c0.2,0.5,0.7,0.8,1,1.3c0.4,0.6,0.6,1.3,0.5,1.9
		c-0.2,1.1-1.4,2.2-2.4,2.7c-0.3,0.2-0.7,0.2-1.1,0.1c-0.7-0.2-1.1-0.8-1.6-1.3c-0.5-0.6-1.2-1.3-2-1c-0.1,0-0.1,0.1-0.2,0.1
		c-1.1,0.7-0.2,1.9,0.3,2.6c0.5,0.7,0.9,1.6,0.6,2.5c-0.2,0.6-0.8,1-1.4,1.3c-0.7,0.4-1.4,0.7-2.1,1c-0.8,0.5-2.2,1.2-2.8,0
		c-0.2-0.4-0.3-0.9-0.4-1.3c-0.2-0.6-0.4-1.5-1.2-1.4c-1.4,0.1-1.2,1.8-0.8,2.7c0.2,0.5,0.3,1.2-0.2,1.5c-0.4,0.2-1,0.2-1.4,0
		c-0.9-0.5-0.6-2-0.7-2.9c-0.1-0.6-0.7-1-1.3-0.8c-0.1,0-0.1,0-0.2,0.1c-1,0.4-0.6,1.3-0.6,2.1c0.1,1.1-0.7,2.1-1.8,2.2
		c-0.7,0-1.4-0.1-2.1-0.3c-1.6-0.4-2.3-2.3-1.7-3.8c0,0,0.1-0.6,0.1-0.6c0.2-0.6-0.2-1.2-0.8-1.3c-0.2-0.1-0.5,0-0.7,0
		c-0.3,0.1-0.6,0.4-0.7,0.7l-0.1,0.6c-0.2,0.8-0.2,2-1.1,2.3c-0.5,0.2-1.1,0-1.6-0.3c-0.7-0.5-1.4-1.3-1.9-2c-0.4-0.6-0.6-1.3-0.7-2
		c0-0.6,0.1-1.2,0.4-1.6c0.3-0.4,0.8-0.7,1.3-0.4c0.5,0.4,0.5,1.1,0.5,1.6c0,0,0,1,1,1.2c1.2,0.2,1.4-1.5,1.7-2.2
		c0.2-0.4,0.5-0.9,0.8-1.2c0.4-0.4,1-0.5,1.6-0.5c0.7,0,1.3,0.4,1.6,1c0.4,0.7,0.3,1.5,0.3,2.3c0,0.7,0,1.6,0.7,2.1
		c0.5,0.3,1.3,0.3,1.7-0.1c0.4-0.4,0.5-1,0.6-1.6c0.1-0.9,0.2-1.8,1-2.4c0.7-0.5,1.5-0.7,2.3-0.8c1-0.2,2.4-0.3,3.3,0.4
		c0.5,0.4,0.9,1,1.2,1.6c0.4,0.7,0.8,1.5,1.1,2.3c0.2,0.6,0.3,1.5,1.1,1.7c0.6,0.2,1.2-0.3,1.3-0.9c0.2-0.7-0.2-1.4-0.4-2
		c-0.3-0.7-0.6-1.4-0.9-2.1c-0.4-1-0.6-2,0.2-2.8c0.4-0.5,1-0.9,1.6-1.1c0.3-0.1,0.6-0.2,1-0.3c1.3-0.2,2.2,0.8,2.9,1.7
		c0.4,0.5,1.1,0.5,1.5,0.2c0.5-0.4,0.5-1.1,0.2-1.5c-0.4-0.5-1.3-1.2-1.4-1.9c-0.1-1.3,1.2-2.2,2.2-2.6c1.7-0.7,2.7,0.1,3.2,0.8
		C161.2,347.5,162,348.9,161.3,349.1z" style="fill:#40bfff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M335.3,88.7c8.2-0.3,10.6-6.2,7.8-13
		c-2.8-6.8-13.3-14.6-19.1-2.7C318.8,83.8,326,89,335.3,88.7z M324.5,80c-0.1-1.8,0.6-3.6,1.2-5.3c0.3-0.8,0.7-1.6,1.2-2.4
		c0.5-0.7,1.1-1.6,1.8-2.1c0.4-0.3,1.3-0.7,1.7-0.1c0.3,0.4,0.3,1,0.1,1.5c-0.2,0.7-0.6,1.2-0.9,1.9c-0.4,1.1-1,2.2-1.4,3.4
		c-0.2,0.6,0.1,1.3,0.8,1.5c0.1,0,0.3,0.1,0.4,0.1c0.5,0,1-0.3,1.1-0.8c0.2-0.7,0.5-1.4,0.9-2.1c0.6-1.1,1.2-2.3,1.9-3.4
		c0.3-0.4,0.6-0.8,1.1-0.9c0.8-0.3,1.7,0,2.2,0.7c0.8,1.2-0.2,2.6-0.8,3.6c-0.6,0.9-1.1,1.8-1.4,2.3c-0.3,0.6-0.1,1.3,0.4,1.6
		c0.2,0.1,0.4,0.2,0.6,0.2c0.4,0,0.8-0.2,1-0.6c0.5-0.8,1-1.6,1.4-2.4c0.3-0.6,0.6-1.4,1.5-1.4c0.8,0,1.3,0.7,1.6,1.4
		c1.2,2.9,1.2,5.6,0.1,7.3c-0.3,0.5-0.9,1.5-1.6,1.4c-1.2-0.2-0.3-2-0.4-2.7c-0.1-0.4-0.4-0.7-0.8-0.9c-0.6-0.2-1.3,0.1-1.5,0.8
		c-0.2,0.7-0.5,1.4-0.8,2c-0.4,0.8-1,1.4-1.9,1.4c-0.4,0-0.9-0.1-1.2-0.4c-0.2-0.2-0.5-0.8-0.5-1.2c0.1-1.3,1.1-2.8-0.2-3.8
		c-0.2-0.2-0.6-0.3-0.9-0.3c-1,0.2-1.4,1.7-2,2.5c-0.9,1.3-2.6,1-3.6,0c0,0-0.1-0.1-0.1-0.1C324.8,81.9,324.5,80.9,324.5,80z" style="fill:#40bfff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M258.2,407.3c9.3,5.9,20.6,8.6,23.1,3
		c2.3-5-3.7-10.4-12.1-18.6C258.4,381,246,399.6,258.2,407.3z M258,404.6c-0.7-0.3-0.3-1.2,0-1.7c0.6-1,1.3-1.9,1.9-2.9
		c0.3-0.5,0.2-1.2-0.4-1.5c-0.5-0.3-1.2-0.2-1.5,0.4c-0.3,0.5-1.5,3.1-2.4,2.1c-0.4-0.4-0.4-1.1-0.4-1.6c0-0.7-0.1-1.4,0.1-2.1
		c0.1-0.3,0.1-0.6,0.2-1c0.3-1.1,0.9-2.2,1.6-3.1c0.8-1,1.9-1.7,3.1-2.2c0.3-0.1,1.4-0.5,1.3,0.2c-0.1,0.5-0.7,1-1,1.4
		c-0.5,0.6-1.3,1.4-1.2,2.2c0.1,0.6,0.8,1.1,1.4,0.8c0.6-0.2,0.9-1,1.2-1.5c0.5-0.8,1.2-1.5,1.8-2.2c0.3-0.4,0.7-0.6,1.1-0.6
		c0.6,0,1.3,0.3,1.7,0.9c0.2,0.3,0.2,0.7,0.1,1c-0.2,0.7-0.8,1.2-1.3,1.7c-0.5,0.5-1.2,1.1-0.8,1.8c0.3,0.7,1.3,0.9,1.8,0.3
		c0.6-0.7,1.2-1.6,2-2c0.7-0.4,1.4,0.1,1.8,0.8c0.8,1.3-0.8,2.3-1.5,3.2c-0.5,0.6-0.9,1.2-1.3,1.8c-0.7,1-1.4,2-1.9,3
		c-0.3,0.6,0,1.2,0.5,1.5c0.2,0.1,0.3,0.1,0.5,0.1c0.4,0,0.8-0.2,1-0.6c0.2-0.5,0.5-0.9,0.7-1.3c1-1.4,1.9-2.9,3.1-4.1
		c0.4-0.4,0.7-0.8,1.3-0.8c0.5,0,0.9,0.3,1.1,0.7c0.5,1-0.2,2-0.7,2.9c-0.3,0.4-0.7,0.9-0.8,1.4c-0.2,0.6,0.1,1.2,0.7,1.4
		c0.1,0,0.2,0.1,0.3,0.1c0.7,0,1.1-0.8,1.4-1.3c0.3-0.5,0.9-1.5,1.6-1.5c0.4,0,0.7,0.3,1,0.6c0.4,0.4,0.5,1.1,0.5,1.6
		c-0.1,0.6-0.6,0.9-0.9,1.4c-0.5,0.7-1,1.4-1.3,2.2c-0.2,0.6,0,1.2,0.6,1.5c0.1,0.1,0.3,0.1,0.4,0.1c0.7,0,0.9-0.6,1.2-1.1
		c0.2-0.5,0.7-1.3,1.3-1.3c0.7-0.1,0.9,1,0.8,1.5c-0.2,2.1-2.9,2.7-4.6,2.7c-0.6,0-1.2-0.1-1.9-0.1c-0.6-0.1-1.2,0-1.7-0.4
		c-0.7-0.4-0.4-1.1-0.5-1.8c-0.2-0.7-0.8-1.2-1.6-0.9c-0.6,0.3-0.9,1.1-1.6,1.3c-0.6,0.2-1.3,0-1.9-0.2c-0.8-0.4-1.6-0.8-2.4-1.1
		c-0.8-0.4-2-0.7-2.2-1.7c-0.2-0.8,0.5-1.5,1-2.1c0.4-0.4,0.4-1.1,0-1.6c-0.4-0.4-1.1-0.4-1.6,0c-0.5,0.5-0.8,1.1-1.2,1.5
		c-0.2,0.2-0.5,0.4-0.9,0.5C258.5,404.7,258.3,404.7,258,404.6z" style="fill:#40bfff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495.7,123.6c-9.1-17.2-18.3-18.4-32.1-17.7
		c-11,0.5-8.9,12.8-0.4,12.3c12.5-0.8,16.7,2,21.9,12.6C488.7,138,501.6,134.9,495.7,123.6z M491.5,132.8c-0.9,0.1-1.8-0.2-2.6-0.6
		c-0.4-0.2-0.8-0.5-1.1-0.8c-0.5-0.6-0.7-1.2-0.1-1.7c0.4-0.4,1-0.5,1.4-0.8c0.5-0.3,0.7-0.8,0.4-1.4c-0.3-0.5-0.9-0.5-1.4-0.3
		c-0.7,0.3-1.6,1-2.4,0.5c-0.8-0.4-0.3-1.5,0.1-2c0.5-0.6,1.2-0.9,1.7-1.4c0.3-0.2,0.5-0.6,0.4-1c0-0.1-0.1-0.3-0.2-0.4
		c-0.4-0.5-1.1-0.4-1.6,0c-0.6,0.4-1.2,1-1.9,1.3c-0.5,0.2-1-0.1-1.2-0.6c-0.2-0.5,0.1-1.1,0.4-1.6c0.3-0.4,0.7-0.7,0.6-1.3
		c0-0.2-0.1-0.3-0.3-0.4c-0.9-0.8-1.6,0.4-2.4,0.6c-1,0.3-1.1-0.9-1-1.6c0.1-0.5,0.5-1.1,0.4-1.6c-0.2-0.9-1.2-0.8-1.8-0.4
		c-0.5,0.4-1,0.7-1.7,0.7c-0.6,0-1.1-0.3-1.7-0.5c-0.8-0.3-1.6-0.6-2.4-0.7c-0.6-0.1-1.1-0.2-1.7-0.3c-0.5-0.1-1.1-0.1-1.5-0.4
		c-0.4-0.4-0.2-0.9-0.3-1.4c-0.1-0.4-0.5-0.8-1-0.7c-0.3,0-0.7,0.3-0.9,0.6c-0.2,0.4-0.2,1-0.5,1.4c-0.3,0.3-0.8,0.4-1.2,0.4
		c-0.5-0.1-0.9-0.3-1.1-0.7c-0.4-0.8-0.1-3.1-1.6-2.7c-0.5,0.1-0.9,0.7-0.7,1.2c0.2,0.8,0.5,2-0.7,2.1c-1.2,0.1-2.5-0.6-3.2-1.5
		c-0.4-0.5-0.6-1.1-0.8-1.7c-0.2-0.7-0.2-1.5,0-2.2c0.1-0.4,0.5-1.3,1.2-1.1c0.5,0.2,0.5,1.1,0.5,1.6c0,0.5,0.5,0.9,1.1,0.9
		c0.6,0,1-0.5,1-1c0-0.8-0.4-1.7-0.2-2.5c0.2-1,1.9-1.1,2.6-0.7c0.9,0.5,0.6,1.8,1,2.6c0.4,1,2,0.6,1.9-0.5c0-0.6-0.2-1.4,0-2
		c0.1-0.3,0.3-0.5,0.6-0.5c0.4-0.1,0.9,0,1.2,0.3c0.3,0.4,0.1,1,0,1.4c-0.1,0.5,0.4,1,0.9,1c1.1,0.1,1-1.1,1.4-1.7
		c0.2-0.3,0.5-0.5,0.9-0.6c0.6-0.2,1.4-0.2,1.7,0.5c0.3,0.7,0.1,1.5,0,2.3c-0.1,0.5-0.2,1-0.2,1.4c-0.1,0.5,0.3,1,0.8,1.1
		c0.6,0.1,1.1-0.3,1.2-0.8c0.2-0.9,0.2-2,0.6-2.9c0.4-0.7,1.3-0.9,2.1-0.8c1.3,0.2,0.7,1.9,0.4,2.7c-0.3,0.8-0.6,1.5-0.8,2.3
		c-0.1,0.3-0.2,0.5-0.3,0.8c-0.2,0.5,0.1,1.1,0.7,1.2c0.8,0.2,1.2-0.4,1.4-1c0.3-0.8,0.5-1.6,0.7-2.4c0.2-0.6,0.4-1.4,1.1-1.6
		c0.4-0.1,0.9-0.1,1.3,0c0.5,0.2,1.1,0.6,1,1.2c-0.1,1-2.1,2.7-0.7,3.5c0.3,0.2,0.8,0.2,1.1,0c0.6-0.4,0.9-1.3,1.8-1.4
		c0.8-0.1,1.5,0.6,1.7,1.3c0.4,1.2-1.3,2-1.4,3.1c-0.1,1,1.1,1.7,1.8,0.8c0.4-0.5,1-1.2,1.7-1.1c0.6,0.1,1,0.5,1.3,1
		c0.3,0.5,0.4,1,0.2,1.6c-0.3,0.8-1.9,1.4-1.4,2.4c0.2,0.5,0.8,0.7,1.3,0.5c0.6-0.3,1.1-0.6,1.8-0.3c1.1,0.4,1.5,2.2,0.3,2.8
		c-0.6,0.3-1.2,0.4-1.1,1.2c0.1,0.5,0.6,0.9,1.1,0.8c0.7-0.1,1.5-0.4,1.9,0.4c0.2,0.4,0.2,1.1,0.1,1.6c-0.3,1.1-1.2,1.9-2.2,2.4
		C492.5,132.7,492,132.8,491.5,132.8z" style="fill:#40bfff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z M143.7,244.4c-0.1,0-0.3,0-0.5,0c-1.3,0.3-1.5,1.3-0.6,2.2
		c1,1.1-0.4,3.7-1.9,2.9c-0.6-0.3-0.9-0.8-1.4-1.2c-0.5-0.4-1.1-0.5-1.6,0.2c-0.5,0.5-0.2,1.1,0.2,1.6c0.6,0.6,1.6,1.3,1.4,2.3
		c-0.2,1-1.4,0.9-2.2,0.7c-0.8-0.2-1.4-0.8-2.1-1.3c-0.4-0.2-0.9-0.2-1.2,0c-0.1,0.1-0.2,0.2-0.3,0.3c-0.4,0.7,0,1.3,0.5,1.7
		c0.7,0.5,1.5,0.8,2,1.5c0.4,0.5,0.2,1.1-0.2,1.4c-0.5,0.4-1.2,0.4-1.8,0.2c-0.6-0.2-1-0.5-1.6-0.1c-0.2,0.1-0.3,0.3-0.4,0.4
		c-0.6,1.2,1,1.5,1.5,2.3c0.7,1-0.6,1.5-1.4,1.6c-0.6,0.1-1.3-0.1-1.8,0.2c-0.8,0.6-0.5,1.5,0.2,2c0.6,0.4,1.1,0.8,1.3,1.5
		c0.2,0.6,0.1,1.3,0,2c0,0.9,0,1.9,0.1,2.8c0.1,0.6,0.2,1.2,0.2,1.9c0.1,0.5,0.3,1.2,0.1,1.7c-0.2,0.6-0.9,0.6-1.4,0.9
		c-0.4,0.3-0.6,0.9-0.4,1.4c0.1,0.3,0.6,0.6,0.9,0.7c0.5,0,1.1-0.2,1.6,0c0.5,0.2,0.7,0.7,0.8,1.1c0.1,0.5,0,1.1-0.4,1.4
		c-0.7,0.8-3.3,1.3-2.3,2.8c0.3,0.5,1,0.7,1.5,0.3c0.8-0.5,1.9-1.3,2.5-0.1c0.6,1.2,0.2,2.8-0.4,3.9c-0.4,0.6-0.9,1.1-1.5,1.5
		c-0.7,0.4-1.5,0.8-2.4,0.8c-0.5,0-1.6,0-1.6-0.8c0-0.6,1-0.9,1.5-1.1c0.6-0.2,0.8-0.9,0.6-1.5c-0.2-0.6-0.9-0.8-1.5-0.6
		c-0.9,0.4-1.7,1.1-2.6,1.2c-1.2,0.2-1.9-1.5-1.7-2.5c0.2-1.2,1.7-1.3,2.4-2c0.9-0.8-0.1-2.3-1.2-1.8c-0.7,0.3-1.4,0.8-2.1,0.7
		c-0.3,0-0.6-0.1-0.8-0.4c-0.2-0.4-0.4-1-0.1-1.3c0.3-0.5,1-0.4,1.5-0.5c0.6-0.1,0.9-0.8,0.8-1.3c-0.3-1.2-1.5-0.6-2.3-0.8
		c-0.4-0.1-0.7-0.4-1-0.7c-0.5-0.6-0.7-1.4-0.1-1.9c0.7-0.6,1.6-0.7,2.4-0.9c0.6-0.1,1.1-0.2,1.5-0.3c0.6-0.1,1-0.7,0.9-1.3
		c-0.1-0.6-0.7-1-1.3-0.9c-1,0.2-2.2,0.6-3.3,0.5c-0.9-0.1-1.5-1-1.6-1.9c-0.2-1.4,1.8-1.5,2.8-1.5c0.9,0,1.8,0,2.8,0
		c0.3,0,0.6,0,0.9,0c0.6,0,1.1-0.6,1-1.2c-0.1-0.9-0.8-1.1-1.6-1.1c-0.9,0-1.9,0.1-2.8,0.2c-0.7,0-1.7,0.2-2.1-0.5
		c-0.2-0.3-0.4-0.9-0.4-1.3c0-0.6,0.3-1.3,0.9-1.5c1.1-0.3,3.6,1.1,4-0.7c0.1-0.4-0.1-0.9-0.4-1.2c-0.7-0.5-1.7-0.5-2.2-1.3
		c-0.4-0.8,0.1-1.8,0.8-2.3c1.2-0.8,2.5,0.5,3.8,0.3c1.1-0.2,1.4-1.8,0.2-2.2c-0.7-0.2-1.6-0.5-1.8-1.3c-0.1-0.6,0.2-1.3,0.6-1.8
		c0.4-0.5,1-0.9,1.6-0.9c1,0,2.1,1.5,3.1,0.5c0.4-0.4,0.5-1.1,0.1-1.5c-0.5-0.5-1-1-1-1.8c0.1-1.4,1.8-2.4,2.9-1.4
		c0.7,0.7,2.3,1.3,2.6-0.2c0.1-1-1.3-1.7-0.7-2.8c1.2-2.3,5.5,2,5.4-1c0-0.9-1-1.7-0.9-2.6c0.2-1.9,3.3-0.6,4.1,0
		c0.8,0.6,1.4,1.6,1.5,2.7C147.3,243.5,145.6,244.4,143.7,244.4z" style="fill:#40bfff;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M281.3,410.3c-2.6,5.5-13.8,2.8-23.1-3
		c-12.2-7.7,0.2-26.3,11-15.6C277.5,399.9,283.6,405.3,281.3,410.3z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M463.6,106c13.9-0.7,23.1,0.5,32.1,17.7
		c5.9,11.2-7,14.4-10.6,7.1c-5.2-10.5-9.4-13.4-21.9-12.6C454.7,118.8,452.5,106.5,463.6,106z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495,340.9c-11,17,0.4,13.3-11.7,30.9
		c-6.5,9.5-16.4,0.6-12.6-6.1c10.6-18.6,0.8-16.5,12.6-32.5C488.4,326.1,500,333.3,495,340.9z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M132.5,351.2c11.1,0.8,14.3,1,22.2-5.3
		c7.1-5.7,12.3,3.3,6.4,9.4c-6.8,7.1-16,11.7-27.4,8.8C123,361.3,122.8,350.5,132.5,351.2z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M324,73.1c5.8-12,16.3-4.1,19.1,2.7
		c2.8,6.8,0.4,12.7-7.8,13C326,89,318.8,83.8,324,73.1z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M483.2,333.2c-11.8,16.1-2,13.9-12.6,32.5
		c-3.8,6.6,6.1,15.6,12.6,6.1c12.1-17.6,0.7-13.9,11.7-30.9C500,333.3,488.4,326.1,483.2,333.2z M478,353.2
		c-0.1-0.7,0.1-1.4,0.3-2.1c0.2-0.8,0.3-1.7,0.5-2.5c0.2-0.8,0.7-1.4,1.7-1.3c0.4,0.1,0.7,0.2,1.1,0.3c0.9,0.2,1.8,0.5,2.8,0.7
		c0.9,0.2,2.8,0.4,2.8-1c0-0.8-0.8-1.2-1.4-1.4c-0.8-0.3-1.5-0.6-2.3-0.9c-1.3-0.5-3.2-1.2-3-2.9c0-0.5,0.3-1.1,0.6-1.5
		c0.5-0.7,1.3-1.2,2.1-1c0.7,0.2,1.3,0.7,2,1c0.5,0.3,1.1,0.1,1.4-0.4c0.3-0.5,0.1-1.2-0.4-1.4c-0.1-0.1-0.3-0.2-0.4-0.2
		c-0.8-0.4-2.4-1-1.9-2.2c0.3-0.7,1-1.1,1.7-0.9c0.7,0.2,1.2,0.8,1.9,1.2c0.5,0.3,1.1,0.1,1.4-0.3c0.3-0.5,0.1-1.1-0.3-1.4
		c-0.4-0.3-0.9-0.6-1.3-0.9c-0.6-0.4-0.6-0.9,0-1.4c0.3-0.2,0.7-0.3,1.1-0.3c1.5-0.2,3.1,0.3,4.3,1.3c0.5,0.4,0.9,0.9,1.2,1.5
		c0.2,0.3,0.3,0.7,0.4,1c0.2,0.8,0.3,1.7-0.3,2.4c-0.6,0.6-1.5,0-2.1-0.2c-0.3-0.1-0.9-0.1-1.1,0.2c-0.5,0.4-0.5,1.1,0,1.5
		c0.4,0.3,1,0.6,1.2,1c0.2,0.4,0.2,0.9-0.1,1.3c-0.2,0.4-0.5,0.8-0.9,1c-0.9,0.5-2-0.3-2.8-0.6c-0.5-0.2-1.2,0-1.4,0.5
		c-0.2,0.4-0.1,0.9,0.3,1.2c0.1,0.1,0.2,0.1,0.2,0.1c0.7,0.3,1.5,0.7,1.8,1.4c0.3,0.7,0.1,1.5-0.1,2.2c-0.2,0.7-0.4,1.5-1.1,1.9
		c-0.5,0.3-1.2,0.4-1.9,0.3c-0.6-0.1-1.2-0.2-1.7-0.3c-0.6-0.1-1.1,0.3-1.2,0.9c-0.1,0.4,0.1,0.8,0.4,1c0.3,0.2,0.7,0.3,1.1,0.3
		c0.5,0.1,0.9,0.1,1.4,0.2c0.5,0,1.1,0,1.5,0.4c0.4,0.4,0.5,1.1,0.3,1.6c-0.6,1.3-2.4,0.1-2.9,1.3c-0.2,0.4-0.1,0.8,0.1,1.2
		c0.2,0.3,0.6,0.4,1,0.5c0.4,0,0.8,0.1,1.2,0.3c1,0.6,0.5,2.3-0.5,2.6c-0.7,0.2-1.4-0.2-2-0.4c-0.5-0.2-1.1,0.1-1.4,0.6
		c-0.2,0.4,0,0.9,0.3,1.2c0.6,0.5,1.7,0.3,2.1,1c0.3,0.5,0.1,1.1-0.3,1.5c-0.6,0.4-1.3,0.3-1.9,0.1c-0.6-0.2-1.4-0.6-1.9,0.1
		c-0.7,1,0.4,1.5,1.1,1.9c0.3,0.2,0.7,0.5,0.8,0.9c0.2,0.6-0.2,1.3-0.5,1.8c-0.4,0.7-1,1.3-1.7,1.7c-0.7,0.4-1.6,0-2.2-0.4
		c-0.3-0.2-0.7-0.5-1-0.7c-0.5-0.3-1.1-0.2-1.5,0.3c-0.2,0.2-0.2,0.6-0.1,0.9c0.3,0.9,2.6,1.1,2.1,2.4c-0.5,1.3-2.5,0.9-3.4,0.4
		c-0.4-0.2-0.8-0.4-1.2-0.7c-0.4-0.4-0.8-0.8-1.1-1.3c-0.3-0.4-0.5-0.9-0.6-1.3c-0.5-1.4-0.3-3.1,1.2-3.9c0.7-0.4,1.4,0.1,2,0.4
		c0.9,0.4,1.7,0.9,2.5,1.5c0.5,0.3,1.1,0.2,1.5-0.3c0.3-0.5,0.2-1.1-0.3-1.5c-0.3-0.2-0.6-0.4-0.9-0.5c-0.7-0.4-1.3-0.7-2-1.1
		c-0.6-0.3-1.2-0.6-1.5-1.2c-0.3-0.5-0.1-1.2,0.3-1.6c0.5-0.6,1.2-0.7,1.9-0.4c0.6,0.2,1.1,0.4,1.7,0.7c0.5,0.3,1.1,0,1.4-0.5
		c0.3-0.5,0-1.2-0.5-1.4c-0.3-0.2-0.7-0.3-1-0.4c-0.6-0.3-1.5-0.5-1.7-1.3c-0.1-0.4,0.2-0.9,0.6-1.1c0.4-0.3,1-0.2,1.4-0.1
		c0.5,0.1,1.1,0.3,1.6,0.5c1.3,0.5,2-1.5,0.8-2c-1-0.4-1.9-0.9-2.5-1.7C478.2,354.1,478.1,353.7,478,353.2z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.7,345.9c-7.9,6.3-11.1,6.1-22.2,5.3
		c-9.6-0.7-9.4,10.1,1.2,12.9c11.3,2.9,20.6-1.7,27.4-8.8C167,349.1,161.7,340.2,154.7,345.9z M161.3,349.1c-0.4,0.1-1-0.5-1.2-0.8
		c-0.3-0.3-0.7-0.4-1.1-0.2c-0.2,0.1-0.3,0.2-0.4,0.3c-0.3,0.4-0.3,0.9-0.1,1.3c0.2,0.5,0.7,0.8,1,1.3c0.4,0.6,0.6,1.3,0.5,1.9
		c-0.2,1.1-1.4,2.2-2.4,2.7c-0.3,0.2-0.7,0.2-1.1,0.1c-0.7-0.2-1.1-0.8-1.6-1.3c-0.5-0.6-1.2-1.3-2-1c-0.1,0-0.1,0.1-0.2,0.1
		c-1.1,0.7-0.2,1.9,0.3,2.6c0.5,0.7,0.9,1.6,0.6,2.5c-0.2,0.6-0.8,1-1.4,1.3c-0.7,0.4-1.4,0.7-2.1,1c-0.8,0.5-2.2,1.2-2.8,0
		c-0.2-0.4-0.3-0.9-0.4-1.3c-0.2-0.6-0.4-1.5-1.2-1.4c-1.4,0.1-1.2,1.8-0.8,2.7c0.2,0.5,0.3,1.2-0.2,1.5c-0.4,0.2-1,0.2-1.4,0
		c-0.9-0.5-0.6-2-0.7-2.9c-0.1-0.6-0.7-1-1.3-0.8c-0.1,0-0.1,0-0.2,0.1c-1,0.4-0.6,1.3-0.6,2.1c0.1,1.1-0.7,2.1-1.8,2.2
		c-0.7,0-1.4-0.1-2.1-0.3c-1.6-0.4-2.3-2.3-1.7-3.8c0,0,0.1-0.6,0.1-0.6c0.2-0.6-0.2-1.2-0.8-1.3c-0.2-0.1-0.5,0-0.7,0
		c-0.3,0.1-0.6,0.4-0.7,0.7l-0.1,0.6c-0.2,0.8-0.2,2-1.1,2.3c-0.5,0.2-1.1,0-1.6-0.3c-0.7-0.5-1.4-1.3-1.9-2c-0.4-0.6-0.6-1.3-0.7-2
		c0-0.6,0.1-1.2,0.4-1.6c0.3-0.4,0.8-0.7,1.3-0.4c0.5,0.4,0.5,1.1,0.5,1.6c0,0,0,1,1,1.2c1.2,0.2,1.4-1.5,1.7-2.2
		c0.2-0.4,0.5-0.9,0.8-1.2c0.4-0.4,1-0.5,1.6-0.5c0.7,0,1.3,0.4,1.6,1c0.4,0.7,0.3,1.5,0.3,2.3c0,0.7,0,1.6,0.7,2.1
		c0.5,0.3,1.3,0.3,1.7-0.1c0.4-0.4,0.5-1,0.6-1.6c0.1-0.9,0.2-1.8,1-2.4c0.7-0.5,1.5-0.7,2.3-0.8c1-0.2,2.4-0.3,3.3,0.4
		c0.5,0.4,0.9,1,1.2,1.6c0.4,0.7,0.8,1.5,1.1,2.3c0.2,0.6,0.3,1.5,1.1,1.7c0.6,0.2,1.2-0.3,1.3-0.9c0.2-0.7-0.2-1.4-0.4-2
		c-0.3-0.7-0.6-1.4-0.9-2.1c-0.4-1-0.6-2,0.2-2.8c0.4-0.5,1-0.9,1.6-1.1c0.3-0.1,0.6-0.2,1-0.3c1.3-0.2,2.2,0.8,2.9,1.7
		c0.4,0.5,1.1,0.5,1.5,0.2c0.5-0.4,0.5-1.1,0.2-1.5c-0.4-0.5-1.3-1.2-1.4-1.9c-0.1-1.3,1.2-2.2,2.2-2.6c1.7-0.7,2.7,0.1,3.2,0.8
		C161.2,347.5,162,348.9,161.3,349.1z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M335.3,88.7c8.2-0.3,10.6-6.2,7.8-13
		c-2.8-6.8-13.3-14.6-19.1-2.7C318.8,83.8,326,89,335.3,88.7z M324.5,80c-0.1-1.8,0.6-3.6,1.2-5.3c0.3-0.8,0.7-1.6,1.2-2.4
		c0.5-0.7,1.1-1.6,1.8-2.1c0.4-0.3,1.3-0.7,1.7-0.1c0.3,0.4,0.3,1,0.1,1.5c-0.2,0.7-0.6,1.2-0.9,1.9c-0.4,1.1-1,2.2-1.4,3.4
		c-0.2,0.6,0.1,1.3,0.8,1.5c0.1,0,0.3,0.1,0.4,0.1c0.5,0,1-0.3,1.1-0.8c0.2-0.7,0.5-1.4,0.9-2.1c0.6-1.1,1.2-2.3,1.9-3.4
		c0.3-0.4,0.6-0.8,1.1-0.9c0.8-0.3,1.7,0,2.2,0.7c0.8,1.2-0.2,2.6-0.8,3.6c-0.6,0.9-1.1,1.8-1.4,2.3c-0.3,0.6-0.1,1.3,0.4,1.6
		c0.2,0.1,0.4,0.2,0.6,0.2c0.4,0,0.8-0.2,1-0.6c0.5-0.8,1-1.6,1.4-2.4c0.3-0.6,0.6-1.4,1.5-1.4c0.8,0,1.3,0.7,1.6,1.4
		c1.2,2.9,1.2,5.6,0.1,7.3c-0.3,0.5-0.9,1.5-1.6,1.4c-1.2-0.2-0.3-2-0.4-2.7c-0.1-0.4-0.4-0.7-0.8-0.9c-0.6-0.2-1.3,0.1-1.5,0.8
		c-0.2,0.7-0.5,1.4-0.8,2c-0.4,0.8-1,1.4-1.9,1.4c-0.4,0-0.9-0.1-1.2-0.4c-0.2-0.2-0.5-0.8-0.5-1.2c0.1-1.3,1.1-2.8-0.2-3.8
		c-0.2-0.2-0.6-0.3-0.9-0.3c-1,0.2-1.4,1.7-2,2.5c-0.9,1.3-2.6,1-3.6,0c0,0-0.1-0.1-0.1-0.1C324.8,81.9,324.5,80.9,324.5,80z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M258.2,407.3c9.3,5.9,20.6,8.6,23.1,3
		c2.3-5-3.7-10.4-12.1-18.6C258.4,381,246,399.6,258.2,407.3z M258,404.6c-0.7-0.3-0.3-1.2,0-1.7c0.6-1,1.3-1.9,1.9-2.9
		c0.3-0.5,0.2-1.2-0.4-1.5c-0.5-0.3-1.2-0.2-1.5,0.4c-0.3,0.5-1.5,3.1-2.4,2.1c-0.4-0.4-0.4-1.1-0.4-1.6c0-0.7-0.1-1.4,0.1-2.1
		c0.1-0.3,0.1-0.6,0.2-1c0.3-1.1,0.9-2.2,1.6-3.1c0.8-1,1.9-1.7,3.1-2.2c0.3-0.1,1.4-0.5,1.3,0.2c-0.1,0.5-0.7,1-1,1.4
		c-0.5,0.6-1.3,1.4-1.2,2.2c0.1,0.6,0.8,1.1,1.4,0.8c0.6-0.2,0.9-1,1.2-1.5c0.5-0.8,1.2-1.5,1.8-2.2c0.3-0.4,0.7-0.6,1.1-0.6
		c0.6,0,1.3,0.3,1.7,0.9c0.2,0.3,0.2,0.7,0.1,1c-0.2,0.7-0.8,1.2-1.3,1.7c-0.5,0.5-1.2,1.1-0.8,1.8c0.3,0.7,1.3,0.9,1.8,0.3
		c0.6-0.7,1.2-1.6,2-2c0.7-0.4,1.4,0.1,1.8,0.8c0.8,1.3-0.8,2.3-1.5,3.2c-0.5,0.6-0.9,1.2-1.3,1.8c-0.7,1-1.4,2-1.9,3
		c-0.3,0.6,0,1.2,0.5,1.5c0.2,0.1,0.3,0.1,0.5,0.1c0.4,0,0.8-0.2,1-0.6c0.2-0.5,0.5-0.9,0.7-1.3c1-1.4,1.9-2.9,3.1-4.1
		c0.4-0.4,0.7-0.8,1.3-0.8c0.5,0,0.9,0.3,1.1,0.7c0.5,1-0.2,2-0.7,2.9c-0.3,0.4-0.7,0.9-0.8,1.4c-0.2,0.6,0.1,1.2,0.7,1.4
		c0.1,0,0.2,0.1,0.3,0.1c0.7,0,1.1-0.8,1.4-1.3c0.3-0.5,0.9-1.5,1.6-1.5c0.4,0,0.7,0.3,1,0.6c0.4,0.4,0.5,1.1,0.5,1.6
		c-0.1,0.6-0.6,0.9-0.9,1.4c-0.5,0.7-1,1.4-1.3,2.2c-0.2,0.6,0,1.2,0.6,1.5c0.1,0.1,0.3,0.1,0.4,0.1c0.7,0,0.9-0.6,1.2-1.1
		c0.2-0.5,0.7-1.3,1.3-1.3c0.7-0.1,0.9,1,0.8,1.5c-0.2,2.1-2.9,2.7-4.6,2.7c-0.6,0-1.2-0.1-1.9-0.1c-0.6-0.1-1.2,0-1.7-0.4
		c-0.7-0.4-0.4-1.1-0.5-1.8c-0.2-0.7-0.8-1.2-1.6-0.9c-0.6,0.3-0.9,1.1-1.6,1.3c-0.6,0.2-1.3,0-1.9-0.2c-0.8-0.4-1.6-0.8-2.4-1.1
		c-0.8-0.4-2-0.7-2.2-1.7c-0.2-0.8,0.5-1.5,1-2.1c0.4-0.4,0.4-1.1,0-1.6c-0.4-0.4-1.1-0.4-1.6,0c-0.5,0.5-0.8,1.1-1.2,1.5
		c-0.2,0.2-0.5,0.4-0.9,0.5C258.5,404.7,258.3,404.7,258,404.6z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495.7,123.6c-9.1-17.2-18.3-18.4-32.1-17.7
		c-11,0.5-8.9,12.8-0.4,12.3c12.5-0.8,16.7,2,21.9,12.6C488.7,138,501.6,134.9,495.7,123.6z M491.5,132.8c-0.9,0.1-1.8-0.2-2.6-0.6
		c-0.4-0.2-0.8-0.5-1.1-0.8c-0.5-0.6-0.7-1.2-0.1-1.7c0.4-0.4,1-0.5,1.4-0.8c0.5-0.3,0.7-0.8,0.4-1.4c-0.3-0.5-0.9-0.5-1.4-0.3
		c-0.7,0.3-1.6,1-2.4,0.5c-0.8-0.4-0.3-1.5,0.1-2c0.5-0.6,1.2-0.9,1.7-1.4c0.3-0.2,0.5-0.6,0.4-1c0-0.1-0.1-0.3-0.2-0.4
		c-0.4-0.5-1.1-0.4-1.6,0c-0.6,0.4-1.2,1-1.9,1.3c-0.5,0.2-1-0.1-1.2-0.6c-0.2-0.5,0.1-1.1,0.4-1.6c0.3-0.4,0.7-0.7,0.6-1.3
		c0-0.2-0.1-0.3-0.3-0.4c-0.9-0.8-1.6,0.4-2.4,0.6c-1,0.3-1.1-0.9-1-1.6c0.1-0.5,0.5-1.1,0.4-1.6c-0.2-0.9-1.2-0.8-1.8-0.4
		c-0.5,0.4-1,0.7-1.7,0.7c-0.6,0-1.1-0.3-1.7-0.5c-0.8-0.3-1.6-0.6-2.4-0.7c-0.6-0.1-1.1-0.2-1.7-0.3c-0.5-0.1-1.1-0.1-1.5-0.4
		c-0.4-0.4-0.2-0.9-0.3-1.4c-0.1-0.4-0.5-0.8-1-0.7c-0.3,0-0.7,0.3-0.9,0.6c-0.2,0.4-0.2,1-0.5,1.4c-0.3,0.3-0.8,0.4-1.2,0.4
		c-0.5-0.1-0.9-0.3-1.1-0.7c-0.4-0.8-0.1-3.1-1.6-2.7c-0.5,0.1-0.9,0.7-0.7,1.2c0.2,0.8,0.5,2-0.7,2.1c-1.2,0.1-2.5-0.6-3.2-1.5
		c-0.4-0.5-0.6-1.1-0.8-1.7c-0.2-0.7-0.2-1.5,0-2.2c0.1-0.4,0.5-1.3,1.2-1.1c0.5,0.2,0.5,1.1,0.5,1.6c0,0.5,0.5,0.9,1.1,0.9
		c0.6,0,1-0.5,1-1c0-0.8-0.4-1.7-0.2-2.5c0.2-1,1.9-1.1,2.6-0.7c0.9,0.5,0.6,1.8,1,2.6c0.4,1,2,0.6,1.9-0.5c0-0.6-0.2-1.4,0-2
		c0.1-0.3,0.3-0.5,0.6-0.5c0.4-0.1,0.9,0,1.2,0.3c0.3,0.4,0.1,1,0,1.4c-0.1,0.5,0.4,1,0.9,1c1.1,0.1,1-1.1,1.4-1.7
		c0.2-0.3,0.5-0.5,0.9-0.6c0.6-0.2,1.4-0.2,1.7,0.5c0.3,0.7,0.1,1.5,0,2.3c-0.1,0.5-0.2,1-0.2,1.4c-0.1,0.5,0.3,1,0.8,1.1
		c0.6,0.1,1.1-0.3,1.2-0.8c0.2-0.9,0.2-2,0.6-2.9c0.4-0.7,1.3-0.9,2.1-0.8c1.3,0.2,0.7,1.9,0.4,2.7c-0.3,0.8-0.6,1.5-0.8,2.3
		c-0.1,0.3-0.2,0.5-0.3,0.8c-0.2,0.5,0.1,1.1,0.7,1.2c0.8,0.2,1.2-0.4,1.4-1c0.3-0.8,0.5-1.6,0.7-2.4c0.2-0.6,0.4-1.4,1.1-1.6
		c0.4-0.1,0.9-0.1,1.3,0c0.5,0.2,1.1,0.6,1,1.2c-0.1,1-2.1,2.7-0.7,3.5c0.3,0.2,0.8,0.2,1.1,0c0.6-0.4,0.9-1.3,1.8-1.4
		c0.8-0.1,1.5,0.6,1.7,1.3c0.4,1.2-1.3,2-1.4,3.1c-0.1,1,1.1,1.7,1.8,0.8c0.4-0.5,1-1.2,1.7-1.1c0.6,0.1,1,0.5,1.3,1
		c0.3,0.5,0.4,1,0.2,1.6c-0.3,0.8-1.9,1.4-1.4,2.4c0.2,0.5,0.8,0.7,1.3,0.5c0.6-0.3,1.1-0.6,1.8-0.3c1.1,0.4,1.5,2.2,0.3,2.8
		c-0.6,0.3-1.2,0.4-1.1,1.2c0.1,0.5,0.6,0.9,1.1,0.8c0.7-0.1,1.5-0.4,1.9,0.4c0.2,0.4,0.2,1.1,0.1,1.6c-0.3,1.1-1.2,1.9-2.2,2.4
		C492.5,132.7,492,132.8,491.5,132.8z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z M143.7,244.4c-0.1,0-0.3,0-0.5,0c-1.3,0.3-1.5,1.3-0.6,2.2
		c1,1.1-0.4,3.7-1.9,2.9c-0.6-0.3-0.9-0.8-1.4-1.2c-0.5-0.4-1.1-0.5-1.6,0.2c-0.5,0.5-0.2,1.1,0.2,1.6c0.6,0.6,1.6,1.3,1.4,2.3
		c-0.2,1-1.4,0.9-2.2,0.7c-0.8-0.2-1.4-0.8-2.1-1.3c-0.4-0.2-0.9-0.2-1.2,0c-0.1,0.1-0.2,0.2-0.3,0.3c-0.4,0.7,0,1.3,0.5,1.7
		c0.7,0.5,1.5,0.8,2,1.5c0.4,0.5,0.2,1.1-0.2,1.4c-0.5,0.4-1.2,0.4-1.8,0.2c-0.6-0.2-1-0.5-1.6-0.1c-0.2,0.1-0.3,0.3-0.4,0.4
		c-0.6,1.2,1,1.5,1.5,2.3c0.7,1-0.6,1.5-1.4,1.6c-0.6,0.1-1.3-0.1-1.8,0.2c-0.8,0.6-0.5,1.5,0.2,2c0.6,0.4,1.1,0.8,1.3,1.5
		c0.2,0.6,0.1,1.3,0,2c0,0.9,0,1.9,0.1,2.8c0.1,0.6,0.2,1.2,0.2,1.9c0.1,0.5,0.3,1.2,0.1,1.7c-0.2,0.6-0.9,0.6-1.4,0.9
		c-0.4,0.3-0.6,0.9-0.4,1.4c0.1,0.3,0.6,0.6,0.9,0.7c0.5,0,1.1-0.2,1.6,0c0.5,0.2,0.7,0.7,0.8,1.1c0.1,0.5,0,1.1-0.4,1.4
		c-0.7,0.8-3.3,1.3-2.3,2.8c0.3,0.5,1,0.7,1.5,0.3c0.8-0.5,1.9-1.3,2.5-0.1c0.6,1.2,0.2,2.8-0.4,3.9c-0.4,0.6-0.9,1.1-1.5,1.5
		c-0.7,0.4-1.5,0.8-2.4,0.8c-0.5,0-1.6,0-1.6-0.8c0-0.6,1-0.9,1.5-1.1c0.6-0.2,0.8-0.9,0.6-1.5c-0.2-0.6-0.9-0.8-1.5-0.6
		c-0.9,0.4-1.7,1.1-2.6,1.2c-1.2,0.2-1.9-1.5-1.7-2.5c0.2-1.2,1.7-1.3,2.4-2c0.9-0.8-0.1-2.3-1.2-1.8c-0.7,0.3-1.4,0.8-2.1,0.7
		c-0.3,0-0.6-0.1-0.8-0.4c-0.2-0.4-0.4-1-0.1-1.3c0.3-0.5,1-0.4,1.5-0.5c0.6-0.1,0.9-0.8,0.8-1.3c-0.3-1.2-1.5-0.6-2.3-0.8
		c-0.4-0.1-0.7-0.4-1-0.7c-0.5-0.6-0.7-1.4-0.1-1.9c0.7-0.6,1.6-0.7,2.4-0.9c0.6-0.1,1.1-0.2,1.5-0.3c0.6-0.1,1-0.7,0.9-1.3
		c-0.1-0.6-0.7-1-1.3-0.9c-1,0.2-2.2,0.6-3.3,0.5c-0.9-0.1-1.5-1-1.6-1.9c-0.2-1.4,1.8-1.5,2.8-1.5c0.9,0,1.8,0,2.8,0
		c0.3,0,0.6,0,0.9,0c0.6,0,1.1-0.6,1-1.2c-0.1-0.9-0.8-1.1-1.6-1.1c-0.9,0-1.9,0.1-2.8,0.2c-0.7,0-1.7,0.2-2.1-0.5
		c-0.2-0.3-0.4-0.9-0.4-1.3c0-0.6,0.3-1.3,0.9-1.5c1.1-0.3,3.6,1.1,4-0.7c0.1-0.4-0.1-0.9-0.4-1.2c-0.7-0.5-1.7-0.5-2.2-1.3
		c-0.4-0.8,0.1-1.8,0.8-2.3c1.2-0.8,2.5,0.5,3.8,0.3c1.1-0.2,1.4-1.8,0.2-2.2c-0.7-0.2-1.6-0.5-1.8-1.3c-0.1-0.6,0.2-1.3,0.6-1.8
		c0.4-0.5,1-0.9,1.6-0.9c1,0,2.1,1.5,3.1,0.5c0.4-0.4,0.5-1.1,0.1-1.5c-0.5-0.5-1-1-1-1.8c0.1-1.4,1.8-2.4,2.9-1.4
		c0.7,0.7,2.3,1.3,2.6-0.2c0.1-1-1.3-1.7-0.7-2.8c1.2-2.3,5.5,2,5.4-1c0-0.9-1-1.7-0.9-2.6c0.2-1.9,3.3-0.6,4.1,0
		c0.8,0.6,1.4,1.6,1.5,2.7C147.3,243.5,145.6,244.4,143.7,244.4z" style="fill:#40bfff;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col lipid_droplets vesicles" descendants="level 0: ['lipid droplet, pval = 0.68643']
level 1: ['ascus lipid droplet, pval = --', 'monolayer-surrounded lipid storage body, pval = --']
level 2: ['monolayer-surrounded lipid storage body outer lipid monolayer, pval = --']" id="lipid_droplets" init-pval="0.68643" min-pval="0.68643" min-pval-children="--" title="Lipid droplets">
<path d="M429.8,340.9c2.3-2,1.8-7.3-1-8.6
		c-4.4-2-9.5,4.8-6.4,8.3C424.3,342.5,427.9,342.7,429.8,340.9z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M428.9,154.2c1.2-3-2.3-7-5.5-5.3
		C417.9,152.1,426.9,159.5,428.9,154.2z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M461.9,284.3c4.8,0.1,4.4-9.6-1-8.3
		C456.6,276.9,457.6,284.1,461.9,284.3z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M244.6,320.3c4.8,0.1,4.4-9.6-1-8.3
		C239.3,312.9,240.3,320.1,244.6,320.3z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M305.8,128.7c4.8,0.1,4.4-9.6-1-8.3
		C300.5,121.4,301.6,128.6,305.8,128.7z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M424.5,345.6c2-4.3-7-7.8-8-2.3
		C415.7,347.6,422.7,349.5,424.5,345.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M354.2,363.6c2.3-2,1.8-7.3-1-8.6
		c-4.4-2-9.5,4.8-6.4,8.3C348.6,365.2,352.2,365.4,354.2,363.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M338.5,132.6c2-4.3-7-7.8-8-2.3
		C329.7,134.6,336.7,136.5,338.5,132.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M322.8,349.6c2-4.3-7-7.8-8-2.3
		C314,351.6,321,353.5,322.8,349.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M309.4,347.9c5,3,9.2-5.6,5.9-9.1
		C310.4,333.6,303.6,344.5,309.4,347.9z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M471.9,215c5,2.6,9.2-4.8,5.9-7.8
		C472.9,202.7,466.1,212.1,471.9,215z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M384.9,132.7c-2.2-3.4-9.5-1.2-7.5,3.1
		C379.4,140.2,388,137.4,384.9,132.7z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M304,347.2c-3.8,4,3.8,6.8,4.1,1.5
		c0.2-3-2.7-3.8-4.4-1.5" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M433.5,346.3c4.4-0.2,3.6-7.8-2-5.9
		c-4.1,1.4-2.4,5.4,1.3,5.8C433.1,346.3,433.3,346.3,433.5,346.3z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M444.2,304.3c4.4-0.2,3.6-7.8-2-5.9
		c-4.1,1.4-2.4,5.4,1.3,5.8C443.7,304.3,444,304.3,444.2,304.3z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M444.7,176.1c4.4-0.2,3.6-7.8-2-5.9
		c-4.1,1.4-2.4,5.4,1.3,5.8C444.2,176.1,444.5,176.1,444.7,176.1z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M277.7,140.1c4.4-0.2,3.6-7.8-2-5.9
		c-4.1,1.4-2.4,5.4,1.3,5.8C277.2,140.1,277.5,140.1,277.7,140.1z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M289.7,368.1c4.4-0.2,3.6-7.8-2-5.9
		c-4.1,1.4-2.4,5.4,1.3,5.8C289.2,368.1,289.5,368.1,289.7,368.1z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M371.4,369.1c4.4-0.2,3.6-7.8-2-5.9
		c-4.1,1.4-2.4,5.4,1.3,5.8C370.9,369.1,371.1,369.1,371.4,369.1z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M440.6,156.1c-0.6-6.1-8.5-1.6-6.5,2.3
		C436.3,162.5,441,160.4,440.6,156.1z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M264.5,352.2c0.8,3.1,6,4.3,7.6,1
		C274.6,347.3,263,346.7,264.5,352.2z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M254,357.6c4.1,4.6,7.8-3.7,3.8-5.7
		C253.6,349.9,251,354.4,254,357.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M474.5,224.3c2.4-3.2-2.3-9.3-5.6-5.9
		C465.6,221.8,471.2,228.8,474.5,224.3z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M481.3,217.6c-0.6-6.1-8.5-1.6-6.5,2.3
		C477,224,481.7,221.9,481.3,217.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M305.3,129.6c2,4.8-5.7,7.6-8.4,4.3
		C293,129.1,303,124.2,305.3,129.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M314.3,133.1c-3.5,1.7-9-3.1-5.3-5.6
		C312.7,124.8,319.1,130.6,314.3,133.1z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M306.9,138.3c-6-1.3-0.7-7.7,2.9-5.5
		C313.7,135.2,311.1,139.1,306.9,138.3z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M402.9,135.7c-3.5,1.7-9-3.1-5.3-5.6
		C401.4,127.5,407.8,133.3,402.9,135.7z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M394.9,141.6c-6-1.3-0.7-7.7,2.9-5.5
		C401.7,138.6,399.1,142.5,394.9,141.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M307,341.6c-0.6-6.1-8.5-1.6-6.5,2.3
		C302.7,348,307.4,345.9,307,341.6z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M312.9,357.4c6-2,2.9-11.2-2.7-8.5
		C305.6,351,307.5,359.2,312.9,357.4z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M436.7,339.9c6-2,2.9-11.2-2.7-8.5
		C429.4,333.5,431.3,341.7,436.7,339.9z" style="fill:#9F3253;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton lysosomes vesicles" descendants="level 0: ['lysosome, pval = 0.57691']
level 1: ['secondary lysosome, pval = --', 'primary lysosome, pval = 0.68643', 'lysosomal lumen, pval = --', 'lysosomal multienzyme complex, pval = --', 'cytolytic granule, pval = --', 'endolysosome, pval = --', 'lysosomal glycocalyx, pval = --', 'lysosomal protein catabolic process, pval = --', 'megasome, pval = --', 'lysosomal membrane, pval = 0.35378', 'mast cell granule, pval = 0.76316', 'proteolysis within lysosome associated with antigen processing and presentation, pval = --', 'UDP-N-acetylglucosamine-lysosomal-enzyme N-acetylglucosaminephosphotransferase complex, pval = --']
level 2: ['autolysosome, pval = --', 'phagolysosome, pval = --', 'azurophil granule, pval = 0.68643', 'endolysosome lumen, pval = --', 'lysosomal matrix, pval = --', 'acrosomal lumen, pval = --', 'cytolytic granule lumen, pval = --', 'endolysosome membrane, pval = --', 'regulation of lysosomal protein catabolic process, pval = --', 'negative regulation of lysosomal protein catabolic process, pval = --', 'positive regulation of lysosomal protein catabolic process, pval = --', 'extrinsic component of lysosome membrane, pval = --', 'integral component of lysosomal membrane, pval = --', 'lumenal side of lysosomal membrane, pval = --', 'cytoplasmic side of lysosomal membrane, pval = --', 'lysosomal proton-transporting V-type ATPase complex, pval = --', 'azurophil granule membrane, pval = --', 'cytolytic granule membrane, pval = --', 'phagolysosome membrane, pval = --', 'lysosomal HOPS complex, pval = --', 'chaperone-mediated autophagy translocation complex, pval = --', 'mast cell degranulation, pval = --']
level 3: ['food vacuole, pval = --', 'phagolysosome vesicle lumen, pval = --', 'phagolysosome vesicle membrane, pval = --', 'azurophil granule lumen, pval = 0.68643', 'lysosomal proton-transporting V-type ATPase, V0 domain, pval = --', 'lysosomal proton-transporting V-type ATPase, V1 domain, pval = --', 'regulation of mast cell degranulation, pval = --', 'positive regulation of mast cell degranulation, pval = --', 'negative regulation of mast cell degranulation, pval = --', 'histamine secretion by mast cell, pval = --', 'serotonin secretion by mast cell, pval = --']
level 4: ['regulation of histamine secretion by mast cell, pval = --', 'modulation of mast cell degranulation in other organism, pval = --', 'positive regulation of histamine secretion by mast cell, pval = --', 'positive regulation of mast cell degranulation in other organism, pval = --', 'negative regulation of histamine secretion by mast cell, pval = --']
level 5: ['envenomation resulting in modulation of mast cell degranulation in other organism, pval = --', 'envenomation resulting in positive regulation of mast cell degranulation in other organism, pval = --']" id="lysosomes_ton" init-pval="0.57691" min-pval="0.35378" min-pval-children="0.35378" title="Lysosomes">
<path d="M501.2,380.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C502.8,387.1,500.4,383.1,501.2,380.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M187.7,141.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C189.3,148.1,186.9,144.1,187.7,141.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M175,382.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C176.7,386.9,174.8,384.6,175,382.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,226.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C190.2,230.9,188.3,228.6,188.5,226.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M426,364.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C427.7,368.9,425.8,366.6,426,364.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M370.6,108.7c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C372.3,113.4,370.4,111.1,370.6,108.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M235.5,107.7c-2-0.4-2.7-2.8-2.2-4.8
		c0.4-1.7,2.1-2.8,3.8-2.3c3.2,0.9,3.1,6.4-0.2,7.1C236.4,107.7,235.9,107.8,235.5,107.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M180.1,315.4c-2.4-0.3-4.5-3.8-1.7-5.2
		c2.4-1.2,5.9,2.4,3.8,4.6C181.6,315.3,180.8,315.5,180.1,315.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M110.6,298.9c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C115.1,300.2,112.1,300.7,110.6,298.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M77.4,374.2c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C81.8,375.6,78.9,376.1,77.4,374.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M151.6,195.1c0.2-2.2,2.1-4.1,4.6-2.1
		c2.3,1.8,2,7-1.8,6.2C152.2,198.7,151.4,196.8,151.6,195.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M537.2,295.8c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C537.4,301.4,535.8,297.6,537.2,295.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M388.5,55.1c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C388.7,60.7,387.1,56.8,388.5,55.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M473.6,318c1.8-3.7,6.5-1.1,4.7,3
		C476.3,325.4,471.9,321.4,473.6,318z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M440.7,97.8c-0.9-3.1,3.5-7,4.8-2.1
		C446.6,99.6,442.1,102.3,440.7,97.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M418,393c1.3-3.1,7.5-2.5,5.2,2
		C421.4,398.4,416.5,396.4,418,393z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M482.3,140.9c3.6,0.7,4,5.1,0.4,6.2
		C477.8,148.6,477.4,139.9,482.3,140.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col lysosomes vesicles" descendants="level 0: ['lysosome, pval = 0.57691']
level 1: ['secondary lysosome, pval = --', 'primary lysosome, pval = 0.68643', 'lysosomal lumen, pval = --', 'lysosomal multienzyme complex, pval = --', 'cytolytic granule, pval = --', 'endolysosome, pval = --', 'lysosomal glycocalyx, pval = --', 'lysosomal protein catabolic process, pval = --', 'megasome, pval = --', 'lysosomal membrane, pval = 0.35378', 'mast cell granule, pval = 0.76316', 'proteolysis within lysosome associated with antigen processing and presentation, pval = --', 'UDP-N-acetylglucosamine-lysosomal-enzyme N-acetylglucosaminephosphotransferase complex, pval = --']
level 2: ['autolysosome, pval = --', 'phagolysosome, pval = --', 'azurophil granule, pval = 0.68643', 'endolysosome lumen, pval = --', 'lysosomal matrix, pval = --', 'acrosomal lumen, pval = --', 'cytolytic granule lumen, pval = --', 'endolysosome membrane, pval = --', 'regulation of lysosomal protein catabolic process, pval = --', 'negative regulation of lysosomal protein catabolic process, pval = --', 'positive regulation of lysosomal protein catabolic process, pval = --', 'extrinsic component of lysosome membrane, pval = --', 'integral component of lysosomal membrane, pval = --', 'lumenal side of lysosomal membrane, pval = --', 'cytoplasmic side of lysosomal membrane, pval = --', 'lysosomal proton-transporting V-type ATPase complex, pval = --', 'azurophil granule membrane, pval = --', 'cytolytic granule membrane, pval = --', 'phagolysosome membrane, pval = --', 'lysosomal HOPS complex, pval = --', 'chaperone-mediated autophagy translocation complex, pval = --', 'mast cell degranulation, pval = --']
level 3: ['food vacuole, pval = --', 'phagolysosome vesicle lumen, pval = --', 'phagolysosome vesicle membrane, pval = --', 'azurophil granule lumen, pval = 0.68643', 'lysosomal proton-transporting V-type ATPase, V0 domain, pval = --', 'lysosomal proton-transporting V-type ATPase, V1 domain, pval = --', 'regulation of mast cell degranulation, pval = --', 'positive regulation of mast cell degranulation, pval = --', 'negative regulation of mast cell degranulation, pval = --', 'histamine secretion by mast cell, pval = --', 'serotonin secretion by mast cell, pval = --']
level 4: ['regulation of histamine secretion by mast cell, pval = --', 'modulation of mast cell degranulation in other organism, pval = --', 'positive regulation of histamine secretion by mast cell, pval = --', 'positive regulation of mast cell degranulation in other organism, pval = --', 'negative regulation of histamine secretion by mast cell, pval = --']
level 5: ['envenomation resulting in modulation of mast cell degranulation in other organism, pval = --', 'envenomation resulting in positive regulation of mast cell degranulation in other organism, pval = --']" id="lysosomes" init-pval="0.57691" min-pval="0.35378" min-pval-children="0.35378" title="Lysosomes">
<path d="M501.2,380.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C502.8,387.1,500.4,383.1,501.2,380.7z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M187.7,141.7c0.1-0.2,0.2-0.5,0.3-0.7
		c1.1-2.1,3.8-1.7,4.9,0.2c0.8,1.5,0.7,3.9-0.8,4.9C189.3,148.1,186.9,144.1,187.7,141.7z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M175,382.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C176.7,386.9,174.8,384.6,175,382.2z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,226.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C190.2,230.9,188.3,228.6,188.5,226.2z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M426,364.2c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C427.7,368.9,425.8,366.6,426,364.2z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M370.6,108.7c0-0.5,0.2-1.1,0.5-1.6
		c1.1-2.1,4.1-3.7,6-1.7c1.9,2.1,0.4,5.7-2,6.7C372.3,113.4,370.4,111.1,370.6,108.7z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M235.5,107.7c-2-0.4-2.7-2.8-2.2-4.8
		c0.4-1.7,2.1-2.8,3.8-2.3c3.2,0.9,3.1,6.4-0.2,7.1C236.4,107.7,235.9,107.8,235.5,107.7z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M180.1,315.4c-2.4-0.3-4.5-3.8-1.7-5.2
		c2.4-1.2,5.9,2.4,3.8,4.6C181.6,315.3,180.8,315.5,180.1,315.4z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M110.6,298.9c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C115.1,300.2,112.1,300.7,110.6,298.9z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M77.4,374.2c-0.1-0.1-0.2-0.3-0.3-0.4
		c-0.9-1.5-1.1-3.9,0.8-4.7c2.3-0.9,4.3,1.5,4.2,3.7C81.8,375.6,78.9,376.1,77.4,374.2z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M151.6,195.1c0.2-2.2,2.1-4.1,4.6-2.1
		c2.3,1.8,2,7-1.8,6.2C152.2,198.7,151.4,196.8,151.6,195.1z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M537.2,295.8c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C537.4,301.4,535.8,297.6,537.2,295.8z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M388.5,55.1c0.4-0.5,1-0.8,1.9-0.8
		c2.9-0.1,4.4,4.9,1.2,5.7C388.7,60.7,387.1,56.8,388.5,55.1z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M473.6,318c1.8-3.7,6.5-1.1,4.7,3
		C476.3,325.4,471.9,321.4,473.6,318z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M440.7,97.8c-0.9-3.1,3.5-7,4.8-2.1
		C446.6,99.6,442.1,102.3,440.7,97.8z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M418,393c1.3-3.1,7.5-2.5,5.2,2
		C421.4,398.4,416.5,396.4,418,393z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M482.3,140.9c3.6,0.7,4,5.1,0.4,6.2
		C477.8,148.6,477.4,139.9,482.3,140.9z" style="fill:#D06986;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton peroxisomes vesicles" descendants="level 0: ['peroxisome, pval = 0.40992']
level 1: ['P1 peroxisome, pval = --', 'glyoxysome, pval = --', 'P2 peroxisome, pval = --', 'P4 peroxisome, pval = --', 'P3 peroxisome, pval = --', 'P6 peroxisome, pval = --', 'P5 peroxisome, pval = --', 'maintenance of protein location in peroxisome, pval = --', 'peroxisomal membrane, pval = --', 'peroxisomal matrix, pval = 0.43974', 'glycosome, pval = --', 'mannosome, pval = --']
level 2: ['intrinsic component of peroxisomal membrane, pval = --', 'extrinsic component of intraperoxisomal membrane, pval = --', 'glyoxysomal membrane, pval = --', 'glycosome membrane, pval = --', 'glycosome lumen, pval = --', 'glyoxysomal lumen, pval = --']
level 3: ['integral component of peroxisomal membrane, pval = --']" id="peroxisomes_ton" init-pval="0.40992" min-pval="0.40992" min-pval-children="0.43974" title="Peroxisomes">
<path d="M558.5,272.6c0.7-0.4,3.9-5,2.8-5.7
		c-0.7-0.4-2.2,2.6-2.6,3.2C558.3,270.8,556.3,274,558.5,272.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M525.6,202.2c-0.1-2.3-7.6-0.5-5.7,1.5
		c0.8,0.9,2.6,0.8,3.6,0.4C524.1,203.8,525.7,203,525.6,202.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M566,160c0.5-2.1-7.1,1.7-3.7,2.8
		C563.9,163.3,565.7,161.4,566,160z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M494,264.8c0.3-1,0.4-5.2-1.4-3.3
		C491.4,262.7,493,267.9,494,264.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M539.1,244.8c0,0.2,0.2,0.4,0.6,0.7
		c1,0.6,2.6,0.4,3.6,0.1c0.8-0.3,2.8-1.4,1.4-2.3C543.6,242.6,539.1,243.7,539.1,244.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M517.7,129.2c0.1-1.9-0.8-3.2-2.1-1.2
		C514.2,130.3,517.5,132.2,517.7,129.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M478.8,200.8c1,1.1,2.4,1.6,3.2,0.4
		c1-1.7-3.7-5.4-4.3-3.1C477.5,198.9,478,200,478.8,200.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.2,305.5c1.4,0.6,3.2-3.8,1.5-4.3
		c-1.6-0.4-2.8,2.7-1.9,3.9C517.9,305.3,518.1,305.4,518.2,305.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M469.3,147.7c0.2,0.1,0.5,0.1,1-0.1
		c2.5-1.3,0.3-7.7-0.9-3.4C469.2,144.8,468.6,147.2,469.3,147.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M427.9,76.5c0,0.2,0.2,0.5,0.6,0.7
		c2.5,1.4,6.6-3.9,2.4-2.6C430.3,74.9,427.9,75.7,427.9,76.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M70.3,341.6c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C72.8,343,71.1,341.2,70.3,341.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M352.8,409.1c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C355.3,410.5,353.6,408.7,352.8,409.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M543.1,184.3c0.1-0.5,0-1-0.3-1.5
		c-0.6-1-4.2-4.4-3.6-1.5C539.4,182.1,542.6,187,543.1,184.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M563.8,194c0.1,0.2,0.5,0.5,1.2,0.6
		c0.7,0.2,4.6,0.1,4.1-1.4c-0.3-1.1-2.5-0.8-3.3-0.7C564.7,192.7,563.5,193.4,563.8,194z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M412.8,449c0.7,0.5,6.2,1.5,6.3,0.3
		c0.1-0.8-3.3-1-4-1.1C414.4,448.1,410.8,447.5,412.8,449z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M464.1,390.6c2.1-1-2.6-7.2-3.7-4.6
		c-0.5,1.1,0.3,2.7,1.1,3.5C462,389.8,463.4,390.9,464.1,390.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.9,410.6c2.1-0.4-4.4-5.8-4-2.2
		C515.1,409.9,517.6,410.8,518.9,410.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M394.1,386.8c1.1-0.1,5-1.7,2.5-2.6
		C394.9,383.5,390.8,387.1,394.1,386.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M430.5,420.1c-0.2,0.1-0.3,0.3-0.4,0.8
		c-0.1,1.2,0.6,2.5,1.4,3.4c0.6,0.6,2.4,2,2.7,0.3C434.4,423.3,431.5,419.6,430.5,420.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M527.7,353.9c1.8-0.6,2.6-2,0.3-2.4
		C525.3,351.1,524.9,354.9,527.7,353.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M407.2,354.4c1.8-0.6,2.6-2,0.3-2.4
		C404.8,351.6,404.4,355.4,407.2,354.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M165.7,328.9c1.8-0.6,2.6-2,0.3-2.4
		C163.3,326.1,162.9,329.9,165.7,328.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M446.5,347.1c-0.6,1.3-0.5,2.9,0.9,3.1
		c1.9,0.3,3.5-5.6,1.1-5.2C447.7,345.2,446.9,346.1,446.5,347.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366.6,425.4c0,1.5,4.8,1.4,4.5-0.4
		c-0.2-1.7-3.6-1.5-4.4-0.2C366.6,425,366.6,425.2,366.6,425.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M491.3,317c0,0.2,0.1,0.5,0.5,0.8
		c2.2,1.8,7.1-2.8,2.8-2.2C493.9,315.8,491.5,316.2,491.3,317z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M487.5,399.3c0.5-0.1,0.9-0.5,1.2-0.9
		c0.7-0.9,2.3-5.6-0.1-3.9C488,395,484.9,400,487.5,399.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.9,422.2c-0.2,0.2-0.2,0.6-0.1,1.3
		c0.1,0.7,1.7,4.2,2.9,3.2c0.9-0.8-0.3-2.6-0.7-3.3C488.5,422.5,487.3,421.7,486.9,422.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M152.6,427c0.5,0.7,5.4,3.4,5.9,2.3
		c0.3-0.7-2.8-1.9-3.4-2.3C154.4,426.6,151.1,424.9,152.6,427z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M219.7,387.7c2.3-0.3-0.2-7.6-2-5.6
		c-0.8,0.9-0.5,2.7-0.1,3.6C217.9,386.3,218.9,387.8,219.7,387.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M265.4,424c2.1,0.3-2.3-6.9-3.1-3.4
		C261.9,422.2,264,423.9,265.4,424z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.5,362c1.1,0.2,5.2-0.1,3.2-1.7
		C156.3,359.2,151.2,361.2,154.5,362z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,405.1c-0.2,0-0.4,0.2-0.6,0.6
		c-0.5,1.1-0.2,2.6,0.3,3.6c0.3,0.8,1.6,2.6,2.4,1.2C181.2,409.4,179.6,405,178.5,405.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M301.2,384.9c1.9,0,3.1-1.1,1-2.2
		C299.8,381.5,298.2,385,301.2,384.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,244.2c1.9,0,3.1-1.1,1-2.2
		C177.1,240.8,175.5,244.2,178.5,244.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M181,176c1.9,0,3.1-1.1,1-2.2
		C179.6,172.6,178,176.1,181,176z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M356.6,27.9c1.9,0,3.1-1.1,1-2.2
		C355.2,24.4,353.6,27.9,356.6,27.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.2,164.9c1.9,0,3.1-1.1,1-2.2
		C484.8,161.5,483.2,165,486.2,164.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M470.2,85.4c1.9,0,3.1-1.1,1-2.2
		C468.8,82,467.2,85.5,470.2,85.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M509.5,281.9c1.9,0,3.1-1.1,1-2.2
		C508.1,278.5,506.5,281.9,509.5,281.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M216.7,340.9c-1,1.1-1.4,2.6-0.1,3.2
		c1.7,0.9,5.1-4.2,2.7-4.6C218.5,339.5,217.5,340.1,216.7,340.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M100.6,356.7c-0.4,1.5,4.1,2.8,4.4,1.1
		c0.3-1.7-3-2.6-4.1-1.5C100.7,356.4,100.6,356.6,100.6,356.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M268.8,326.6c-0.1,0.2-0.1,0.5,0.2,0.9
		c1.6,2.4,7.6-0.4,3.3-1.2C271.6,326.2,269.2,325.8,268.8,326.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M239.1,403.4c0.5,0,1-0.1,1.5-0.5
		c0.9-0.6,4-4.5,1.1-3.7C241,399.5,236.4,403.2,239.1,403.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M231.4,424.9c-0.2,0.1-0.4,0.5-0.5,1.2
		c-0.1,0.7,0.3,4.6,1.8,4c1.1-0.4,0.6-2.5,0.4-3.3C232.7,425.7,231.9,424.6,231.4,424.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M112.5,138.4c-0.8,0.4-4.1,4.8-3.1,5.5
		c0.7,0.4,2.3-2.5,2.7-3.1C112.6,140.3,114.8,137.2,112.5,138.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M141.8,210.5c0,2.3,7.6,0.9,5.8-1.2
		c-0.8-0.9-2.6-0.9-3.6-0.6C143.4,208.9,141.8,209.6,141.8,210.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M99.4,250.5c-0.6,2,7.2-1.3,3.8-2.6
		C101.7,247.4,99.7,249.2,99.4,250.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M176.6,149.5c-0.4,1-0.7,5.2,1.2,3.4
		C179.1,151.8,177.8,146.5,176.6,149.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M130.5,167.2c0-0.2-0.1-0.5-0.5-0.7
		c-1-0.6-2.5-0.6-3.6-0.3c-0.8,0.2-2.8,1.2-1.5,2.2C125.9,169.2,130.4,168.3,130.5,167.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M146.1,283.8c-0.2,1.9,0.7,3.2,2,1.3
		C149.7,282.9,146.5,280.8,146.1,283.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,214.2c-0.9-1.1-2.3-1.8-3.2-0.6
		c-1.1,1.6,3.5,5.6,4.1,3.3C189.7,216.1,189.3,215,188.5,214.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.4,107.7c-1.4-0.6-3.4,3.7-1.7,4.2
		c1.6,0.5,3-2.6,2.1-3.8C154.7,107.9,154.6,107.7,154.4,107.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M195.3,267.7c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C195.3,270.6,196.1,268.2,195.3,267.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M63.6,300.5c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C63.6,303.3,64.3,301,63.6,300.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M26.1,331c-0.1-0.2-0.3-0.4-0.8-0.6
		c-2.8-0.7-5.4,5.4-1.7,3.1C24.2,333.2,26.3,331.8,26.1,331z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M523.7,61.8c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C523.6,64.7,524.4,62.3,523.7,61.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M451.6,313.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C451.6,316.8,452.3,314.4,451.6,313.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M169.5,362.4c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C169.4,365.3,170.2,362.9,169.5,362.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M562.5,372.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C562.4,375.8,563.2,373.4,562.5,372.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M123.5,227.4c-0.1,0.5,0,1,0.2,1.5
		c0.5,1,3.9,4.6,3.5,1.6C127.1,229.8,124.1,224.8,123.5,227.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M103.3,216.7c-0.1-0.2-0.5-0.5-1.2-0.7
		c-0.7-0.2-4.6-0.3-4.2,1.2c0.3,1.1,2.4,0.9,3.2,0.8C102.3,217.9,103.6,217.3,103.3,216.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M350.9,43.1c-0.6-0.6-5.7-2.9-6.1-1.7
		c-0.3,0.7,3,1.7,3.6,1.9C349.2,43.7,352.6,45,350.9,43.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M287.9,88.7c-2.3,0.5,0.9,7.6,2.5,5.3
		c0.7-1,0.3-2.7-0.3-3.6C289.9,89.9,288.7,88.5,287.9,88.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.9,57c-2.1-0.1,3,6.6,3.4,3.1
		C242.6,58.5,240.3,57,238.9,57z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M355.4,108c-1.1-0.1-5.2,0.6-3,2
		C353.8,110.9,358.7,108.4,355.4,108z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M320.1,54.3c0.2,0,0.4-0.2,0.5-0.7
		c0.3-1.1-0.1-2.6-0.6-3.6c-0.4-0.7-1.9-2.5-2.5-0.9C317,50.2,318.9,54.5,320.1,54.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M217.8,110.3c-1.9,0.2-3,1.4-0.8,2.3
		C219.5,113.6,220.7,110,217.8,110.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M295.5,135c0.8-1.2,1.2-2.7-0.2-3.2
		c-1.8-0.7-4.6,4.7-2.2,4.8C293.8,136.6,294.8,135.9,295.5,135z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M390.8,76.5c0.3-1.5-4.4-2.4-4.5-0.6
		c-0.1,1.7,3.2,2.3,4.2,1.1C390.7,76.8,390.7,76.7,390.8,76.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M245,154.3c0.1-0.2,0-0.5-0.3-0.9
		c-1.8-2.2-7.6,1.1-3.2,1.5C242.2,154.9,244.7,155.1,245,154.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M267.1,75c-0.5,0-1,0.2-1.4,0.6
		c-0.9,0.7-3.5,4.9-0.8,3.8C265.6,79,269.8,74.9,267.1,75z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M272.7,52.8c0.2-0.2,0.4-0.6,0.4-1.3
		c0-0.7-0.8-4.5-2.1-3.8c-1,0.5-0.3,2.6-0.1,3.3C271.3,52.1,272.2,53.2,272.7,52.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col peroxisomes vesicles" descendants="level 0: ['peroxisome, pval = 0.40992']
level 1: ['P1 peroxisome, pval = --', 'glyoxysome, pval = --', 'P2 peroxisome, pval = --', 'P4 peroxisome, pval = --', 'P3 peroxisome, pval = --', 'P6 peroxisome, pval = --', 'P5 peroxisome, pval = --', 'maintenance of protein location in peroxisome, pval = --', 'peroxisomal membrane, pval = --', 'peroxisomal matrix, pval = 0.43974', 'glycosome, pval = --', 'mannosome, pval = --']
level 2: ['intrinsic component of peroxisomal membrane, pval = --', 'extrinsic component of intraperoxisomal membrane, pval = --', 'glyoxysomal membrane, pval = --', 'glycosome membrane, pval = --', 'glycosome lumen, pval = --', 'glyoxysomal lumen, pval = --']
level 3: ['integral component of peroxisomal membrane, pval = --']" id="peroxisomes" init-pval="0.40992" min-pval="0.40992" min-pval-children="0.43974" title="Peroxisomes">
<path d="M558.5,272.6c0.7-0.4,3.9-5,2.8-5.7
		c-0.7-0.4-2.2,2.6-2.6,3.2C558.3,270.8,556.3,274,558.5,272.6z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M525.6,202.2c-0.1-2.3-7.6-0.5-5.7,1.5
		c0.8,0.9,2.6,0.8,3.6,0.4C524.1,203.8,525.7,203,525.6,202.2z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M566,160c0.5-2.1-7.1,1.7-3.7,2.8
		C563.9,163.3,565.7,161.4,566,160z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M494,264.8c0.3-1,0.4-5.2-1.4-3.3
		C491.4,262.7,493,267.9,494,264.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M539.1,244.8c0,0.2,0.2,0.4,0.6,0.7
		c1,0.6,2.6,0.4,3.6,0.1c0.8-0.3,2.8-1.4,1.4-2.3C543.6,242.6,539.1,243.7,539.1,244.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M517.7,129.2c0.1-1.9-0.8-3.2-2.1-1.2
		C514.2,130.3,517.5,132.2,517.7,129.2z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M478.8,200.8c1,1.1,2.4,1.6,3.2,0.4
		c1-1.7-3.7-5.4-4.3-3.1C477.5,198.9,478,200,478.8,200.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.2,305.5c1.4,0.6,3.2-3.8,1.5-4.3
		c-1.6-0.4-2.8,2.7-1.9,3.9C517.9,305.3,518.1,305.4,518.2,305.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M469.3,147.7c0.2,0.1,0.5,0.1,1-0.1
		c2.5-1.3,0.3-7.7-0.9-3.4C469.2,144.8,468.6,147.2,469.3,147.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M427.9,76.5c0,0.2,0.2,0.5,0.6,0.7
		c2.5,1.4,6.6-3.9,2.4-2.6C430.3,74.9,427.9,75.7,427.9,76.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M70.3,341.6c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C72.8,343,71.1,341.2,70.3,341.6z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M352.8,409.1c-0.2,0.1-0.4,0.4-0.4,0.9
		c-0.2,2.9,6.4,4.2,3.4,1C355.3,410.5,353.6,408.7,352.8,409.1z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M543.1,184.3c0.1-0.5,0-1-0.3-1.5
		c-0.6-1-4.2-4.4-3.6-1.5C539.4,182.1,542.6,187,543.1,184.3z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M563.8,194c0.1,0.2,0.5,0.5,1.2,0.6
		c0.7,0.2,4.6,0.1,4.1-1.4c-0.3-1.1-2.5-0.8-3.3-0.7C564.7,192.7,563.5,193.4,563.8,194z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M412.8,449c0.7,0.5,6.2,1.5,6.3,0.3
		c0.1-0.8-3.3-1-4-1.1C414.4,448.1,410.8,447.5,412.8,449z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M464.1,390.6c2.1-1-2.6-7.2-3.7-4.6
		c-0.5,1.1,0.3,2.7,1.1,3.5C462,389.8,463.4,390.9,464.1,390.6z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518.9,410.6c2.1-0.4-4.4-5.8-4-2.2
		C515.1,409.9,517.6,410.8,518.9,410.6z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M394.1,386.8c1.1-0.1,5-1.7,2.5-2.6
		C394.9,383.5,390.8,387.1,394.1,386.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M430.5,420.1c-0.2,0.1-0.3,0.3-0.4,0.8
		c-0.1,1.2,0.6,2.5,1.4,3.4c0.6,0.6,2.4,2,2.7,0.3C434.4,423.3,431.5,419.6,430.5,420.1z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M527.7,353.9c1.8-0.6,2.6-2,0.3-2.4
		C525.3,351.1,524.9,354.9,527.7,353.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M407.2,354.4c1.8-0.6,2.6-2,0.3-2.4
		C404.8,351.6,404.4,355.4,407.2,354.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M165.7,328.9c1.8-0.6,2.6-2,0.3-2.4
		C163.3,326.1,162.9,329.9,165.7,328.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M446.5,347.1c-0.6,1.3-0.5,2.9,0.9,3.1
		c1.9,0.3,3.5-5.6,1.1-5.2C447.7,345.2,446.9,346.1,446.5,347.1z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366.6,425.4c0,1.5,4.8,1.4,4.5-0.4
		c-0.2-1.7-3.6-1.5-4.4-0.2C366.6,425,366.6,425.2,366.6,425.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M491.3,317c0,0.2,0.1,0.5,0.5,0.8
		c2.2,1.8,7.1-2.8,2.8-2.2C493.9,315.8,491.5,316.2,491.3,317z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M487.5,399.3c0.5-0.1,0.9-0.5,1.2-0.9
		c0.7-0.9,2.3-5.6-0.1-3.9C488,395,484.9,400,487.5,399.3z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.9,422.2c-0.2,0.2-0.2,0.6-0.1,1.3
		c0.1,0.7,1.7,4.2,2.9,3.2c0.9-0.8-0.3-2.6-0.7-3.3C488.5,422.5,487.3,421.7,486.9,422.2z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M152.6,427c0.5,0.7,5.4,3.4,5.9,2.3
		c0.3-0.7-2.8-1.9-3.4-2.3C154.4,426.6,151.1,424.9,152.6,427z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M219.7,387.7c2.3-0.3-0.2-7.6-2-5.6
		c-0.8,0.9-0.5,2.7-0.1,3.6C217.9,386.3,218.9,387.8,219.7,387.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M265.4,424c2.1,0.3-2.3-6.9-3.1-3.4
		C261.9,422.2,264,423.9,265.4,424z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.5,362c1.1,0.2,5.2-0.1,3.2-1.7
		C156.3,359.2,151.2,361.2,154.5,362z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,405.1c-0.2,0-0.4,0.2-0.6,0.6
		c-0.5,1.1-0.2,2.6,0.3,3.6c0.3,0.8,1.6,2.6,2.4,1.2C181.2,409.4,179.6,405,178.5,405.1z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M301.2,384.9c1.9,0,3.1-1.1,1-2.2
		C299.8,381.5,298.2,385,301.2,384.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M178.5,244.2c1.9,0,3.1-1.1,1-2.2
		C177.1,240.8,175.5,244.2,178.5,244.2z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M181,176c1.9,0,3.1-1.1,1-2.2
		C179.6,172.6,178,176.1,181,176z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M356.6,27.9c1.9,0,3.1-1.1,1-2.2
		C355.2,24.4,353.6,27.9,356.6,27.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M486.2,164.9c1.9,0,3.1-1.1,1-2.2
		C484.8,161.5,483.2,165,486.2,164.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M470.2,85.4c1.9,0,3.1-1.1,1-2.2
		C468.8,82,467.2,85.5,470.2,85.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M509.5,281.9c1.9,0,3.1-1.1,1-2.2
		C508.1,278.5,506.5,281.9,509.5,281.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M216.7,340.9c-1,1.1-1.4,2.6-0.1,3.2
		c1.7,0.9,5.1-4.2,2.7-4.6C218.5,339.5,217.5,340.1,216.7,340.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M100.6,356.7c-0.4,1.5,4.1,2.8,4.4,1.1
		c0.3-1.7-3-2.6-4.1-1.5C100.7,356.4,100.6,356.6,100.6,356.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M268.8,326.6c-0.1,0.2-0.1,0.5,0.2,0.9
		c1.6,2.4,7.6-0.4,3.3-1.2C271.6,326.2,269.2,325.8,268.8,326.6z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M239.1,403.4c0.5,0,1-0.1,1.5-0.5
		c0.9-0.6,4-4.5,1.1-3.7C241,399.5,236.4,403.2,239.1,403.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M231.4,424.9c-0.2,0.1-0.4,0.5-0.5,1.2
		c-0.1,0.7,0.3,4.6,1.8,4c1.1-0.4,0.6-2.5,0.4-3.3C232.7,425.7,231.9,424.6,231.4,424.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M112.5,138.4c-0.8,0.4-4.1,4.8-3.1,5.5
		c0.7,0.4,2.3-2.5,2.7-3.1C112.6,140.3,114.8,137.2,112.5,138.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M141.8,210.5c0,2.3,7.6,0.9,5.8-1.2
		c-0.8-0.9-2.6-0.9-3.6-0.6C143.4,208.9,141.8,209.6,141.8,210.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M99.4,250.5c-0.6,2,7.2-1.3,3.8-2.6
		C101.7,247.4,99.7,249.2,99.4,250.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M176.6,149.5c-0.4,1-0.7,5.2,1.2,3.4
		C179.1,151.8,177.8,146.5,176.6,149.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M130.5,167.2c0-0.2-0.1-0.5-0.5-0.7
		c-1-0.6-2.5-0.6-3.6-0.3c-0.8,0.2-2.8,1.2-1.5,2.2C125.9,169.2,130.4,168.3,130.5,167.2z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M146.1,283.8c-0.2,1.9,0.7,3.2,2,1.3
		C149.7,282.9,146.5,280.8,146.1,283.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M188.5,214.2c-0.9-1.1-2.3-1.8-3.2-0.6
		c-1.1,1.6,3.5,5.6,4.1,3.3C189.7,216.1,189.3,215,188.5,214.2z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.4,107.7c-1.4-0.6-3.4,3.7-1.7,4.2
		c1.6,0.5,3-2.6,2.1-3.8C154.7,107.9,154.6,107.7,154.4,107.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M195.3,267.7c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C195.3,270.6,196.1,268.2,195.3,267.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M63.6,300.5c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C63.6,303.3,64.3,301,63.6,300.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M26.1,331c-0.1-0.2-0.3-0.4-0.8-0.6
		c-2.8-0.7-5.4,5.4-1.7,3.1C24.2,333.2,26.3,331.8,26.1,331z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M523.7,61.8c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C523.6,64.7,524.4,62.3,523.7,61.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M451.6,313.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C451.6,316.8,452.3,314.4,451.6,313.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M169.5,362.4c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C169.4,365.3,170.2,362.9,169.5,362.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M562.5,372.9c-0.2-0.1-0.5-0.1-1,0.1
		c-2.6,1.2-0.7,7.6,0.7,3.4C562.4,375.8,563.2,373.4,562.5,372.9z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M123.5,227.4c-0.1,0.5,0,1,0.2,1.5
		c0.5,1,3.9,4.6,3.5,1.6C127.1,229.8,124.1,224.8,123.5,227.4z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M103.3,216.7c-0.1-0.2-0.5-0.5-1.2-0.7
		c-0.7-0.2-4.6-0.3-4.2,1.2c0.3,1.1,2.4,0.9,3.2,0.8C102.3,217.9,103.6,217.3,103.3,216.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M350.9,43.1c-0.6-0.6-5.7-2.9-6.1-1.7
		c-0.3,0.7,3,1.7,3.6,1.9C349.2,43.7,352.6,45,350.9,43.1z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M287.9,88.7c-2.3,0.5,0.9,7.6,2.5,5.3
		c0.7-1,0.3-2.7-0.3-3.6C289.9,89.9,288.7,88.5,287.9,88.7z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.9,57c-2.1-0.1,3,6.6,3.4,3.1
		C242.6,58.5,240.3,57,238.9,57z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M355.4,108c-1.1-0.1-5.2,0.6-3,2
		C353.8,110.9,358.7,108.4,355.4,108z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M320.1,54.3c0.2,0,0.4-0.2,0.5-0.7
		c0.3-1.1-0.1-2.6-0.6-3.6c-0.4-0.7-1.9-2.5-2.5-0.9C317,50.2,318.9,54.5,320.1,54.3z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M217.8,110.3c-1.9,0.2-3,1.4-0.8,2.3
		C219.5,113.6,220.7,110,217.8,110.3z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M295.5,135c0.8-1.2,1.2-2.7-0.2-3.2
		c-1.8-0.7-4.6,4.7-2.2,4.8C293.8,136.6,294.8,135.9,295.5,135z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M390.8,76.5c0.3-1.5-4.4-2.4-4.5-0.6
		c-0.1,1.7,3.2,2.3,4.2,1.1C390.7,76.8,390.7,76.7,390.8,76.5z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M245,154.3c0.1-0.2,0-0.5-0.3-0.9
		c-1.8-2.2-7.6,1.1-3.2,1.5C242.2,154.9,244.7,155.1,245,154.3z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M267.1,75c-0.5,0-1,0.2-1.4,0.6
		c-0.9,0.7-3.5,4.9-0.8,3.8C265.6,79,269.8,74.9,267.1,75z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M272.7,52.8c0.2-0.2,0.4-0.6,0.4-1.3
		c0-0.7-0.8-4.5-2.1-3.8c-1,0.5-0.3,2.6-0.1,3.3C271.3,52.1,272.2,53.2,272.7,52.8z" style="fill:#E583A2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton endosomes vesicles" descendants="level 0: ['endosome, pval = 0.00915']
level 1: ['early endosome, pval = 0.1586', 'tubular endosome, pval = --', 'late endosome, pval = 0.23172', 'endolysosome, pval = --', 'proteolysis within endosome associated with antigen processing and presentation, pval = --', 'endosome lumen, pval = 0.66415', 'presynaptic endosome, pval = --', 'endosome membrane, pval = 0.10877', 'postsynaptic endosome, pval = --', 'endosomal signal transduction, pval = --', 'sorting endosome, pval = --', 'ESCRT complex, pval = --', 'TRAPPII protein complex, pval = --', 'recycling endosome, pval = 0.36956', 'pinosome, pval = --', 'CORVET complex, pval = --', 'EGO complex, pval = --']
level 2: ['early endosome membrane, pval = 0.36956', 'early endosome lumen, pval = 0.43974', 'postsynaptic early endosome, pval = --', 'multivesicular body, pval = 0.19304', 'late endosome lumen, pval = --', 'late endosome membrane, pval = 0.59248', 'Mon1-Ccz1 complex, pval = --', 'endolysosome membrane, pval = --', 'endolysosome lumen, pval = --', 'recycling endosome lumen, pval = --', 'presynaptic endosome membrane, pval = --', 'postsynaptic endosome membrane, pval = --', 'ESCRT IV complex, pval = --', 'recycling endosome membrane, pval = 0.3878', 'Vps55/Vps68 complex, pval = --', 'ESCRT III complex, pval = --', 'ESCRT II complex, pval = --', 'ESCRT I complex, pval = --', 'intrinsic component of endosome membrane, pval = --', 'extrinsic component of endosome membrane, pval = --', 'intralumenal vesicle formation, pval = --', 'cytoplasmic side of endosome membrane, pval = --', 'lumenal side of endosome membrane, pval = --', 'postsynaptic recycling endosome, pval = --', 'ESCRT-0 complex, pval = --', 'basolateral recycling endosome, pval = --', 'apical recycling endosome, pval = --', 'peri-centrosomal recycling endosome, pval = --', 'EARP complex, pval = --', 'micropinosome, pval = --', 'macropinosome, pval = --']
level 3: ['cytoplasmic side of early endosome membrane, pval = --', 'lumenal side of early endosome membrane, pval = --', 'postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle, pval = 0.43974', 'multivesicular body lumen, pval = --', 'multivesicular body membrane, pval = --', 'endosomal scaffold complex, pval = --', 'Ragulator complex, pval = --', 'cytoplasmic side of late endosome membrane, pval = --', 'lumenal side of late endosome membrane, pval = --', 'intrinsic component of presynaptic endosome membrane, pval = --', 'extrinsic component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic endosome membrane, pval = --', 'postsynaptic recycling endosome membrane, pval = --', 'integral component of endosome membrane, pval = --', 'positive regulation of intralumenal vesicle formation, pval = --', 'negative regulation of intralumenal vesicle formation, pval = --', 'regulation of intralumenal vesicle formation, pval = --']
level 4: ['intrinsic component of postsynaptic early endosome membrane, pval = --', 'extrinsic component of postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle membrane, pval = --', 'multivesicular body, internal vesicle lumen, pval = 0.43974', 'multivesicular body HOPS complex, pval = --', 'anchored component of presynaptic endosome membrane, pval = --', 'integral component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic recycling endosome membrane, pval = --', 'anchored component of postsynaptic endosome membrane, pval = --', 'integral component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic recycling endosome membrane, pval = --']
level 5: ['anchored component of postsynaptic early endosome membrane, pval = --', 'integral component of postsynaptic early endosome membrane, pval = --', 'anchored component of postsynaptic recycling endosome membrane, pval = --', 'integral component of postsynaptic recycling endosome membrane, pval = --']" id="endosomes_ton" init-pval="0.00915" min-pval="0.00915" min-pval-children="0.10877" title="Endosomes">
<path d="M433.2,118.5c0.4-1.4,1.6-2.4,3.3-1.4
		c2.7,1.6,0.7,6.8-2.2,4.7C433.2,121,432.9,119.6,433.2,118.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M126.4,341.3c0.1-0.8,0.7-1.6,1.6-1.7
		c1.7-0.3,3,1.5,2.5,3.1c-0.5,1.8-2.6,2.5-3.7,0.9C126.4,342.9,126.2,342.1,126.4,341.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M351.6,60.2c-0.1-2.7,5.3-4.1,5.3,0.4
		C356.9,65.2,351.7,63.1,351.6,60.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M453.6,74.5c0.7-3.3,6.3-3.6,4.8,1.6
		C456.9,81,452.8,78.5,453.6,74.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518,196.7c1.5-3.1,5.7-1.8,5.2,1.5
		c-0.7,3.8-5.6,3.3-5.4-0.5C517.8,197.3,517.9,196.9,518,196.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M313.2,421.4c0.2-2.8,3.5-3.6,4.4-0.7
		C318.7,424.1,312.9,426,313.2,421.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M391.5,442.8c0.9-2.8,4.4-1.7,3.7,1.2
		C394.2,447.7,390.7,445.4,391.5,442.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M215.9,456.1c-1.9-1.4,0-5.5,2.6-3.6
		C221,454.4,218.3,457.8,215.9,456.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M56.3,242.3c-1.8-2.7,1.7-6.2,4-2.7
		C62.7,243.1,58.2,245.3,56.3,242.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.8,82.3c-1.8-2.7,1.7-6.2,4-2.7
		C245.2,83.1,240.7,85.3,238.8,82.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M568,295.7c-1.8-2.7,1.7-6.2,4-2.7
		C574.4,296.5,569.9,298.7,568,295.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M186.1,403.1c0.1,0,0.1,0,0.2-0.1
		c2.2-0.7,4.8,3,2,4C186,408,183.5,404.1,186.1,403.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M124.8,207.8c0.3-0.4,0.7-0.8,1.2-1
		c3.2-1.1,4.3,4,1.2,5C124.7,212.7,123.7,209.6,124.8,207.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M190.7,113.3c0.6-1,2.1-1.8,3.5-1
		c2.4,1.2,1,4.9-1.7,4.3C190.2,116,190,114.4,190.7,113.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M405,50.6c-0.3-3.2,4.2-4.7,4.9-1
		C410.5,53.4,405.3,54,405,50.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M590.3,366.4c-1.7-1.8-0.1-6.4,2.7-3.4
		C595.2,365.4,592.8,369,590.3,366.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M545.9,231.4c-0.9-2.6,3.8-6,4.8-2
		c0.7,2.6-2.5,5-4.5,2.7C546.1,231.8,546,231.6,545.9,231.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M514.4,318c-0.3-0.7-0.2-1.6,0.7-2.4
		c2.2-2.2,5.8,1.1,3.6,3.3C517.3,320.2,515,319.4,514.4,318z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M542.9,277.5c-0.7-2.5,3-3.5,3.7-0.8
		c0.7,2.5-2.8,3.4-3.7,1C543,277.6,543,277.6,542.9,277.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M560.5,343.6c-0.9-2.5,2.8-4.8,3.9-1.4
		C565.6,345.6,561.5,346.5,560.5,343.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M534.9,400.4c-1.9-2.4,0.9-6.1,3.4-3.5
		C540.9,399.7,537,403.1,534.9,400.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M465.3,410.1c-0.4-0.8-0.3-1.8,0.5-2.8
		c1.9-2.1,5.7,0.1,4.2,2.8C468.7,412.6,466.1,411.8,465.3,410.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M460.1,437.9c1.4-2.7,5.9-0.8,4,2.5
		C462.1,443.9,458.5,440.8,460.1,437.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M278.2,443.2c0.1-3.1,4.9-3.5,4.6,0.8
		C282.4,448.7,278.1,446.2,278.2,443.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M221.7,404.5c0.4-3.3,4.4-2.3,4.3,0.6
		C225.8,408.2,221.4,407.8,221.7,404.5C221.7,404.5,221.7,404.5,221.7,404.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M127,387.8c1.7,0.7,2.1,4,0.3,4.5
		c-1.9,0.5-4.5-3.9-1.3-4.6C126.4,387.6,126.7,387.7,127,387.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M51.3,383.2c0.1-0.1,0.2-0.2,0.3-0.3
		c2.4-1.9,5.1,2.2,3.2,3.9C53,388.5,49.4,385.3,51.3,383.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M45.5,327.8c-0.7-1.2,0.5-3.1,2.3-3
		c0.1,0,0.3,0,0.4,0c2.5,0.4,2.4,4.4-0.9,4C46.4,328.7,45.8,328.3,45.5,327.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M148.6,139.1c0.1-0.4,0.5-0.7,1-0.9
		c1.6-0.5,3.4,2,2.4,3.3C150.7,143.6,148,140.8,148.6,139.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col endosomes vesicles" descendants="level 0: ['endosome, pval = 0.00915']
level 1: ['early endosome, pval = 0.1586', 'tubular endosome, pval = --', 'late endosome, pval = 0.23172', 'endolysosome, pval = --', 'proteolysis within endosome associated with antigen processing and presentation, pval = --', 'endosome lumen, pval = 0.66415', 'presynaptic endosome, pval = --', 'endosome membrane, pval = 0.10877', 'postsynaptic endosome, pval = --', 'endosomal signal transduction, pval = --', 'sorting endosome, pval = --', 'ESCRT complex, pval = --', 'TRAPPII protein complex, pval = --', 'recycling endosome, pval = 0.36956', 'pinosome, pval = --', 'CORVET complex, pval = --', 'EGO complex, pval = --']
level 2: ['early endosome membrane, pval = 0.36956', 'early endosome lumen, pval = 0.43974', 'postsynaptic early endosome, pval = --', 'multivesicular body, pval = 0.19304', 'late endosome lumen, pval = --', 'late endosome membrane, pval = 0.59248', 'Mon1-Ccz1 complex, pval = --', 'endolysosome membrane, pval = --', 'endolysosome lumen, pval = --', 'recycling endosome lumen, pval = --', 'presynaptic endosome membrane, pval = --', 'postsynaptic endosome membrane, pval = --', 'ESCRT IV complex, pval = --', 'recycling endosome membrane, pval = 0.3878', 'Vps55/Vps68 complex, pval = --', 'ESCRT III complex, pval = --', 'ESCRT II complex, pval = --', 'ESCRT I complex, pval = --', 'intrinsic component of endosome membrane, pval = --', 'extrinsic component of endosome membrane, pval = --', 'intralumenal vesicle formation, pval = --', 'cytoplasmic side of endosome membrane, pval = --', 'lumenal side of endosome membrane, pval = --', 'postsynaptic recycling endosome, pval = --', 'ESCRT-0 complex, pval = --', 'basolateral recycling endosome, pval = --', 'apical recycling endosome, pval = --', 'peri-centrosomal recycling endosome, pval = --', 'EARP complex, pval = --', 'micropinosome, pval = --', 'macropinosome, pval = --']
level 3: ['cytoplasmic side of early endosome membrane, pval = --', 'lumenal side of early endosome membrane, pval = --', 'postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle, pval = 0.43974', 'multivesicular body lumen, pval = --', 'multivesicular body membrane, pval = --', 'endosomal scaffold complex, pval = --', 'Ragulator complex, pval = --', 'cytoplasmic side of late endosome membrane, pval = --', 'lumenal side of late endosome membrane, pval = --', 'intrinsic component of presynaptic endosome membrane, pval = --', 'extrinsic component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic endosome membrane, pval = --', 'postsynaptic recycling endosome membrane, pval = --', 'integral component of endosome membrane, pval = --', 'positive regulation of intralumenal vesicle formation, pval = --', 'negative regulation of intralumenal vesicle formation, pval = --', 'regulation of intralumenal vesicle formation, pval = --']
level 4: ['intrinsic component of postsynaptic early endosome membrane, pval = --', 'extrinsic component of postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle membrane, pval = --', 'multivesicular body, internal vesicle lumen, pval = 0.43974', 'multivesicular body HOPS complex, pval = --', 'anchored component of presynaptic endosome membrane, pval = --', 'integral component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic recycling endosome membrane, pval = --', 'anchored component of postsynaptic endosome membrane, pval = --', 'integral component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic recycling endosome membrane, pval = --']
level 5: ['anchored component of postsynaptic early endosome membrane, pval = --', 'integral component of postsynaptic early endosome membrane, pval = --', 'anchored component of postsynaptic recycling endosome membrane, pval = --', 'integral component of postsynaptic recycling endosome membrane, pval = --']" id="endosomes" init-pval="0.00915" min-pval="0.00915" min-pval-children="0.10877" title="Endosomes">
<path d="M433.2,118.5c0.4-1.4,1.6-2.4,3.3-1.4
		c2.7,1.6,0.7,6.8-2.2,4.7C433.2,121,432.9,119.6,433.2,118.5z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M126.4,341.3c0.1-0.8,0.7-1.6,1.6-1.7
		c1.7-0.3,3,1.5,2.5,3.1c-0.5,1.8-2.6,2.5-3.7,0.9C126.4,342.9,126.2,342.1,126.4,341.3z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M351.6,60.2c-0.1-2.7,5.3-4.1,5.3,0.4
		C356.9,65.2,351.7,63.1,351.6,60.2z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M453.6,74.5c0.7-3.3,6.3-3.6,4.8,1.6
		C456.9,81,452.8,78.5,453.6,74.5z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M518,196.7c1.5-3.1,5.7-1.8,5.2,1.5
		c-0.7,3.8-5.6,3.3-5.4-0.5C517.8,197.3,517.9,196.9,518,196.7z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M313.2,421.4c0.2-2.8,3.5-3.6,4.4-0.7
		C318.7,424.1,312.9,426,313.2,421.4z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M391.5,442.8c0.9-2.8,4.4-1.7,3.7,1.2
		C394.2,447.7,390.7,445.4,391.5,442.8z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M215.9,456.1c-1.9-1.4,0-5.5,2.6-3.6
		C221,454.4,218.3,457.8,215.9,456.1z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M56.3,242.3c-1.8-2.7,1.7-6.2,4-2.7
		C62.7,243.1,58.2,245.3,56.3,242.3z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M238.8,82.3c-1.8-2.7,1.7-6.2,4-2.7
		C245.2,83.1,240.7,85.3,238.8,82.3z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M568,295.7c-1.8-2.7,1.7-6.2,4-2.7
		C574.4,296.5,569.9,298.7,568,295.7z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M186.1,403.1c0.1,0,0.1,0,0.2-0.1
		c2.2-0.7,4.8,3,2,4C186,408,183.5,404.1,186.1,403.1z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M124.8,207.8c0.3-0.4,0.7-0.8,1.2-1
		c3.2-1.1,4.3,4,1.2,5C124.7,212.7,123.7,209.6,124.8,207.8z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M190.7,113.3c0.6-1,2.1-1.8,3.5-1
		c2.4,1.2,1,4.9-1.7,4.3C190.2,116,190,114.4,190.7,113.3z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M405,50.6c-0.3-3.2,4.2-4.7,4.9-1
		C410.5,53.4,405.3,54,405,50.6z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M590.3,366.4c-1.7-1.8-0.1-6.4,2.7-3.4
		C595.2,365.4,592.8,369,590.3,366.4z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M545.9,231.4c-0.9-2.6,3.8-6,4.8-2
		c0.7,2.6-2.5,5-4.5,2.7C546.1,231.8,546,231.6,545.9,231.4z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M514.4,318c-0.3-0.7-0.2-1.6,0.7-2.4
		c2.2-2.2,5.8,1.1,3.6,3.3C517.3,320.2,515,319.4,514.4,318z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M542.9,277.5c-0.7-2.5,3-3.5,3.7-0.8
		c0.7,2.5-2.8,3.4-3.7,1C543,277.6,543,277.6,542.9,277.5z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M560.5,343.6c-0.9-2.5,2.8-4.8,3.9-1.4
		C565.6,345.6,561.5,346.5,560.5,343.6z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M534.9,400.4c-1.9-2.4,0.9-6.1,3.4-3.5
		C540.9,399.7,537,403.1,534.9,400.4z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M465.3,410.1c-0.4-0.8-0.3-1.8,0.5-2.8
		c1.9-2.1,5.7,0.1,4.2,2.8C468.7,412.6,466.1,411.8,465.3,410.1z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M460.1,437.9c1.4-2.7,5.9-0.8,4,2.5
		C462.1,443.9,458.5,440.8,460.1,437.9z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M278.2,443.2c0.1-3.1,4.9-3.5,4.6,0.8
		C282.4,448.7,278.1,446.2,278.2,443.2z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M221.7,404.5c0.4-3.3,4.4-2.3,4.3,0.6
		C225.8,408.2,221.4,407.8,221.7,404.5C221.7,404.5,221.7,404.5,221.7,404.5z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M127,387.8c1.7,0.7,2.1,4,0.3,4.5
		c-1.9,0.5-4.5-3.9-1.3-4.6C126.4,387.6,126.7,387.7,127,387.8z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M51.3,383.2c0.1-0.1,0.2-0.2,0.3-0.3
		c2.4-1.9,5.1,2.2,3.2,3.9C53,388.5,49.4,385.3,51.3,383.2z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M45.5,327.8c-0.7-1.2,0.5-3.1,2.3-3
		c0.1,0,0.3,0,0.4,0c2.5,0.4,2.4,4.4-0.9,4C46.4,328.7,45.8,328.3,45.5,327.8z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M148.6,139.1c0.1-0.4,0.5-0.7,1-0.9
		c1.6-0.5,3.4,2,2.4,3.3C150.7,143.6,148,140.8,148.6,139.1z" style="fill:#F5B4CF;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton endoplasmic_reticulum" descendants="level 0: ['endoplasmic reticulum, pval = 0.10307']
level 1: ['oligosaccharyltransferase complex, pval = --', 'MHC class I peptide loading complex, pval = 0.43974', 'rough endoplasmic reticulum, pval = --', 'smooth endoplasmic reticulum, pval = 0.43974', 'maintenance of protein localization in endoplasmic reticulum, pval = --', 'GPI-anchor transamidase complex, pval = --', 'endoplasmic reticulum calcium ion homeostasis, pval = --', 'endoplasmic reticulum quality control compartment, pval = --', 'plasmodesmatal endoplasmic reticulum, pval = --', 'egasyn-beta-glucuronidase complex, pval = --', 'glucosidase II complex, pval = --', 'extrinsic component of endoplasmic reticulum membrane, pval = 0.43974', 'VCP-NPL4-UFD1 AAA ATPase complex, pval = --', 'protein glycosylation in endoplasmic reticulum, pval = --', 'Dsl1/NZR complex, pval = --', 'fatty acid elongase complex, pval = --', 'perinuclear endoplasmic reticulum, pval = 0.19304', 'SREBP-SCAP-Insig complex, pval = --', 'membrane stack, pval = --', 'endoplasmic reticulum exit site, pval = --', 'endoplasmic reticulum chaperone complex, pval = 0.43974', 'Ig heavy chain-bound endoplasmic reticulum chaperone complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase complex, pval = --', 'GET complex, pval = --', 'glycosylphosphatidylinositol-mannosyltransferase I complex, pval = --', 'endoplasmic reticulum lumen, pval = 0.87374', 'UDP-N-acetylglucosamine transferase complex, pval = --', 'ERMES complex, pval = --', 'procollagen-proline 4-dioxygenase complex, pval = --', 'sarcoplasmic reticulum, pval = 0.22736', 'endoplasmic reticulum subcompartment, pval = 0.0018', 'glycosylphosphatidylinositol-mannosyltransferase II complex, pval = --', 'ER ubiquitin ligase complex, pval = --', 'plasmodesmatal desmotubule, pval = --']
level 2: ['oligosaccharyltransferase I complex, pval = --', 'oligosaccharyltransferase II complex, pval = --', 'oligosaccharyltransferase III complex, pval = --', 'Tapasin-ERp57 complex, pval = --', 'TAP complex, pval = --', 'rough endoplasmic reticulum membrane, pval = --', 'rough endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum membrane, pval = --', 'smooth endoplasmic reticulum calcium ion homeostasis, pval = --', 'smooth endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum cisterna, pval = --', 'protein retention in ER lumen, pval = --', 'SREBP-SCAP complex retention in endoplasmic reticulum, pval = --', 'positive regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'endoplasmic reticulum mannose trimming, pval = --', 'protein galactosylation in endoplasmic reticulum, pval = --', 'perinuclear endoplasmic reticulum membrane, pval = 0.43974', 'perinuclear endoplasmic reticulum lumen, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt4p homodimer complex, pval = --', 'sarcoplasmic reticulum lumen, pval = --', 'cortical endoplasmic reticulum lumen, pval = --', 'luminal surveillance complex, pval = --', 'mannosyl-oligosaccharide 1,2-alpha-mannosidase complex, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(III) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(II) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(I) type, pval = --', 'sarcoplasmic reticulum membrane, pval = 0.43974', 'longitudinal sarcoplasmic reticulum, pval = --', 'terminal cisterna, pval = --', 'subrhabdomeral cisterna, pval = --', 'endoplasmic reticulum tubular network, pval = --', 'endoplasmic reticulum cisternal network, pval = --', 'CUE1-UBC7 ubiquitin-conjugating enzyme complex, pval = --', 'Hrd1p ubiquitin ligase complex, pval = --', 'Doa10p ubiquitin ligase complex, pval = --']
level 3: ['signal recognition particle receptor complex, pval = --', 'translocon complex, pval = --', 'endoplasmic reticulum Sec complex, pval = --', 'lumenal side of rough endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of rough endoplasmic reticulum membrane, pval = --', 'lumenal side of smooth endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of smooth endoplasmic reticulum membrane, pval = --', 'positive regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'hypolemmal cisterna, pval = --', 'trimming of terminal mannose on B branch, pval = --', 'trimming of second mannose on A branch, pval = --', 'trimming of first mannose on A branch, pval = --', 'trimming of terminal mannose on C branch, pval = --', 'longitudinal sarcoplasmic reticulum lumen, pval = --', 'terminal cisterna lumen, pval = --', 'phospholamban complex, pval = --', 'ryanodine receptor complex, pval = --', 'junctional sarcoplasmic reticulum membrane, pval = --', 'free sarcoplasmic reticulum membrane, pval = --', 'cortical endoplasmic reticulum, pval = --', 'rough endoplasmic reticulum cisterna, pval = --', 'Hrd1p ubiquitin ligase ERAD-M complex, pval = --', 'Hrd1p ubiquitin ligase ERAD-L complex, pval = --']
level 4: ['Sec61 translocon complex, pval = --', 'Ssh1 translocon complex, pval = --', 'Sec62/Sec63 complex, pval = --', 'sub-surface cisterna, pval = --']" id="endoplasmic_reticulum_ton" init-pval="0.10307" min-pval="0.0018" min-pval-children="0.0018" title="Endoplasmic reticulum">
<g>
<path d="M425.7,314.6c0.1,0,0.3,0,0.4,0
			c0.3-0.1,0.7-0.2,0.9-0.5c3.1-2.8,4.6-4.9,6.9-9.5c0.4-0.9,1.2-2.2,2.2-3.8c2.4-4,5.6-9.4,7.1-13.8c2.4-6.9,5.7-17.3,6.8-27.6
			c0.2-1.7,0.8-3.7,2.9-3.3c0.8,0.2,1.6,0.7,2.1,1.4c0.4,0.5,0.7,1,1.4,1.2c0.2,0.1,0.3,0.1,0.5,0.1c0.9,0,1.7-0.7,1.9-1.6
			c0.7-3.4,0.8-10.8,0.1-18c-0.6-6.7-3.8-23.1-5.6-28.8c-0.3-0.8-1-1.4-1.9-1.4c-0.2,0-0.4,0-0.6,0.1c-0.9,0.3-1.5,1.2-1.3,2.2
			c0,0.4-0.1,0.8-0.4,1.1c-0.4,0.4-1.4,0.8-1.9,0.7c-1.3-0.1-1.7-1.2-1.9-2.3c-0.1-0.4-0.2-0.8-0.3-1.1l-0.3-0.9
			c-0.4-1.1-0.7-2.3-1.1-3.4c-0.4-1.1-0.8-2.2-1.2-3.2c-0.4-1-0.9-2.1-1.4-3.1c-0.5-1-1-2.1-1.6-3.2c-0.6-1.1-1.2-2.2-1.8-3.3
			c-0.7-1.1-1.4-2.3-2.1-3.4c-0.8-1.2-1.6-2.3-2.5-3.6c-0.9-1.3-1.9-2.6-2.9-3.9l-0.2-0.2c-1-1.3-2-2.6-3-3.8
			c-10.8-13.7-23.2-24.6-34-29.9c-4.4-2.2-12.2-4.3-18-5.5c-6-1.3-12.2-1.7-16.3-1.9c-1.4-0.1-2.6-0.2-3.3-0.2
			c-1.7-0.2-3.5-0.2-5.5-0.2l-0.8,0c-1.9,0-4.1,0-5.9-0.2l-0.1,0c-1,0-1.9,0.8-1.9,1.8c-0.1,1.1,0.7,2,1.8,2.1
			c2,0.2,4.1,0.2,6.2,0.2c2.1,0,4.2,0.1,6,0.2c0.5,0,1.2,0.1,2.1,0.1c3.6,0.2,9.5,0.6,15.2,1.6c4.9,0.8,14,3,19.1,5.6
			c7.4,3.7,19.1,11.7,32.6,28.8c12.5,15.9,16,23.9,19.4,38.6c0.2,0.9,1,1.5,1.9,1.5c0.5,0,1-0.2,1.3-0.5c0.8-0.8,0.3-2.3,1.2-3
			c1.7-1.2,3.3,0.7,3.6,2.2c0,0.2,0.1,0.4,0.1,0.6c0.3,1.2,0.5,2.5,0.8,3.7l0.1,0.3c0.1,0.6,0.2,1.2,0.3,1.8c0.2,1,0.4,2,0.5,3
			c0.1,0.8,0.3,1.5,0.4,2.2c0.3,1.4,0.5,2.9,0.8,4.7l0.1,0.6c0.2,1.1,0.3,2.2,0.4,3.3l0,0.4c0.1,1.6,0.2,3.3,0.3,4.9
			c0,1,0.2,1.9,0.2,2.9c0.1,1.8-0.5,4.3-2.9,4.1c-1.9-0.2-2-2.5-2.1-4c0-1-0.1-2.1-0.1-3.1c-0.1-1.9-0.2-4.1-0.4-6.3
			c-0.1-1.8-0.3-3.6-0.6-5.5c-0.2-1.4-0.4-2.7-0.6-4.1c-0.1-0.8-0.3-1.6-0.4-2.5c-0.2-0.9-1-1.6-1.9-1.6c-0.1,0-0.2,0-0.3,0
			c-0.5,0.1-0.9,0.3-1.2,0.8c-0.3,0.4-0.4,1-0.3,1.5c0.3,1.9,3.2,18.7,1.3,33.1c-1,8-3.9,18.5-5.6,23.4c-2.1,6.2-5.5,11.7-7.9,15.8
			c-0.9,1.6-1.7,2.9-2.2,3.9c-2.2,4.4-3.5,6.1-6.1,8.4c-0.4,0.3-0.6,0.8-0.6,1.3c0,0.5,0.2,1,0.5,1.4
			C424.7,314.4,425.2,314.6,425.7,314.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M335.1,349.2c3.5,0,6.7-0.2,9.5-0.5
			c1.9-0.1,3.8-0.3,5.4-0.3c0.1,0,0.2,0,0.3,0c0.9-0.2,1.6-1,1.6-2c0-1.1-0.9-1.9-2-1.9c-1.8,0-3.6,0.2-5.5,0.3l-0.1,0
			c-2.8,0.2-5.9,0.5-9.3,0.5c-1.6,0-3.1-0.1-4.6-0.2l-0.1,0c-1,0-1.9,0.8-2,1.8c0,0.5,0.1,1,0.5,1.4c0.3,0.4,0.8,0.6,1.3,0.7
			C331.9,349.2,333.4,349.2,335.1,349.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M257.4,198c0.1,0,0.3,0,0.4,0
			c0.6-0.1,1-0.5,1.3-1c0.2-0.3,0.3-0.6,0.5-0.8c0.4-0.7,0.7-1.4,1.1-2c0.4-0.6,0.7-1.1,1.1-1.7c0.7-1,1.3-2.1,1.9-3.1
			c0.3-0.6,0.6-1.3,1.3-1.6c0.8-0.3,1.6,0.4,1.7,1.1c0.1,0.4,0,0.7,0,1.1c0,0.2-0.2,1,0,1.2c0.6,0.8,1.6-0.8,2-1.2
			c0.5-0.5,1.1-1,1.6-1.6l0.1-0.2c0.3-0.4,0.7-0.7,0.9-1.2l0.1-0.2c0.5-0.6,0.8-1.2,0.7-1.7c-0.1-0.1-0.2-0.4-0.7-0.4
			c-0.3,0-0.6,0.1-0.8,0.1c-0.1,0-0.2,0-0.2,0.1c-0.2,0-0.5,0.1-0.8,0.1c-0.8,0-1.2-0.2-1.4-0.7c-0.3-0.7,0.2-1.6,0.6-2.1
			c0.3-0.5,0.8-0.9,1.2-1.3c0.3-0.3,0.6-0.6,0.8-0.9l0.2-0.2c1.3-1.4,2.5-2.8,3.8-4.1c2.9-3,6.1-5.8,9.6-8.7c2-1.7,4-3.3,6.3-5
			c1.1-0.9,3.8-2.6,4.3-0.2c0.2,0.7-0.2,1.3-0.4,1.9c-0.2,0.8,0.5,0.7,1,0.6c0.3-0.1,0.7-0.2,1-0.4c0.3-0.2,0.7-0.6,1-0.7
			c0.2-0.1,0.6-0.2,0.9-0.4c0.8-0.3,1.4-0.7,2.1-1.2c0.5-0.4,1.3-0.5,1.4-1.2c0.1-1.2-1.9-0.7-2.5-0.8c-0.9-0.2-1.3-1.2-1.1-2
			c0.2-1,1.2-1.5,2.1-1.9c0.8-0.3,1.7-0.7,2.5-1.1l0.2-0.1c0.8-0.3,1.5-0.7,2.2-1c3.2-1.5,6.3-2.7,9-3.7c2.6-1,5.3-1.9,8.3-2.8
			c2.3-0.7,4.7-1.3,6.9-1.9c1.8-0.4,3.3-0.8,4.7-1.1c0.5-0.1,0.9-0.4,1.2-0.8c0.3-0.4,0.4-0.9,0.3-1.4c-0.2-0.9-1-1.5-1.9-1.5
			c-0.1,0-0.3,0-0.4,0c-6.6,1.3-29.3,6.6-45.2,18.7l-0.6,0.4c-1.5,1.2-3,2.3-4.5,3.5c-1.4,1.2-2.8,2.3-4.2,3.5
			c-1.4,1.2-2.7,2.4-3.9,3.5c-1.3,1.2-2.5,2.4-3.6,3.5c-0.3,0.3-0.6,0.6-0.8,0.9c-0.7,0.7-1.4,1.6-2.4,1.9c-1.5,0.3-2.8-1.2-2.3-2.6
			c0.4-1.2,1.2-2.1,2.1-2.9c0.8-0.7,1.5-1.4,2.3-2.2c1.3-1.3,2.7-2.6,4-3.8l1.1-1c10.3-9.2,16.9-14.2,28.5-19.5
			c0.5-0.2,0.8-0.6,1-1.1c0.2-0.5,0.2-1-0.1-1.5c-0.3-0.7-1-1.2-1.8-1.2c-0.3,0-0.6,0.1-0.8,0.2c-1.3,0.6-2.6,1.2-4,1.9
			c-0.4,0.2-0.8,0.4-1.2,0.6c-0.8,0.4-1.4,0.7-2.1,1.1c-1.1,0.6-2.2,1.3-3.2,2c-1,0.7-2,1.3-2.9,1.8c-0.9,0.5-2,0.6-2.8,0
			c-0.8-0.5-1.4-1.5-1.3-2.5c0.1-0.8,0.8-1.5,1-2.2c0.2-0.6,0.1-1.4-0.2-2c-0.6-1-2.5-1.1-3.3-0.4c-7.5,5.8-13.6,10.8-18.2,15.1
			c-0.4,0.4-0.6,0.9-0.6,1.4c0,0.5,0.2,1.1,0.5,1.5c0.4,0.4,0.9,0.7,1.5,0.7c0.5,0,1-0.2,1.4-0.5c1.4-1.3,2.9-2.6,4.2-3.7l1.4-1.2
			c0.9-0.8,1.8-1.5,2.7-2.2l0.8-0.7c0.6-0.5,1.1-0.9,1.7-1.4c1.1-0.8,3-1.7,4.3-0.9c1.3,0.8,0.9,2.7,0.3,3.8c-0.7,1.2-2,1.8-3.1,2.6
			c-1.2,0.9-2.3,1.9-3.5,2.9c0,0-2.9,2.6-2.9,2.6c-9.5,8.5-16.6,16.3-19.9,22.1c-0.3,0.5-0.3,1-0.2,1.5c0.1,0.5,0.5,0.9,0.9,1.2
			c0.3,0.2,0.6,0.3,1,0.3c0.7,0,1.4-0.4,1.7-1c0.7-1.2,1.5-2.3,2.3-3.4c0.2-0.3,0.4-0.5,0.5-0.8c0.7-1,1.9-2.1,3.2-1.8
			c1.8,0.5,1.6,3.1,0.8,4.3c-0.7,1-1.5,1.8-2.2,2.8c-0.7,1-1.3,2-1.9,3c-1.3,2-2.5,4-3.7,6.2c-0.2,0.5-0.3,1-0.2,1.5
			c0.1,0.5,0.5,0.9,0.9,1.2C256.8,197.9,257.1,198,257.4,198z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M448.2,203.2c0.1,0,0.3,0,0.4,0
			c0.1,0,0.1,0,0.2-0.1c0.9-0.3,1.3-1.2,1.4-2.1c0.1-1.4-0.4-2.7-0.5-4.1c-0.1-1.3,0.3-2.8,1.9-2.8c1.6,0,2.8,1.4,3.7,2.5
			c0.4,0.5,1,0.8,1.6,0.8c0.4,0,0.9-0.1,1.2-0.4c0.4-0.3,0.7-0.8,0.8-1.3c0.1-0.5-0.1-1.1-0.4-1.5c-1.7-2.2-2.6-4.7-3.5-7.1
			c-0.5-1.3-1-2.6-1.5-3.7c-0.3-0.7-1-1.1-1.8-1.1c-0.3,0-0.6,0.1-0.9,0.2c-0.5,0.2-0.8,0.7-1,1.2c-0.2,0.5-0.1,1.1,0.1,1.5l0.1,0.3
			c0.7,1.3,1.9,3.6,0.3,4.7c-0.4,0.3-1.1,0.4-1.6,0.5c-2,0.1-3.3-2.1-4.2-3.6c-0.3-0.5-0.6-0.9-0.8-1.3c-0.6-1-1.2-2.1-1.8-3.2
			c-1-1.7-1.9-3-2.8-4.4c-0.4-0.5-1-0.8-1.6-0.8c-0.4,0-0.8,0.1-1.2,0.4c-0.4,0.3-0.7,0.8-0.8,1.3c-0.1,0.5,0.1,1,0.4,1.5
			c3.1,4.2,8.1,13.7,10.5,21.2C446.6,202.6,447.3,203.2,448.2,203.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M466.7,251.6c1,0,1.9-0.8,2-1.8
			c1-10.2-2.2-23.8-3.5-28.9c-0.2-0.7-0.3-1.2-0.4-1.5c-0.2-0.9-1-1.6-2-1.6c-0.1,0-0.3,0-0.4,0c-1.1,0.2-1.8,1.3-1.5,2.4
			c0.1,0.4,0.2,0.9,0.4,1.6c1.2,4.9,4.3,18,3.4,27.6c-0.1,0.5,0.1,1.1,0.5,1.5c0.3,0.4,0.8,0.7,1.4,0.7
			C466.6,251.6,466.7,251.6,466.7,251.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366,345.6c-0.1-1.1,0.7-2,1.8-2.1
			c1.1-0.1,3.1,0,3.2-1.5c0.2-2.5-3.1-2-4.6-1.9c-1.5,0.1-3,0.2-4.6,0.4c-2,0.2-3.9,0.3-5.9,0.5c-3,0.2-6,0.3-8.9,0.4l-2.1,0.1
			c-1,0-1.9-0.8-1.9-1.8c0-0.5,0.2-1,0.5-1.4c0.3-0.4,0.8-0.6,1.3-0.6l1.2,0c1,0,2.1-0.1,3.1-0.1c1.1-0.1,2.1-0.1,3.3-0.2
			c1.7-0.1,3.6-0.2,5.4-0.3l0.4,0c0.7,0,1.5-0.1,2.2-0.2c0.5-0.1,1.1-0.6,1-1.1c0-0.8-0.8-1.5-1.5-1.8c-0.4-0.2-0.8-0.4-0.9-0.8
			c-0.1-0.2,0-0.5,0.2-0.7c0.6-0.7,1.6-0.6,2.3-0.7c0.7-0.1,1.3-0.5,2-0.5c0.2,0,0.4,0,0.6,0c0.2,0,0.3,0,0.5,0c0.6,0,1.2,0,1.7,0.3
			c0.1,0.1,0.3,0.1,0.4,0.2c0.2,0.1,0.3,0.2,0.6,0.3l0.1,0.1l0,0.1c0.2,0.6-1.2,1.6-1.5,2.1c-0.4,0.7-0.4,1.6,0.4,1.9
			c0.2,0.1,0.5,0.1,0.8,0.1c0.3,0,0.7,0,1-0.1l0.1,0c0.6-0.1,1.3-0.2,1.9-0.3l0.2,0c0.8-0.1,1.6-0.3,2.4-0.4c1.5-0.2,3-0.4,4.4-0.7
			c2.8-0.6,5.9-1.3,8.9-2.1c2-0.6,4-1.2,5.6-1.8c1-0.3,2-0.7,3.2-1.2l1-0.4c2.9-1.2,5.9-2.4,8.7-3.9c0.1-0.1,0.3-0.1,0.4-0.2
			c0.4-0.2,0.8-0.4,1-0.7c0.9-1-0.1-2.4-1.2-2.6c-0.4-0.1-0.7-0.1-1-0.3c-0.1-0.1-0.2-0.3-0.2-0.5c0-0.7,0.8-1.6,1-1.8
			c0.9-0.9,2.4-1.3,3.6-1.7l0.2-0.1l0.2-0.1c0.2-0.1,0.6-0.2,0.9-0.2c0.4,0,0.7,0.2,0.8,0.6c0.1,0.7-0.3,1.4-0.2,2.1
			c0.1,0.6,0.6,1.1,1.2,1.1c0.4,0,0.8-0.2,1.3-0.5c0.7-0.4,1.3-0.9,2-1.3c0.9-0.6,2.1-1.5,3.2-2.5c0.4-0.3,0.8-0.5,1.3-0.5
			c0.6,0,1.1,0.2,1.4,0.7c0.3,0.4,0.5,0.9,0.5,1.4c0,0.5-0.3,1-0.7,1.3c-1.1,1-2.3,1.9-3.7,2.8c-0.7,0.5-1.3,0.9-2,1.3
			c-1.2,0.8-5.7,2.7-4,4.8c1.1,1.4,2.7,0.5,3.8-0.3c0.4-0.3,0.9-0.6,1.4-0.9c1-0.5,2-0.6,2.7,0.4c0.6,0.9,0.5,2.1-0.4,2.8
			c-12.7,9.3-30.9,15.5-50,17c-0.1,0-0.1,0-0.2,0C366.9,347.4,366,346.6,366,345.6z M403.9,329.9c-1.6,0.3-2.9,1.1-4.4,1.7l-0.6,0.3
			c-2.3,1-4.8,2-7.5,2.9c-1.4,0.5-2.9,1-4.4,1.4c-2.8,0.8-5.6,1.5-8.4,2.1c-0.9,0.2-2.1,0.4-2.6,1.3c-0.4,0.8-0.1,1.6,0.6,2.1
			c0.4,0.3,1,0.4,1.5,0.5c1.5,0.1,3.1-0.4,4.5-0.8c0.9-0.2,1.9-0.5,2.8-0.7c0.8-0.2,1.6-0.4,2.4-0.6c2.1-0.5,4.6-1.2,7.1-2.1
			c2.3-0.8,4.6-1.7,7-2.8c1.1-0.5,2.1-0.9,3.1-1.4c0.8-0.4,1.6-0.8,1.7-1.8c0.1-0.9-0.5-1.9-1.4-2
			C404.8,329.8,404.3,329.9,403.9,329.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366.7,355.7c1.7-0.1,6.1-0.3,11.2-1.2
			c0.5-0.1,1-0.4,1.3-0.8c0.3-0.4,0.4-1,0.3-1.5c-0.2-1-1-1.6-2-1.6c-0.1,0-0.3,0-0.4,0c-4.9,0.9-9,1.1-10.6,1.2c-1.1,0-2,1-2,2.1
			C364.7,354.8,365.6,355.7,366.7,355.7L366.7,355.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M274.7,334.9c-7.3-3.4-16.4-11.8-25.1-23.1
			c-10.1-13.1-12.2-23-14.9-35.6l-0.5-2.2c-0.1-0.5,0-1,0.3-1.5c0.3-0.4,0.7-0.7,1.2-0.9c0.1,0,0.3,0,0.4,0c0.9,0,1.7,0.6,1.9,1.6
			c0.3,1.2,0.4,2.5,1.3,3.4c0.6,0.6,1.5,0.7,2.2,0.3c1.6-0.9,1.1-2.5,0.8-4c-0.2-0.8-0.3-1.6-0.5-2.3c-0.4-2-0.8-4-1.2-6.2
			c-0.1-0.5,0-1,0.3-1.4c0.3-0.4,0.7-0.7,1.2-0.8c0.1,0,0.3,0,0.4,0c0.9,0,1.7,0.6,1.9,1.5c0.4,2.1,0.8,4.2,1.2,6.3
			c1.8,9.7,3.4,18.1,11.1,29.9c9.5,14.5,15.1,19.5,22.8,23.9c0.9,0.5,1.2,1.7,0.7,2.6c-0.3,0.5-0.7,0.8-1.3,0.9c-0.1,0-0.2,0-0.4,0
			c-0.3,0-0.7-0.1-1-0.3c-0.7-0.4-1.4-0.8-2.1-1.3c-0.8-0.6-2.4-0.9-3.2-0.2c-0.7,0.6-0.9,1.9-0.5,2.6c0.5,1,1.6,1.5,2.5,2
			c0.7,0.4,1.4,0.8,2.1,1.1c0.5,0.2,0.8,0.6,1,1.1c0.2,0.5,0.2,1-0.1,1.5c-0.3,0.7-1,1.1-1.8,1.1C275.3,335.1,275,335,274.7,334.9z
			 M241.6,282.4c-1.5,0.9-1,3.2-0.6,4.6c0.3,1.1,0.8,2.2,1.2,3.3c0.9,2.4,1.8,4.4,2.6,6.3c1,2.3,2,4.2,3.2,6.1
			c1.3,2.2,2.9,4.4,4.7,6.7c1.2,1.6,2.3,2.9,3.3,4c2.3,2.7,4.1,4.7,5.9,6.5c0.5,0.5,0.9,1,1.4,1.5c0.8,0.9,1.8,2,3,2.3
			c1,0.2,2.1-0.3,2.3-1.3c0.5-2.1-1.7-3.2-2.9-4.4c-0.7-0.7-1.3-1.4-1.9-2.1c-1.2-1.4-2.4-2.8-3.5-4.3c-2.6-3.4-5-7-6.8-9.7
			c-1.1-1.7-2.1-3.4-3-5c-1.6-2.9-3.1-6-4.2-9c-0.4-1.1-0.8-2.3-1.1-3.5c-0.3-1-1.2-2-2.2-2.3C242.6,282,242.1,282.1,241.6,282.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M265.8,340.5c0.7,0,1.3-0.3,1.6-0.8
			c0.6-0.9,0.4-2.2-0.5-2.8c-4.7-3.3-9.8-8.1-12.3-11c-0.4-0.4-0.9-0.7-1.5-0.7c-0.5,0-1,0.2-1.3,0.5c-0.4,0.4-0.7,0.8-0.7,1.4
			c0,0.5,0.1,1.1,0.5,1.5c2.7,3.1,8,8.1,13,11.6C265,340.3,265.4,340.5,265.8,340.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M234,256c1.1,0,2-0.9,2-2
			c0-1.8,0.1-3.5,0.2-5.2c0.3-3.5,0.7-6.7,1.2-9.9c0.2-0.9,0.3-1.9,0.4-2.8l0.1-0.5c0.1-0.6,0.2-1.2,0.3-1.7
			c0.1-0.4,0.1-0.9,0.2-1.3c0.2-1,0.6-2,1.6-2.6c1-0.6,2.2-0.4,2.8,0.6c0.7,1,0.5,2.6,0.3,3.8c-0.4,2.6-0.8,5.1-1.1,7.7
			c-0.3,2.1-0.5,4.3-0.6,6.4c0,0.5,0.1,1,0.5,1.4c0.3,0.4,0.8,0.6,1.3,0.6c0.3,0,0.4,0,0.5,0c0.9-0.2,1.5-0.9,1.5-1.8
			c0.5-8.5,2.8-21.7,4.3-27.6c0.1-0.5,0-1-0.2-1.5c-0.3-0.4-0.7-0.8-1.2-0.9c-0.2,0-0.3-0.1-0.5-0.1c-0.9,0-1.7,0.6-1.9,1.5
			c-0.3,1.2-0.4,2.5-1,3.7c-0.5,1-1.5,2-2.7,1.9c-1.9-0.2-2-2.4-1.7-3.8c0.1-0.6,0.2-1.2,0.3-1.8c0.5-2.7,1-5.4,1.1-8.2
			c0-1.1-0.8-2-1.9-2c-1.1,0-2,0.8-2,1.9c-0.1,3-1,8-2.1,13.7C234,234.3,232,245,232,254C232,255.1,232.9,256,234,256z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M226.5,237.6c1,0,1.9-0.8,2-1.8
			c0.3-2.3,1.2-8.2,2.5-14.5c0.1-0.5,0-1.1-0.3-1.5c-0.3-0.5-0.8-0.8-1.3-0.9c-0.1,0-0.3,0-0.4,0c-1,0-1.8,0.7-2,1.6
			c-1.3,6.4-2.3,12.5-2.6,14.8c-0.1,1.1,0.6,2.1,1.8,2.3C226.3,237.6,226.4,237.6,226.5,237.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
</g>
<g class="col endoplasmic_reticulum" descendants="level 0: ['endoplasmic reticulum, pval = 0.10307']
level 1: ['oligosaccharyltransferase complex, pval = --', 'MHC class I peptide loading complex, pval = 0.43974', 'rough endoplasmic reticulum, pval = --', 'smooth endoplasmic reticulum, pval = 0.43974', 'maintenance of protein localization in endoplasmic reticulum, pval = --', 'GPI-anchor transamidase complex, pval = --', 'endoplasmic reticulum calcium ion homeostasis, pval = --', 'endoplasmic reticulum quality control compartment, pval = --', 'plasmodesmatal endoplasmic reticulum, pval = --', 'egasyn-beta-glucuronidase complex, pval = --', 'glucosidase II complex, pval = --', 'extrinsic component of endoplasmic reticulum membrane, pval = 0.43974', 'VCP-NPL4-UFD1 AAA ATPase complex, pval = --', 'protein glycosylation in endoplasmic reticulum, pval = --', 'Dsl1/NZR complex, pval = --', 'fatty acid elongase complex, pval = --', 'perinuclear endoplasmic reticulum, pval = 0.19304', 'SREBP-SCAP-Insig complex, pval = --', 'membrane stack, pval = --', 'endoplasmic reticulum exit site, pval = --', 'endoplasmic reticulum chaperone complex, pval = 0.43974', 'Ig heavy chain-bound endoplasmic reticulum chaperone complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase complex, pval = --', 'GET complex, pval = --', 'glycosylphosphatidylinositol-mannosyltransferase I complex, pval = --', 'endoplasmic reticulum lumen, pval = 0.87374', 'UDP-N-acetylglucosamine transferase complex, pval = --', 'ERMES complex, pval = --', 'procollagen-proline 4-dioxygenase complex, pval = --', 'sarcoplasmic reticulum, pval = 0.22736', 'endoplasmic reticulum subcompartment, pval = 0.0018', 'glycosylphosphatidylinositol-mannosyltransferase II complex, pval = --', 'ER ubiquitin ligase complex, pval = --', 'plasmodesmatal desmotubule, pval = --']
level 2: ['oligosaccharyltransferase I complex, pval = --', 'oligosaccharyltransferase II complex, pval = --', 'oligosaccharyltransferase III complex, pval = --', 'Tapasin-ERp57 complex, pval = --', 'TAP complex, pval = --', 'rough endoplasmic reticulum membrane, pval = --', 'rough endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum membrane, pval = --', 'smooth endoplasmic reticulum calcium ion homeostasis, pval = --', 'smooth endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum cisterna, pval = --', 'protein retention in ER lumen, pval = --', 'SREBP-SCAP complex retention in endoplasmic reticulum, pval = --', 'positive regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'endoplasmic reticulum mannose trimming, pval = --', 'protein galactosylation in endoplasmic reticulum, pval = --', 'perinuclear endoplasmic reticulum membrane, pval = 0.43974', 'perinuclear endoplasmic reticulum lumen, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt4p homodimer complex, pval = --', 'sarcoplasmic reticulum lumen, pval = --', 'cortical endoplasmic reticulum lumen, pval = --', 'luminal surveillance complex, pval = --', 'mannosyl-oligosaccharide 1,2-alpha-mannosidase complex, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(III) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(II) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(I) type, pval = --', 'sarcoplasmic reticulum membrane, pval = 0.43974', 'longitudinal sarcoplasmic reticulum, pval = --', 'terminal cisterna, pval = --', 'subrhabdomeral cisterna, pval = --', 'endoplasmic reticulum tubular network, pval = --', 'endoplasmic reticulum cisternal network, pval = --', 'CUE1-UBC7 ubiquitin-conjugating enzyme complex, pval = --', 'Hrd1p ubiquitin ligase complex, pval = --', 'Doa10p ubiquitin ligase complex, pval = --']
level 3: ['signal recognition particle receptor complex, pval = --', 'translocon complex, pval = --', 'endoplasmic reticulum Sec complex, pval = --', 'lumenal side of rough endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of rough endoplasmic reticulum membrane, pval = --', 'lumenal side of smooth endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of smooth endoplasmic reticulum membrane, pval = --', 'positive regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'hypolemmal cisterna, pval = --', 'trimming of terminal mannose on B branch, pval = --', 'trimming of second mannose on A branch, pval = --', 'trimming of first mannose on A branch, pval = --', 'trimming of terminal mannose on C branch, pval = --', 'longitudinal sarcoplasmic reticulum lumen, pval = --', 'terminal cisterna lumen, pval = --', 'phospholamban complex, pval = --', 'ryanodine receptor complex, pval = --', 'junctional sarcoplasmic reticulum membrane, pval = --', 'free sarcoplasmic reticulum membrane, pval = --', 'cortical endoplasmic reticulum, pval = --', 'rough endoplasmic reticulum cisterna, pval = --', 'Hrd1p ubiquitin ligase ERAD-M complex, pval = --', 'Hrd1p ubiquitin ligase ERAD-L complex, pval = --']
level 4: ['Sec61 translocon complex, pval = --', 'Ssh1 translocon complex, pval = --', 'Sec62/Sec63 complex, pval = --', 'sub-surface cisterna, pval = --']" id="endoplasmic_reticulum" init-pval="0.10307" min-pval="0.0018" min-pval-children="0.0018" title="Endoplasmic reticulum">
<g>
<path d="M425.7,314.6c0.1,0,0.3,0,0.4,0
			c0.3-0.1,0.7-0.2,0.9-0.5c3.1-2.8,4.6-4.9,6.9-9.5c0.4-0.9,1.2-2.2,2.2-3.8c2.4-4,5.6-9.4,7.1-13.8c2.4-6.9,5.7-17.3,6.8-27.6
			c0.2-1.7,0.8-3.7,2.9-3.3c0.8,0.2,1.6,0.7,2.1,1.4c0.4,0.5,0.7,1,1.4,1.2c0.2,0.1,0.3,0.1,0.5,0.1c0.9,0,1.7-0.7,1.9-1.6
			c0.7-3.4,0.8-10.8,0.1-18c-0.6-6.7-3.8-23.1-5.6-28.8c-0.3-0.8-1-1.4-1.9-1.4c-0.2,0-0.4,0-0.6,0.1c-0.9,0.3-1.5,1.2-1.3,2.2
			c0,0.4-0.1,0.8-0.4,1.1c-0.4,0.4-1.4,0.8-1.9,0.7c-1.3-0.1-1.7-1.2-1.9-2.3c-0.1-0.4-0.2-0.8-0.3-1.1l-0.3-0.9
			c-0.4-1.1-0.7-2.3-1.1-3.4c-0.4-1.1-0.8-2.2-1.2-3.2c-0.4-1-0.9-2.1-1.4-3.1c-0.5-1-1-2.1-1.6-3.2c-0.6-1.1-1.2-2.2-1.8-3.3
			c-0.7-1.1-1.4-2.3-2.1-3.4c-0.8-1.2-1.6-2.3-2.5-3.6c-0.9-1.3-1.9-2.6-2.9-3.9l-0.2-0.2c-1-1.3-2-2.6-3-3.8
			c-10.8-13.7-23.2-24.6-34-29.9c-4.4-2.2-12.2-4.3-18-5.5c-6-1.3-12.2-1.7-16.3-1.9c-1.4-0.1-2.6-0.2-3.3-0.2
			c-1.7-0.2-3.5-0.2-5.5-0.2l-0.8,0c-1.9,0-4.1,0-5.9-0.2l-0.1,0c-1,0-1.9,0.8-1.9,1.8c-0.1,1.1,0.7,2,1.8,2.1
			c2,0.2,4.1,0.2,6.2,0.2c2.1,0,4.2,0.1,6,0.2c0.5,0,1.2,0.1,2.1,0.1c3.6,0.2,9.5,0.6,15.2,1.6c4.9,0.8,14,3,19.1,5.6
			c7.4,3.7,19.1,11.7,32.6,28.8c12.5,15.9,16,23.9,19.4,38.6c0.2,0.9,1,1.5,1.9,1.5c0.5,0,1-0.2,1.3-0.5c0.8-0.8,0.3-2.3,1.2-3
			c1.7-1.2,3.3,0.7,3.6,2.2c0,0.2,0.1,0.4,0.1,0.6c0.3,1.2,0.5,2.5,0.8,3.7l0.1,0.3c0.1,0.6,0.2,1.2,0.3,1.8c0.2,1,0.4,2,0.5,3
			c0.1,0.8,0.3,1.5,0.4,2.2c0.3,1.4,0.5,2.9,0.8,4.7l0.1,0.6c0.2,1.1,0.3,2.2,0.4,3.3l0,0.4c0.1,1.6,0.2,3.3,0.3,4.9
			c0,1,0.2,1.9,0.2,2.9c0.1,1.8-0.5,4.3-2.9,4.1c-1.9-0.2-2-2.5-2.1-4c0-1-0.1-2.1-0.1-3.1c-0.1-1.9-0.2-4.1-0.4-6.3
			c-0.1-1.8-0.3-3.6-0.6-5.5c-0.2-1.4-0.4-2.7-0.6-4.1c-0.1-0.8-0.3-1.6-0.4-2.5c-0.2-0.9-1-1.6-1.9-1.6c-0.1,0-0.2,0-0.3,0
			c-0.5,0.1-0.9,0.3-1.2,0.8c-0.3,0.4-0.4,1-0.3,1.5c0.3,1.9,3.2,18.7,1.3,33.1c-1,8-3.9,18.5-5.6,23.4c-2.1,6.2-5.5,11.7-7.9,15.8
			c-0.9,1.6-1.7,2.9-2.2,3.9c-2.2,4.4-3.5,6.1-6.1,8.4c-0.4,0.3-0.6,0.8-0.6,1.3c0,0.5,0.2,1,0.5,1.4
			C424.7,314.4,425.2,314.6,425.7,314.6z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M335.1,349.2c3.5,0,6.7-0.2,9.5-0.5
			c1.9-0.1,3.8-0.3,5.4-0.3c0.1,0,0.2,0,0.3,0c0.9-0.2,1.6-1,1.6-2c0-1.1-0.9-1.9-2-1.9c-1.8,0-3.6,0.2-5.5,0.3l-0.1,0
			c-2.8,0.2-5.9,0.5-9.3,0.5c-1.6,0-3.1-0.1-4.6-0.2l-0.1,0c-1,0-1.9,0.8-2,1.8c0,0.5,0.1,1,0.5,1.4c0.3,0.4,0.8,0.6,1.3,0.7
			C331.9,349.2,333.4,349.2,335.1,349.2z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M257.4,198c0.1,0,0.3,0,0.4,0
			c0.6-0.1,1-0.5,1.3-1c0.2-0.3,0.3-0.6,0.5-0.8c0.4-0.7,0.7-1.4,1.1-2c0.4-0.6,0.7-1.1,1.1-1.7c0.7-1,1.3-2.1,1.9-3.1
			c0.3-0.6,0.6-1.3,1.3-1.6c0.8-0.3,1.6,0.4,1.7,1.1c0.1,0.4,0,0.7,0,1.1c0,0.2-0.2,1,0,1.2c0.6,0.8,1.6-0.8,2-1.2
			c0.5-0.5,1.1-1,1.6-1.6l0.1-0.2c0.3-0.4,0.7-0.7,0.9-1.2l0.1-0.2c0.5-0.6,0.8-1.2,0.7-1.7c-0.1-0.1-0.2-0.4-0.7-0.4
			c-0.3,0-0.6,0.1-0.8,0.1c-0.1,0-0.2,0-0.2,0.1c-0.2,0-0.5,0.1-0.8,0.1c-0.8,0-1.2-0.2-1.4-0.7c-0.3-0.7,0.2-1.6,0.6-2.1
			c0.3-0.5,0.8-0.9,1.2-1.3c0.3-0.3,0.6-0.6,0.8-0.9l0.2-0.2c1.3-1.4,2.5-2.8,3.8-4.1c2.9-3,6.1-5.8,9.6-8.7c2-1.7,4-3.3,6.3-5
			c1.1-0.9,3.8-2.6,4.3-0.2c0.2,0.7-0.2,1.3-0.4,1.9c-0.2,0.8,0.5,0.7,1,0.6c0.3-0.1,0.7-0.2,1-0.4c0.3-0.2,0.7-0.6,1-0.7
			c0.2-0.1,0.6-0.2,0.9-0.4c0.8-0.3,1.4-0.7,2.1-1.2c0.5-0.4,1.3-0.5,1.4-1.2c0.1-1.2-1.9-0.7-2.5-0.8c-0.9-0.2-1.3-1.2-1.1-2
			c0.2-1,1.2-1.5,2.1-1.9c0.8-0.3,1.7-0.7,2.5-1.1l0.2-0.1c0.8-0.3,1.5-0.7,2.2-1c3.2-1.5,6.3-2.7,9-3.7c2.6-1,5.3-1.9,8.3-2.8
			c2.3-0.7,4.7-1.3,6.9-1.9c1.8-0.4,3.3-0.8,4.7-1.1c0.5-0.1,0.9-0.4,1.2-0.8c0.3-0.4,0.4-0.9,0.3-1.4c-0.2-0.9-1-1.5-1.9-1.5
			c-0.1,0-0.3,0-0.4,0c-6.6,1.3-29.3,6.6-45.2,18.7l-0.6,0.4c-1.5,1.2-3,2.3-4.5,3.5c-1.4,1.2-2.8,2.3-4.2,3.5
			c-1.4,1.2-2.7,2.4-3.9,3.5c-1.3,1.2-2.5,2.4-3.6,3.5c-0.3,0.3-0.6,0.6-0.8,0.9c-0.7,0.7-1.4,1.6-2.4,1.9c-1.5,0.3-2.8-1.2-2.3-2.6
			c0.4-1.2,1.2-2.1,2.1-2.9c0.8-0.7,1.5-1.4,2.3-2.2c1.3-1.3,2.7-2.6,4-3.8l1.1-1c10.3-9.2,16.9-14.2,28.5-19.5
			c0.5-0.2,0.8-0.6,1-1.1c0.2-0.5,0.2-1-0.1-1.5c-0.3-0.7-1-1.2-1.8-1.2c-0.3,0-0.6,0.1-0.8,0.2c-1.3,0.6-2.6,1.2-4,1.9
			c-0.4,0.2-0.8,0.4-1.2,0.6c-0.8,0.4-1.4,0.7-2.1,1.1c-1.1,0.6-2.2,1.3-3.2,2c-1,0.7-2,1.3-2.9,1.8c-0.9,0.5-2,0.6-2.8,0
			c-0.8-0.5-1.4-1.5-1.3-2.5c0.1-0.8,0.8-1.5,1-2.2c0.2-0.6,0.1-1.4-0.2-2c-0.6-1-2.5-1.1-3.3-0.4c-7.5,5.8-13.6,10.8-18.2,15.1
			c-0.4,0.4-0.6,0.9-0.6,1.4c0,0.5,0.2,1.1,0.5,1.5c0.4,0.4,0.9,0.7,1.5,0.7c0.5,0,1-0.2,1.4-0.5c1.4-1.3,2.9-2.6,4.2-3.7l1.4-1.2
			c0.9-0.8,1.8-1.5,2.7-2.2l0.8-0.7c0.6-0.5,1.1-0.9,1.7-1.4c1.1-0.8,3-1.7,4.3-0.9c1.3,0.8,0.9,2.7,0.3,3.8c-0.7,1.2-2,1.8-3.1,2.6
			c-1.2,0.9-2.3,1.9-3.5,2.9c0,0-2.9,2.6-2.9,2.6c-9.5,8.5-16.6,16.3-19.9,22.1c-0.3,0.5-0.3,1-0.2,1.5c0.1,0.5,0.5,0.9,0.9,1.2
			c0.3,0.2,0.6,0.3,1,0.3c0.7,0,1.4-0.4,1.7-1c0.7-1.2,1.5-2.3,2.3-3.4c0.2-0.3,0.4-0.5,0.5-0.8c0.7-1,1.9-2.1,3.2-1.8
			c1.8,0.5,1.6,3.1,0.8,4.3c-0.7,1-1.5,1.8-2.2,2.8c-0.7,1-1.3,2-1.9,3c-1.3,2-2.5,4-3.7,6.2c-0.2,0.5-0.3,1-0.2,1.5
			c0.1,0.5,0.5,0.9,0.9,1.2C256.8,197.9,257.1,198,257.4,198z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M448.2,203.2c0.1,0,0.3,0,0.4,0
			c0.1,0,0.1,0,0.2-0.1c0.9-0.3,1.3-1.2,1.4-2.1c0.1-1.4-0.4-2.7-0.5-4.1c-0.1-1.3,0.3-2.8,1.9-2.8c1.6,0,2.8,1.4,3.7,2.5
			c0.4,0.5,1,0.8,1.6,0.8c0.4,0,0.9-0.1,1.2-0.4c0.4-0.3,0.7-0.8,0.8-1.3c0.1-0.5-0.1-1.1-0.4-1.5c-1.7-2.2-2.6-4.7-3.5-7.1
			c-0.5-1.3-1-2.6-1.5-3.7c-0.3-0.7-1-1.1-1.8-1.1c-0.3,0-0.6,0.1-0.9,0.2c-0.5,0.2-0.8,0.7-1,1.2c-0.2,0.5-0.1,1.1,0.1,1.5l0.1,0.3
			c0.7,1.3,1.9,3.6,0.3,4.7c-0.4,0.3-1.1,0.4-1.6,0.5c-2,0.1-3.3-2.1-4.2-3.6c-0.3-0.5-0.6-0.9-0.8-1.3c-0.6-1-1.2-2.1-1.8-3.2
			c-1-1.7-1.9-3-2.8-4.4c-0.4-0.5-1-0.8-1.6-0.8c-0.4,0-0.8,0.1-1.2,0.4c-0.4,0.3-0.7,0.8-0.8,1.3c-0.1,0.5,0.1,1,0.4,1.5
			c3.1,4.2,8.1,13.7,10.5,21.2C446.6,202.6,447.3,203.2,448.2,203.2z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M466.7,251.6c1,0,1.9-0.8,2-1.8
			c1-10.2-2.2-23.8-3.5-28.9c-0.2-0.7-0.3-1.2-0.4-1.5c-0.2-0.9-1-1.6-2-1.6c-0.1,0-0.3,0-0.4,0c-1.1,0.2-1.8,1.3-1.5,2.4
			c0.1,0.4,0.2,0.9,0.4,1.6c1.2,4.9,4.3,18,3.4,27.6c-0.1,0.5,0.1,1.1,0.5,1.5c0.3,0.4,0.8,0.7,1.4,0.7
			C466.6,251.6,466.7,251.6,466.7,251.6z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366,345.6c-0.1-1.1,0.7-2,1.8-2.1
			c1.1-0.1,3.1,0,3.2-1.5c0.2-2.5-3.1-2-4.6-1.9c-1.5,0.1-3,0.2-4.6,0.4c-2,0.2-3.9,0.3-5.9,0.5c-3,0.2-6,0.3-8.9,0.4l-2.1,0.1
			c-1,0-1.9-0.8-1.9-1.8c0-0.5,0.2-1,0.5-1.4c0.3-0.4,0.8-0.6,1.3-0.6l1.2,0c1,0,2.1-0.1,3.1-0.1c1.1-0.1,2.1-0.1,3.3-0.2
			c1.7-0.1,3.6-0.2,5.4-0.3l0.4,0c0.7,0,1.5-0.1,2.2-0.2c0.5-0.1,1.1-0.6,1-1.1c0-0.8-0.8-1.5-1.5-1.8c-0.4-0.2-0.8-0.4-0.9-0.8
			c-0.1-0.2,0-0.5,0.2-0.7c0.6-0.7,1.6-0.6,2.3-0.7c0.7-0.1,1.3-0.5,2-0.5c0.2,0,0.4,0,0.6,0c0.2,0,0.3,0,0.5,0c0.6,0,1.2,0,1.7,0.3
			c0.1,0.1,0.3,0.1,0.4,0.2c0.2,0.1,0.3,0.2,0.6,0.3l0.1,0.1l0,0.1c0.2,0.6-1.2,1.6-1.5,2.1c-0.4,0.7-0.4,1.6,0.4,1.9
			c0.2,0.1,0.5,0.1,0.8,0.1c0.3,0,0.7,0,1-0.1l0.1,0c0.6-0.1,1.3-0.2,1.9-0.3l0.2,0c0.8-0.1,1.6-0.3,2.4-0.4c1.5-0.2,3-0.4,4.4-0.7
			c2.8-0.6,5.9-1.3,8.9-2.1c2-0.6,4-1.2,5.6-1.8c1-0.3,2-0.7,3.2-1.2l1-0.4c2.9-1.2,5.9-2.4,8.7-3.9c0.1-0.1,0.3-0.1,0.4-0.2
			c0.4-0.2,0.8-0.4,1-0.7c0.9-1-0.1-2.4-1.2-2.6c-0.4-0.1-0.7-0.1-1-0.3c-0.1-0.1-0.2-0.3-0.2-0.5c0-0.7,0.8-1.6,1-1.8
			c0.9-0.9,2.4-1.3,3.6-1.7l0.2-0.1l0.2-0.1c0.2-0.1,0.6-0.2,0.9-0.2c0.4,0,0.7,0.2,0.8,0.6c0.1,0.7-0.3,1.4-0.2,2.1
			c0.1,0.6,0.6,1.1,1.2,1.1c0.4,0,0.8-0.2,1.3-0.5c0.7-0.4,1.3-0.9,2-1.3c0.9-0.6,2.1-1.5,3.2-2.5c0.4-0.3,0.8-0.5,1.3-0.5
			c0.6,0,1.1,0.2,1.4,0.7c0.3,0.4,0.5,0.9,0.5,1.4c0,0.5-0.3,1-0.7,1.3c-1.1,1-2.3,1.9-3.7,2.8c-0.7,0.5-1.3,0.9-2,1.3
			c-1.2,0.8-5.7,2.7-4,4.8c1.1,1.4,2.7,0.5,3.8-0.3c0.4-0.3,0.9-0.6,1.4-0.9c1-0.5,2-0.6,2.7,0.4c0.6,0.9,0.5,2.1-0.4,2.8
			c-12.7,9.3-30.9,15.5-50,17c-0.1,0-0.1,0-0.2,0C366.9,347.4,366,346.6,366,345.6z M403.9,329.9c-1.6,0.3-2.9,1.1-4.4,1.7l-0.6,0.3
			c-2.3,1-4.8,2-7.5,2.9c-1.4,0.5-2.9,1-4.4,1.4c-2.8,0.8-5.6,1.5-8.4,2.1c-0.9,0.2-2.1,0.4-2.6,1.3c-0.4,0.8-0.1,1.6,0.6,2.1
			c0.4,0.3,1,0.4,1.5,0.5c1.5,0.1,3.1-0.4,4.5-0.8c0.9-0.2,1.9-0.5,2.8-0.7c0.8-0.2,1.6-0.4,2.4-0.6c2.1-0.5,4.6-1.2,7.1-2.1
			c2.3-0.8,4.6-1.7,7-2.8c1.1-0.5,2.1-0.9,3.1-1.4c0.8-0.4,1.6-0.8,1.7-1.8c0.1-0.9-0.5-1.9-1.4-2
			C404.8,329.8,404.3,329.9,403.9,329.9z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M366.7,355.7c1.7-0.1,6.1-0.3,11.2-1.2
			c0.5-0.1,1-0.4,1.3-0.8c0.3-0.4,0.4-1,0.3-1.5c-0.2-1-1-1.6-2-1.6c-0.1,0-0.3,0-0.4,0c-4.9,0.9-9,1.1-10.6,1.2c-1.1,0-2,1-2,2.1
			C364.7,354.8,365.6,355.7,366.7,355.7L366.7,355.7z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M274.7,334.9c-7.3-3.4-16.4-11.8-25.1-23.1
			c-10.1-13.1-12.2-23-14.9-35.6l-0.5-2.2c-0.1-0.5,0-1,0.3-1.5c0.3-0.4,0.7-0.7,1.2-0.9c0.1,0,0.3,0,0.4,0c0.9,0,1.7,0.6,1.9,1.6
			c0.3,1.2,0.4,2.5,1.3,3.4c0.6,0.6,1.5,0.7,2.2,0.3c1.6-0.9,1.1-2.5,0.8-4c-0.2-0.8-0.3-1.6-0.5-2.3c-0.4-2-0.8-4-1.2-6.2
			c-0.1-0.5,0-1,0.3-1.4c0.3-0.4,0.7-0.7,1.2-0.8c0.1,0,0.3,0,0.4,0c0.9,0,1.7,0.6,1.9,1.5c0.4,2.1,0.8,4.2,1.2,6.3
			c1.8,9.7,3.4,18.1,11.1,29.9c9.5,14.5,15.1,19.5,22.8,23.9c0.9,0.5,1.2,1.7,0.7,2.6c-0.3,0.5-0.7,0.8-1.3,0.9c-0.1,0-0.2,0-0.4,0
			c-0.3,0-0.7-0.1-1-0.3c-0.7-0.4-1.4-0.8-2.1-1.3c-0.8-0.6-2.4-0.9-3.2-0.2c-0.7,0.6-0.9,1.9-0.5,2.6c0.5,1,1.6,1.5,2.5,2
			c0.7,0.4,1.4,0.8,2.1,1.1c0.5,0.2,0.8,0.6,1,1.1c0.2,0.5,0.2,1-0.1,1.5c-0.3,0.7-1,1.1-1.8,1.1C275.3,335.1,275,335,274.7,334.9z
			 M241.6,282.4c-1.5,0.9-1,3.2-0.6,4.6c0.3,1.1,0.8,2.2,1.2,3.3c0.9,2.4,1.8,4.4,2.6,6.3c1,2.3,2,4.2,3.2,6.1
			c1.3,2.2,2.9,4.4,4.7,6.7c1.2,1.6,2.3,2.9,3.3,4c2.3,2.7,4.1,4.7,5.9,6.5c0.5,0.5,0.9,1,1.4,1.5c0.8,0.9,1.8,2,3,2.3
			c1,0.2,2.1-0.3,2.3-1.3c0.5-2.1-1.7-3.2-2.9-4.4c-0.7-0.7-1.3-1.4-1.9-2.1c-1.2-1.4-2.4-2.8-3.5-4.3c-2.6-3.4-5-7-6.8-9.7
			c-1.1-1.7-2.1-3.4-3-5c-1.6-2.9-3.1-6-4.2-9c-0.4-1.1-0.8-2.3-1.1-3.5c-0.3-1-1.2-2-2.2-2.3C242.6,282,242.1,282.1,241.6,282.4z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M265.8,340.5c0.7,0,1.3-0.3,1.6-0.8
			c0.6-0.9,0.4-2.2-0.5-2.8c-4.7-3.3-9.8-8.1-12.3-11c-0.4-0.4-0.9-0.7-1.5-0.7c-0.5,0-1,0.2-1.3,0.5c-0.4,0.4-0.7,0.8-0.7,1.4
			c0,0.5,0.1,1.1,0.5,1.5c2.7,3.1,8,8.1,13,11.6C265,340.3,265.4,340.5,265.8,340.5z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M234,256c1.1,0,2-0.9,2-2
			c0-1.8,0.1-3.5,0.2-5.2c0.3-3.5,0.7-6.7,1.2-9.9c0.2-0.9,0.3-1.9,0.4-2.8l0.1-0.5c0.1-0.6,0.2-1.2,0.3-1.7
			c0.1-0.4,0.1-0.9,0.2-1.3c0.2-1,0.6-2,1.6-2.6c1-0.6,2.2-0.4,2.8,0.6c0.7,1,0.5,2.6,0.3,3.8c-0.4,2.6-0.8,5.1-1.1,7.7
			c-0.3,2.1-0.5,4.3-0.6,6.4c0,0.5,0.1,1,0.5,1.4c0.3,0.4,0.8,0.6,1.3,0.6c0.3,0,0.4,0,0.5,0c0.9-0.2,1.5-0.9,1.5-1.8
			c0.5-8.5,2.8-21.7,4.3-27.6c0.1-0.5,0-1-0.2-1.5c-0.3-0.4-0.7-0.8-1.2-0.9c-0.2,0-0.3-0.1-0.5-0.1c-0.9,0-1.7,0.6-1.9,1.5
			c-0.3,1.2-0.4,2.5-1,3.7c-0.5,1-1.5,2-2.7,1.9c-1.9-0.2-2-2.4-1.7-3.8c0.1-0.6,0.2-1.2,0.3-1.8c0.5-2.7,1-5.4,1.1-8.2
			c0-1.1-0.8-2-1.9-2c-1.1,0-2,0.8-2,1.9c-0.1,3-1,8-2.1,13.7C234,234.3,232,245,232,254C232,255.1,232.9,256,234,256z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M226.5,237.6c1,0,1.9-0.8,2-1.8
			c0.3-2.3,1.2-8.2,2.5-14.5c0.1-0.5,0-1.1-0.3-1.5c-0.3-0.5-0.8-0.8-1.3-0.9c-0.1,0-0.3,0-0.4,0c-1,0-1.8,0.7-2,1.6
			c-1.3,6.4-2.3,12.5-2.6,14.8c-0.1,1.1,0.6,2.1,1.8,2.3C226.3,237.6,226.4,237.6,226.5,237.6z" style="fill:#8F83BD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
</g>
<g class="ton golgi_apparatus" descendants="level 0: ['Golgi apparatus, pval = 0.02368']
level 1: ['Golgi subcompartment, pval = 0.19764', 'Golgi transport complex, pval = 0.43974', 'Golgi calcium ion homeostasis, pval = --', 'exomer complex, pval = --', 'cis-Golgi network, pval = 0.43974', 'RIC1-RGP1 guanyl-nucleotide exchange factor complex, pval = --', 'postsynaptic Golgi apparatus, pval = --', 'TRAPPII protein complex, pval = --', 'TRAPPIII protein complex, pval = --', 'inositol phosphoceramide synthase complex, pval = --', 'protein glycosylation in Golgi, pval = --', 'Golgi-associated vesicle lumen, pval = --', 'Golgi lumen, pval = 0.89614', 'Golgi membrane, pval = 0.14987', 'Golgi apparatus mannose trimming, pval = --', 'GARP complex, pval = --', 'clathrin coat of trans-Golgi network vesicle, pval = --', 'COPI vesicle coat, pval = --', 'acroblast, pval = --']
level 2: ['trans-Golgi network, pval = 0.59248', 'Golgi cisterna, pval = 0.43974', 'Golgi stack, pval = 0.43974', 'cis-Golgi network membrane, pval = --', 'negative regulation of protein glycosylation in Golgi, pval = --', 'positive regulation of protein glycosylation in Golgi, pval = --', 'regulation of protein glycosylation in Golgi, pval = --', 'protein galactosylation in Golgi, pval = --', 'Golgi stack lumen, pval = --', 'trans-Golgi network transport vesicle lumen, pval = --', 'Golgi vesicle budding, pval = --', 'intrinsic component of Golgi membrane, pval = 0.68643', 'extrinsic component of Golgi membrane, pval = --', 'lumenal side of Golgi membrane, pval = --', 'cytoplasmic side of Golgi membrane, pval = --', 'trans-Golgi network membrane, pval = 0.68643', 'Golgi cisterna membrane, pval = 0.43974', 'AP-1 adaptor complex, pval = --']
level 3: ['Golgi medial cisterna, pval = --', 'Golgi trans cisterna, pval = --', 'Golgi cis cisterna, pval = --', 'COPII vesicles tethering complex, pval = --', 'Golgi transport vesicle coating, pval = --', 'COPII-coated vesicle budding, pval = --', 'Golgi vesicle bud deformation and release, pval = --', 'COPI-coated vesicle budding, pval = --', 'intrinsic component of Golgi trans cisterna membrane, pval = --', 'intrinsic component of Golgi medial cisterna membrane, pval = --', 'intrinsic component of Golgi cis cisterna membrane, pval = --', 'integral component of Golgi membrane, pval = 0.68643', 'Golgi medial cisterna membrane, pval = --', 'Golgi cis cisterna membrane, pval = --', 'Golgi trans cisterna membrane, pval = --']
level 4: ['alpha-1,6-mannosyltransferase complex, pval = --', 'Golgi membrane priming complex assembly, pval = --', 'clathrin coating of Golgi vesicle, pval = --', 'COPI coating of Golgi vesicle, pval = --', 'Golgi membrane coat protein complex assembly, pval = --', 'COPII-coated vesicle cargo loading, pval = --', 'COPII vesicle coating, pval = --', 'integral component of Golgi trans cisterna membrane, pval = --', 'integral component of Golgi medial cisterna membrane, pval = --', 'integral component of Golgi cis cisterna membrane, pval = --']
level 5: ['clathrin coating of Golgi vesicle, plasma membrane to endosome targeting, pval = --', 'clathrin coating of Golgi vesicle, trans-Golgi to endosome targeting, pval = --', 'COPI coating of Golgi vesicle, inter-Golgi cisterna, pval = --', 'COPI coating of Golgi vesicle, cis-Golgi to rough ER, pval = --', 'negative regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of COPII vesicle coating, pval = --']" id="golgi_apparatus_ton" init-pval="0.02368" min-pval="0.02368" min-pval-children="0.14987" title="Golgi apparatus">
<path d="M213.1,238.5c0.3,0.1,0.6,0.1,1,0.1
		c1.6-0.1,3.2-1.7,3.9-3c0.9-1.8,0.4-4.2,0.2-6.1c-0.4-2.7-0.3-5.5-0.1-8.3c0.1-1.5,0.3-3.1,0.4-4.6c0.3-3.1,0.3-6.3,0.9-9.4
		c0.7-3.9,1.8-7.7,3.1-11.5c2.7-7.8,7.2-15,11.3-22.1c3.2-5.7,6.8-11.1,11.6-15.5c1.8-1.6,3.8-3,5.9-4.2c1.8-1,4.2-1.8,5.6-3.4
		c1.9-2,2.5-7.1-1.3-7c-3.6,0-6.7,3-9.1,5.3c-2.7,2.5-4.8,5.6-6.9,8.6c-1.1,1.7-2.2,3.3-3.4,5c-1.6,2.3-3.1,4.8-4.6,7.2
		c-1.8,2.9-3.4,5.8-5.1,8.8c-0.9,1.7-1.9,3.3-2.8,5c-3.1,5.4-5.2,11.4-6.7,17.4c-1.5,6.1-2.4,12.2-3.2,18.4c-0.5,3.7-1,7.4-1.9,11
		C211.3,232.3,210,237.8,213.1,238.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M203.4,224.9c0.7,1.2,2.2,1.2,3.2,0.4
		c1.3-1.1,1.7-3,1.7-4.6c0-1.9-0.3-3.8-0.3-5.6c0-3,0.6-5.9,1.4-8.8c2.7-9.6,7.9-18.3,12.8-26.9c1.5-2.6,3-5.2,4.6-7.8
		c1.5-2.5,2.9-5.1,4.3-7.6c2.4-4.4,4.8-8.8,7.9-12.9c2.2-2.9,4.8-5.2,7.4-7.7c1.1-1,2-2.3,1.9-3.9c-0.2-1.6-1.7-2.4-3.2-2.3
		c-2.2,0.2-4,1.7-5.1,3.5c-1.7,2.8-3.5,5.6-5.1,8.4c-1.6,2.7-3.2,5.5-4.7,8.3c-0.8,1.5-1.7,3.1-2.5,4.6c-1.3,2.5-2.7,5.1-4.2,7.5
		c-4.5,7.5-10.3,17.6-15.2,28.1c-2.3,5.1-3.2,10.6-4.4,16c-0.3,1.6-0.7,3.3-0.9,4.9C202.8,220.7,202.3,222.9,203.4,224.9
		C203.4,224.8,203.4,224.9,203.4,224.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M199.3,212.7c0.6,0,1.3-0.2,1.8-0.8
		c1.2-1.4,1.5-3.4,1.9-5.1c0.8-4.1,2-8,3.4-11.9c2.5-6.7,5.9-13.8,10.8-22.1c2.5-4.3,5.3-8.7,8.1-13c1.8-2.7,3.4-5.6,5-8.3
		c1.9-3.3,4.4-6.5,4.7-10.5c0.1-1.3-0.2-3-1.8-3.2c-3-0.4-4.2,3.1-4.9,5.3c-0.4,1.2-0.6,2.5-1,3.8c-0.6,2-1.5,3.9-2.4,5.8
		c-1.2,2.4-2.6,4.8-4.1,7.1c-2.9,4.6-5.9,9.3-8.6,14.1c-1.3,2.2-2.5,4.5-3.6,6.8c-1.9,3.9-4.1,7.9-5.4,12.1
		c-0.8,2.7-2.1,5.2-3.3,7.7c-1.1,2.3-2.4,4.6-3,7c-0.3,1.3-0.2,2.8,0.5,4C197.8,212.2,198.5,212.7,199.3,212.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M195.2,194.8c3.4,0,5.1-4.7,6-7.3
		c1.7-4.7,4-9.1,6.6-13.4c2.6-4.4,5.4-8.8,8-13.2c1.4-2.3,3-4.5,4.3-6.9c0.9-1.7,1.9-4.1,1.2-6c-0.8-2.4-3.2-1.4-4.3,0.2
		c-1.5,2.2-2.1,5-3.1,7.4c-1.2,2.9-2.6,5.8-4.3,8.5c-2.7,4.4-5.4,8.7-8.2,13c-1.7,2.7-3.5,5.2-5.4,7.8c-1.5,2.1-3.5,4.6-3.6,7.3
		c0,1.2,0.8,2.3,2,2.5C194.7,194.7,195,194.8,195.2,194.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M198.4,173.1c3.9-1.8,5.5-6.2,7.6-9.7
		c0.7-1.2,1.5-2.4,2.3-3.6c0.8-1.3,1.6-2.6,2.3-4c0.8-1.7,2.8-6.7-0.7-7.1c-2.1-0.3-3.1,2.4-3.6,4c-0.7,2.4-1.1,4.8-2.2,7
		c-0.9,1.7-1.9,3.4-3.2,4.7c-1.5,1.5-3.3,2.1-5.1,3.3c-0.8,0.5-1.5,1.1-2,1.9c-0.8,1.4-0.4,3,1.1,3.7
		C196.1,173.7,197.3,173.6,198.4,173.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M232.7,183.3c-2.5,5.3-4.9,10.6-6.5,16.3
		c-0.6,2.3-1.1,4.7-0.6,7.1c0.4,1.6,1.9,3.6,3.6,2.4c1.6-1.1,1.4-2.9,1.4-4.6c0-3.3,0.8-6.6,1.8-9.8c1.4-4.6,3-9.2,5.5-13.3
		c1.7-2.8,3.7-5.4,6-7.6c2.1-2,4.7-3.5,7.2-4.9c1.5-0.9,3.6-2.3,2.8-4.3c-0.6-1.6-2.4-1.9-3.9-1.7c-2.3,0.4-4.3,1.7-5.9,3.2
		c-1.7,1.7-3.1,3.2-4.5,5.2c-1.1,1.6-2.2,3.2-3.1,4.8C235.1,178.4,233.9,180.9,232.7,183.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col golgi_apparatus" descendants="level 0: ['Golgi apparatus, pval = 0.02368']
level 1: ['Golgi subcompartment, pval = 0.19764', 'Golgi transport complex, pval = 0.43974', 'Golgi calcium ion homeostasis, pval = --', 'exomer complex, pval = --', 'cis-Golgi network, pval = 0.43974', 'RIC1-RGP1 guanyl-nucleotide exchange factor complex, pval = --', 'postsynaptic Golgi apparatus, pval = --', 'TRAPPII protein complex, pval = --', 'TRAPPIII protein complex, pval = --', 'inositol phosphoceramide synthase complex, pval = --', 'protein glycosylation in Golgi, pval = --', 'Golgi-associated vesicle lumen, pval = --', 'Golgi lumen, pval = 0.89614', 'Golgi membrane, pval = 0.14987', 'Golgi apparatus mannose trimming, pval = --', 'GARP complex, pval = --', 'clathrin coat of trans-Golgi network vesicle, pval = --', 'COPI vesicle coat, pval = --', 'acroblast, pval = --']
level 2: ['trans-Golgi network, pval = 0.59248', 'Golgi cisterna, pval = 0.43974', 'Golgi stack, pval = 0.43974', 'cis-Golgi network membrane, pval = --', 'negative regulation of protein glycosylation in Golgi, pval = --', 'positive regulation of protein glycosylation in Golgi, pval = --', 'regulation of protein glycosylation in Golgi, pval = --', 'protein galactosylation in Golgi, pval = --', 'Golgi stack lumen, pval = --', 'trans-Golgi network transport vesicle lumen, pval = --', 'Golgi vesicle budding, pval = --', 'intrinsic component of Golgi membrane, pval = 0.68643', 'extrinsic component of Golgi membrane, pval = --', 'lumenal side of Golgi membrane, pval = --', 'cytoplasmic side of Golgi membrane, pval = --', 'trans-Golgi network membrane, pval = 0.68643', 'Golgi cisterna membrane, pval = 0.43974', 'AP-1 adaptor complex, pval = --']
level 3: ['Golgi medial cisterna, pval = --', 'Golgi trans cisterna, pval = --', 'Golgi cis cisterna, pval = --', 'COPII vesicles tethering complex, pval = --', 'Golgi transport vesicle coating, pval = --', 'COPII-coated vesicle budding, pval = --', 'Golgi vesicle bud deformation and release, pval = --', 'COPI-coated vesicle budding, pval = --', 'intrinsic component of Golgi trans cisterna membrane, pval = --', 'intrinsic component of Golgi medial cisterna membrane, pval = --', 'intrinsic component of Golgi cis cisterna membrane, pval = --', 'integral component of Golgi membrane, pval = 0.68643', 'Golgi medial cisterna membrane, pval = --', 'Golgi cis cisterna membrane, pval = --', 'Golgi trans cisterna membrane, pval = --']
level 4: ['alpha-1,6-mannosyltransferase complex, pval = --', 'Golgi membrane priming complex assembly, pval = --', 'clathrin coating of Golgi vesicle, pval = --', 'COPI coating of Golgi vesicle, pval = --', 'Golgi membrane coat protein complex assembly, pval = --', 'COPII-coated vesicle cargo loading, pval = --', 'COPII vesicle coating, pval = --', 'integral component of Golgi trans cisterna membrane, pval = --', 'integral component of Golgi medial cisterna membrane, pval = --', 'integral component of Golgi cis cisterna membrane, pval = --']
level 5: ['clathrin coating of Golgi vesicle, plasma membrane to endosome targeting, pval = --', 'clathrin coating of Golgi vesicle, trans-Golgi to endosome targeting, pval = --', 'COPI coating of Golgi vesicle, inter-Golgi cisterna, pval = --', 'COPI coating of Golgi vesicle, cis-Golgi to rough ER, pval = --', 'negative regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of COPII vesicle coating, pval = --']" id="golgi_apparatus" init-pval="0.02368" min-pval="0.02368" min-pval-children="0.14987" title="Golgi apparatus">
<path d="M213.1,238.5c0.3,0.1,0.6,0.1,1,0.1
		c1.6-0.1,3.2-1.7,3.9-3c0.9-1.8,0.4-4.2,0.2-6.1c-0.4-2.7-0.3-5.5-0.1-8.3c0.1-1.5,0.3-3.1,0.4-4.6c0.3-3.1,0.3-6.3,0.9-9.4
		c0.7-3.9,1.8-7.7,3.1-11.5c2.7-7.8,7.2-15,11.3-22.1c3.2-5.7,6.8-11.1,11.6-15.5c1.8-1.6,3.8-3,5.9-4.2c1.8-1,4.2-1.8,5.6-3.4
		c1.9-2,2.5-7.1-1.3-7c-3.6,0-6.7,3-9.1,5.3c-2.7,2.5-4.8,5.6-6.9,8.6c-1.1,1.7-2.2,3.3-3.4,5c-1.6,2.3-3.1,4.8-4.6,7.2
		c-1.8,2.9-3.4,5.8-5.1,8.8c-0.9,1.7-1.9,3.3-2.8,5c-3.1,5.4-5.2,11.4-6.7,17.4c-1.5,6.1-2.4,12.2-3.2,18.4c-0.5,3.7-1,7.4-1.9,11
		C211.3,232.3,210,237.8,213.1,238.5z" style="fill:#D99EBE;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M203.4,224.9c0.7,1.2,2.2,1.2,3.2,0.4
		c1.3-1.1,1.7-3,1.7-4.6c0-1.9-0.3-3.8-0.3-5.6c0-3,0.6-5.9,1.4-8.8c2.7-9.6,7.9-18.3,12.8-26.9c1.5-2.6,3-5.2,4.6-7.8
		c1.5-2.5,2.9-5.1,4.3-7.6c2.4-4.4,4.8-8.8,7.9-12.9c2.2-2.9,4.8-5.2,7.4-7.7c1.1-1,2-2.3,1.9-3.9c-0.2-1.6-1.7-2.4-3.2-2.3
		c-2.2,0.2-4,1.7-5.1,3.5c-1.7,2.8-3.5,5.6-5.1,8.4c-1.6,2.7-3.2,5.5-4.7,8.3c-0.8,1.5-1.7,3.1-2.5,4.6c-1.3,2.5-2.7,5.1-4.2,7.5
		c-4.5,7.5-10.3,17.6-15.2,28.1c-2.3,5.1-3.2,10.6-4.4,16c-0.3,1.6-0.7,3.3-0.9,4.9C202.8,220.7,202.3,222.9,203.4,224.9
		C203.4,224.8,203.4,224.9,203.4,224.9z" style="fill:#D99EBE;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M199.3,212.7c0.6,0,1.3-0.2,1.8-0.8
		c1.2-1.4,1.5-3.4,1.9-5.1c0.8-4.1,2-8,3.4-11.9c2.5-6.7,5.9-13.8,10.8-22.1c2.5-4.3,5.3-8.7,8.1-13c1.8-2.7,3.4-5.6,5-8.3
		c1.9-3.3,4.4-6.5,4.7-10.5c0.1-1.3-0.2-3-1.8-3.2c-3-0.4-4.2,3.1-4.9,5.3c-0.4,1.2-0.6,2.5-1,3.8c-0.6,2-1.5,3.9-2.4,5.8
		c-1.2,2.4-2.6,4.8-4.1,7.1c-2.9,4.6-5.9,9.3-8.6,14.1c-1.3,2.2-2.5,4.5-3.6,6.8c-1.9,3.9-4.1,7.9-5.4,12.1
		c-0.8,2.7-2.1,5.2-3.3,7.7c-1.1,2.3-2.4,4.6-3,7c-0.3,1.3-0.2,2.8,0.5,4C197.8,212.2,198.5,212.7,199.3,212.7z" style="fill:#D99EBE;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M195.2,194.8c3.4,0,5.1-4.7,6-7.3
		c1.7-4.7,4-9.1,6.6-13.4c2.6-4.4,5.4-8.8,8-13.2c1.4-2.3,3-4.5,4.3-6.9c0.9-1.7,1.9-4.1,1.2-6c-0.8-2.4-3.2-1.4-4.3,0.2
		c-1.5,2.2-2.1,5-3.1,7.4c-1.2,2.9-2.6,5.8-4.3,8.5c-2.7,4.4-5.4,8.7-8.2,13c-1.7,2.7-3.5,5.2-5.4,7.8c-1.5,2.1-3.5,4.6-3.6,7.3
		c0,1.2,0.8,2.3,2,2.5C194.7,194.7,195,194.8,195.2,194.8z" style="fill:#D99EBE;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M198.4,173.1c3.9-1.8,5.5-6.2,7.6-9.7
		c0.7-1.2,1.5-2.4,2.3-3.6c0.8-1.3,1.6-2.6,2.3-4c0.8-1.7,2.8-6.7-0.7-7.1c-2.1-0.3-3.1,2.4-3.6,4c-0.7,2.4-1.1,4.8-2.2,7
		c-0.9,1.7-1.9,3.4-3.2,4.7c-1.5,1.5-3.3,2.1-5.1,3.3c-0.8,0.5-1.5,1.1-2,1.9c-0.8,1.4-0.4,3,1.1,3.7
		C196.1,173.7,197.3,173.6,198.4,173.1z" style="fill:#D99EBE;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M232.7,183.3c-2.5,5.3-4.9,10.6-6.5,16.3
		c-0.6,2.3-1.1,4.7-0.6,7.1c0.4,1.6,1.9,3.6,3.6,2.4c1.6-1.1,1.4-2.9,1.4-4.6c0-3.3,0.8-6.6,1.8-9.8c1.4-4.6,3-9.2,5.5-13.3
		c1.7-2.8,3.7-5.4,6-7.6c2.1-2,4.7-3.5,7.2-4.9c1.5-0.9,3.6-2.3,2.8-4.3c-0.6-1.6-2.4-1.9-3.9-1.7c-2.3,0.4-4.3,1.7-5.9,3.2
		c-1.7,1.7-3.1,3.2-4.5,5.2c-1.1,1.6-2.2,3.2-3.1,4.8C235.1,178.4,233.9,180.9,232.7,183.3z" style="fill:#D99EBE;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton nucleoplasm nucleus" descendants="level 0: ['nucleoplasm, pval = 0.00942']
level 1: ['Fanconi anaemia nuclear complex, pval = 0.07809', 'DNA replication preinitiation complex, pval = --', 'transcription elongation factor complex, pval = 0.82468', 'nuclear body, pval = 0.19345', 'nucleoplasmic periphery of the nuclear pore complex, pval = --', 'survivin complex, pval = --', 'mRNA cleavage factor complex, pval = --', 'histone deacetylase complex, pval = 0.44375', 'RNA polymerase II, holoenzyme, pval = 0.93038', 'mediator complex, pval = --', 'histone methyltransferase complex, pval = 0.2377', 'nuclear pre-replicative complex, pval = --', 'histone acetyltransferase complex, pval = --', 'nuclear transcriptional repressor complex, pval = 0.68643', 'MAML3-RBP-Jkappa-ICN1 complex, pval = --', 'MAML2-RBP-Jkappa-ICN1 complex, pval = --', 'MAML2-RBP-Jkappa-ICN2 complex, pval = --', 'MAML2-RBP-Jkappa-ICN3 complex, pval = --', 'MAML2-RBP-Jkappa-ICN4 complex, pval = --', 'DUBm complex, pval = --', 'MAML3-RBP-Jkappa-ICN2 complex, pval = --', 'MAML3-RBP-Jkappa-ICN3 complex, pval = --', 'MAML3-RBP-Jkappa-ICN4 complex, pval = --', 'FANCM-MHF complex, pval = --', 'nuclear RNA export factor complex, pval = --']
level 2: ['CMG complex, pval = --', 'GINS complex, pval = --', 'cyclin/CDK positive transcription elongation factor complex, pval = --', 'FACT complex, pval = --', 'nucleoplasmic THO complex, pval = --', 'NELF complex, pval = --', 'DSIF complex, pval = --', 'Cdc73/Paf1 complex, pval = --', 'super elongation complex, pval = --', 'elongin complex, pval = --', 'interchromatin granule, pval = --', 'Cajal body, pval = 0.68643', 'nuclear dicing body, pval = --', 'paraspeckles, pval = --', 'cleavage body, pval = --', 'Gemini of coiled bodies, pval = --', 'sphere organelle, pval = --', 'Mmi1 nuclear focus, pval = --', 'histone locus body, pval = --', 'PML body, pval = 0.23172', 'LYSP100-associated nuclear domain, pval = --', 'nuclear speck, pval = 0.86024', 'mRNA cleavage stimulating factor complex, pval = --', 'mRNA cleavage and polyadenylation specificity factor complex, pval = --', 'Rpd3L-Expanded complex, pval = --', 'Snt2C complex, pval = --', 'Set3 complex, pval = --', 'HDA1 complex, pval = --', 'SHREC complex, pval = --', 'Sin3-type complex, pval = 0.59248', 'NuRD complex, pval = 0.68643', 'Swr1 complex, pval = --', &quot;Clr6 histone deacetylase complex I'', pval = --&quot;, 'SHREC2 complex, pval = --', 'transcription factor TFIIA complex, pval = --', 'transcription factor TFIIF complex, pval = --', 'transcription factor TFIIE complex, pval = --', 'transcription factor TFIIH holo complex, pval = 0.90208', 'RNA polymerase II, core complex, pval = --', 'transcription factor TFIID complex, pval = 0.90208', 'transcription factor TFTC complex, pval = --', 'MECO complex, pval = --', 'MLL1/2 complex, pval = --', 'MLL3/4 complex, pval = 0.40992', 'CLRC ubiquitin ligase complex, pval = --', 'nucleosomal methylation activator complex, pval = --', 'ESC/E(Z) complex, pval = 0.68643', 'Set1C/COMPASS complex, pval = --', 'SAGA-type complex, pval = --', 'H3 histone acetyltransferase complex, pval = --', 'H4 histone acetyltransferase complex, pval = --', 'Beta-catenin-ICAT complex, pval = --', 'Cry-Per complex, pval = --', 'chromatin silencing complex, pval = --', 'RNA polymerase I transcription repressor complex, pval = --', 'RNA polymerase V transcription repressor complex, pval = --', 'RNA polymerase IV transcription repressor complex, pval = --', 'RNA polymerase III transcription repressor complex, pval = --', 'RNA polymerase II transcription repressor complex, pval = --', 'SAGA complex, pval = --', 'SLIK (SAGA-like) complex, pval = --']
level 3: ['nuclear replication fork, pval = 0.82468', 'P-TEFb complex, pval = --', 'CTDK-1 complex, pval = --', 'box H/ACA scaRNP complex, pval = --', 'omega speckle, pval = --', 'Rpd3S complex, pval = --', 'Rpd3L complex, pval = --', 'Sin3 complex, pval = 0.59248', 'core TFIIH complex portion of holo TFIIH complex, pval = --', 'transcription factor TFIIK complex, pval = --', 'TAF4B-containing transcription factor TFIID complex, pval = --', 'MLL1 complex, pval = --', 'PCAF complex, pval = --', 'STAGA complex, pval = --', 'Hpa2 acetyltransferase complex, pval = --', 'histone H3-K14 acetyltransferase complex, pval = --', 'NuA3 histone acetyltransferase complex, pval = --', 'MOZ/MORF histone acetyltransferase complex, pval = --', 'Ada2/Gcn5/Ada3 transcription activator complex, pval = --', 'MSL complex, pval = --', 'NSL complex, pval = --', 'H4/H2A histone acetyltransferase complex, pval = --', 'eNoSc complex, pval = --', 'chromatin lock complex, pval = --', 'RENT complex, pval = --', 'negative cofactor 2 complex, pval = --', 'DRM complex, pval = --', 'Tle3-Aes complex, pval = --', 'Mad-Max-mSin3B complex, pval = --', 'Mad-Max complex, pval = --', 'Mad-Max-mSin3A complex, pval = --']
level 4: ['replication fork protection complex, pval = --', 'nuclear replisome, pval = --', 'Mst2 histone acetyltransferase complex, pval = --', 'NuA3b histone acetyltransferase complex, pval = --', 'NuA3a histone acetyltransferase complex, pval = --', 'NuA4 histone acetyltransferase complex, pval = --']
level 5: ['delta DNA polymerase complex, pval = --', 'DNA replication factor A complex, pval = --', 'alpha DNA polymerase:primase complex, pval = --', 'nuclear DNA replication factor C complex, pval = --', 'Piccolo NuA4 histone acetyltransferase complex, pval = --']
level 6: ['DNA replication factor C core complex, pval = --', 'SAS acetyltransferase complex, pval = --']" id="nucleoplasm_ton" init-pval="0.00942" min-pval="0.00942" min-pval-children="0.07809" title="Nucleoplasm">
<path d="M409.3,174.2c20.1,21.7,35.6,44.5,27.5,92.6
		c-8,48.1-45.2,57.7-60.9,60.6c-15.6,2.8-71.8,11.1-96.7-14.1c-25-25.2-32.7-45.4-22.9-88c9.8-42.6,42.6-64.8,71-71.2
		C355.7,147.7,389,152.3,409.3,174.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col nucleoplasm nucleus" descendants="level 0: ['nucleoplasm, pval = 0.00942']
level 1: ['Fanconi anaemia nuclear complex, pval = 0.07809', 'DNA replication preinitiation complex, pval = --', 'transcription elongation factor complex, pval = 0.82468', 'nuclear body, pval = 0.19345', 'nucleoplasmic periphery of the nuclear pore complex, pval = --', 'survivin complex, pval = --', 'mRNA cleavage factor complex, pval = --', 'histone deacetylase complex, pval = 0.44375', 'RNA polymerase II, holoenzyme, pval = 0.93038', 'mediator complex, pval = --', 'histone methyltransferase complex, pval = 0.2377', 'nuclear pre-replicative complex, pval = --', 'histone acetyltransferase complex, pval = --', 'nuclear transcriptional repressor complex, pval = 0.68643', 'MAML3-RBP-Jkappa-ICN1 complex, pval = --', 'MAML2-RBP-Jkappa-ICN1 complex, pval = --', 'MAML2-RBP-Jkappa-ICN2 complex, pval = --', 'MAML2-RBP-Jkappa-ICN3 complex, pval = --', 'MAML2-RBP-Jkappa-ICN4 complex, pval = --', 'DUBm complex, pval = --', 'MAML3-RBP-Jkappa-ICN2 complex, pval = --', 'MAML3-RBP-Jkappa-ICN3 complex, pval = --', 'MAML3-RBP-Jkappa-ICN4 complex, pval = --', 'FANCM-MHF complex, pval = --', 'nuclear RNA export factor complex, pval = --']
level 2: ['CMG complex, pval = --', 'GINS complex, pval = --', 'cyclin/CDK positive transcription elongation factor complex, pval = --', 'FACT complex, pval = --', 'nucleoplasmic THO complex, pval = --', 'NELF complex, pval = --', 'DSIF complex, pval = --', 'Cdc73/Paf1 complex, pval = --', 'super elongation complex, pval = --', 'elongin complex, pval = --', 'interchromatin granule, pval = --', 'Cajal body, pval = 0.68643', 'nuclear dicing body, pval = --', 'paraspeckles, pval = --', 'cleavage body, pval = --', 'Gemini of coiled bodies, pval = --', 'sphere organelle, pval = --', 'Mmi1 nuclear focus, pval = --', 'histone locus body, pval = --', 'PML body, pval = 0.23172', 'LYSP100-associated nuclear domain, pval = --', 'nuclear speck, pval = 0.86024', 'mRNA cleavage stimulating factor complex, pval = --', 'mRNA cleavage and polyadenylation specificity factor complex, pval = --', 'Rpd3L-Expanded complex, pval = --', 'Snt2C complex, pval = --', 'Set3 complex, pval = --', 'HDA1 complex, pval = --', 'SHREC complex, pval = --', 'Sin3-type complex, pval = 0.59248', 'NuRD complex, pval = 0.68643', 'Swr1 complex, pval = --', &quot;Clr6 histone deacetylase complex I'', pval = --&quot;, 'SHREC2 complex, pval = --', 'transcription factor TFIIA complex, pval = --', 'transcription factor TFIIF complex, pval = --', 'transcription factor TFIIE complex, pval = --', 'transcription factor TFIIH holo complex, pval = 0.90208', 'RNA polymerase II, core complex, pval = --', 'transcription factor TFIID complex, pval = 0.90208', 'transcription factor TFTC complex, pval = --', 'MECO complex, pval = --', 'MLL1/2 complex, pval = --', 'MLL3/4 complex, pval = 0.40992', 'CLRC ubiquitin ligase complex, pval = --', 'nucleosomal methylation activator complex, pval = --', 'ESC/E(Z) complex, pval = 0.68643', 'Set1C/COMPASS complex, pval = --', 'SAGA-type complex, pval = --', 'H3 histone acetyltransferase complex, pval = --', 'H4 histone acetyltransferase complex, pval = --', 'Beta-catenin-ICAT complex, pval = --', 'Cry-Per complex, pval = --', 'chromatin silencing complex, pval = --', 'RNA polymerase I transcription repressor complex, pval = --', 'RNA polymerase V transcription repressor complex, pval = --', 'RNA polymerase IV transcription repressor complex, pval = --', 'RNA polymerase III transcription repressor complex, pval = --', 'RNA polymerase II transcription repressor complex, pval = --', 'SAGA complex, pval = --', 'SLIK (SAGA-like) complex, pval = --']
level 3: ['nuclear replication fork, pval = 0.82468', 'P-TEFb complex, pval = --', 'CTDK-1 complex, pval = --', 'box H/ACA scaRNP complex, pval = --', 'omega speckle, pval = --', 'Rpd3S complex, pval = --', 'Rpd3L complex, pval = --', 'Sin3 complex, pval = 0.59248', 'core TFIIH complex portion of holo TFIIH complex, pval = --', 'transcription factor TFIIK complex, pval = --', 'TAF4B-containing transcription factor TFIID complex, pval = --', 'MLL1 complex, pval = --', 'PCAF complex, pval = --', 'STAGA complex, pval = --', 'Hpa2 acetyltransferase complex, pval = --', 'histone H3-K14 acetyltransferase complex, pval = --', 'NuA3 histone acetyltransferase complex, pval = --', 'MOZ/MORF histone acetyltransferase complex, pval = --', 'Ada2/Gcn5/Ada3 transcription activator complex, pval = --', 'MSL complex, pval = --', 'NSL complex, pval = --', 'H4/H2A histone acetyltransferase complex, pval = --', 'eNoSc complex, pval = --', 'chromatin lock complex, pval = --', 'RENT complex, pval = --', 'negative cofactor 2 complex, pval = --', 'DRM complex, pval = --', 'Tle3-Aes complex, pval = --', 'Mad-Max-mSin3B complex, pval = --', 'Mad-Max complex, pval = --', 'Mad-Max-mSin3A complex, pval = --']
level 4: ['replication fork protection complex, pval = --', 'nuclear replisome, pval = --', 'Mst2 histone acetyltransferase complex, pval = --', 'NuA3b histone acetyltransferase complex, pval = --', 'NuA3a histone acetyltransferase complex, pval = --', 'NuA4 histone acetyltransferase complex, pval = --']
level 5: ['delta DNA polymerase complex, pval = --', 'DNA replication factor A complex, pval = --', 'alpha DNA polymerase:primase complex, pval = --', 'nuclear DNA replication factor C complex, pval = --', 'Piccolo NuA4 histone acetyltransferase complex, pval = --']
level 6: ['DNA replication factor C core complex, pval = --', 'SAS acetyltransferase complex, pval = --']" id="nucleoplasm" init-pval="0.00942" min-pval="0.00942" min-pval-children="0.07809" title="Nucleoplasm">
<path d="M409.3,174.2c20.1,21.7,35.6,44.5,27.5,92.6
		c-8,48.1-45.2,57.7-60.9,60.6c-15.6,2.8-71.8,11.1-96.7-14.1c-25-25.2-32.7-45.4-22.9-88c9.8-42.6,42.6-64.8,71-71.2
		C355.7,147.7,389,152.3,409.3,174.2z" style="fill:#8AC0E2;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton nuclear_membrane" descendants="level 0: ['nuclear membrane, pval = 0.42226']
level 1: ['nuclear inner membrane, pval = 0.43974', 'nuclear outer membrane, pval = --', '5-lipoxygenase complex, pval = --']
level 2: ['intrinsic component of nuclear inner membrane, pval = --', 'Asi complex, pval = --', 'intrinsic component of nuclear outer membrane, pval = --', 'extrinsic component of nuclear outer membrane, pval = --']
level 3: ['integral component of nuclear inner membrane, pval = --', 'integral component of nuclear outer membrane, pval = --']
level 4: ['nuclear membrane complex Bqt3-Bqt4, pval = --']" id="nuclear_membrane_ton" init-pval="0.42226" min-pval="0.42226" min-pval-children="0.43974" title="Nuclear membrane">
<path d="M410.6,175c0.5,0,1-0.2,1.4-0.6
		c0.9-0.8,0.9-2.1,0.1-3c-14.4-15.5-36.2-24-61.3-24c-7,0-14.1,0.7-21.1,2c-1.1,0.2-1.9,1.3-1.7,2.5c0.2,1.1,1.3,1.9,2.5,1.7
		c6.7-1.3,13.5-1.9,20.3-1.9c24,0,44.7,8.1,58.2,22.7C409.5,174.8,410.1,175,410.6,175z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M438.6,269.2c1,0,1.9-0.7,2.1-1.8
		c7.7-46.2-6-70-22.1-88.7c-0.8-0.9-2.1-1-3-0.2c-0.9,0.8-1,2.1-0.2,3c15.4,18,28.6,40.8,21.1,85.3c-0.2,1.1,0.6,2.2,1.7,2.4
		C438.4,269.2,438.5,269.2,438.6,269.2z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M385.7,329.1c0.2,0,0.4,0,0.6-0.1
		c26.4-7.4,44.3-24.7,51.8-50.1c0.3-1.1-0.3-2.3-1.4-2.6c-1.1-0.3-2.3,0.3-2.6,1.4c-7.2,24.3-23.7,40.2-49,47.3
		c-1.1,0.3-1.8,1.5-1.5,2.6C383.9,328.5,384.7,329.1,385.7,329.1z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M337.6,334.9c16.6,0,31.6-2.3,39.1-3.6
		c1.1-0.2,1.9-1.3,1.7-2.4c-0.2-1.1-1.3-1.9-2.4-1.7c-7.3,1.3-22.1,3.6-38.3,3.6c-27,0-46.6-5.9-58.1-17.7c-0.8-0.8-2.1-0.8-3,0
		c-0.8,0.8-0.8,2.1,0,3C288.8,328.5,309.4,334.9,337.6,334.9z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M270.5,308.6c0.5,0,1-0.2,1.4-0.5
		c0.9-0.8,1-2.1,0.2-3c-15.5-17.7-23.8-35.8-17.3-71.4c0.2-1.1-0.5-2.2-1.7-2.4c-1.1-0.2-2.2,0.5-2.4,1.7
		c-6.8,37.2,1.9,56.3,18.2,74.9C269.3,308.4,269.9,308.6,270.5,308.6z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M254.5,226.9c1,0,1.8-0.7,2-1.6
		c11.7-50.9,53.7-66.3,66.3-69.9c1.1-0.3,1.8-1.5,1.5-2.6c-0.3-1.1-1.5-1.8-2.6-1.5c-13.2,3.7-57.1,19.8-69.3,73
		c-0.3,1.1,0.4,2.3,1.6,2.5C254.2,226.9,254.4,226.9,254.5,226.9z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col nuclear_membrane" descendants="level 0: ['nuclear membrane, pval = 0.42226']
level 1: ['nuclear inner membrane, pval = 0.43974', 'nuclear outer membrane, pval = --', '5-lipoxygenase complex, pval = --']
level 2: ['intrinsic component of nuclear inner membrane, pval = --', 'Asi complex, pval = --', 'intrinsic component of nuclear outer membrane, pval = --', 'extrinsic component of nuclear outer membrane, pval = --']
level 3: ['integral component of nuclear inner membrane, pval = --', 'integral component of nuclear outer membrane, pval = --']
level 4: ['nuclear membrane complex Bqt3-Bqt4, pval = --']" id="nuclear_membrane" init-pval="0.42226" min-pval="0.42226" min-pval-children="0.43974" title="Nuclear membrane">
<path d="M410.6,175c0.5,0,1-0.2,1.4-0.6
		c0.9-0.8,0.9-2.1,0.1-3c-14.4-15.5-36.2-24-61.3-24c-7,0-14.1,0.7-21.1,2c-1.1,0.2-1.9,1.3-1.7,2.5c0.2,1.1,1.3,1.9,2.5,1.7
		c6.7-1.3,13.5-1.9,20.3-1.9c24,0,44.7,8.1,58.2,22.7C409.5,174.8,410.1,175,410.6,175z" style="fill:#7D64A1;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M438.6,269.2c1,0,1.9-0.7,2.1-1.8
		c7.7-46.2-6-70-22.1-88.7c-0.8-0.9-2.1-1-3-0.2c-0.9,0.8-1,2.1-0.2,3c15.4,18,28.6,40.8,21.1,85.3c-0.2,1.1,0.6,2.2,1.7,2.4
		C438.4,269.2,438.5,269.2,438.6,269.2z" style="fill:#7D64A1;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M385.7,329.1c0.2,0,0.4,0,0.6-0.1
		c26.4-7.4,44.3-24.7,51.8-50.1c0.3-1.1-0.3-2.3-1.4-2.6c-1.1-0.3-2.3,0.3-2.6,1.4c-7.2,24.3-23.7,40.2-49,47.3
		c-1.1,0.3-1.8,1.5-1.5,2.6C383.9,328.5,384.7,329.1,385.7,329.1z" style="fill:#7D64A1;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M337.6,334.9c16.6,0,31.6-2.3,39.1-3.6
		c1.1-0.2,1.9-1.3,1.7-2.4c-0.2-1.1-1.3-1.9-2.4-1.7c-7.3,1.3-22.1,3.6-38.3,3.6c-27,0-46.6-5.9-58.1-17.7c-0.8-0.8-2.1-0.8-3,0
		c-0.8,0.8-0.8,2.1,0,3C288.8,328.5,309.4,334.9,337.6,334.9z" style="fill:#7D64A1;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M270.5,308.6c0.5,0,1-0.2,1.4-0.5
		c0.9-0.8,1-2.1,0.2-3c-15.5-17.7-23.8-35.8-17.3-71.4c0.2-1.1-0.5-2.2-1.7-2.4c-1.1-0.2-2.2,0.5-2.4,1.7
		c-6.8,37.2,1.9,56.3,18.2,74.9C269.3,308.4,269.9,308.6,270.5,308.6z" style="fill:#7D64A1;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M254.5,226.9c1,0,1.8-0.7,2-1.6
		c11.7-50.9,53.7-66.3,66.3-69.9c1.1-0.3,1.8-1.5,1.5-2.6c-0.3-1.1-1.5-1.8-2.6-1.5c-13.2,3.7-57.1,19.8-69.3,73
		c-0.3,1.1,0.4,2.3,1.6,2.5C254.2,226.9,254.4,226.9,254.5,226.9z" style="fill:#7D64A1;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton nuclear_bodies" descendants="level 0: ['nuclear body, pval = 0.19345']
level 1: ['interchromatin granule, pval = --', 'Cajal body, pval = 0.68643', 'nuclear dicing body, pval = --', 'paraspeckles, pval = --', 'cleavage body, pval = --', 'Gemini of coiled bodies, pval = --', 'sphere organelle, pval = --', 'Mmi1 nuclear focus, pval = --', 'histone locus body, pval = --', 'PML body, pval = 0.23172', 'LYSP100-associated nuclear domain, pval = --', 'nuclear speck, pval = 0.86024']
level 2: ['box H/ACA scaRNP complex, pval = --', 'omega speckle, pval = --']" id="nuclear_bodies_ton" init-pval="0.19345" min-pval="0.19345" min-pval-children="0.23172" title="Nuclear bodies">
<path d="M335.8,253.4c-1.4-1.2-1.7-4.3,0.5-4.8
		c1.6-0.3,3.7,0.6,4.3,2.1C341.6,253.6,337.9,255.3,335.8,253.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M293.5,283.9c-1.5-2.4,2.6-4.8,4.2-2.4
		C299.6,284.4,295.3,286.8,293.5,283.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M328.9,189.8c0.5-1.8,2.2-2.3,3.7-1.4
		c1.9,1.1,2,3.7-0.1,4.7C330.2,194.2,328.2,192.2,328.9,189.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M354.5,217.2c-1.6-1.2-1.5-3.2,0.1-4.4
		c1.9-1.5,4.6-0.5,4.7,1.9C359.5,217.6,356.7,218.8,354.5,217.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M415.1,251c-0.1,0.6-0.6,1.2-1.7,1.6
		c-1.9,0.6-4-1.1-3.1-3.2C411.5,246.3,415.5,248.9,415.1,251z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M323.2,318.5c-0.1,0.4-0.4,0.7-1.1,1
		c-1.2,0.4-2.4-0.7-1.9-2C320.9,315.6,323.4,317.2,323.2,318.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M361.7,185c-0.1,0.4-0.4,0.7-1.1,1
		c-1.2,0.4-2.4-0.7-1.9-2C359.4,182.1,361.9,183.7,361.7,185z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M401.9,213c-3.8,0.2-4.5-7.7,0.7-5.9
		C405.7,208.2,405.6,212.8,401.9,213z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M393.2,176.5c-2.5,1.6-6.1-3.9-1.9-4.5
		C393.8,171.6,395.6,175,393.2,176.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M367.4,167.5c-2.4,0-5.2-4.1-1.3-5.4
		c2-0.7,4.1,1.9,3.4,3.7C369,167,368.2,167.5,367.4,167.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M332.1,293c-0.7-1.4-0.3-3.2,2.1-3.9
		c3.2-0.9,8.2,3.4,3.6,5.6C335.2,295.9,332.9,294.6,332.1,293z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M386.1,310.3c-1.3-0.2-2.4-1.3-1.9-3.2
		c0.7-2.5,5.9-3.7,5.6,0.3C389.5,309.6,387.6,310.5,386.1,310.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M261.3,270.6c-0.4-0.5-0.5-1.1-0.5-1.8
		c-0.1-3.3,4-4.6,4.8-1.1C266.3,270.4,263.7,273.8,261.3,270.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M293.6,215c-0.3-0.5-0.6-1.1-0.6-1.8
		c-0.2-4.5,5.5-1.5,4.9,1.2C297.2,217.1,294.7,216.7,293.6,215z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col nuclear_bodies" descendants="level 0: ['nuclear body, pval = 0.19345']
level 1: ['interchromatin granule, pval = --', 'Cajal body, pval = 0.68643', 'nuclear dicing body, pval = --', 'paraspeckles, pval = --', 'cleavage body, pval = --', 'Gemini of coiled bodies, pval = --', 'sphere organelle, pval = --', 'Mmi1 nuclear focus, pval = --', 'histone locus body, pval = --', 'PML body, pval = 0.23172', 'LYSP100-associated nuclear domain, pval = --', 'nuclear speck, pval = 0.86024']
level 2: ['box H/ACA scaRNP complex, pval = --', 'omega speckle, pval = --']" id="nuclear_bodies" init-pval="0.19345" min-pval="0.19345" min-pval-children="0.23172" title="Nuclear bodies">
<path d="M335.8,253.4c-1.4-1.2-1.7-4.3,0.5-4.8
		c1.6-0.3,3.7,0.6,4.3,2.1C341.6,253.6,337.9,255.3,335.8,253.4z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M293.5,283.9c-1.5-2.4,2.6-4.8,4.2-2.4
		C299.6,284.4,295.3,286.8,293.5,283.9z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M328.9,189.8c0.5-1.8,2.2-2.3,3.7-1.4
		c1.9,1.1,2,3.7-0.1,4.7C330.2,194.2,328.2,192.2,328.9,189.8z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M354.5,217.2c-1.6-1.2-1.5-3.2,0.1-4.4
		c1.9-1.5,4.6-0.5,4.7,1.9C359.5,217.6,356.7,218.8,354.5,217.2z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M415.1,251c-0.1,0.6-0.6,1.2-1.7,1.6
		c-1.9,0.6-4-1.1-3.1-3.2C411.5,246.3,415.5,248.9,415.1,251z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M323.2,318.5c-0.1,0.4-0.4,0.7-1.1,1
		c-1.2,0.4-2.4-0.7-1.9-2C320.9,315.6,323.4,317.2,323.2,318.5z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M361.7,185c-0.1,0.4-0.4,0.7-1.1,1
		c-1.2,0.4-2.4-0.7-1.9-2C359.4,182.1,361.9,183.7,361.7,185z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M401.9,213c-3.8,0.2-4.5-7.7,0.7-5.9
		C405.7,208.2,405.6,212.8,401.9,213z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M393.2,176.5c-2.5,1.6-6.1-3.9-1.9-4.5
		C393.8,171.6,395.6,175,393.2,176.5z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M367.4,167.5c-2.4,0-5.2-4.1-1.3-5.4
		c2-0.7,4.1,1.9,3.4,3.7C369,167,368.2,167.5,367.4,167.5z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M332.1,293c-0.7-1.4-0.3-3.2,2.1-3.9
		c3.2-0.9,8.2,3.4,3.6,5.6C335.2,295.9,332.9,294.6,332.1,293z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M386.1,310.3c-1.3-0.2-2.4-1.3-1.9-3.2
		c0.7-2.5,5.9-3.7,5.6,0.3C389.5,309.6,387.6,310.5,386.1,310.3z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M261.3,270.6c-0.4-0.5-0.5-1.1-0.5-1.8
		c-0.1-3.3,4-4.6,4.8-1.1C266.3,270.4,263.7,273.8,261.3,270.6z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M293.6,215c-0.3-0.5-0.6-1.1-0.6-1.8
		c-0.2-4.5,5.5-1.5,4.9,1.2C297.2,217.1,294.7,216.7,293.6,215z" style="fill:#B6D9E8;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton nuclear_speckles" descendants="level 0: [None]" id="nuclear_speckles_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Nuclear speckles">
<path d="M405.3,185.5c0,0.1,0.1,0.2,0.1,0.3
		c0.5,1.5,0.2,3.9,1.3,5.1c1.3,1.4,5.2,0.5,4.6,3.5c-0.7,3.1-5.3,1.8-6.7,4.2c-1,1.6,0.3,5.2-1.7,6c-1.1,0.5-2.1,0-3.2-0.1
		c-1.3-0.1-2.6,0.3-3.9,0.3c-2.9-0.1-1-2-1.9-3.8c-0.8-1.4-4.5-2.8-2.8-4.9c1.3-1.6,5.6-1.7,4.8-4.8c-0.4-1.9-2.8-3.8-1.2-5.9
		c1.6-2.1,4.2-0.5,6.5-1C402.7,183.9,404.6,184,405.3,185.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M418.9,198.3c0.4-0.1,0.8-0.2,1.3-0.1
		c1.1,0.3,3.5,0.9,4.4,1.6c1.5,1.2,0.4,2.8,0.3,4.3c-0.1,1,0.2,1.8,0.5,2.7c0.3,1,0.1,1.4-0.7,2.1c-0.8,0.7-2.2,0.8-2.8,1.4
		c-0.7,0.8,0.3,2,0.5,2.7c0.2,1.1,0.1,1.4-0.5,2.3c-0.9,1.6-2.2,2.9-4.2,2.9c-2.2,0-5.6-0.5-7.7-1.2c-1.5-0.5-1.2-2.1-1.4-3.3
		c-0.2-1.5-1.2-3-1.2-4.6c0-1.7,1.9-2.4,2.3-3.9c0.8-3.1-3.9-4.7-2.8-7.5c0.6-1.4,2-1,3.2-1.2c1.6-0.2,3.7-1.9,4.8,0
		c0.5,0.8,0.5,1.6,1.4,2C417.2,199.2,418,198.6,418.9,198.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M371,212.2c-0.4,0.2-0.8,0.4-1.2,0.4
		c-0.6,0.1-1.3,0-1.9,0.2c-0.8,0.4-1.1,1.4-2.1,1.6c-1.5,0.3-2.4-1.2-3.4-2.1c-1.2-1.2-3.1-0.6-4.5-1.1c-1.6-0.6-0.8-1.8-1.7-2.9
		c-0.9-1-2.8-0.2-3.6-1.3c-0.7-0.9,0.1-2.5-0.1-3.6c-0.2-1.3-0.9-2.5-1-3.8c-0.2-1.8,0.8-1.8,1.9-2.9c1.5-1.6,0.8-5.3,3.8-4.9
		c1.9,0.3,4.2,2.8,4.8-0.2c0.4-1.9,1.6-5.3,4.1-3c1.1,1,1.7,0.9,3.2,0.7c1.6-0.2,1.7,0.1,1.6,1.6c-0.1,1.5,0.3,3.2,2.1,3.2
		c2,0,3.2-1,5.1,0.4c2.4,1.8,0.9,4.5,1.3,6.9c0.3,1.9,1.2,3.1-0.8,4.3c-2.1,1.3-1.9,1.6-1.2,3.8c0.5,1.6,0.1,3.4-1.9,3
		c-1.3-0.2-2.5-0.9-3.9-0.7C371.3,212.1,371.1,212.2,371,212.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M433.4,239.6c-0.1,0.1-0.2,0.3-0.3,0.4
		c-0.8,0.9-2.4,1-3.3,1.8c-1.4,1.3-2.6,5.1-5.1,2.8c-0.9-0.8-2-6.1-4.2-3.7c-1.1,1.3,0.1,2.8-1.5,4c-1,0.8-2.2,0.7-3.5,0.9
		c-1.9,0.3-8.8,1.2-7.8-3c0.5-2.1,5.3-2.8,2.1-5.6c-1.3-1.2-3.5-1.6-4.6-3.1c-0.9-1.3-1-4.7,0.1-6c1.7-1.9,4.3-0.8,6.4-1.4
		c2-0.6,1.4-2.3,2.7-3.3c2.1-1.6,4.1,0.9,4.9,2.6c0.6,1.4,1.6,3.2,3.4,2.8c1.6-0.3,2.8-2.2,4.5-2.4c2.8-0.3,1.8,3.7,2.7,5.4
		C431.2,234.1,435.9,236,433.4,239.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M292.5,267.7c0,0.2,0.1,0.3,0.1,0.5
		c0.1,1.5-1.4,2.9-0.6,4.5c0.6,1.1,1.9,1.6,2.7,2.6c1.1,1.5,0.4,2.6-1.2,3c-1.2,0.3-2.9,1-4,1c-1.4,0-2.2-1.5-3.6-1.6
		c-1.6,0-2.3,1.9-1.9,3.2c0.4,1.4,2,2,2.8,3.2c1.6,2.7-0.3,5.7-3.1,6.3c-1.7,0.4-3.2-0.4-4.8-0.1c-1.5,0.2-2.5,1.2-4,0.1
		c-2.1-1.5-0.6-3.8-0.6-5.7c0-2.6-3-1.5-4.6-1.7c-1.4-0.2-2.6-1-2.6-2.4c0-1,1.1-2.3,0.7-3.3c-0.4-1.1-1.9-1.4-1.9-2.7
		c0-1,0.8-1.8,1.4-2.4c0.2-0.2,0.9-0.6,1-0.9c0.2-0.5-0.3-0.9-0.2-1.4c0.3-2.8,3.2-2.1,5-1.7c4.2,0.9,2.4-3.2,3.9-5.3
		c0.6-0.8,1.8-1.2,2.6-1.8c0.8-0.7,1.4-1.9,2.6-1.3c1.1,0.5,1.2,2.1,1.8,3c0.6,0.9,1.9,1.5,2.9,1.7
		C289.1,265.3,291.9,264.4,292.5,267.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M302.8,308.5c-0.1,0.1-0.2,0.3-0.3,0.4
		c-0.9,1.1-2.7,1.2-3.5,2.5c-0.8,1.4-0.8,3.3-2.9,3.3c-2.7,0.1-4.4-3.1-7.3-1.7c-1.4,0.7-2.8,1.6-4.4,0.7c-1.1-0.6-1.1-1.7-1.8-2.5
		c-1.3-1.6-4-1-3-3.7c0.6-1.7,2.8-2.3,1.2-4.1c-1.5-1.6-3.4-2.8-1.9-5.1c2.3-3.5,6.5-7,9.1-1.8c0.3,0.7-0.1,1.6,0.6,2.3
		c0.6,0.5,1.3,0.5,2,0.3c1.6-0.5,2.6-2.1,4.1-2.8c4-1.8,2.9,2.7,4.2,4.8C300.1,303.3,305,304.7,302.8,308.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M349.1,214.5c-0.1,0.1-0.2,0.2-0.3,0.3
		c-1.3,1.5-0.9,4.5-2.9,5.5c-2.7,1.3-5.5-2-8.2-1.8c-2,0.2-4.2,1.4-3.7-1.9c0.5-3.8-7.2,0.2-7.5-4.1c-0.1-1.8,0.8-3.5,2.6-3.6
		c4-0.3,0.9-4.3,1.2-7.1c0.4-3,3.2-2.8,5.5-2.5c1.1,0.1,2,0.2,3.1-0.1c0.8-0.2,1.9-1.1,2.7-0.9c2.1,0.4,0.5,3.5,2,4.5
		c1.9,1.3,3.5-2.5,4.7-3.2c2.7-1.6,3.8,2.2,3.8,4c0,0.7,0,1.5-0.3,2.2c-0.3,0.5-0.9,0.8-1.1,1.4c-0.8,1.6,0.5,2.6,0.6,4
		C351.5,212.9,350,213.6,349.1,214.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M354,272.7c-0.2,0.3-0.4,0.5-0.7,0.9
		c-1.6,1.9-1,4.6-2.1,6.6c-1.3,2.4-4.9,2.5-5.1-0.6c-0.1-1.4,0.6-2.7-1-3.5c-1.2-0.6-3.1,0-3.8,1.2c-1,1.7,0.7,5.6-2.8,5.2
		c-1.5-0.2-5.6-3.1-6.8-4.1c-2-1.6-1.4-4.9,0-7c1.1-1.7,3.1-2.8,2.8-5.1c-0.3-2.1-1.2-4,0.5-5.9c1.5-1.7,3.2-0.4,4.7-1.3
		c1.4-0.9,0.7-3.2,2-4.2c1.3-1,3.5,0.3,5,0.1c2.2-0.2,5.2-1,6.4,1.4c0.7,1.6,1.2,1.9,2.9,2.2c1.4,0.2,2.3,0.2,2.7,1.8
		c0.2,0.9,0.7,2.6,0.3,3.5c-1.1,2.4-5.9,1.5-5.2,4.9C354.2,270.9,354.8,271.5,354,272.7z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M373.1,323.1c-0.1,0-0.1,0.1-0.2,0.1
		c-1.1,0.7-2,0.9-2.8,2.1c-1.2,1.8-2.2,2.2-4.3,1.5c-2.2-0.7-2.1-1.7-3.1-3.4c-1.1-1.8-3.2-1.1-4.9-1.3c-2.1-0.3-2-2-2.6-3.5
		c-0.6-1.3-1.9-2.9-0.5-4.3c0.8-0.8,2.1-0.5,3-1c1.4-0.9,0.7-2.1,0.5-3.4c-0.3-2.4,2.1-3,3.9-3c2.1,0,3.8,1.8,5.8,1.8
		c3.5,0,2.1-3.3,3.9-5c1.1-1,3.4-1.8,4.8-1.3c1.8,0.6,1.6,2.4,1.4,3.9c-0.2,2.1,0.3,4.6,2.9,4.3c1.4-0.1,3-1.2,3.1,0.9
		c0,1-0.4,2-0.3,3c0.1,0.8,0.5,1.8,0.4,2.6c-0.3,2.3-2.5,0.6-3.9,0.9c-1.8,0.4-1.8,0.8-1.9,2.5c-0.1,1.5-0.9,1.9-2.3,1.7
		C374.6,322.1,374.1,322.5,373.1,323.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M400.1,310c-1.1,0.5-2.1,1.1-2.9,1.4
		c-0.9,0.3-2.2,0.7-3,0.1c-1.1-0.9-0.5-2.6-0.8-3.8c-0.4-1.9-3.3-3.1-3.3-4.9c0-2.8,4.3-3,5.1-4.9c0.5-1.3-1.9-2.9-2-4.1
		c-0.1-1.4,1.9-1.8,2.9-2.2c2-0.6,3.4-1.5,4.5,0.5c1.3,2.3,4.1,1,4.8,3.8c0.3,1.2,0.8,1.9,2.3,1.7c1.4-0.2,2.2-0.8,3.1,0.6
		c0.8,1.2,1.6,4.1,0.7,5.5c-1.8,2.9-6.2,0-6.4,3.9c-0.1,1.5-0.9,1.9-2.3,1.7C402.1,309.1,401.1,309.5,400.1,310z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M327.1,189.2c0,0.2,0.1,0.3,0.1,0.5
		c0.8,3.6-2.4,2.7-3.5,5.5c-1.1,2.7-0.6,4.5-4.2,3.3c-1.7-0.5-1.6-0.9-3.5,0c-3.9,1.8-6.6-1.7-8.1-4.8c-0.7-1.4-3.1-2-3.3-3.6
		c-0.2-2,2.1-3,2-5c-0.1-1.2-0.7-2.3,0-3.5c0.7-1.1,2.2-0.9,2.8-1.9c1.3-2.2-1.7-4.1,1.5-5.4c3.6-1.6,3,1.6,4.7,3.1
		c3.9,3.4,6.2-3.2,9.2-2.4c1,0.3,2.9,3.5,3.5,4.3c1.2,1.7,5.2,4.2,1.1,6C326.6,186.6,326.5,186.5,327.1,189.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M302.2,204.1c-0.7,0.9-1.1,1.9-1.5,3
		c-1,2.5-4.5,1.1-6.7,1.5c-1.7,0.3-3.4,1-4.3,2.5c-0.5,0.9-0.4,1.4-1.2,2c-0.8,0.5-1.8,0.6-2.6,0.4c-1.6-0.4-2.5-1.6-3.8-2.3
		c-0.9-0.5-2.1-0.9-2.5-2c-0.5-1.3,0.6-2.1,0.8-3.3c0.1-1.3-0.7-2.1-0.9-3.3c-0.1-1.1,0.4-1.8-0.1-2.9c-1.2-2.3-4,0-4.9-2.1
		c-0.8-1.9,1.2-6.1,3.1-6.7c1.5-0.5,3.5,1.3,5-0.1c1.3-1.3-0.6-3.9,1-5.3c0.8-0.7,4-1.3,4.9-0.8c1.6,0.8,0.1,3.3,0.7,4.4
		c0.8,1.8,2.5,1,3.5-0.3c0.9-1.1,0.6-3.7,2.7-3.6c0.9,0,1.2,0.9,2.1,1.1c0.9,0.1,2.6-0.5,3.4-0.8c1.6-0.5,4.4-1,4.6,1.4
		c0.1,1.5-1.3,2.7-0.7,4.2c0.5,1.2,2.5,2.1,2.3,3.5c-0.2,2.1-2.8,2-3.7,3.3c-0.6,0.9,0.3,1.6,0.6,2.4c0.6,1.6,0.3,2.1-0.9,3.1
		C302.6,203.6,302.4,203.8,302.2,204.1z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M348.5,246.9c-0.1,0.1-0.2,0.2-0.2,0.3
		c-1.4,1.7-3,2.6-4.5,0.2c-1.5-2.4-2.5-0.7-4.7-0.7c-2.6,0-4.3-0.6-5.7-3c-0.8-1.4-2.9-6.8-0.3-7.9c1.8-0.8,9.2,1.4,7.2-3.4
		c-0.5-1.2-1.9-1.5-2.7-2.3c-0.7-0.7-2.2-2.8-1.6-3.9c0.8-1.5,2.8-1.3,4.1-1.9c1.7-0.8,0.6-3,2.1-3.7c0.8-0.3,3,0.2,3.7,0.6
		c1.7,0.9,2.7,4.9,5,3.9c1.2-0.5,1.6-1.9,2.7-2.5c3.9-2,4.7,2.4,6.1,4.5c0.7,1.1,2.3,2.1,2.3,3.6c0,2.3-2.4,3.6-2.7,5.7
		c-0.3,1.7,3.5,4.6,1.1,5.9c-0.7,0.4-1.9-0.1-2.7,0.4c-0.6,0.4-0.7,1-0.9,1.6c-0.5,1.5-0.7,2.9-2.7,2.5
		C352,246.6,350.2,245.2,348.5,246.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M365.6,260.5c-0.3-0.1-0.5-0.2-0.8-0.1
		c-2.2,0.1-3.5,2-5.4-0.2c-0.6-0.8-0.5-0.9-1.5-1.2c-0.8-0.2-1.5-0.4-2.3-0.5c-2.3-0.6-3-2.7-3.5-4.8c-1-4.1,2.8-4.3,4.3-7.4
		c0.6-1.2,0.2-2.8,2.1-2.8c2.2,0,3.7,0.8,5.7-0.5c1.2-0.8,2.3-2.1,3.8-0.9c0.7,0.6,0.8,1.7,1.5,2.4c0.7,0.7,1.6,0.8,2.5,1.2
		c1.9,0.9,2.2,2.2,1.8,4c-0.5,2.4,0.3,4.1,0.9,6.3c0.5,1.6-0.1,3.1-2,2.9c-1.3-0.1-2.5-1.5-3.7-0.6c-1,0.7-1,2.3-2.4,2.4
		C366.3,260.7,366,260.6,365.6,260.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M384.2,183.9c-0.1,0-0.3,0-0.4,0
		c-0.8,0.1-2.4,0.2-2.9,1c-0.5,0.8,0.4,1.6,0,2.4c-1.1,1.9-3.4-0.2-4.1-1.2c-1.3-1.8-1.9-3.2-4-4.2c-1.3-0.6-3.6-2-2.7-3.7
		c0.7-1.3,2.5-1.6,3.4-2.7c1-1.3-1.8-3.2-2.2-4.3c-0.7-1.8,0.1-3.1,2-3.7c1-0.3,2.5,0,3.2-0.9c0.7-0.8,0.1-2.2,0.5-3.1
		c1.6-3,6.6,1.1,8.6,1.1c1,0,1.7-0.8,2.7-0.8c4,0.2,1.4,3.8,3,5.7c1.5,1.8,4.4,2.5,5.4,4.8c0.6,1.2,0.2,2.5,0.6,3.8
		c0.3,1.3,1,2,0.7,3.5c-0.4,2.1-2.1,2-3.3,3.5c-0.9,1.1-0.4,2.5-1.5,3.4c-1.8,1.4-3-0.8-4.1-1.9
		C387.6,185.3,386.2,183.9,384.2,183.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M413.2,293.5c-0.1,0.1-0.3,0.2-0.4,0.3
		c-2.9,2.3-3.5-1-6.5-0.8c-2.9,0.2-4.3,1.4-4.8-2.3c-0.3-1.8,0.1-1.8-1.5-3.1c-1.5-1.3-1.9-1.2-1.8-3.2c0.2-1.9-0.4-5.9,1.5-6.9
		c1.9-1.1,4,1.7,5.7,0.2c2-1.7,2.4-6.2,5.7-5.9c1.3,0.1,1.8,1.6,2.9,1.7c2.5,0.2,2.9-3.3,5.5-1c3,2.6-0.1,3.3-0.7,5.5
		c-1.4,5,5.6,4.2,6.2,7.3c0.2,1-1.9,4.1-2.3,5c-1,1.8-1.5,6.5-4.9,3.6C415.3,292,415.3,291.8,413.2,293.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M399.7,273.8c-0.1,0.1-0.3,0.2-0.4,0.3
		c-2.9,2.3-3.5-1-6.5-0.8c-2.9,0.2-4.3,1.4-4.8-2.3c-0.3-1.8,0.1-1.8-1.5-3.1c-1.5-1.3-1.9-1.2-1.8-3.2c0.2-1.9-0.4-5.9,1.5-6.9
		c1.9-1.1,4,1.7,5.7,0.2c2-1.7,2.4-6.2,5.7-5.9c1.3,0.1,1.8,1.6,2.9,1.7c2.5,0.2,2.9-3.3,5.5-1c3,2.6-0.1,3.3-0.7,5.5
		c-1.4,5,5.6,4.2,6.2,7.3c0.2,1-1.9,4.1-2.3,5c-1,1.8-1.5,6.5-4.9,3.6C401.9,272.2,401.9,272.1,399.7,273.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M290.6,221.6c0.1,0.1,0.3,0.2,0.4,0.3
		c3.1,1.9,0.3,3.6,1.5,6.4c1.2,2.6,2.8,3.5-0.5,5.3c-1.6,0.9-1.7,0.5-2.4,2.5c-0.6,1.9-0.4,2.2-2.4,2.8c-1.5,0.4-2.9,1.2-4.5,1.4
		c-1.9,0.3-2.6-0.7-4-1.6c-0.9-0.6-1.6-1.5-2.5-2.1c-2-1.2-4.7,0.7-5.7-2c-1-2.7,0.7-3.3,2.9-3.5c1.2-0.1,2.7-1.3,3-2.5
		c0.5-1.9-1.4-2.4-2.6-3.3c-1.1-0.8-1.6-1.9-0.6-3c1-1.1,2.4-0.6,3.2-1.9c0.7-1.1,0.7-2.6,0.9-3.9c0.3-2.9,3.3-5.7,6.2-3.9
		c0.8,0.5,1.3,1.1,2.2,1.2c0.7,0.1,1.5-0.1,2.2,0c1.6,0.4,1.8,1.9,1.3,3.3C288.4,220.1,288.2,220.2,290.6,221.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M320.3,215.2c2.2,1.4,2.9,2.1,1.8,4.5
		c-0.7,1.7,0.2,3.9-0.9,5.4c-1.2,1.7-2.9,1-4.6,1.2c-1.8,0.3-2.3,1.6-3.2,3.1c-0.5,1-1,2.1-2.3,1.9c-2-0.4-2.9-2.5-4.7-3.3
		c-1.4-0.7-3.6-1.1-4-2.9c-0.2-1.3,0.5-2.4-0.1-3.7c-0.6-1.3-2.2-2-2.3-3.6c-0.1-1,0.5-2.4,1.5-2.9c0.9-0.4,1.8,0,2.4-1
		c0.8-1.2,0.4-5.3,2.5-5.3c1.4,0,2.2,1.8,3.4,2.1c1.7,0.4,2.3-1.6,3.3-2.5c2.1-1.9,7.7-0.4,6.6,3.1c-0.5,1.6-2.3,1.9-0.4,3.5
		C319.7,214.8,320,215,320.3,215.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M351,305.4c1.2,0.5,4,0.9,4.1,2.6
		c0.1,2.2-5.2,3.2-5.9,5.2c-1.7,4.5,9.2,5.4,6.2,10.1c-0.9,1.4-2.2,0.8-3.5,1.1c-1.3,0.3-1.7,1.6-2.8,2.2c-1.3,0.7-2.3-0.3-3.5-0.7
		c-2.4-0.8-4.2,2.3-6.4,0.5c-1.9-1.6,0.2-4.9-2-6.2c-2.6-1.5-7.5-0.1-8.5-3.7c-0.4-1.5,1-2.6,0.7-3.8c-0.4-1.5-1.8-1.6-2.7-2.6
		c-0.7-0.8-0.7-2.4-0.4-3.4c0.2-0.8,0.8-1.5,1.7-1.7c1.8-0.4,3.8,1.7,5.6,0.3c1.6-1.2,1.3-4.4,1.8-6.2c0.8-2.7,2.7-1.3,4.7-1.3
		c1.9-0.1,2.2-2.6,3.9-3.3c2.7-1.2,5.6,1.3,5.5,4C349.2,301.2,347.7,303.9,351,305.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M310.6,307.9c-0.8-0.9-2.2-0.4-3.2-1
		c-1.7-0.9-2.3-2.7-1.7-4.5c0.5-1.5,3.9-3.6,1.9-5.5c-1.1-1-2.1-0.5-2.5-2.5c-0.3-1.4-0.1-2.6,1.2-3.3c1.6-0.8,2.6-1.1,3.6-2.6
		c0.9-1.4,2.3-3.3,4.1-1.9c1,0.7,1.9,1.8,3.1,2.1c1.2,0.2,1-0.4,1.9-1c1.5-1,5.9-0.3,7,1c2.2,2.6-2.3,5.3,0.9,7.8
		c0.9,0.7,1.8,0.7,2.3,2c0.5,1.2,0.1,3.1,0.1,4.3c0.1,1.4-0.7,1.7-2.1,1.9c-1.3,0.2-2.2,0.2-2.1,1.7c0,0.9,1,2.7-0.1,3.4
		c-1,0.7-2.6-0.7-3.6-0.9c-1.5-0.3-1.9,0.5-2.4,1.6c-1.3,2.8-4.8,1.5-6.9,0.3C311.2,310.1,311.4,308.7,310.6,307.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M277.9,261.8c-0.3,0.2-0.6,0.4-1,0.7
		c-1.2,0.9-2.5,1.9-3.5,0.2c-0.6-1-0.5-3.2-1.9-3.7c-0.7-0.2-1.6,0.9-2.1,1.3c-2.6,2.2-4.7,0.7-5.6-2.1c-0.9-2.7-1.6-4.4-4.8-4.6
		c-3.4-0.2-4.3-2.9-3.3-6c0.7-2.3-1.9-6.5,1.5-7.5c1.6-0.5,3.4,0.2,5-0.2c1.7-0.4,2.4-1.6,3.2-3c1.2-2.2,2.1-1.4,4.3-0.8
		c1.4,0.4,2.6,0.3,4,0.3c1.9,0,2.2,0.6,2.9,2.2c0.6,1.4,2.4,3,2.1,4.6c-0.4,2.1-2.6,2.8-2.6,5.2c0,1-0.3,2.2,0.3,3.1
		c0.6,0.9,1.7,1,2.5,1.7c1.5,1.3,1.6,4.4,1.3,6.2C280,260.5,279,261.1,277.9,261.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M429,270.6c-1.2-0.5-1.9-2.2-3.2-1
		c-0.7,0.6-0.5,2.2-1,3c-2.1,3.9-4.6-1.3-6.9-1.7c-1.5-0.3-2.6,0-2.8-1.8c-0.2-1.6,0.7-3.2,0-4.7c-0.7-1.5-2.2-2.6-3.1-4
		c-1.6-2.8-0.6-4.9,2.5-5.8c1.4-0.4,2.2-1,2.7-2.3c0.6-1.3,0.5-2.7,2.2-3c0.8-0.1,1.6,0.3,2.4,0.1c0.6-0.2,0.8-0.8,1.3-1
		c1.7-0.7,4.5-0.2,5.1,1.7c0.6,1.6-0.8,3.9-0.2,5.3c1,2.3,3.7,0,5.3,0.1c2,0.2,2.6,2.8,2,4.4c-0.8,2.2-3.1,2.8-3,5.5
		C432.5,267,432.4,272.2,429,270.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M357.9,163.4c0.9,0.8,2.2,1.7,2.3,2.9
		c0.1,1.4-1.5,2.2-1.1,3.7c0.3,1.1,2.5,2.3,2.1,3.4c-0.2,0.5-1.4,0.9-1.9,1.2c-0.8,0.6-1.5,1.2-1.9,2.2c-0.8,1.9,0.2,4-0.7,5.8
		c-0.4,0.8-0.9,1.5-1.3,2.4c-0.4,0.9-0.6,1.6-1.7,1.8c-1.9,0.4-5-0.7-5.7-2.7c-0.4-1.3,0.5-3.2-1.4-3.5c-1.5-0.2-2.5,1.6-3.8,1.2
		c-1.4-0.4-2.3-2.5-3.3-3.4c-0.7-0.6-1.8-1-2.4-1.7c-1-1.2-0.9-2.7-2.4-3.4c-1.5-0.7-4-1.3-4.6-3.1c-0.5-1.5,0-3.3,1.6-3.8
		c1.9-0.5,3.1-0.9,4.6-2.2c1.5-1.3,3.1-2.6,5.2-1.8c2.1,0.9,3.6,1.5,6,1.1c2.3-0.4,4.3-2,6.8-1.8c1.4,0.1,2.5,0.4,3.6,1.4
		C357.7,163.2,357.8,163.3,357.9,163.4z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M319.6,174.2c1.5,0.5,3.4,1.9,4.4,0.1
		c0.5-0.9,0.4-1.8,1.6-2c1-0.2,3.2,1,3.8-0.3c0.3-0.5-0.3-1.4-0.4-1.9c-0.1-0.9,0.2-1.5,0.6-2.2c0.9-1.4,0.2-1.9-0.6-3.1
		c-0.7-1.1,0.1-2.6-0.6-3.6c-0.7-1-3-1-4.1-1.3c-1.3-0.3-3-0.7-4.1,0.5c-0.9,0.9-0.8,2-2.2,2.3c-4.5,0.7-1.1,3.5-0.8,6.2
		c0.1,1.5-1.2,1.7-1,3.2C316.6,174.1,318.1,173.6,319.6,174.2z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M285.3,241.8c-1,0.1-2,1.1-2.4,2.1
		c-0.6,1.4,0.2,1.8,1,2.7c1.8,2.1-1.8,2.5-1.9,4.6c0,1.8,1.8,2.6,2.8,3.8c0.9,1.1,1.1,2.1,2.8,1.7c3.8-0.7,1.9-4.2,3.7-6.5
		c0.7-0.9,2.3-0.7,3-1.7c0.6-1-0.3-2-0.7-2.9c-0.4-1-0.6-3-1.6-3.7c-0.9-0.6-2,0.1-2.9,0.4c-1.2,0.4-1.8,0.2-3-0.4
		C285.8,241.8,285.5,241.8,285.3,241.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M340.6,185.8c-1.1,0-2,0-2.5,1.3
		c-0.8,2.2,1.1,3.5,2.7,4.4c1,0.6,0.9,1,0.8,2.1c-0.2,1.9,0.8,2,2.3,1.8c1.1-0.2,2.8,0.3,3.7-0.6c0.9-0.9-0.1-2.4,1.1-3.3
		c1.1-0.9,2.7-2.2,0.8-3.4c-0.8-0.5-1.6-0.5-2.1-1.5c-0.3-0.6-0.1-1.7-0.7-2.1c-1.3-0.8-3.4,1.3-4.7,1.4
		C341.5,185.9,341.1,185.9,340.6,185.8z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M303.8,277.9c-0.1,0-0.1,0.1-0.2,0.1
		c-1,1-1,2.6,0.5,2.8c1.3,0.2,2-0.8,2.4,0.9c0.3,1.7,0.2,5.7,2.9,4.9c1.5-0.4,2.1-1.9,3.5-2.4c0.9-0.4,2.3-0.3,3.3-0.4
		c2.7-0.1,1.6-2,1.5-3.7c-0.2-1.7,0.8-3.9-1.9-3.7c-2,0.2-2.7,0-4.1-1.4c-1.8-1.8-3,0.5-4.5,1.5
		C306.1,277.3,304.7,277.2,303.8,277.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col nuclear_speckles" descendants="level 0: [None]" id="nuclear_speckles" init-pval="--" min-pval="--" min-pval-children="--" title="Nuclear speckles">
<path d="M405.3,185.5c0,0.1,0.1,0.2,0.1,0.3
		c0.5,1.5,0.2,3.9,1.3,5.1c1.3,1.4,5.2,0.5,4.6,3.5c-0.7,3.1-5.3,1.8-6.7,4.2c-1,1.6,0.3,5.2-1.7,6c-1.1,0.5-2.1,0-3.2-0.1
		c-1.3-0.1-2.6,0.3-3.9,0.3c-2.9-0.1-1-2-1.9-3.8c-0.8-1.4-4.5-2.8-2.8-4.9c1.3-1.6,5.6-1.7,4.8-4.8c-0.4-1.9-2.8-3.8-1.2-5.9
		c1.6-2.1,4.2-0.5,6.5-1C402.7,183.9,404.6,184,405.3,185.5z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M418.9,198.3c0.4-0.1,0.8-0.2,1.3-0.1
		c1.1,0.3,3.5,0.9,4.4,1.6c1.5,1.2,0.4,2.8,0.3,4.3c-0.1,1,0.2,1.8,0.5,2.7c0.3,1,0.1,1.4-0.7,2.1c-0.8,0.7-2.2,0.8-2.8,1.4
		c-0.7,0.8,0.3,2,0.5,2.7c0.2,1.1,0.1,1.4-0.5,2.3c-0.9,1.6-2.2,2.9-4.2,2.9c-2.2,0-5.6-0.5-7.7-1.2c-1.5-0.5-1.2-2.1-1.4-3.3
		c-0.2-1.5-1.2-3-1.2-4.6c0-1.7,1.9-2.4,2.3-3.9c0.8-3.1-3.9-4.7-2.8-7.5c0.6-1.4,2-1,3.2-1.2c1.6-0.2,3.7-1.9,4.8,0
		c0.5,0.8,0.5,1.6,1.4,2C417.2,199.2,418,198.6,418.9,198.3z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M371,212.2c-0.4,0.2-0.8,0.4-1.2,0.4
		c-0.6,0.1-1.3,0-1.9,0.2c-0.8,0.4-1.1,1.4-2.1,1.6c-1.5,0.3-2.4-1.2-3.4-2.1c-1.2-1.2-3.1-0.6-4.5-1.1c-1.6-0.6-0.8-1.8-1.7-2.9
		c-0.9-1-2.8-0.2-3.6-1.3c-0.7-0.9,0.1-2.5-0.1-3.6c-0.2-1.3-0.9-2.5-1-3.8c-0.2-1.8,0.8-1.8,1.9-2.9c1.5-1.6,0.8-5.3,3.8-4.9
		c1.9,0.3,4.2,2.8,4.8-0.2c0.4-1.9,1.6-5.3,4.1-3c1.1,1,1.7,0.9,3.2,0.7c1.6-0.2,1.7,0.1,1.6,1.6c-0.1,1.5,0.3,3.2,2.1,3.2
		c2,0,3.2-1,5.1,0.4c2.4,1.8,0.9,4.5,1.3,6.9c0.3,1.9,1.2,3.1-0.8,4.3c-2.1,1.3-1.9,1.6-1.2,3.8c0.5,1.6,0.1,3.4-1.9,3
		c-1.3-0.2-2.5-0.9-3.9-0.7C371.3,212.1,371.1,212.2,371,212.2z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M433.4,239.6c-0.1,0.1-0.2,0.3-0.3,0.4
		c-0.8,0.9-2.4,1-3.3,1.8c-1.4,1.3-2.6,5.1-5.1,2.8c-0.9-0.8-2-6.1-4.2-3.7c-1.1,1.3,0.1,2.8-1.5,4c-1,0.8-2.2,0.7-3.5,0.9
		c-1.9,0.3-8.8,1.2-7.8-3c0.5-2.1,5.3-2.8,2.1-5.6c-1.3-1.2-3.5-1.6-4.6-3.1c-0.9-1.3-1-4.7,0.1-6c1.7-1.9,4.3-0.8,6.4-1.4
		c2-0.6,1.4-2.3,2.7-3.3c2.1-1.6,4.1,0.9,4.9,2.6c0.6,1.4,1.6,3.2,3.4,2.8c1.6-0.3,2.8-2.2,4.5-2.4c2.8-0.3,1.8,3.7,2.7,5.4
		C431.2,234.1,435.9,236,433.4,239.6z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M292.5,267.7c0,0.2,0.1,0.3,0.1,0.5
		c0.1,1.5-1.4,2.9-0.6,4.5c0.6,1.1,1.9,1.6,2.7,2.6c1.1,1.5,0.4,2.6-1.2,3c-1.2,0.3-2.9,1-4,1c-1.4,0-2.2-1.5-3.6-1.6
		c-1.6,0-2.3,1.9-1.9,3.2c0.4,1.4,2,2,2.8,3.2c1.6,2.7-0.3,5.7-3.1,6.3c-1.7,0.4-3.2-0.4-4.8-0.1c-1.5,0.2-2.5,1.2-4,0.1
		c-2.1-1.5-0.6-3.8-0.6-5.7c0-2.6-3-1.5-4.6-1.7c-1.4-0.2-2.6-1-2.6-2.4c0-1,1.1-2.3,0.7-3.3c-0.4-1.1-1.9-1.4-1.9-2.7
		c0-1,0.8-1.8,1.4-2.4c0.2-0.2,0.9-0.6,1-0.9c0.2-0.5-0.3-0.9-0.2-1.4c0.3-2.8,3.2-2.1,5-1.7c4.2,0.9,2.4-3.2,3.9-5.3
		c0.6-0.8,1.8-1.2,2.6-1.8c0.8-0.7,1.4-1.9,2.6-1.3c1.1,0.5,1.2,2.1,1.8,3c0.6,0.9,1.9,1.5,2.9,1.7
		C289.1,265.3,291.9,264.4,292.5,267.7z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M302.8,308.5c-0.1,0.1-0.2,0.3-0.3,0.4
		c-0.9,1.1-2.7,1.2-3.5,2.5c-0.8,1.4-0.8,3.3-2.9,3.3c-2.7,0.1-4.4-3.1-7.3-1.7c-1.4,0.7-2.8,1.6-4.4,0.7c-1.1-0.6-1.1-1.7-1.8-2.5
		c-1.3-1.6-4-1-3-3.7c0.6-1.7,2.8-2.3,1.2-4.1c-1.5-1.6-3.4-2.8-1.9-5.1c2.3-3.5,6.5-7,9.1-1.8c0.3,0.7-0.1,1.6,0.6,2.3
		c0.6,0.5,1.3,0.5,2,0.3c1.6-0.5,2.6-2.1,4.1-2.8c4-1.8,2.9,2.7,4.2,4.8C300.1,303.3,305,304.7,302.8,308.5z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M349.1,214.5c-0.1,0.1-0.2,0.2-0.3,0.3
		c-1.3,1.5-0.9,4.5-2.9,5.5c-2.7,1.3-5.5-2-8.2-1.8c-2,0.2-4.2,1.4-3.7-1.9c0.5-3.8-7.2,0.2-7.5-4.1c-0.1-1.8,0.8-3.5,2.6-3.6
		c4-0.3,0.9-4.3,1.2-7.1c0.4-3,3.2-2.8,5.5-2.5c1.1,0.1,2,0.2,3.1-0.1c0.8-0.2,1.9-1.1,2.7-0.9c2.1,0.4,0.5,3.5,2,4.5
		c1.9,1.3,3.5-2.5,4.7-3.2c2.7-1.6,3.8,2.2,3.8,4c0,0.7,0,1.5-0.3,2.2c-0.3,0.5-0.9,0.8-1.1,1.4c-0.8,1.6,0.5,2.6,0.6,4
		C351.5,212.9,350,213.6,349.1,214.5z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M354,272.7c-0.2,0.3-0.4,0.5-0.7,0.9
		c-1.6,1.9-1,4.6-2.1,6.6c-1.3,2.4-4.9,2.5-5.1-0.6c-0.1-1.4,0.6-2.7-1-3.5c-1.2-0.6-3.1,0-3.8,1.2c-1,1.7,0.7,5.6-2.8,5.2
		c-1.5-0.2-5.6-3.1-6.8-4.1c-2-1.6-1.4-4.9,0-7c1.1-1.7,3.1-2.8,2.8-5.1c-0.3-2.1-1.2-4,0.5-5.9c1.5-1.7,3.2-0.4,4.7-1.3
		c1.4-0.9,0.7-3.2,2-4.2c1.3-1,3.5,0.3,5,0.1c2.2-0.2,5.2-1,6.4,1.4c0.7,1.6,1.2,1.9,2.9,2.2c1.4,0.2,2.3,0.2,2.7,1.8
		c0.2,0.9,0.7,2.6,0.3,3.5c-1.1,2.4-5.9,1.5-5.2,4.9C354.2,270.9,354.8,271.5,354,272.7z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M373.1,323.1c-0.1,0-0.1,0.1-0.2,0.1
		c-1.1,0.7-2,0.9-2.8,2.1c-1.2,1.8-2.2,2.2-4.3,1.5c-2.2-0.7-2.1-1.7-3.1-3.4c-1.1-1.8-3.2-1.1-4.9-1.3c-2.1-0.3-2-2-2.6-3.5
		c-0.6-1.3-1.9-2.9-0.5-4.3c0.8-0.8,2.1-0.5,3-1c1.4-0.9,0.7-2.1,0.5-3.4c-0.3-2.4,2.1-3,3.9-3c2.1,0,3.8,1.8,5.8,1.8
		c3.5,0,2.1-3.3,3.9-5c1.1-1,3.4-1.8,4.8-1.3c1.8,0.6,1.6,2.4,1.4,3.9c-0.2,2.1,0.3,4.6,2.9,4.3c1.4-0.1,3-1.2,3.1,0.9
		c0,1-0.4,2-0.3,3c0.1,0.8,0.5,1.8,0.4,2.6c-0.3,2.3-2.5,0.6-3.9,0.9c-1.8,0.4-1.8,0.8-1.9,2.5c-0.1,1.5-0.9,1.9-2.3,1.7
		C374.6,322.1,374.1,322.5,373.1,323.1z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M400.1,310c-1.1,0.5-2.1,1.1-2.9,1.4
		c-0.9,0.3-2.2,0.7-3,0.1c-1.1-0.9-0.5-2.6-0.8-3.8c-0.4-1.9-3.3-3.1-3.3-4.9c0-2.8,4.3-3,5.1-4.9c0.5-1.3-1.9-2.9-2-4.1
		c-0.1-1.4,1.9-1.8,2.9-2.2c2-0.6,3.4-1.5,4.5,0.5c1.3,2.3,4.1,1,4.8,3.8c0.3,1.2,0.8,1.9,2.3,1.7c1.4-0.2,2.2-0.8,3.1,0.6
		c0.8,1.2,1.6,4.1,0.7,5.5c-1.8,2.9-6.2,0-6.4,3.9c-0.1,1.5-0.9,1.9-2.3,1.7C402.1,309.1,401.1,309.5,400.1,310z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M327.1,189.2c0,0.2,0.1,0.3,0.1,0.5
		c0.8,3.6-2.4,2.7-3.5,5.5c-1.1,2.7-0.6,4.5-4.2,3.3c-1.7-0.5-1.6-0.9-3.5,0c-3.9,1.8-6.6-1.7-8.1-4.8c-0.7-1.4-3.1-2-3.3-3.6
		c-0.2-2,2.1-3,2-5c-0.1-1.2-0.7-2.3,0-3.5c0.7-1.1,2.2-0.9,2.8-1.9c1.3-2.2-1.7-4.1,1.5-5.4c3.6-1.6,3,1.6,4.7,3.1
		c3.9,3.4,6.2-3.2,9.2-2.4c1,0.3,2.9,3.5,3.5,4.3c1.2,1.7,5.2,4.2,1.1,6C326.6,186.6,326.5,186.5,327.1,189.2z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M302.2,204.1c-0.7,0.9-1.1,1.9-1.5,3
		c-1,2.5-4.5,1.1-6.7,1.5c-1.7,0.3-3.4,1-4.3,2.5c-0.5,0.9-0.4,1.4-1.2,2c-0.8,0.5-1.8,0.6-2.6,0.4c-1.6-0.4-2.5-1.6-3.8-2.3
		c-0.9-0.5-2.1-0.9-2.5-2c-0.5-1.3,0.6-2.1,0.8-3.3c0.1-1.3-0.7-2.1-0.9-3.3c-0.1-1.1,0.4-1.8-0.1-2.9c-1.2-2.3-4,0-4.9-2.1
		c-0.8-1.9,1.2-6.1,3.1-6.7c1.5-0.5,3.5,1.3,5-0.1c1.3-1.3-0.6-3.9,1-5.3c0.8-0.7,4-1.3,4.9-0.8c1.6,0.8,0.1,3.3,0.7,4.4
		c0.8,1.8,2.5,1,3.5-0.3c0.9-1.1,0.6-3.7,2.7-3.6c0.9,0,1.2,0.9,2.1,1.1c0.9,0.1,2.6-0.5,3.4-0.8c1.6-0.5,4.4-1,4.6,1.4
		c0.1,1.5-1.3,2.7-0.7,4.2c0.5,1.2,2.5,2.1,2.3,3.5c-0.2,2.1-2.8,2-3.7,3.3c-0.6,0.9,0.3,1.6,0.6,2.4c0.6,1.6,0.3,2.1-0.9,3.1
		C302.6,203.6,302.4,203.8,302.2,204.1z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M348.5,246.9c-0.1,0.1-0.2,0.2-0.2,0.3
		c-1.4,1.7-3,2.6-4.5,0.2c-1.5-2.4-2.5-0.7-4.7-0.7c-2.6,0-4.3-0.6-5.7-3c-0.8-1.4-2.9-6.8-0.3-7.9c1.8-0.8,9.2,1.4,7.2-3.4
		c-0.5-1.2-1.9-1.5-2.7-2.3c-0.7-0.7-2.2-2.8-1.6-3.9c0.8-1.5,2.8-1.3,4.1-1.9c1.7-0.8,0.6-3,2.1-3.7c0.8-0.3,3,0.2,3.7,0.6
		c1.7,0.9,2.7,4.9,5,3.9c1.2-0.5,1.6-1.9,2.7-2.5c3.9-2,4.7,2.4,6.1,4.5c0.7,1.1,2.3,2.1,2.3,3.6c0,2.3-2.4,3.6-2.7,5.7
		c-0.3,1.7,3.5,4.6,1.1,5.9c-0.7,0.4-1.9-0.1-2.7,0.4c-0.6,0.4-0.7,1-0.9,1.6c-0.5,1.5-0.7,2.9-2.7,2.5
		C352,246.6,350.2,245.2,348.5,246.9z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M365.6,260.5c-0.3-0.1-0.5-0.2-0.8-0.1
		c-2.2,0.1-3.5,2-5.4-0.2c-0.6-0.8-0.5-0.9-1.5-1.2c-0.8-0.2-1.5-0.4-2.3-0.5c-2.3-0.6-3-2.7-3.5-4.8c-1-4.1,2.8-4.3,4.3-7.4
		c0.6-1.2,0.2-2.8,2.1-2.8c2.2,0,3.7,0.8,5.7-0.5c1.2-0.8,2.3-2.1,3.8-0.9c0.7,0.6,0.8,1.7,1.5,2.4c0.7,0.7,1.6,0.8,2.5,1.2
		c1.9,0.9,2.2,2.2,1.8,4c-0.5,2.4,0.3,4.1,0.9,6.3c0.5,1.6-0.1,3.1-2,2.9c-1.3-0.1-2.5-1.5-3.7-0.6c-1,0.7-1,2.3-2.4,2.4
		C366.3,260.7,366,260.6,365.6,260.5z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M384.2,183.9c-0.1,0-0.3,0-0.4,0
		c-0.8,0.1-2.4,0.2-2.9,1c-0.5,0.8,0.4,1.6,0,2.4c-1.1,1.9-3.4-0.2-4.1-1.2c-1.3-1.8-1.9-3.2-4-4.2c-1.3-0.6-3.6-2-2.7-3.7
		c0.7-1.3,2.5-1.6,3.4-2.7c1-1.3-1.8-3.2-2.2-4.3c-0.7-1.8,0.1-3.1,2-3.7c1-0.3,2.5,0,3.2-0.9c0.7-0.8,0.1-2.2,0.5-3.1
		c1.6-3,6.6,1.1,8.6,1.1c1,0,1.7-0.8,2.7-0.8c4,0.2,1.4,3.8,3,5.7c1.5,1.8,4.4,2.5,5.4,4.8c0.6,1.2,0.2,2.5,0.6,3.8
		c0.3,1.3,1,2,0.7,3.5c-0.4,2.1-2.1,2-3.3,3.5c-0.9,1.1-0.4,2.5-1.5,3.4c-1.8,1.4-3-0.8-4.1-1.9
		C387.6,185.3,386.2,183.9,384.2,183.9z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M413.2,293.5c-0.1,0.1-0.3,0.2-0.4,0.3
		c-2.9,2.3-3.5-1-6.5-0.8c-2.9,0.2-4.3,1.4-4.8-2.3c-0.3-1.8,0.1-1.8-1.5-3.1c-1.5-1.3-1.9-1.2-1.8-3.2c0.2-1.9-0.4-5.9,1.5-6.9
		c1.9-1.1,4,1.7,5.7,0.2c2-1.7,2.4-6.2,5.7-5.9c1.3,0.1,1.8,1.6,2.9,1.7c2.5,0.2,2.9-3.3,5.5-1c3,2.6-0.1,3.3-0.7,5.5
		c-1.4,5,5.6,4.2,6.2,7.3c0.2,1-1.9,4.1-2.3,5c-1,1.8-1.5,6.5-4.9,3.6C415.3,292,415.3,291.8,413.2,293.5z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M399.7,273.8c-0.1,0.1-0.3,0.2-0.4,0.3
		c-2.9,2.3-3.5-1-6.5-0.8c-2.9,0.2-4.3,1.4-4.8-2.3c-0.3-1.8,0.1-1.8-1.5-3.1c-1.5-1.3-1.9-1.2-1.8-3.2c0.2-1.9-0.4-5.9,1.5-6.9
		c1.9-1.1,4,1.7,5.7,0.2c2-1.7,2.4-6.2,5.7-5.9c1.3,0.1,1.8,1.6,2.9,1.7c2.5,0.2,2.9-3.3,5.5-1c3,2.6-0.1,3.3-0.7,5.5
		c-1.4,5,5.6,4.2,6.2,7.3c0.2,1-1.9,4.1-2.3,5c-1,1.8-1.5,6.5-4.9,3.6C401.9,272.2,401.9,272.1,399.7,273.8z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M290.6,221.6c0.1,0.1,0.3,0.2,0.4,0.3
		c3.1,1.9,0.3,3.6,1.5,6.4c1.2,2.6,2.8,3.5-0.5,5.3c-1.6,0.9-1.7,0.5-2.4,2.5c-0.6,1.9-0.4,2.2-2.4,2.8c-1.5,0.4-2.9,1.2-4.5,1.4
		c-1.9,0.3-2.6-0.7-4-1.6c-0.9-0.6-1.6-1.5-2.5-2.1c-2-1.2-4.7,0.7-5.7-2c-1-2.7,0.7-3.3,2.9-3.5c1.2-0.1,2.7-1.3,3-2.5
		c0.5-1.9-1.4-2.4-2.6-3.3c-1.1-0.8-1.6-1.9-0.6-3c1-1.1,2.4-0.6,3.2-1.9c0.7-1.1,0.7-2.6,0.9-3.9c0.3-2.9,3.3-5.7,6.2-3.9
		c0.8,0.5,1.3,1.1,2.2,1.2c0.7,0.1,1.5-0.1,2.2,0c1.6,0.4,1.8,1.9,1.3,3.3C288.4,220.1,288.2,220.2,290.6,221.6z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M320.3,215.2c2.2,1.4,2.9,2.1,1.8,4.5
		c-0.7,1.7,0.2,3.9-0.9,5.4c-1.2,1.7-2.9,1-4.6,1.2c-1.8,0.3-2.3,1.6-3.2,3.1c-0.5,1-1,2.1-2.3,1.9c-2-0.4-2.9-2.5-4.7-3.3
		c-1.4-0.7-3.6-1.1-4-2.9c-0.2-1.3,0.5-2.4-0.1-3.7c-0.6-1.3-2.2-2-2.3-3.6c-0.1-1,0.5-2.4,1.5-2.9c0.9-0.4,1.8,0,2.4-1
		c0.8-1.2,0.4-5.3,2.5-5.3c1.4,0,2.2,1.8,3.4,2.1c1.7,0.4,2.3-1.6,3.3-2.5c2.1-1.9,7.7-0.4,6.6,3.1c-0.5,1.6-2.3,1.9-0.4,3.5
		C319.7,214.8,320,215,320.3,215.2z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M351,305.4c1.2,0.5,4,0.9,4.1,2.6
		c0.1,2.2-5.2,3.2-5.9,5.2c-1.7,4.5,9.2,5.4,6.2,10.1c-0.9,1.4-2.2,0.8-3.5,1.1c-1.3,0.3-1.7,1.6-2.8,2.2c-1.3,0.7-2.3-0.3-3.5-0.7
		c-2.4-0.8-4.2,2.3-6.4,0.5c-1.9-1.6,0.2-4.9-2-6.2c-2.6-1.5-7.5-0.1-8.5-3.7c-0.4-1.5,1-2.6,0.7-3.8c-0.4-1.5-1.8-1.6-2.7-2.6
		c-0.7-0.8-0.7-2.4-0.4-3.4c0.2-0.8,0.8-1.5,1.7-1.7c1.8-0.4,3.8,1.7,5.6,0.3c1.6-1.2,1.3-4.4,1.8-6.2c0.8-2.7,2.7-1.3,4.7-1.3
		c1.9-0.1,2.2-2.6,3.9-3.3c2.7-1.2,5.6,1.3,5.5,4C349.2,301.2,347.7,303.9,351,305.4z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M310.6,307.9c-0.8-0.9-2.2-0.4-3.2-1
		c-1.7-0.9-2.3-2.7-1.7-4.5c0.5-1.5,3.9-3.6,1.9-5.5c-1.1-1-2.1-0.5-2.5-2.5c-0.3-1.4-0.1-2.6,1.2-3.3c1.6-0.8,2.6-1.1,3.6-2.6
		c0.9-1.4,2.3-3.3,4.1-1.9c1,0.7,1.9,1.8,3.1,2.1c1.2,0.2,1-0.4,1.9-1c1.5-1,5.9-0.3,7,1c2.2,2.6-2.3,5.3,0.9,7.8
		c0.9,0.7,1.8,0.7,2.3,2c0.5,1.2,0.1,3.1,0.1,4.3c0.1,1.4-0.7,1.7-2.1,1.9c-1.3,0.2-2.2,0.2-2.1,1.7c0,0.9,1,2.7-0.1,3.4
		c-1,0.7-2.6-0.7-3.6-0.9c-1.5-0.3-1.9,0.5-2.4,1.6c-1.3,2.8-4.8,1.5-6.9,0.3C311.2,310.1,311.4,308.7,310.6,307.9z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M277.9,261.8c-0.3,0.2-0.6,0.4-1,0.7
		c-1.2,0.9-2.5,1.9-3.5,0.2c-0.6-1-0.5-3.2-1.9-3.7c-0.7-0.2-1.6,0.9-2.1,1.3c-2.6,2.2-4.7,0.7-5.6-2.1c-0.9-2.7-1.6-4.4-4.8-4.6
		c-3.4-0.2-4.3-2.9-3.3-6c0.7-2.3-1.9-6.5,1.5-7.5c1.6-0.5,3.4,0.2,5-0.2c1.7-0.4,2.4-1.6,3.2-3c1.2-2.2,2.1-1.4,4.3-0.8
		c1.4,0.4,2.6,0.3,4,0.3c1.9,0,2.2,0.6,2.9,2.2c0.6,1.4,2.4,3,2.1,4.6c-0.4,2.1-2.6,2.8-2.6,5.2c0,1-0.3,2.2,0.3,3.1
		c0.6,0.9,1.7,1,2.5,1.7c1.5,1.3,1.6,4.4,1.3,6.2C280,260.5,279,261.1,277.9,261.8z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M429,270.6c-1.2-0.5-1.9-2.2-3.2-1
		c-0.7,0.6-0.5,2.2-1,3c-2.1,3.9-4.6-1.3-6.9-1.7c-1.5-0.3-2.6,0-2.8-1.8c-0.2-1.6,0.7-3.2,0-4.7c-0.7-1.5-2.2-2.6-3.1-4
		c-1.6-2.8-0.6-4.9,2.5-5.8c1.4-0.4,2.2-1,2.7-2.3c0.6-1.3,0.5-2.7,2.2-3c0.8-0.1,1.6,0.3,2.4,0.1c0.6-0.2,0.8-0.8,1.3-1
		c1.7-0.7,4.5-0.2,5.1,1.7c0.6,1.6-0.8,3.9-0.2,5.3c1,2.3,3.7,0,5.3,0.1c2,0.2,2.6,2.8,2,4.4c-0.8,2.2-3.1,2.8-3,5.5
		C432.5,267,432.4,272.2,429,270.6z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M357.9,163.4c0.9,0.8,2.2,1.7,2.3,2.9
		c0.1,1.4-1.5,2.2-1.1,3.7c0.3,1.1,2.5,2.3,2.1,3.4c-0.2,0.5-1.4,0.9-1.9,1.2c-0.8,0.6-1.5,1.2-1.9,2.2c-0.8,1.9,0.2,4-0.7,5.8
		c-0.4,0.8-0.9,1.5-1.3,2.4c-0.4,0.9-0.6,1.6-1.7,1.8c-1.9,0.4-5-0.7-5.7-2.7c-0.4-1.3,0.5-3.2-1.4-3.5c-1.5-0.2-2.5,1.6-3.8,1.2
		c-1.4-0.4-2.3-2.5-3.3-3.4c-0.7-0.6-1.8-1-2.4-1.7c-1-1.2-0.9-2.7-2.4-3.4c-1.5-0.7-4-1.3-4.6-3.1c-0.5-1.5,0-3.3,1.6-3.8
		c1.9-0.5,3.1-0.9,4.6-2.2c1.5-1.3,3.1-2.6,5.2-1.8c2.1,0.9,3.6,1.5,6,1.1c2.3-0.4,4.3-2,6.8-1.8c1.4,0.1,2.5,0.4,3.6,1.4
		C357.7,163.2,357.8,163.3,357.9,163.4z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M319.6,174.2c1.5,0.5,3.4,1.9,4.4,0.1
		c0.5-0.9,0.4-1.8,1.6-2c1-0.2,3.2,1,3.8-0.3c0.3-0.5-0.3-1.4-0.4-1.9c-0.1-0.9,0.2-1.5,0.6-2.2c0.9-1.4,0.2-1.9-0.6-3.1
		c-0.7-1.1,0.1-2.6-0.6-3.6c-0.7-1-3-1-4.1-1.3c-1.3-0.3-3-0.7-4.1,0.5c-0.9,0.9-0.8,2-2.2,2.3c-4.5,0.7-1.1,3.5-0.8,6.2
		c0.1,1.5-1.2,1.7-1,3.2C316.6,174.1,318.1,173.6,319.6,174.2z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M285.3,241.8c-1,0.1-2,1.1-2.4,2.1
		c-0.6,1.4,0.2,1.8,1,2.7c1.8,2.1-1.8,2.5-1.9,4.6c0,1.8,1.8,2.6,2.8,3.8c0.9,1.1,1.1,2.1,2.8,1.7c3.8-0.7,1.9-4.2,3.7-6.5
		c0.7-0.9,2.3-0.7,3-1.7c0.6-1-0.3-2-0.7-2.9c-0.4-1-0.6-3-1.6-3.7c-0.9-0.6-2,0.1-2.9,0.4c-1.2,0.4-1.8,0.2-3-0.4
		C285.8,241.8,285.5,241.8,285.3,241.8z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M340.6,185.8c-1.1,0-2,0-2.5,1.3
		c-0.8,2.2,1.1,3.5,2.7,4.4c1,0.6,0.9,1,0.8,2.1c-0.2,1.9,0.8,2,2.3,1.8c1.1-0.2,2.8,0.3,3.7-0.6c0.9-0.9-0.1-2.4,1.1-3.3
		c1.1-0.9,2.7-2.2,0.8-3.4c-0.8-0.5-1.6-0.5-2.1-1.5c-0.3-0.6-0.1-1.7-0.7-2.1c-1.3-0.8-3.4,1.3-4.7,1.4
		C341.5,185.9,341.1,185.9,340.6,185.8z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M303.8,277.9c-0.1,0-0.1,0.1-0.2,0.1
		c-1,1-1,2.6,0.5,2.8c1.3,0.2,2-0.8,2.4,0.9c0.3,1.7,0.2,5.7,2.9,4.9c1.5-0.4,2.1-1.9,3.5-2.4c0.9-0.4,2.3-0.3,3.3-0.4
		c2.7-0.1,1.6-2,1.5-3.7c-0.2-1.7,0.8-3.9-1.9-3.7c-2,0.2-2.7,0-4.1-1.4c-1.8-1.8-3,0.5-4.5,1.5
		C306.1,277.3,304.7,277.2,303.8,277.9z" style="fill:#60A2C4;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton nucleoli" descendants="level 0: ['nucleolus, pval = 0.00453']
level 1: ['PeBoW complex, pval = --', 'Pwp2p-containing subcomplex of 90S preribosome, pval = --', 'RENT complex, pval = --', 'nucleolar chromatin, pval = --', 'nucleolar exosome (RNase complex), pval = --', 'B-WICH complex, pval = --', 'nucleolar preribosome, pval = --', 'granular component, pval = --', 'dense fibrillar component, pval = --', 'fibrillar center, pval = 0.94537', 'RNA polymerase I complex, pval = --', 'RNA polymerase I transcription factor complex, pval = --', 'nucleolar ribonuclease P complex, pval = --', 'eNoSc complex, pval = --', 't-UTP complex, pval = --', 'UTP-C complex, pval = --', 'box C/D snoRNP complex, pval = --', 'Mpp10 complex, pval = --', 'box H/ACA snoRNP complex, pval = --']
level 2: ['nucleolus organizer region, pval = --', 'rDNA protrusion, pval = --', 'nucleolus-associated heterochromatin, pval = --', 'RNA polymerase transcription factor SL1 complex, pval = --', 'RNA polymerase I core factor complex, pval = --', 'RNA polymerase I upstream activating factor complex, pval = --', '90S preribosome, pval = --', 'small-subunit processome, pval = --']
level 3: ['Noc4p-Nop14p complex, pval = --', 'Noc1p-Noc2p complex, pval = --']" id="nucleoli_ton" init-pval="0.00453" min-pval="0.00453" min-pval-children="0.94537" title="Nucleoli">
<path d="M333.7,254.1c-0.6-2.4-0.6-4.9-1.4-7.3
		c-2.4-7.1-10.2-9.2-17-9.3c-2.1,0-4.4,0.2-6.4,0.7c-4.1,1.1-7.9,2.3-10.4,5.9c-2.7,3.9-5,8.3-4.6,13.2c0.1,1.4,0.5,2.7,1.1,4
		c1,2.1,1.9,4.1,3.5,5.8c2.5,2.7,7,5,10.7,5.4c5.1,0.5,10.8,0.9,15.7-1.1c3.6-1.5,8.2-5.2,9.2-9.2
		C334.6,259.6,334.3,256.7,333.7,254.1C333.7,254.2,333.7,254.1,333.7,254.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M397.6,220.8c-1.9-2.2-3.8-3.3-6.3-4.8
		c-2.1-1.2-3.5-2.4-5.8-2.8c-2.5-0.5-5-0.8-7.5-0.4c-5,0.9-10.3,2.6-13.1,7.2c-2.6,4.2-2.3,10.5,0.2,14.6c2.3,3.8,6.4,5.7,10.1,7.9
		c3.4,2.1,6.3,5.6,10.4,6.2c3.3,0.4,7.8-0.9,10.4-2.9c1.8-1.4,3.1-3.3,4-5.3c0.8-1.8,1.2-3.8,1.7-5.7c0.6-2.5,0.6-4.8-0.1-7.3
		C400.6,225.1,399.3,222.8,397.6,220.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M390.1,278.4c0-0.1-0.1-0.2-0.1-0.4
		c-1.1-3.5-4.8-7.9-7.8-10.1c-2.6-1.8-5.8-2.4-8.9-2.5c-3.3-0.1-5.5,1.4-8.5,2.6c-2.4,0.9-4.7,2.2-6.8,3.7c-3.7,2.7-6.3,6.7-7,11.2
		c-0.4,2.1-0.3,4.3,0,6.5c0.4,2.5,0.9,5,2.5,7.1c1.6,2,3.9,3.9,6.4,4.8c3.7,1.3,9.6,1.5,13.3,0.1c3.1-1.2,5.8-3.2,8.6-5.1
		c2.2-1.4,4.6-2.9,6.3-5c1.3-1.6,2-4.3,2.4-6.2C390.9,283.1,390.7,280.6,390.1,278.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col nucleoli" descendants="level 0: ['nucleolus, pval = 0.00453']
level 1: ['PeBoW complex, pval = --', 'Pwp2p-containing subcomplex of 90S preribosome, pval = --', 'RENT complex, pval = --', 'nucleolar chromatin, pval = --', 'nucleolar exosome (RNase complex), pval = --', 'B-WICH complex, pval = --', 'nucleolar preribosome, pval = --', 'granular component, pval = --', 'dense fibrillar component, pval = --', 'fibrillar center, pval = 0.94537', 'RNA polymerase I complex, pval = --', 'RNA polymerase I transcription factor complex, pval = --', 'nucleolar ribonuclease P complex, pval = --', 'eNoSc complex, pval = --', 't-UTP complex, pval = --', 'UTP-C complex, pval = --', 'box C/D snoRNP complex, pval = --', 'Mpp10 complex, pval = --', 'box H/ACA snoRNP complex, pval = --']
level 2: ['nucleolus organizer region, pval = --', 'rDNA protrusion, pval = --', 'nucleolus-associated heterochromatin, pval = --', 'RNA polymerase transcription factor SL1 complex, pval = --', 'RNA polymerase I core factor complex, pval = --', 'RNA polymerase I upstream activating factor complex, pval = --', '90S preribosome, pval = --', 'small-subunit processome, pval = --']
level 3: ['Noc4p-Nop14p complex, pval = --', 'Noc1p-Noc2p complex, pval = --']" id="nucleoli" init-pval="0.00453" min-pval="0.00453" min-pval-children="0.94537" title="Nucleoli">
<path d="M333.7,254.1c-0.6-2.4-0.6-4.9-1.4-7.3
		c-2.4-7.1-10.2-9.2-17-9.3c-2.1,0-4.4,0.2-6.4,0.7c-4.1,1.1-7.9,2.3-10.4,5.9c-2.7,3.9-5,8.3-4.6,13.2c0.1,1.4,0.5,2.7,1.1,4
		c1,2.1,1.9,4.1,3.5,5.8c2.5,2.7,7,5,10.7,5.4c5.1,0.5,10.8,0.9,15.7-1.1c3.6-1.5,8.2-5.2,9.2-9.2
		C334.6,259.6,334.3,256.7,333.7,254.1C333.7,254.2,333.7,254.1,333.7,254.1z" style="fill:#CEEAF9;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M397.6,220.8c-1.9-2.2-3.8-3.3-6.3-4.8
		c-2.1-1.2-3.5-2.4-5.8-2.8c-2.5-0.5-5-0.8-7.5-0.4c-5,0.9-10.3,2.6-13.1,7.2c-2.6,4.2-2.3,10.5,0.2,14.6c2.3,3.8,6.4,5.7,10.1,7.9
		c3.4,2.1,6.3,5.6,10.4,6.2c3.3,0.4,7.8-0.9,10.4-2.9c1.8-1.4,3.1-3.3,4-5.3c0.8-1.8,1.2-3.8,1.7-5.7c0.6-2.5,0.6-4.8-0.1-7.3
		C400.6,225.1,399.3,222.8,397.6,220.8z" style="fill:#CEEAF9;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M390.1,278.4c0-0.1-0.1-0.2-0.1-0.4
		c-1.1-3.5-4.8-7.9-7.8-10.1c-2.6-1.8-5.8-2.4-8.9-2.5c-3.3-0.1-5.5,1.4-8.5,2.6c-2.4,0.9-4.7,2.2-6.8,3.7c-3.7,2.7-6.3,6.7-7,11.2
		c-0.4,2.1-0.3,4.3,0,6.5c0.4,2.5,0.9,5,2.5,7.1c1.6,2,3.9,3.9,6.4,4.8c3.7,1.3,9.6,1.5,13.3,0.1c3.1-1.2,5.8-3.2,8.6-5.1
		c2.2-1.4,4.6-2.9,6.3-5c1.3-1.6,2-4.3,2.4-6.2C390.9,283.1,390.7,280.6,390.1,278.4z" style="fill:#CEEAF9;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton nucleoli_fibrillar_center" descendants="level 0: [None]" id="nucleoli_fibrillar_center_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Nucleoli fibrillar center">
<path d="M303.2,254.8c0.2,0.8,0.1,1.1,1.1,1.2
		c1.5,0.2,3.2,0.9,3.8-0.9c0.3-0.9,1.4-1.1,1.2-2.2c-0.2-0.8-0.9-2.2-1.7-2.5c-1.1-0.5-2.3-0.8-3.4,0c-0.5,0.4-1.1,0.7-1.4,1.1
		c-0.8,1.1,0.2,2,0.5,3.1C303.2,254.8,303.2,254.8,303.2,254.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M308.8,250.3c1.5,1,2.6,0.8,4.3,0.5
		c0.6-0.1,1.5-0.1,1.9-0.7c0.4-0.5,0.7-1.2,0.5-1.9c-0.1-0.4-0.4-0.8-0.5-1.2c-0.1-0.5,0.1-1-0.1-1.5c-0.4-0.7-1.3-0.9-2-1
		c-0.7-0.1-1.6-0.2-2.3-0.2c-0.8,0.1-1.1,0.9-1.6,1.3c-0.6,0.5-0.9,0.8-1,1.6c-0.1,0.9-0.2,2.1,0.5,2.8
		C308.6,250.1,308.7,250.2,308.8,250.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M317.7,251.4c1.1,0.3,1.8,1.7,3.1,1.7
		c0.6,0,1.2-0.2,1.9-0.2c0.6,0,1.6,0.2,2-0.4c0.8-1.1-0.3-2.6-0.7-3.6c-0.5-1.4-0.7-2.2-2.1-2.9c-0.6-0.3-1.6-1-2.3-1
		c-0.8,0-1.8,0.7-2.5,1c-0.8,0.4-1.1,1.2-1.1,2c0,0.7,0,1.6,0.2,2.2C316.3,251,317,251.2,317.7,251.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M311.2,257.3c0.4,0.4,0.6,0.8,1.2,1
		c0.6,0.2,1.3-0.1,1.9-0.2c0.7-0.1,1.4-0.6,2-0.8c0.5-0.2,1.2-0.3,1.6-0.8c1-1.3-0.8-3.2-1.4-4.3c-0.4-0.7-1-1.5-1.8-1.4
		c-0.5,0.1-0.9,0.1-1.4,0.3c-0.5,0.1-0.6,0.4-1,0.6c-0.7,0.5-1.6,0.2-2.3,0.8c-0.5,0.4-0.4,0.9-0.3,1.5c0.1,0.5,0,0.9-0.1,1.4
		C309.4,256.4,310.6,256.7,311.2,257.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M375.8,225.6c0.9,0.4,1.9,0.6,2.6,0.5
		c0.9-0.1,1.5-0.8,2.4-1c0.9-0.2,1.4-0.7,1.5-1.6c0.1-1-0.2-2.2-0.7-3c-0.5-0.8-1.8-1.8-2.9-1.8c-0.5,0-0.6,0.3-1,0.5
		c-0.5,0.2-0.9,0-1.4,0c-0.7,0-1.5,0.5-1.8,1.1c-0.2,0.3-0.2,0.6-0.3,0.9c-0.1,0.3-0.4,0.6-0.5,0.9c-0.2,0.6-0.2,1.8,0.2,2.3
		C374.3,224.8,375,225.2,375.8,225.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M375.7,234.3c0.6,0.1,1.1,0.1,1.7,0.2
		c1.2,0.3,2.2-0.6,3.3-1.1c1.1-0.4,2.6-1.1,2.6-2.4c0-0.5-0.4-0.9-0.6-1.3c-0.8-2-2.2-3.9-5-3.1c-0.9,0.3-1.9,0.7-2.6,1.3
		c-0.7,0.6-1,1.5-1.5,2.2c-1,1.2-0.6,2.3,0.3,3.1c0.4,0.4,0.9,0.8,1.5,1C375.6,234.3,375.7,234.3,375.7,234.3z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M380.6,239.8c0.3,0.3,0.7,0.4,1.2,0.4
		c1.1-0.2,1.4-0.1,2.3,0.5c0.6,0.4,1.6,0.3,2.1-0.3c0.3-0.3,0.1-0.8,0.1-1.2c0-0.5,0.1-0.9,0.2-1.4c0.1-0.7-0.1-1-0.5-1.6
		c-0.3-0.5-0.4-1.1-0.7-1.6c-0.4-0.7-1.3-0.7-2-0.7c-0.7,0-1.8-0.4-2.5,0c-0.7,0.4-1.1,0.9-1.1,1.7c-0.1,1.1-0.9,2-0.1,3.1
		C379.9,239,380.2,239.5,380.6,239.8z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M384.4,229.5c0.6,0.8,1.7,0.3,2.6,0
		c1.5-0.5,5.8,0.2,4.4-2.9c-0.3-0.7-0.7-1.3-0.9-2c-0.4-1.1-0.9-2.7-2.3-3c-1.1-0.3-2.2-0.8-3.3-0.3c-0.8,0.4-1.7,1.3-1.9,2.2
		c-0.1,0.7-0.7,1-0.8,1.8c-0.1,0.6,0.1,2.1,0.5,2.6c0.2,0.3,0.7,0.3,1,0.5c0.3,0.3,0.5,0.7,0.7,1.1
		C384.4,229.4,384.4,229.5,384.4,229.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M386.9,235.9c0.3,0.6,0.7,1.4,1.3,1.7
		c0.4,0.2,1.1,0.3,1.6,0.4c0.6,0.1,1.5,0.4,2.1,0.2c0.6-0.2,1.1-1.1,1.4-1.7c0.2-0.4,0.5-0.5,0.6-1c0.1-0.6-0.1-1.2,0-1.8
		c0.5-2.3-1.4-4.2-3.7-4.2c-0.6,0-0.9,0.4-1.4,0.7c-0.5,0.3-1.1,0.4-1.6,0.8c-0.4,0.4-0.5,1.2-0.7,1.7
		C386.1,233.8,386.4,234.9,386.9,235.9z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M364.1,287.6c1.3-0.9,1.1-2.7,0.9-4.1
		c-0.1-0.7,0-1.7-0.4-2.4c-0.6-1.1-2-2.4-3.3-2.3c-1,0.1-1.3,1.1-1.9,1.9c-1.2,1.5-1.1,3.1,0,4.8c0.3,0.4,0.9,0.8,1,1.2
		c0.5,1.2,2.5,1.6,3.5,1.1C363.9,287.7,364,287.6,364.1,287.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M369.3,296.6c0.2,0,0.3,0,0.5,0
		c1.2,0.2,2.5,0.5,3.2-0.7c0.7-1.2,0.5-2.1,0.6-3.3c0.1-0.8,0.3-1.7-0.3-2.4c-0.3-0.4-0.9-0.5-1.4-0.7c-0.8-0.4-1.7-0.4-2.6-0.4
		c-0.9,0-1.6-0.4-2.5-0.4c-1.3,0.1-1.4,0.9-2.2,1.6c-0.6,0.6-1.3,1.1-1.4,2c-0.1,1.1,0.8,1.5,1.1,2.4c0.4,1.1,1.2,1.7,2.2,2
		c0.5,0.2,0.9,0.3,1.4,0.3C368.4,296.8,368.9,296.7,369.3,296.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M378.1,289.2c0.2-0.2,0.4-0.5,0.7-0.7
		c0.6-0.4,1.2-0.6,1.9-0.8c0.6-0.1,1.3-0.3,1.4-1c0.1-0.5-0.2-1-0.3-1.5c-0.3-1.1-0.4-1.6-1.4-2.2c-0.8-0.5-1.7-0.9-2.7-1
		c-1-0.1-1.7,0.6-2.6,0.7c-0.9,0.1-1.4,0.8-1.4,1.6c0,0.9-0.4,1.6-0.5,2.5c0,0.6,0.3,1.7,0.8,2.1c1.2,1,2.6,1.8,3.9,0.4
		C378,289.3,378.1,289.3,378.1,289.2z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M371.4,287.7c0.8-0.2,0.9-1.2,1-1.9
		c0.1-0.4,0.2-0.8,0.3-1.1c0.2-0.7,0.7-1.5,0.6-2.2c-0.1-0.9-1.2-1-1.9-1.3c-0.4-0.1-0.6-0.2-1-0.1c-1.2,0.2-2.6,0.1-3.7,0.7
		c-1.1,0.6-0.9,1.9-0.9,3c0,0.8-0.4,1.6,0.3,2.2c0.7,0.7,2.3,1.7,3.4,1.5c0.3-0.1,0.5-0.4,0.7-0.5c0.3-0.2,0.7-0.1,1.1-0.1
		C371.3,287.7,371.3,287.7,371.4,287.7z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M370.4,280.5c0.3-0.2,0.7-0.6,0.9-0.9
		c0.4-0.6,0.6-1.3,0.6-2c0.1-1.2-0.4-3.6-1.7-4.1c-0.7-0.3-1.4-0.1-2.1,0c-0.6,0.1-0.8,0.4-1.4,0.7c-0.8,0.4-1.8,0.5-2.5,1.1
		c-1,0.8-1,3.4-0.1,4.2c0.3,0.3,0.7,0.4,1,0.6c0.8,0.5,1.8,0.7,2.7,0.8c0.8,0.1,1.7,0.2,2.4-0.2
		C370.3,280.6,370.3,280.6,370.4,280.5z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M377.5,281.6c0.6-0.1,1.6-0.2,2.1-0.6
		c0.3-0.3,0.7-0.9,0.9-1.3c0.3-0.5,0.9-1.2,0.9-1.8c0-0.6-0.6-1.4-1-1.9c-0.3-0.4-0.3-0.6-0.7-0.9c-0.5-0.3-1.1-0.3-1.6-0.6
		c-2-1.3-4.4-0.2-5.2,1.9c-0.2,0.6,0.1,1,0.1,1.5c0.1,0.6-0.1,1.2,0.2,1.8c0.2,0.5,0.9,0.9,1.4,1.2
		C375.3,281.6,376.4,281.7,377.5,281.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M317.4,261.6c0.5,1.2,2.1,1.6,3.2,1.5
		c0.8-0.1,2-0.1,2.7-0.7c0.6-0.5,0.8-1.9,1-2.7c0.3-0.9,0-1.7-0.4-2.5c-0.4-0.9-1-1.2-1.7-1.9c-0.7-0.7-1.4-0.8-2.4-0.6
		c-0.8,0.2-0.9,0.6-1.3,1.3c-0.5,1.1-1.8,1.5-2.3,2.6c-0.2,0.6,0.1,1.1,0.4,1.6C317.1,260.7,317.2,261.1,317.4,261.6z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M308.6,264.4c0.1,0.9,0.8,1.2,1.6,1.3
		c0.8,0.1,1.8,0.7,2.7,0.5c0.5-0.1,0.7-0.5,1.1-0.8c0.4-0.4,0.9-0.6,1.4-0.9c1.2-0.7,0.7-1.8,0.2-2.8c-0.4-0.8-1-2.4-2-2.8
		c-0.8-0.4-3.1-0.5-3.8,0c-1,0.6-0.9,1.5-0.8,2.5C308.9,262.5,308.4,263.4,308.6,264.4z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M302.3,262.1c0.1,0.1,0.2,0.2,0.3,0.3
		c1,1,2.4,0.5,3.6,0.6c1.3,0.1,2.4,0,2.2-1.7c-0.1-0.7,0-1.3,0.2-1.9c0.1-0.6,0.2-1.2-0.4-1.5c-0.4-0.3-0.9-0.4-1.4-0.6
		c-0.4-0.2-0.6-0.6-1.1-0.7c-0.6-0.1-1.4-0.1-1.9,0.1c-0.6,0.2-0.9,0.7-1.4,1C301.1,258.6,301.6,261,302.3,262.1z" style="fill:#F6F6F6;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col nucleoli_fibrillar_center" descendants="level 0: [None]" id="nucleoli_fibrillar_center" init-pval="--" min-pval="--" min-pval-children="--" title="Nucleoli fibrillar center">
<path d="M303.2,254.8c0.2,0.8,0.1,1.1,1.1,1.2
		c1.5,0.2,3.2,0.9,3.8-0.9c0.3-0.9,1.4-1.1,1.2-2.2c-0.2-0.8-0.9-2.2-1.7-2.5c-1.1-0.5-2.3-0.8-3.4,0c-0.5,0.4-1.1,0.7-1.4,1.1
		c-0.8,1.1,0.2,2,0.5,3.1C303.2,254.8,303.2,254.8,303.2,254.8z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M308.8,250.3c1.5,1,2.6,0.8,4.3,0.5
		c0.6-0.1,1.5-0.1,1.9-0.7c0.4-0.5,0.7-1.2,0.5-1.9c-0.1-0.4-0.4-0.8-0.5-1.2c-0.1-0.5,0.1-1-0.1-1.5c-0.4-0.7-1.3-0.9-2-1
		c-0.7-0.1-1.6-0.2-2.3-0.2c-0.8,0.1-1.1,0.9-1.6,1.3c-0.6,0.5-0.9,0.8-1,1.6c-0.1,0.9-0.2,2.1,0.5,2.8
		C308.6,250.1,308.7,250.2,308.8,250.3z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M317.7,251.4c1.1,0.3,1.8,1.7,3.1,1.7
		c0.6,0,1.2-0.2,1.9-0.2c0.6,0,1.6,0.2,2-0.4c0.8-1.1-0.3-2.6-0.7-3.6c-0.5-1.4-0.7-2.2-2.1-2.9c-0.6-0.3-1.6-1-2.3-1
		c-0.8,0-1.8,0.7-2.5,1c-0.8,0.4-1.1,1.2-1.1,2c0,0.7,0,1.6,0.2,2.2C316.3,251,317,251.2,317.7,251.4z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M311.2,257.3c0.4,0.4,0.6,0.8,1.2,1
		c0.6,0.2,1.3-0.1,1.9-0.2c0.7-0.1,1.4-0.6,2-0.8c0.5-0.2,1.2-0.3,1.6-0.8c1-1.3-0.8-3.2-1.4-4.3c-0.4-0.7-1-1.5-1.8-1.4
		c-0.5,0.1-0.9,0.1-1.4,0.3c-0.5,0.1-0.6,0.4-1,0.6c-0.7,0.5-1.6,0.2-2.3,0.8c-0.5,0.4-0.4,0.9-0.3,1.5c0.1,0.5,0,0.9-0.1,1.4
		C309.4,256.4,310.6,256.7,311.2,257.3z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M375.8,225.6c0.9,0.4,1.9,0.6,2.6,0.5
		c0.9-0.1,1.5-0.8,2.4-1c0.9-0.2,1.4-0.7,1.5-1.6c0.1-1-0.2-2.2-0.7-3c-0.5-0.8-1.8-1.8-2.9-1.8c-0.5,0-0.6,0.3-1,0.5
		c-0.5,0.2-0.9,0-1.4,0c-0.7,0-1.5,0.5-1.8,1.1c-0.2,0.3-0.2,0.6-0.3,0.9c-0.1,0.3-0.4,0.6-0.5,0.9c-0.2,0.6-0.2,1.8,0.2,2.3
		C374.3,224.8,375,225.2,375.8,225.6z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M375.7,234.3c0.6,0.1,1.1,0.1,1.7,0.2
		c1.2,0.3,2.2-0.6,3.3-1.1c1.1-0.4,2.6-1.1,2.6-2.4c0-0.5-0.4-0.9-0.6-1.3c-0.8-2-2.2-3.9-5-3.1c-0.9,0.3-1.9,0.7-2.6,1.3
		c-0.7,0.6-1,1.5-1.5,2.2c-1,1.2-0.6,2.3,0.3,3.1c0.4,0.4,0.9,0.8,1.5,1C375.6,234.3,375.7,234.3,375.7,234.3z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M380.6,239.8c0.3,0.3,0.7,0.4,1.2,0.4
		c1.1-0.2,1.4-0.1,2.3,0.5c0.6,0.4,1.6,0.3,2.1-0.3c0.3-0.3,0.1-0.8,0.1-1.2c0-0.5,0.1-0.9,0.2-1.4c0.1-0.7-0.1-1-0.5-1.6
		c-0.3-0.5-0.4-1.1-0.7-1.6c-0.4-0.7-1.3-0.7-2-0.7c-0.7,0-1.8-0.4-2.5,0c-0.7,0.4-1.1,0.9-1.1,1.7c-0.1,1.1-0.9,2-0.1,3.1
		C379.9,239,380.2,239.5,380.6,239.8z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M384.4,229.5c0.6,0.8,1.7,0.3,2.6,0
		c1.5-0.5,5.8,0.2,4.4-2.9c-0.3-0.7-0.7-1.3-0.9-2c-0.4-1.1-0.9-2.7-2.3-3c-1.1-0.3-2.2-0.8-3.3-0.3c-0.8,0.4-1.7,1.3-1.9,2.2
		c-0.1,0.7-0.7,1-0.8,1.8c-0.1,0.6,0.1,2.1,0.5,2.6c0.2,0.3,0.7,0.3,1,0.5c0.3,0.3,0.5,0.7,0.7,1.1
		C384.4,229.4,384.4,229.5,384.4,229.5z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M386.9,235.9c0.3,0.6,0.7,1.4,1.3,1.7
		c0.4,0.2,1.1,0.3,1.6,0.4c0.6,0.1,1.5,0.4,2.1,0.2c0.6-0.2,1.1-1.1,1.4-1.7c0.2-0.4,0.5-0.5,0.6-1c0.1-0.6-0.1-1.2,0-1.8
		c0.5-2.3-1.4-4.2-3.7-4.2c-0.6,0-0.9,0.4-1.4,0.7c-0.5,0.3-1.1,0.4-1.6,0.8c-0.4,0.4-0.5,1.2-0.7,1.7
		C386.1,233.8,386.4,234.9,386.9,235.9z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M364.1,287.6c1.3-0.9,1.1-2.7,0.9-4.1
		c-0.1-0.7,0-1.7-0.4-2.4c-0.6-1.1-2-2.4-3.3-2.3c-1,0.1-1.3,1.1-1.9,1.9c-1.2,1.5-1.1,3.1,0,4.8c0.3,0.4,0.9,0.8,1,1.2
		c0.5,1.2,2.5,1.6,3.5,1.1C363.9,287.7,364,287.6,364.1,287.6z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M369.3,296.6c0.2,0,0.3,0,0.5,0
		c1.2,0.2,2.5,0.5,3.2-0.7c0.7-1.2,0.5-2.1,0.6-3.3c0.1-0.8,0.3-1.7-0.3-2.4c-0.3-0.4-0.9-0.5-1.4-0.7c-0.8-0.4-1.7-0.4-2.6-0.4
		c-0.9,0-1.6-0.4-2.5-0.4c-1.3,0.1-1.4,0.9-2.2,1.6c-0.6,0.6-1.3,1.1-1.4,2c-0.1,1.1,0.8,1.5,1.1,2.4c0.4,1.1,1.2,1.7,2.2,2
		c0.5,0.2,0.9,0.3,1.4,0.3C368.4,296.8,368.9,296.7,369.3,296.6z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M378.1,289.2c0.2-0.2,0.4-0.5,0.7-0.7
		c0.6-0.4,1.2-0.6,1.9-0.8c0.6-0.1,1.3-0.3,1.4-1c0.1-0.5-0.2-1-0.3-1.5c-0.3-1.1-0.4-1.6-1.4-2.2c-0.8-0.5-1.7-0.9-2.7-1
		c-1-0.1-1.7,0.6-2.6,0.7c-0.9,0.1-1.4,0.8-1.4,1.6c0,0.9-0.4,1.6-0.5,2.5c0,0.6,0.3,1.7,0.8,2.1c1.2,1,2.6,1.8,3.9,0.4
		C378,289.3,378.1,289.3,378.1,289.2z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M371.4,287.7c0.8-0.2,0.9-1.2,1-1.9
		c0.1-0.4,0.2-0.8,0.3-1.1c0.2-0.7,0.7-1.5,0.6-2.2c-0.1-0.9-1.2-1-1.9-1.3c-0.4-0.1-0.6-0.2-1-0.1c-1.2,0.2-2.6,0.1-3.7,0.7
		c-1.1,0.6-0.9,1.9-0.9,3c0,0.8-0.4,1.6,0.3,2.2c0.7,0.7,2.3,1.7,3.4,1.5c0.3-0.1,0.5-0.4,0.7-0.5c0.3-0.2,0.7-0.1,1.1-0.1
		C371.3,287.7,371.3,287.7,371.4,287.7z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M370.4,280.5c0.3-0.2,0.7-0.6,0.9-0.9
		c0.4-0.6,0.6-1.3,0.6-2c0.1-1.2-0.4-3.6-1.7-4.1c-0.7-0.3-1.4-0.1-2.1,0c-0.6,0.1-0.8,0.4-1.4,0.7c-0.8,0.4-1.8,0.5-2.5,1.1
		c-1,0.8-1,3.4-0.1,4.2c0.3,0.3,0.7,0.4,1,0.6c0.8,0.5,1.8,0.7,2.7,0.8c0.8,0.1,1.7,0.2,2.4-0.2
		C370.3,280.6,370.3,280.6,370.4,280.5z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M377.5,281.6c0.6-0.1,1.6-0.2,2.1-0.6
		c0.3-0.3,0.7-0.9,0.9-1.3c0.3-0.5,0.9-1.2,0.9-1.8c0-0.6-0.6-1.4-1-1.9c-0.3-0.4-0.3-0.6-0.7-0.9c-0.5-0.3-1.1-0.3-1.6-0.6
		c-2-1.3-4.4-0.2-5.2,1.9c-0.2,0.6,0.1,1,0.1,1.5c0.1,0.6-0.1,1.2,0.2,1.8c0.2,0.5,0.9,0.9,1.4,1.2
		C375.3,281.6,376.4,281.7,377.5,281.6z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M317.4,261.6c0.5,1.2,2.1,1.6,3.2,1.5
		c0.8-0.1,2-0.1,2.7-0.7c0.6-0.5,0.8-1.9,1-2.7c0.3-0.9,0-1.7-0.4-2.5c-0.4-0.9-1-1.2-1.7-1.9c-0.7-0.7-1.4-0.8-2.4-0.6
		c-0.8,0.2-0.9,0.6-1.3,1.3c-0.5,1.1-1.8,1.5-2.3,2.6c-0.2,0.6,0.1,1.1,0.4,1.6C317.1,260.7,317.2,261.1,317.4,261.6z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M308.6,264.4c0.1,0.9,0.8,1.2,1.6,1.3
		c0.8,0.1,1.8,0.7,2.7,0.5c0.5-0.1,0.7-0.5,1.1-0.8c0.4-0.4,0.9-0.6,1.4-0.9c1.2-0.7,0.7-1.8,0.2-2.8c-0.4-0.8-1-2.4-2-2.8
		c-0.8-0.4-3.1-0.5-3.8,0c-1,0.6-0.9,1.5-0.8,2.5C308.9,262.5,308.4,263.4,308.6,264.4z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M302.3,262.1c0.1,0.1,0.2,0.2,0.3,0.3
		c1,1,2.4,0.5,3.6,0.6c1.3,0.1,2.4,0,2.2-1.7c-0.1-0.7,0-1.3,0.2-1.9c0.1-0.6,0.2-1.2-0.4-1.5c-0.4-0.3-0.9-0.4-1.4-0.6
		c-0.4-0.2-0.6-0.6-1.1-0.7c-0.6-0.1-1.4-0.1-1.9,0.1c-0.6,0.2-0.9,0.7-1.4,1C301.1,258.6,301.6,261,302.3,262.1z" style="fill:#7BBCD6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton rods_and_rings" descendants="level 0: [None]" id="rods_and_rings_ton" init-pval="--" min-pval="--" min-pval-children="--" title="Rods and rings">
<path d="M214.9,93.5c-2.5-2.7-7.7-2.5-10.7-0.9
		c-1.8,1-3.2,2.5-3.4,4.6c-0.3,2.1,0.4,5.9,1.6,7.6c1.2,1.6,2.9,2.7,4.9,2.8c2.8,0.3,5.1-0.3,6.7-1.6c1.6-1.3,2.7-3.4,2.9-6.2
		C217.2,97.5,216.4,95.2,214.9,93.5z M209.9,103.3c-1.8,0.4-3.8-0.4-4.6-2.7c-0.7-2.2-0.3-4.6,2.1-5.4c1.9-0.6,4.7,0.4,5.3,2.5
		C213.5,99.9,212.2,102.7,209.9,103.3z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M535.6,273.1c-3-5.9-15.3-1.5-16,4.4
		c-0.3,2.2,0.4,6.3,1.8,8.1c1.2,1.7,5,3.3,7,3.7c2.2,0.4,4.3-0.2,5.9-1.8c3.5-3.5,2.9-8.9,1.8-13.3
		C536,273.9,535.8,273.5,535.6,273.1z M529.4,284.5c-2.1-0.1-4.5-1.6-5-3.4c-0.8-2.4-0.4-5.1,2.3-5.9c2.1-0.6,5.1,0.5,5.9,2.7
		c0.7,2.1,0.7,4.9-1.4,6.2C530.7,284.4,530.1,284.5,529.4,284.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M202,360.9c0.3,0,0.5,0,0.7,0.1
		c0.7,0.3,0.8,1.4,1.2,2c0.9,1.3,4.1,10.1,6.6,8.9c2.8-1.3-0.9-7.4-1.7-8.9c-0.9-1.6-2.2-2.1-3.6-3.3c-1-0.9-1-1.9-2.5-2.1
		c-1.2-0.2-2.9,0.6-4.1,0.8c-1.8,0.3-6.7,0-7.6,2.2c-1.6,4,6,1.8,7.5,1.5C199.2,361.9,200.9,361,202,360.9z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M407.6,390.6c0.3,1,0.4,2,0.5,3
		c0.2,2.1,0,4.2,0.5,6.3c0.5,2,3.1,4.2,5.1,4.4c0.9,0.1,2.4,0.3,2.6-1c0.2-1.1-1-1.9-1.9-2.3c-2-1.1-2.5-3.2-2.7-5.3
		c-0.1-1.3-0.3-2.6-0.6-3.8c-0.2-1.1-0.1-2.3-0.4-3.4c-0.5-2.3-2.1-4.5-4.3-5.3c-1-0.4-2.3-0.6-3.1,0.2c-0.7,0.6-0.6,1.5,0,2.2
		c0.6,0.7,1.7,0.8,2.4,1.4c1,0.8,1.5,2.2,1.8,3.4C407.5,390.5,407.5,390.5,407.6,390.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M409.6,99.9c0.3,0.4,0.8,0.6,1.6,0.5
		c1.2-0.2,1.4-1.2,1.7-2.1c0.6-2,1.8-3.7,3-5.3c0.9-1.2,2.7-3.8,0.7-4.9c-1.5-0.8-3.4,1.8-4.2,2.8C411.8,91.8,408.3,98.1,409.6,99.9
		z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M422.4,101.6c0.3,1,0.8,1.9,1.8,2.1
		c1.2,0.3,2-0.9,2.1-1.9c0.5-3.3-2.2-5.4-3.7-8c-0.8-1.3-1.1-2.4-1.5-3.7c-0.5-1.6-2.2-3-3.2-1c-1.9,3.6,2.4,7.7,3.9,10.7
		C422,100.3,422.2,101,422.4,101.6z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col rods_and_rings" descendants="level 0: [None]" id="rods_and_rings" init-pval="--" min-pval="--" min-pval-children="--" title="Rods and rings">
<path d="M214.8,93.7c-2.5-2.7-7.7-2.5-10.7-0.9
		c-1.8,1-3.2,2.5-3.4,4.6c-0.3,2.1,0.4,5.9,1.6,7.6c1.2,1.6,2.9,2.7,4.9,2.8c2.8,0.3,5.1-0.3,6.7-1.6c1.6-1.3,2.7-3.4,2.9-6.2
		C217.1,97.7,216.4,95.4,214.8,93.7z M209.8,103.5c-1.8,0.4-3.8-0.4-4.6-2.7c-0.7-2.2-0.3-4.6,2.1-5.4c1.9-0.6,4.7,0.4,5.3,2.5
		C213.4,100.1,212.1,102.9,209.8,103.5z" style="fill:#F49739;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M535.6,273.1c-3-5.9-15.3-1.5-16,4.4
		c-0.3,2.2,0.4,6.3,1.8,8.1c1.2,1.7,5,3.3,7,3.7c2.2,0.4,4.3-0.2,5.9-1.8c3.5-3.5,2.9-8.9,1.8-13.3
		C536,273.9,535.8,273.5,535.6,273.1z M529.4,284.5c-2.1-0.1-4.5-1.6-5-3.4c-0.8-2.4-0.4-5.1,2.3-5.9c2.1-0.6,5.1,0.5,5.9,2.7
		c0.7,2.1,0.7,4.9-1.4,6.2C530.7,284.4,530.1,284.5,529.4,284.5z" style="fill:#F49739;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M202,360.9c0.3,0,0.5,0,0.7,0.1
		c0.7,0.3,0.8,1.4,1.2,2c0.9,1.3,4.1,10.1,6.6,8.9c2.8-1.3-0.9-7.4-1.7-8.9c-0.9-1.6-2.2-2.1-3.6-3.3c-1-0.9-1-1.9-2.5-2.1
		c-1.2-0.2-2.9,0.6-4.1,0.8c-1.8,0.3-6.7,0-7.6,2.2c-1.6,4,6,1.8,7.5,1.5C199.2,361.9,200.9,361,202,360.9z" style="fill:#F49739;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M407.6,390.6c0.3,1,0.4,2,0.5,3
		c0.2,2.1,0,4.2,0.5,6.3c0.5,2,3.1,4.2,5.1,4.4c0.9,0.1,2.4,0.3,2.6-1c0.2-1.1-1-1.9-1.9-2.3c-2-1.1-2.5-3.2-2.7-5.3
		c-0.1-1.3-0.3-2.6-0.6-3.8c-0.2-1.1-0.1-2.3-0.4-3.4c-0.5-2.3-2.1-4.5-4.3-5.3c-1-0.4-2.3-0.6-3.1,0.2c-0.7,0.6-0.6,1.5,0,2.2
		c0.6,0.7,1.7,0.8,2.4,1.4c1,0.8,1.5,2.2,1.8,3.4C407.5,390.5,407.5,390.5,407.6,390.6z" style="fill:#F49739;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M409.6,99.9c0.3,0.4,0.8,0.6,1.6,0.5
		c1.2-0.2,1.4-1.2,1.7-2.1c0.6-2,1.8-3.7,3-5.3c0.9-1.2,2.7-3.8,0.7-4.9c-1.5-0.8-3.4,1.8-4.2,2.8C411.8,91.8,408.3,98.1,409.6,99.9
		z" style="fill:#F49739;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M422.4,101.6c0.3,1,0.8,1.9,1.8,2.1
		c1.2,0.3,2-0.9,2.1-1.9c0.5-3.3-2.2-5.4-3.7-8c-0.8-1.3-1.1-2.4-1.5-3.7c-0.5-1.6-2.2-3-3.2-1c-1.9,3.6,2.4,7.7,3.9,10.7
		C422,100.3,422.2,101,422.4,101.6z" style="fill:#F49739;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton mitochondria" descendants="level 0: ['mitochondrion, pval = 0.16438']
level 1: ['mitochondrion-associated adherens complex, pval = --', 'mitochondrial protein catabolic process, pval = --', 'mitochondrial translation, pval = --', 'mitochondrial protein complex, pval = 0.82468', 'mitochondrial RNA localization, pval = --', 'mitochondrial proton-transporting ATP synthase complex assembly, pval = --', 'mitochondrial respiratory chain complex IV assembly, pval = --', 'mitochondrial DNA packaging, pval = --', 'mitochondrial ATP synthesis coupled electron transport, pval = --', 'mitochondrial ATP synthesis coupled proton transport, pval = --', 'mitochondrial depolarization, pval = --', 'Nebenkern, pval = --', 'mitochondrial derivative, pval = --', 'mitochondrial protein processing, pval = --', 'mitochondrial nucleoid organization, pval = --', 'mitochondrial translational elongation, pval = --', 'mitochondrial translational initiation, pval = --', 'mitochondrial translational termination, pval = --', 'mitochondrial matrix, pval = 0.23874', 'maintenance of protein location in mitochondrion, pval = --', 'mitochondrial gene expression, pval = --', 'mitochondrial DNA replication, pval = --', 'mitochondrial envelope, pval = 0.15438', 'mitochondrial intracristal space, pval = --', 'kinetoplast, pval = --', 'mitochondrial calcium ion homeostasis, pval = --', 'mitochondrial DNA metabolic process, pval = --', 'interfibrillar mitochondrion, pval = --', 'subsarcolemmal mitochondrion, pval = --', 'mitochondrial respiratory chain complex III assembly, pval = --', 'mitochondrial respiratory chain complex II assembly, pval = --', 'mitochondrial RNA metabolic process, pval = --', 'ocelloid, pval = --', 'mitochondrial membrane organization, pval = --']
level 2: ['regulation of mitochondrial translation, pval = --', 'tRNA aminoacylation for mitochondrial protein translation, pval = --', 'positive regulation of mitochondrial translation, pval = --', 'negative regulation of mitochondrial translation, pval = --', 'mitochondrial mRNA editing complex, pval = --', 'mitochondrial glutamate synthase complex (NADH), pval = --', 'mitochondrial electron transfer flavoprotein complex, pval = --', 'Ecsit-NDUFAF1 complex, pval = --', 'mitochondrial pyruvate dehydrogenase complex, pval = --', 'mitochondrial fatty acid beta-oxidation multienzyme complex, pval = --', '3-methylcrotonyl-CoA carboxylase complex, mitochondrial, pval = --', 'mitochondrial ribonuclease P complex, pval = --', 'mitochondrial creatine kinase complex, pval = --', 'mitochondrial processing peptidase complex, pval = --', 'mitochondrial permeability transition pore complex, pval = --', 'gamma DNA polymerase complex, pval = --', 'mitochondrial small ribosomal subunit, pval = --', 'mitochondrial large ribosomal subunit, pval = --', 'mitochondrial endopeptidase Clp complex, pval = --', 'inner mitochondrial membrane protein complex, pval = 0.43974', 'mitochondrial pyruvate dehydrogenase (lipoamide) phosphatase complex, pval = --', 'NAGS/NAGK complex, pval = --', 'mitochondrial degradosome, pval = --', 'mitochondrial DNA-directed RNA polymerase complex, pval = --', 'mitochondrial tricarboxylic acid cycle enzyme complex, pval = --', 'outer mitochondrial membrane protein complex, pval = --', 'mitochondrial intermembrane space protein transporter complex, pval = --', 'pole plasm mitochondrial rRNA localization, pval = --', 'mitochondrial electron transport, cytochrome c to oxygen, pval = --', 'mitochondrial electron transport, ubiquinol to cytochrome c, pval = --', 'mitochondrial electron transport, NADH to ubiquinone, pval = --', 'mitochondrial electron transport, succinate to ubiquinone, pval = --', 'negative regulation of mitochondrial ATP synthesis coupled electron transport, pval = --', 'regulation of mitochondrial ATP synthesis coupled electron transport, pval = --', 'positive regulation of mitochondrial ATP synthesis coupled electron transport, pval = --', 'negative regulation of mitochondrial ATP synthesis coupled proton transport, pval = --', 'regulation of mitochondrial ATP synthesis coupled proton transport, pval = --', 'negative regulation of mitochondrial depolarization, pval = --', 'positive regulation of mitochondrial depolarization, pval = --', 'regulation of mitochondrial depolarization, pval = --', 'major mitochondrial derivative, pval = --', 'minor mitochondrial derivative, pval = --', 'protein processing involved in protein targeting to mitochondrion, pval = --', 'mitochondrial nucleoid organization involved in mitochondrial fission, pval = --', 'positive regulation of mitochondrial translational elongation, pval = --', 'negative regulation of mitochondrial translational elongation, pval = --', 'regulation of mitochondrial translational elongation, pval = --', 'positive regulation of mitochondrial translational initiation, pval = --', 'negative regulation of mitochondrial translational initiation, pval = --', 'regulation of mitochondrial translational initiation, pval = --', 'mitochondrial nucleoid, pval = --', 'mitochondrial ribosome, pval = --', 'PAM complex, Tim23 associated import motor, pval = --', 'mitochondrial RNA processing, pval = --', 'mitochondrial transcription, pval = --', 'regulation of mitochondrial gene expression, pval = --', 'negative regulation of mitochondrial DNA replication, pval = --', 'positive regulation of mitochondrial DNA replication, pval = --', 'regulation of mitochondrial DNA replication, pval = --', 'mitochondrial membrane, pval = 0.21246', 'mitochondrial intermembrane space, pval = 0.0846', 'negative regulation of mitochondrial calcium ion concentration, pval = --', 'positive regulation of mitochondrial calcium ion concentration, pval = --', 'mitochondrial DNA catabolic process, pval = --', 'mitochondrial DNA repair, pval = --', 'regulation of mitochondrial DNA metabolic process, pval = --', 'negative regulation of mitochondrial DNA metabolic process, pval = --', 'positive regulation of mitochondrial DNA metabolic process, pval = --', 'mitochondrion DNA recombination, pval = --', 'transcription initiation from mitochondrial promoter, pval = --', 'transcription elongation from mitochondrial promoter, pval = --', 'termination of mitochondrial transcription, pval = --', 'mitochondrial RNA catabolic process, pval = --', 'mitochondrial RNA modification, pval = --', 'regulation of mitochondrial membrane permeability, pval = --', 'establishment of protein localization to mitochondrial membrane, pval = --', 'mitochondrial membrane fusion, pval = --', 'protein insertion into mitochondrial membrane, pval = --', 'outer mitochondrial membrane organization, pval = --', 'inner mitochondrial membrane organization, pval = --']
level 3: ['mitochondrial glutamyl-tRNA aminoacylation, pval = --', 'mitochondrial glutaminyl-tRNA aminoacylation, pval = --', 'mitochondrial cysteinyl-tRNA aminoacylation, pval = --', 'mitochondrial aspartyl-tRNA aminoacylation, pval = --', 'mitochondrial asparaginyl-tRNA aminoacylation, pval = --', 'mitochondrial arginyl-tRNA aminoacylation, pval = --', 'mitochondrial alanyl-tRNA aminoacylation, pval = --', 'mitochondrial threonyl-tRNA aminoacylation, pval = --', 'mitochondrial seryl-tRNA aminoacylation, pval = --', 'mitochondrial prolyl-tRNA aminoacylation, pval = --', 'mitochondrial phenylalanyl-tRNA aminoacylation, pval = --', 'mitochondrial methionyl-tRNA aminoacylation, pval = --', 'mitochondrial lysyl-tRNA aminoacylation, pval = --', 'mitochondrial leucyl-tRNA aminoacylation, pval = --', 'mitochondrial isoleucyl-tRNA aminoacylation, pval = --', 'mitochondrial histidyl-tRNA aminoacylation, pval = --', 'mitochondrial glycyl-tRNA aminoacylation, pval = --', 'mitochondrial valyl-tRNA aminoacylation, pval = --', 'mitochondrial tyrosyl-tRNA aminoacylation, pval = --', 'mitochondrial tryptophanyl-tRNA aminoacylation, pval = --', 'positive regulation of mitochondrial translation in response to stress, pval = --', 'heterotetrameric decaprenyl diphosphate synthase complex, pval = --', 'i-AAA complex, pval = --', 'mitochondrial respiratory chain complex I, peripheral segment, pval = --', 'mitochondrial respiratory chain complex I, membrane segment, pval = --', 'MIB complex, pval = --', 'mitochondrial respiratory chain complex III, pval = --', 'm-AAA complex, pval = --', 'TIM23 mitochondrial import inner membrane translocase complex, pval = --', 'mitochondrial respiratory chain complex II, succinate dehydrogenase complex (ubiquinone), pval = --', 'mitochondrial respiratory chain complex I, pval = --', 'mitochondrial proton-transporting ATP synthase, central stalk, pval = --', 'mitochondrial proton-transporting ATP synthase complex, pval = --', 'mitochondrial proton-transporting ATP synthase, catalytic core, pval = --', 'mitochondrial respiratory chain complex IV, pval = --', 'INA complex, pval = --', 'mitochondrial inner membrane assembly complex, pval = --', 'mitochondrial proton-transporting ATP synthase, stator stalk, pval = --', 'mitochondrial proton-transporting ATP synthase complex, catalytic core F(1), pval = --', 'mitochondrial proton-transporting ATP synthase complex, coupling factor F(o), pval = --', 'MICOS complex, pval = --', 'mitochondrial respiratory chain complex IV pre-assembly complex, pval = --', 'mitochondrial prohibitin complex, pval = --', 'uniplex complex, pval = --', 'mitochondrial inner membrane peptidase complex, pval = --', 'TIM22 mitochondrial import inner membrane insertion complex, pval = 0.43974', 'mitochondrial alpha-ketoglutarate dehydrogenase complex, pval = --', 'mitochondrial isocitrate dehydrogenase complex (NAD+), pval = --', 'succinate-CoA ligase complex (GDP-forming), pval = --', 'mitochondrial outer membrane translocase complex, pval = --', 'SAM complex, pval = --', 'Cdc48p-Npl4p-Vms1p AAA ATPase complex, pval = --', 'ERMES complex, pval = --', 'pole plasm mitochondrial srRNA localization, pval = --', 'pole plasm mitochondrial lrRNA localization, pval = --', 'negative regulation of mitochondrial electron transport, NADH to ubiquinone, pval = --', 'regulation of mitochondrial electron transport, NADH to ubiquinone, pval = --', 'positive regulation of mitochondrial electron transport, NADH to ubiquinone, pval = --', 'negative regulation of protein processing involved in protein targeting to mitochondrion, pval = --', 'regulation of protein processing involved in protein targeting to mitochondrion, pval = --', 'mitochondrial chromosome, pval = --', &quot;mitochondrial tRNA 3'-trailer cleavage, endonucleolytic, pval = --&quot;, 'mitochondrial mRNA processing, pval = --', 'mitochondrial tRNA processing, pval = --', &quot;mitochondrial RNA 5'-end processing, pval = --&quot;, &quot;mitochondrial RNA 3'-end processing, pval = --&quot;, 'mitochondrial single subunit type RNA polymerase activity, pval = --', 'regulation of mitochondrial transcription, pval = --', 'positive regulation of mitochondrial transcription, pval = --', 'mitochondrial rRNA transcription, pval = --', 'mitochondrial sequence-specific DNA-binding transcription factor activity, pval = --', 'contact site, pval = --', 'bridge contact site, pval = --', 'mitochondrial membrane scission site, pval = --', 'mitochondrial outer membrane, pval = 0.32699', 'mitochondrial inner membrane, pval = 0.49914', 'intrinsic component of mitochondrial membrane, pval = 0.40992', 'mitochondrial RNA polymerase termination site sequence-specific DNA binding, pval = --', 'mitochondrial RNA surveillance, pval = --', 'regulation of mitochondrial RNA catabolic process, pval = --', 'positive regulation of mitochondrial RNA catabolic process, pval = --', 'negative regulation of mitochondrial RNA catabolic process, pval = --', 'mitochondrial mRNA catabolic process, pval = --', 'mitochondrial tRNA modification, pval = --', 'mitochondrial mRNA modification, pval = --', 'negative regulation of mitochondrial membrane permeability, pval = --', 'positive regulation of mitochondrial membrane permeability, pval = --', 'regulation of mitochondrial membrane permeability involved in programmed necrotic cell death, pval = --', 'regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway, pval = --', 'regulation of mitochondrial membrane permeability involved in apoptotic process, pval = --', 'protein insertion into mitochondrial inner membrane, pval = --', 'protein insertion into mitochondrial outer membrane, pval = --', 'establishment of protein localization to mitochondrial membrane involved in mitochondrial fission, pval = --', 'mitochondrial inner membrane fusion, pval = --', 'mitochondrial outer membrane fusion, pval = --', 'protein insertion into mitochondrial membrane involved in apoptotic signaling pathway, pval = --', 'mitochondrial outer membrane translocase complex assembly, pval = --', 'cristae formation, pval = --']
level 4: ['mitochondrial oxoglutarate dehydrogenase complex, pval = --', 'mitochondrial polycistronic RNA processing, pval = --', &quot;mitochondrial mRNA 5'-end processing, pval = --&quot;, &quot;mitochondrial mRNA 3'-end processing, pval = --&quot;, &quot;mitochondrial tRNA 3'-end processing, pval = --&quot;, &quot;mitochondrial tRNA 5'-end processing, pval = --&quot;, 'cytoplasmic side of mitochondrial outer membrane, pval = --', 'intrinsic component of mitochondrial outer membrane, pval = 0.43974', 'extrinsic component of mitochondrial outer membrane, pval = --', 'mitochondrial crista, pval = --', 'matrix side of mitochondrial inner membrane, pval = --', 'mitochondrial inner boundary membrane, pval = --', 'mitochondrial crista junction, pval = --', 'mitochondrial respirasome, pval = --', 'intrinsic component of mitochondrial inner membrane, pval = 0.43974', 'extrinsic component of mitochondrial inner membrane, pval = --', 'integral component of mitochondrial membrane, pval = 0.40992', 'mitochondrial mRNA surveillance, pval = --', 'mitochondrial ncRNA surveillance, pval = --', 'regulation of mitochondrial mRNA catabolic process, pval = --', 'positive regulation of mitochondrial mRNA catabolic process, pval = --', 'negative regulation of mitochondrial mRNA catabolic process, pval = --', 'mitochondrial tRNA thio-modification, pval = --', 'mitochondrial tRNA pseudouridine synthesis, pval = --', 'mitochondrial tRNA methylation, pval = --', 'mitochondrial tRNA threonylcarbamoyladenosine modification, pval = --', 'mitochondrial tRNA wobble uridine modification, pval = --', 'negative regulation of mitochondrial membrane permeability involved in apoptotic process, pval = --', 'mitochondrial outer membrane permeabilization involved in programmed cell death, pval = --', 'regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway, pval = --', 'positive regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway, pval = --', 'negative regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway, pval = --', 'positive regulation of mitochondrial membrane permeability involved in apoptotic process, pval = --', 'protein insertion into mitochondrial inner membrane from matrix, pval = --', 'negative regulation of protein import into mitochondrial outer membrane, pval = --', 'positive regulation of protein import into mitochondrial outer membrane, pval = --', 'regulation of protein import into mitochondrial outer membrane, pval = --', 'positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway, pval = --', 'negative regulation of cristae formation, pval = --', 'positive regulation of cristae formation, pval = --', 'regulation of cristae formation, pval = --']
level 5: ['mitochondrial mRNA polyadenylation, pval = --', 'integral component of mitochondrial outer membrane, pval = 0.43974', 'extrinsic component of matrix side of mitochondrial inner membrane, pval = --', 'integral component of mitochondrial inner membrane, pval = 0.43974', 'mitochondrial tRNA wobble position uridine thiolation, pval = --', 'mitochondrial outer membrane permeabilization, pval = --']" id="mitochondria_ton" init-pval="0.16438" min-pval="0.0846" min-pval-children="0.0846" title="Mitochondria">
<path d="M281.3,410.3c-2.6,5.5-13.8,2.8-23.1-3
		c-12.2-7.7,0.2-26.3,11-15.6C277.5,399.9,283.6,405.3,281.3,410.3z" style="fill:#F6F6F6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M463.6,106c13.9-0.7,23.1,0.5,32.1,17.7
		c5.9,11.2-7,14.4-10.6,7.1c-5.2-10.5-9.4-13.4-21.9-12.6C454.7,118.8,452.5,106.5,463.6,106z" style="fill:#F6F6F6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495,340.9c-11,17,0.4,13.3-11.7,30.9
		c-6.5,9.5-16.4,0.6-12.6-6.1c10.6-18.6,0.8-16.5,12.6-32.5C488.4,326.1,500,333.3,495,340.9z" style="fill:#F6F6F6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M132.5,351.2c11.1,0.8,14.3,1,22.2-5.3
		c7.1-5.7,12.3,3.3,6.4,9.4c-6.8,7.1-16,11.7-27.4,8.8C123,361.3,122.8,350.5,132.5,351.2z" style="fill:#F6F6F6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M324,73.1c5.8-12,16.3-4.1,19.1,2.7
		c2.8,6.8,0.4,12.7-7.8,13C326,89,318.8,83.8,324,73.1z" style="fill:#F6F6F6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M483.2,333.2c-11.8,16.1-2,13.9-12.6,32.5
		c-3.8,6.6,6.1,15.6,12.6,6.1c12.1-17.6,0.7-13.9,11.7-30.9C500,333.3,488.4,326.1,483.2,333.2z M478,353.2
		c-0.1-0.7,0.1-1.4,0.3-2.1c0.2-0.8,0.3-1.7,0.5-2.5c0.2-0.8,0.7-1.4,1.7-1.3c0.4,0.1,0.7,0.2,1.1,0.3c0.9,0.2,1.8,0.5,2.8,0.7
		c0.9,0.2,2.8,0.4,2.8-1c0-0.8-0.8-1.2-1.4-1.4c-0.8-0.3-1.5-0.6-2.3-0.9c-1.3-0.5-3.2-1.2-3-2.9c0-0.5,0.3-1.1,0.6-1.5
		c0.5-0.7,1.3-1.2,2.1-1c0.7,0.2,1.3,0.7,2,1c0.5,0.3,1.1,0.1,1.4-0.4c0.3-0.5,0.1-1.2-0.4-1.4c-0.1-0.1-0.3-0.2-0.4-0.2
		c-0.8-0.4-2.4-1-1.9-2.2c0.3-0.7,1-1.1,1.7-0.9c0.7,0.2,1.2,0.8,1.9,1.2c0.5,0.3,1.1,0.1,1.4-0.3c0.3-0.5,0.1-1.1-0.3-1.4
		c-0.4-0.3-0.9-0.6-1.3-0.9c-0.6-0.4-0.6-0.9,0-1.4c0.3-0.2,0.7-0.3,1.1-0.3c1.5-0.2,3.1,0.3,4.3,1.3c0.5,0.4,0.9,0.9,1.2,1.5
		c0.2,0.3,0.3,0.7,0.4,1c0.2,0.8,0.3,1.7-0.3,2.4c-0.6,0.6-1.5,0-2.1-0.2c-0.3-0.1-0.9-0.1-1.1,0.2c-0.5,0.4-0.5,1.1,0,1.5
		c0.4,0.3,1,0.6,1.2,1c0.2,0.4,0.2,0.9-0.1,1.3c-0.2,0.4-0.5,0.8-0.9,1c-0.9,0.5-2-0.3-2.8-0.6c-0.5-0.2-1.2,0-1.4,0.5
		c-0.2,0.4-0.1,0.9,0.3,1.2c0.1,0.1,0.2,0.1,0.2,0.1c0.7,0.3,1.5,0.7,1.8,1.4c0.3,0.7,0.1,1.5-0.1,2.2c-0.2,0.7-0.4,1.5-1.1,1.9
		c-0.5,0.3-1.2,0.4-1.9,0.3c-0.6-0.1-1.2-0.2-1.7-0.3c-0.6-0.1-1.1,0.3-1.2,0.9c-0.1,0.4,0.1,0.8,0.4,1c0.3,0.2,0.7,0.3,1.1,0.3
		c0.5,0.1,0.9,0.1,1.4,0.2c0.5,0,1.1,0,1.5,0.4c0.4,0.4,0.5,1.1,0.3,1.6c-0.6,1.3-2.4,0.1-2.9,1.3c-0.2,0.4-0.1,0.8,0.1,1.2
		c0.2,0.3,0.6,0.4,1,0.5c0.4,0,0.8,0.1,1.2,0.3c1,0.6,0.5,2.3-0.5,2.6c-0.7,0.2-1.4-0.2-2-0.4c-0.5-0.2-1.1,0.1-1.4,0.6
		c-0.2,0.4,0,0.9,0.3,1.2c0.6,0.5,1.7,0.3,2.1,1c0.3,0.5,0.1,1.1-0.3,1.5c-0.6,0.4-1.3,0.3-1.9,0.1c-0.6-0.2-1.4-0.6-1.9,0.1
		c-0.7,1,0.4,1.5,1.1,1.9c0.3,0.2,0.7,0.5,0.8,0.9c0.2,0.6-0.2,1.3-0.5,1.8c-0.4,0.7-1,1.3-1.7,1.7c-0.7,0.4-1.6,0-2.2-0.4
		c-0.3-0.2-0.7-0.5-1-0.7c-0.5-0.3-1.1-0.2-1.5,0.3c-0.2,0.2-0.2,0.6-0.1,0.9c0.3,0.9,2.6,1.1,2.1,2.4c-0.5,1.3-2.5,0.9-3.4,0.4
		c-0.4-0.2-0.8-0.4-1.2-0.7c-0.4-0.4-0.8-0.8-1.1-1.3c-0.3-0.4-0.5-0.9-0.6-1.3c-0.5-1.4-0.3-3.1,1.2-3.9c0.7-0.4,1.4,0.1,2,0.4
		c0.9,0.4,1.7,0.9,2.5,1.5c0.5,0.3,1.1,0.2,1.5-0.3c0.3-0.5,0.2-1.1-0.3-1.5c-0.3-0.2-0.6-0.4-0.9-0.5c-0.7-0.4-1.3-0.7-2-1.1
		c-0.6-0.3-1.2-0.6-1.5-1.2c-0.3-0.5-0.1-1.2,0.3-1.6c0.5-0.6,1.2-0.7,1.9-0.4c0.6,0.2,1.1,0.4,1.7,0.7c0.5,0.3,1.1,0,1.4-0.5
		c0.3-0.5,0-1.2-0.5-1.4c-0.3-0.2-0.7-0.3-1-0.4c-0.6-0.3-1.5-0.5-1.7-1.3c-0.1-0.4,0.2-0.9,0.6-1.1c0.4-0.3,1-0.2,1.4-0.1
		c0.5,0.1,1.1,0.3,1.6,0.5c1.3,0.5,2-1.5,0.8-2c-1-0.4-1.9-0.9-2.5-1.7C478.2,354.1,478.1,353.7,478,353.2z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.7,345.9c-7.9,6.3-11.1,6.1-22.2,5.3
		c-9.6-0.7-9.4,10.1,1.2,12.9c11.3,2.9,20.6-1.7,27.4-8.8C167,349.1,161.7,340.2,154.7,345.9z M161.3,349.1c-0.4,0.1-1-0.5-1.2-0.8
		c-0.3-0.3-0.7-0.4-1.1-0.2c-0.2,0.1-0.3,0.2-0.4,0.3c-0.3,0.4-0.3,0.9-0.1,1.3c0.2,0.5,0.7,0.8,1,1.3c0.4,0.6,0.6,1.3,0.5,1.9
		c-0.2,1.1-1.4,2.2-2.4,2.7c-0.3,0.2-0.7,0.2-1.1,0.1c-0.7-0.2-1.1-0.8-1.6-1.3c-0.5-0.6-1.2-1.3-2-1c-0.1,0-0.1,0.1-0.2,0.1
		c-1.1,0.7-0.2,1.9,0.3,2.6c0.5,0.7,0.9,1.6,0.6,2.5c-0.2,0.6-0.8,1-1.4,1.3c-0.7,0.4-1.4,0.7-2.1,1c-0.8,0.5-2.2,1.2-2.8,0
		c-0.2-0.4-0.3-0.9-0.4-1.3c-0.2-0.6-0.4-1.5-1.2-1.4c-1.4,0.1-1.2,1.8-0.8,2.7c0.2,0.5,0.3,1.2-0.2,1.5c-0.4,0.2-1,0.2-1.4,0
		c-0.9-0.5-0.6-2-0.7-2.9c-0.1-0.6-0.7-1-1.3-0.8c-0.1,0-0.1,0-0.2,0.1c-1,0.4-0.6,1.3-0.6,2.1c0.1,1.1-0.7,2.1-1.8,2.2
		c-0.7,0-1.4-0.1-2.1-0.3c-1.6-0.4-2.3-2.3-1.7-3.8c0,0,0.1-0.6,0.1-0.6c0.2-0.6-0.2-1.2-0.8-1.3c-0.2-0.1-0.5,0-0.7,0
		c-0.3,0.1-0.6,0.4-0.7,0.7l-0.1,0.6c-0.2,0.8-0.2,2-1.1,2.3c-0.5,0.2-1.1,0-1.6-0.3c-0.7-0.5-1.4-1.3-1.9-2c-0.4-0.6-0.6-1.3-0.7-2
		c0-0.6,0.1-1.2,0.4-1.6c0.3-0.4,0.8-0.7,1.3-0.4c0.5,0.4,0.5,1.1,0.5,1.6c0,0,0,1,1,1.2c1.2,0.2,1.4-1.5,1.7-2.2
		c0.2-0.4,0.5-0.9,0.8-1.2c0.4-0.4,1-0.5,1.6-0.5c0.7,0,1.3,0.4,1.6,1c0.4,0.7,0.3,1.5,0.3,2.3c0,0.7,0,1.6,0.7,2.1
		c0.5,0.3,1.3,0.3,1.7-0.1c0.4-0.4,0.5-1,0.6-1.6c0.1-0.9,0.2-1.8,1-2.4c0.7-0.5,1.5-0.7,2.3-0.8c1-0.2,2.4-0.3,3.3,0.4
		c0.5,0.4,0.9,1,1.2,1.6c0.4,0.7,0.8,1.5,1.1,2.3c0.2,0.6,0.3,1.5,1.1,1.7c0.6,0.2,1.2-0.3,1.3-0.9c0.2-0.7-0.2-1.4-0.4-2
		c-0.3-0.7-0.6-1.4-0.9-2.1c-0.4-1-0.6-2,0.2-2.8c0.4-0.5,1-0.9,1.6-1.1c0.3-0.1,0.6-0.2,1-0.3c1.3-0.2,2.2,0.8,2.9,1.7
		c0.4,0.5,1.1,0.5,1.5,0.2c0.5-0.4,0.5-1.1,0.2-1.5c-0.4-0.5-1.3-1.2-1.4-1.9c-0.1-1.3,1.2-2.2,2.2-2.6c1.7-0.7,2.7,0.1,3.2,0.8
		C161.2,347.5,162,348.9,161.3,349.1z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M335.3,88.7c8.2-0.3,10.6-6.2,7.8-13
		c-2.8-6.8-13.3-14.6-19.1-2.7C318.8,83.8,326,89,335.3,88.7z M324.5,80c-0.1-1.8,0.6-3.6,1.2-5.3c0.3-0.8,0.7-1.6,1.2-2.4
		c0.5-0.7,1.1-1.6,1.8-2.1c0.4-0.3,1.3-0.7,1.7-0.1c0.3,0.4,0.3,1,0.1,1.5c-0.2,0.7-0.6,1.2-0.9,1.9c-0.4,1.1-1,2.2-1.4,3.4
		c-0.2,0.6,0.1,1.3,0.8,1.5c0.1,0,0.3,0.1,0.4,0.1c0.5,0,1-0.3,1.1-0.8c0.2-0.7,0.5-1.4,0.9-2.1c0.6-1.1,1.2-2.3,1.9-3.4
		c0.3-0.4,0.6-0.8,1.1-0.9c0.8-0.3,1.7,0,2.2,0.7c0.8,1.2-0.2,2.6-0.8,3.6c-0.6,0.9-1.1,1.8-1.4,2.3c-0.3,0.6-0.1,1.3,0.4,1.6
		c0.2,0.1,0.4,0.2,0.6,0.2c0.4,0,0.8-0.2,1-0.6c0.5-0.8,1-1.6,1.4-2.4c0.3-0.6,0.6-1.4,1.5-1.4c0.8,0,1.3,0.7,1.6,1.4
		c1.2,2.9,1.2,5.6,0.1,7.3c-0.3,0.5-0.9,1.5-1.6,1.4c-1.2-0.2-0.3-2-0.4-2.7c-0.1-0.4-0.4-0.7-0.8-0.9c-0.6-0.2-1.3,0.1-1.5,0.8
		c-0.2,0.7-0.5,1.4-0.8,2c-0.4,0.8-1,1.4-1.9,1.4c-0.4,0-0.9-0.1-1.2-0.4c-0.2-0.2-0.5-0.8-0.5-1.2c0.1-1.3,1.1-2.8-0.2-3.8
		c-0.2-0.2-0.6-0.3-0.9-0.3c-1,0.2-1.4,1.7-2,2.5c-0.9,1.3-2.6,1-3.6,0c0,0-0.1-0.1-0.1-0.1C324.8,81.9,324.5,80.9,324.5,80z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M258.2,407.3c9.3,5.9,20.6,8.6,23.1,3
		c2.3-5-3.7-10.4-12.1-18.6C258.4,381,246,399.6,258.2,407.3z M258,404.6c-0.7-0.3-0.3-1.2,0-1.7c0.6-1,1.3-1.9,1.9-2.9
		c0.3-0.5,0.2-1.2-0.4-1.5c-0.5-0.3-1.2-0.2-1.5,0.4c-0.3,0.5-1.5,3.1-2.4,2.1c-0.4-0.4-0.4-1.1-0.4-1.6c0-0.7-0.1-1.4,0.1-2.1
		c0.1-0.3,0.1-0.6,0.2-1c0.3-1.1,0.9-2.2,1.6-3.1c0.8-1,1.9-1.7,3.1-2.2c0.3-0.1,1.4-0.5,1.3,0.2c-0.1,0.5-0.7,1-1,1.4
		c-0.5,0.6-1.3,1.4-1.2,2.2c0.1,0.6,0.8,1.1,1.4,0.8c0.6-0.2,0.9-1,1.2-1.5c0.5-0.8,1.2-1.5,1.8-2.2c0.3-0.4,0.7-0.6,1.1-0.6
		c0.6,0,1.3,0.3,1.7,0.9c0.2,0.3,0.2,0.7,0.1,1c-0.2,0.7-0.8,1.2-1.3,1.7c-0.5,0.5-1.2,1.1-0.8,1.8c0.3,0.7,1.3,0.9,1.8,0.3
		c0.6-0.7,1.2-1.6,2-2c0.7-0.4,1.4,0.1,1.8,0.8c0.8,1.3-0.8,2.3-1.5,3.2c-0.5,0.6-0.9,1.2-1.3,1.8c-0.7,1-1.4,2-1.9,3
		c-0.3,0.6,0,1.2,0.5,1.5c0.2,0.1,0.3,0.1,0.5,0.1c0.4,0,0.8-0.2,1-0.6c0.2-0.5,0.5-0.9,0.7-1.3c1-1.4,1.9-2.9,3.1-4.1
		c0.4-0.4,0.7-0.8,1.3-0.8c0.5,0,0.9,0.3,1.1,0.7c0.5,1-0.2,2-0.7,2.9c-0.3,0.4-0.7,0.9-0.8,1.4c-0.2,0.6,0.1,1.2,0.7,1.4
		c0.1,0,0.2,0.1,0.3,0.1c0.7,0,1.1-0.8,1.4-1.3c0.3-0.5,0.9-1.5,1.6-1.5c0.4,0,0.7,0.3,1,0.6c0.4,0.4,0.5,1.1,0.5,1.6
		c-0.1,0.6-0.6,0.9-0.9,1.4c-0.5,0.7-1,1.4-1.3,2.2c-0.2,0.6,0,1.2,0.6,1.5c0.1,0.1,0.3,0.1,0.4,0.1c0.7,0,0.9-0.6,1.2-1.1
		c0.2-0.5,0.7-1.3,1.3-1.3c0.7-0.1,0.9,1,0.8,1.5c-0.2,2.1-2.9,2.7-4.6,2.7c-0.6,0-1.2-0.1-1.9-0.1c-0.6-0.1-1.2,0-1.7-0.4
		c-0.7-0.4-0.4-1.1-0.5-1.8c-0.2-0.7-0.8-1.2-1.6-0.9c-0.6,0.3-0.9,1.1-1.6,1.3c-0.6,0.2-1.3,0-1.9-0.2c-0.8-0.4-1.6-0.8-2.4-1.1
		c-0.8-0.4-2-0.7-2.2-1.7c-0.2-0.8,0.5-1.5,1-2.1c0.4-0.4,0.4-1.1,0-1.6c-0.4-0.4-1.1-0.4-1.6,0c-0.5,0.5-0.8,1.1-1.2,1.5
		c-0.2,0.2-0.5,0.4-0.9,0.5C258.5,404.7,258.3,404.7,258,404.6z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495.7,123.6c-9.1-17.2-18.3-18.4-32.1-17.7
		c-11,0.5-8.9,12.8-0.4,12.3c12.5-0.8,16.7,2,21.9,12.6C488.7,138,501.6,134.9,495.7,123.6z M491.5,132.8c-0.9,0.1-1.8-0.2-2.6-0.6
		c-0.4-0.2-0.8-0.5-1.1-0.8c-0.5-0.6-0.7-1.2-0.1-1.7c0.4-0.4,1-0.5,1.4-0.8c0.5-0.3,0.7-0.8,0.4-1.4c-0.3-0.5-0.9-0.5-1.4-0.3
		c-0.7,0.3-1.6,1-2.4,0.5c-0.8-0.4-0.3-1.5,0.1-2c0.5-0.6,1.2-0.9,1.7-1.4c0.3-0.2,0.5-0.6,0.4-1c0-0.1-0.1-0.3-0.2-0.4
		c-0.4-0.5-1.1-0.4-1.6,0c-0.6,0.4-1.2,1-1.9,1.3c-0.5,0.2-1-0.1-1.2-0.6c-0.2-0.5,0.1-1.1,0.4-1.6c0.3-0.4,0.7-0.7,0.6-1.3
		c0-0.2-0.1-0.3-0.3-0.4c-0.9-0.8-1.6,0.4-2.4,0.6c-1,0.3-1.1-0.9-1-1.6c0.1-0.5,0.5-1.1,0.4-1.6c-0.2-0.9-1.2-0.8-1.8-0.4
		c-0.5,0.4-1,0.7-1.7,0.7c-0.6,0-1.1-0.3-1.7-0.5c-0.8-0.3-1.6-0.6-2.4-0.7c-0.6-0.1-1.1-0.2-1.7-0.3c-0.5-0.1-1.1-0.1-1.5-0.4
		c-0.4-0.4-0.2-0.9-0.3-1.4c-0.1-0.4-0.5-0.8-1-0.7c-0.3,0-0.7,0.3-0.9,0.6c-0.2,0.4-0.2,1-0.5,1.4c-0.3,0.3-0.8,0.4-1.2,0.4
		c-0.5-0.1-0.9-0.3-1.1-0.7c-0.4-0.8-0.1-3.1-1.6-2.7c-0.5,0.1-0.9,0.7-0.7,1.2c0.2,0.8,0.5,2-0.7,2.1c-1.2,0.1-2.5-0.6-3.2-1.5
		c-0.4-0.5-0.6-1.1-0.8-1.7c-0.2-0.7-0.2-1.5,0-2.2c0.1-0.4,0.5-1.3,1.2-1.1c0.5,0.2,0.5,1.1,0.5,1.6c0,0.5,0.5,0.9,1.1,0.9
		c0.6,0,1-0.5,1-1c0-0.8-0.4-1.7-0.2-2.5c0.2-1,1.9-1.1,2.6-0.7c0.9,0.5,0.6,1.8,1,2.6c0.4,1,2,0.6,1.9-0.5c0-0.6-0.2-1.4,0-2
		c0.1-0.3,0.3-0.5,0.6-0.5c0.4-0.1,0.9,0,1.2,0.3c0.3,0.4,0.1,1,0,1.4c-0.1,0.5,0.4,1,0.9,1c1.1,0.1,1-1.1,1.4-1.7
		c0.2-0.3,0.5-0.5,0.9-0.6c0.6-0.2,1.4-0.2,1.7,0.5c0.3,0.7,0.1,1.5,0,2.3c-0.1,0.5-0.2,1-0.2,1.4c-0.1,0.5,0.3,1,0.8,1.1
		c0.6,0.1,1.1-0.3,1.2-0.8c0.2-0.9,0.2-2,0.6-2.9c0.4-0.7,1.3-0.9,2.1-0.8c1.3,0.2,0.7,1.9,0.4,2.7c-0.3,0.8-0.6,1.5-0.8,2.3
		c-0.1,0.3-0.2,0.5-0.3,0.8c-0.2,0.5,0.1,1.1,0.7,1.2c0.8,0.2,1.2-0.4,1.4-1c0.3-0.8,0.5-1.6,0.7-2.4c0.2-0.6,0.4-1.4,1.1-1.6
		c0.4-0.1,0.9-0.1,1.3,0c0.5,0.2,1.1,0.6,1,1.2c-0.1,1-2.1,2.7-0.7,3.5c0.3,0.2,0.8,0.2,1.1,0c0.6-0.4,0.9-1.3,1.8-1.4
		c0.8-0.1,1.5,0.6,1.7,1.3c0.4,1.2-1.3,2-1.4,3.1c-0.1,1,1.1,1.7,1.8,0.8c0.4-0.5,1-1.2,1.7-1.1c0.6,0.1,1,0.5,1.3,1
		c0.3,0.5,0.4,1,0.2,1.6c-0.3,0.8-1.9,1.4-1.4,2.4c0.2,0.5,0.8,0.7,1.3,0.5c0.6-0.3,1.1-0.6,1.8-0.3c1.1,0.4,1.5,2.2,0.3,2.8
		c-0.6,0.3-1.2,0.4-1.1,1.2c0.1,0.5,0.6,0.9,1.1,0.8c0.7-0.1,1.5-0.4,1.9,0.4c0.2,0.4,0.2,1.1,0.1,1.6c-0.3,1.1-1.2,1.9-2.2,2.4
		C492.5,132.7,492,132.8,491.5,132.8z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z" style="fill:#F6F6F6;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z M143.7,244.4c-0.1,0-0.3,0-0.5,0c-1.3,0.3-1.5,1.3-0.6,2.2
		c1,1.1-0.4,3.7-1.9,2.9c-0.6-0.3-0.9-0.8-1.4-1.2c-0.5-0.4-1.1-0.5-1.6,0.2c-0.5,0.5-0.2,1.1,0.2,1.6c0.6,0.6,1.6,1.3,1.4,2.3
		c-0.2,1-1.4,0.9-2.2,0.7c-0.8-0.2-1.4-0.8-2.1-1.3c-0.4-0.2-0.9-0.2-1.2,0c-0.1,0.1-0.2,0.2-0.3,0.3c-0.4,0.7,0,1.3,0.5,1.7
		c0.7,0.5,1.5,0.8,2,1.5c0.4,0.5,0.2,1.1-0.2,1.4c-0.5,0.4-1.2,0.4-1.8,0.2c-0.6-0.2-1-0.5-1.6-0.1c-0.2,0.1-0.3,0.3-0.4,0.4
		c-0.6,1.2,1,1.5,1.5,2.3c0.7,1-0.6,1.5-1.4,1.6c-0.6,0.1-1.3-0.1-1.8,0.2c-0.8,0.6-0.5,1.5,0.2,2c0.6,0.4,1.1,0.8,1.3,1.5
		c0.2,0.6,0.1,1.3,0,2c0,0.9,0,1.9,0.1,2.8c0.1,0.6,0.2,1.2,0.2,1.9c0.1,0.5,0.3,1.2,0.1,1.7c-0.2,0.6-0.9,0.6-1.4,0.9
		c-0.4,0.3-0.6,0.9-0.4,1.4c0.1,0.3,0.6,0.6,0.9,0.7c0.5,0,1.1-0.2,1.6,0c0.5,0.2,0.7,0.7,0.8,1.1c0.1,0.5,0,1.1-0.4,1.4
		c-0.7,0.8-3.3,1.3-2.3,2.8c0.3,0.5,1,0.7,1.5,0.3c0.8-0.5,1.9-1.3,2.5-0.1c0.6,1.2,0.2,2.8-0.4,3.9c-0.4,0.6-0.9,1.1-1.5,1.5
		c-0.7,0.4-1.5,0.8-2.4,0.8c-0.5,0-1.6,0-1.6-0.8c0-0.6,1-0.9,1.5-1.1c0.6-0.2,0.8-0.9,0.6-1.5c-0.2-0.6-0.9-0.8-1.5-0.6
		c-0.9,0.4-1.7,1.1-2.6,1.2c-1.2,0.2-1.9-1.5-1.7-2.5c0.2-1.2,1.7-1.3,2.4-2c0.9-0.8-0.1-2.3-1.2-1.8c-0.7,0.3-1.4,0.8-2.1,0.7
		c-0.3,0-0.6-0.1-0.8-0.4c-0.2-0.4-0.4-1-0.1-1.3c0.3-0.5,1-0.4,1.5-0.5c0.6-0.1,0.9-0.8,0.8-1.3c-0.3-1.2-1.5-0.6-2.3-0.8
		c-0.4-0.1-0.7-0.4-1-0.7c-0.5-0.6-0.7-1.4-0.1-1.9c0.7-0.6,1.6-0.7,2.4-0.9c0.6-0.1,1.1-0.2,1.5-0.3c0.6-0.1,1-0.7,0.9-1.3
		c-0.1-0.6-0.7-1-1.3-0.9c-1,0.2-2.2,0.6-3.3,0.5c-0.9-0.1-1.5-1-1.6-1.9c-0.2-1.4,1.8-1.5,2.8-1.5c0.9,0,1.8,0,2.8,0
		c0.3,0,0.6,0,0.9,0c0.6,0,1.1-0.6,1-1.2c-0.1-0.9-0.8-1.1-1.6-1.1c-0.9,0-1.9,0.1-2.8,0.2c-0.7,0-1.7,0.2-2.1-0.5
		c-0.2-0.3-0.4-0.9-0.4-1.3c0-0.6,0.3-1.3,0.9-1.5c1.1-0.3,3.6,1.1,4-0.7c0.1-0.4-0.1-0.9-0.4-1.2c-0.7-0.5-1.7-0.5-2.2-1.3
		c-0.4-0.8,0.1-1.8,0.8-2.3c1.2-0.8,2.5,0.5,3.8,0.3c1.1-0.2,1.4-1.8,0.2-2.2c-0.7-0.2-1.6-0.5-1.8-1.3c-0.1-0.6,0.2-1.3,0.6-1.8
		c0.4-0.5,1-0.9,1.6-0.9c1,0,2.1,1.5,3.1,0.5c0.4-0.4,0.5-1.1,0.1-1.5c-0.5-0.5-1-1-1-1.8c0.1-1.4,1.8-2.4,2.9-1.4
		c0.7,0.7,2.3,1.3,2.6-0.2c0.1-1-1.3-1.7-0.7-2.8c1.2-2.3,5.5,2,5.4-1c0-0.9-1-1.7-0.9-2.6c0.2-1.9,3.3-0.6,4.1,0
		c0.8,0.6,1.4,1.6,1.5,2.7C147.3,243.5,145.6,244.4,143.7,244.4z" style="fill:#DADADA;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col mitochondria" descendants="level 0: ['mitochondrion, pval = 0.16438']
level 1: ['mitochondrion-associated adherens complex, pval = --', 'mitochondrial protein catabolic process, pval = --', 'mitochondrial translation, pval = --', 'mitochondrial protein complex, pval = 0.82468', 'mitochondrial RNA localization, pval = --', 'mitochondrial proton-transporting ATP synthase complex assembly, pval = --', 'mitochondrial respiratory chain complex IV assembly, pval = --', 'mitochondrial DNA packaging, pval = --', 'mitochondrial ATP synthesis coupled electron transport, pval = --', 'mitochondrial ATP synthesis coupled proton transport, pval = --', 'mitochondrial depolarization, pval = --', 'Nebenkern, pval = --', 'mitochondrial derivative, pval = --', 'mitochondrial protein processing, pval = --', 'mitochondrial nucleoid organization, pval = --', 'mitochondrial translational elongation, pval = --', 'mitochondrial translational initiation, pval = --', 'mitochondrial translational termination, pval = --', 'mitochondrial matrix, pval = 0.23874', 'maintenance of protein location in mitochondrion, pval = --', 'mitochondrial gene expression, pval = --', 'mitochondrial DNA replication, pval = --', 'mitochondrial envelope, pval = 0.15438', 'mitochondrial intracristal space, pval = --', 'kinetoplast, pval = --', 'mitochondrial calcium ion homeostasis, pval = --', 'mitochondrial DNA metabolic process, pval = --', 'interfibrillar mitochondrion, pval = --', 'subsarcolemmal mitochondrion, pval = --', 'mitochondrial respiratory chain complex III assembly, pval = --', 'mitochondrial respiratory chain complex II assembly, pval = --', 'mitochondrial RNA metabolic process, pval = --', 'ocelloid, pval = --', 'mitochondrial membrane organization, pval = --']
level 2: ['regulation of mitochondrial translation, pval = --', 'tRNA aminoacylation for mitochondrial protein translation, pval = --', 'positive regulation of mitochondrial translation, pval = --', 'negative regulation of mitochondrial translation, pval = --', 'mitochondrial mRNA editing complex, pval = --', 'mitochondrial glutamate synthase complex (NADH), pval = --', 'mitochondrial electron transfer flavoprotein complex, pval = --', 'Ecsit-NDUFAF1 complex, pval = --', 'mitochondrial pyruvate dehydrogenase complex, pval = --', 'mitochondrial fatty acid beta-oxidation multienzyme complex, pval = --', '3-methylcrotonyl-CoA carboxylase complex, mitochondrial, pval = --', 'mitochondrial ribonuclease P complex, pval = --', 'mitochondrial creatine kinase complex, pval = --', 'mitochondrial processing peptidase complex, pval = --', 'mitochondrial permeability transition pore complex, pval = --', 'gamma DNA polymerase complex, pval = --', 'mitochondrial small ribosomal subunit, pval = --', 'mitochondrial large ribosomal subunit, pval = --', 'mitochondrial endopeptidase Clp complex, pval = --', 'inner mitochondrial membrane protein complex, pval = 0.43974', 'mitochondrial pyruvate dehydrogenase (lipoamide) phosphatase complex, pval = --', 'NAGS/NAGK complex, pval = --', 'mitochondrial degradosome, pval = --', 'mitochondrial DNA-directed RNA polymerase complex, pval = --', 'mitochondrial tricarboxylic acid cycle enzyme complex, pval = --', 'outer mitochondrial membrane protein complex, pval = --', 'mitochondrial intermembrane space protein transporter complex, pval = --', 'pole plasm mitochondrial rRNA localization, pval = --', 'mitochondrial electron transport, cytochrome c to oxygen, pval = --', 'mitochondrial electron transport, ubiquinol to cytochrome c, pval = --', 'mitochondrial electron transport, NADH to ubiquinone, pval = --', 'mitochondrial electron transport, succinate to ubiquinone, pval = --', 'negative regulation of mitochondrial ATP synthesis coupled electron transport, pval = --', 'regulation of mitochondrial ATP synthesis coupled electron transport, pval = --', 'positive regulation of mitochondrial ATP synthesis coupled electron transport, pval = --', 'negative regulation of mitochondrial ATP synthesis coupled proton transport, pval = --', 'regulation of mitochondrial ATP synthesis coupled proton transport, pval = --', 'negative regulation of mitochondrial depolarization, pval = --', 'positive regulation of mitochondrial depolarization, pval = --', 'regulation of mitochondrial depolarization, pval = --', 'major mitochondrial derivative, pval = --', 'minor mitochondrial derivative, pval = --', 'protein processing involved in protein targeting to mitochondrion, pval = --', 'mitochondrial nucleoid organization involved in mitochondrial fission, pval = --', 'positive regulation of mitochondrial translational elongation, pval = --', 'negative regulation of mitochondrial translational elongation, pval = --', 'regulation of mitochondrial translational elongation, pval = --', 'positive regulation of mitochondrial translational initiation, pval = --', 'negative regulation of mitochondrial translational initiation, pval = --', 'regulation of mitochondrial translational initiation, pval = --', 'mitochondrial nucleoid, pval = --', 'mitochondrial ribosome, pval = --', 'PAM complex, Tim23 associated import motor, pval = --', 'mitochondrial RNA processing, pval = --', 'mitochondrial transcription, pval = --', 'regulation of mitochondrial gene expression, pval = --', 'negative regulation of mitochondrial DNA replication, pval = --', 'positive regulation of mitochondrial DNA replication, pval = --', 'regulation of mitochondrial DNA replication, pval = --', 'mitochondrial membrane, pval = 0.21246', 'mitochondrial intermembrane space, pval = 0.0846', 'negative regulation of mitochondrial calcium ion concentration, pval = --', 'positive regulation of mitochondrial calcium ion concentration, pval = --', 'mitochondrial DNA catabolic process, pval = --', 'mitochondrial DNA repair, pval = --', 'regulation of mitochondrial DNA metabolic process, pval = --', 'negative regulation of mitochondrial DNA metabolic process, pval = --', 'positive regulation of mitochondrial DNA metabolic process, pval = --', 'mitochondrion DNA recombination, pval = --', 'transcription initiation from mitochondrial promoter, pval = --', 'transcription elongation from mitochondrial promoter, pval = --', 'termination of mitochondrial transcription, pval = --', 'mitochondrial RNA catabolic process, pval = --', 'mitochondrial RNA modification, pval = --', 'regulation of mitochondrial membrane permeability, pval = --', 'establishment of protein localization to mitochondrial membrane, pval = --', 'mitochondrial membrane fusion, pval = --', 'protein insertion into mitochondrial membrane, pval = --', 'outer mitochondrial membrane organization, pval = --', 'inner mitochondrial membrane organization, pval = --']
level 3: ['mitochondrial glutamyl-tRNA aminoacylation, pval = --', 'mitochondrial glutaminyl-tRNA aminoacylation, pval = --', 'mitochondrial cysteinyl-tRNA aminoacylation, pval = --', 'mitochondrial aspartyl-tRNA aminoacylation, pval = --', 'mitochondrial asparaginyl-tRNA aminoacylation, pval = --', 'mitochondrial arginyl-tRNA aminoacylation, pval = --', 'mitochondrial alanyl-tRNA aminoacylation, pval = --', 'mitochondrial threonyl-tRNA aminoacylation, pval = --', 'mitochondrial seryl-tRNA aminoacylation, pval = --', 'mitochondrial prolyl-tRNA aminoacylation, pval = --', 'mitochondrial phenylalanyl-tRNA aminoacylation, pval = --', 'mitochondrial methionyl-tRNA aminoacylation, pval = --', 'mitochondrial lysyl-tRNA aminoacylation, pval = --', 'mitochondrial leucyl-tRNA aminoacylation, pval = --', 'mitochondrial isoleucyl-tRNA aminoacylation, pval = --', 'mitochondrial histidyl-tRNA aminoacylation, pval = --', 'mitochondrial glycyl-tRNA aminoacylation, pval = --', 'mitochondrial valyl-tRNA aminoacylation, pval = --', 'mitochondrial tyrosyl-tRNA aminoacylation, pval = --', 'mitochondrial tryptophanyl-tRNA aminoacylation, pval = --', 'positive regulation of mitochondrial translation in response to stress, pval = --', 'heterotetrameric decaprenyl diphosphate synthase complex, pval = --', 'i-AAA complex, pval = --', 'mitochondrial respiratory chain complex I, peripheral segment, pval = --', 'mitochondrial respiratory chain complex I, membrane segment, pval = --', 'MIB complex, pval = --', 'mitochondrial respiratory chain complex III, pval = --', 'm-AAA complex, pval = --', 'TIM23 mitochondrial import inner membrane translocase complex, pval = --', 'mitochondrial respiratory chain complex II, succinate dehydrogenase complex (ubiquinone), pval = --', 'mitochondrial respiratory chain complex I, pval = --', 'mitochondrial proton-transporting ATP synthase, central stalk, pval = --', 'mitochondrial proton-transporting ATP synthase complex, pval = --', 'mitochondrial proton-transporting ATP synthase, catalytic core, pval = --', 'mitochondrial respiratory chain complex IV, pval = --', 'INA complex, pval = --', 'mitochondrial inner membrane assembly complex, pval = --', 'mitochondrial proton-transporting ATP synthase, stator stalk, pval = --', 'mitochondrial proton-transporting ATP synthase complex, catalytic core F(1), pval = --', 'mitochondrial proton-transporting ATP synthase complex, coupling factor F(o), pval = --', 'MICOS complex, pval = --', 'mitochondrial respiratory chain complex IV pre-assembly complex, pval = --', 'mitochondrial prohibitin complex, pval = --', 'uniplex complex, pval = --', 'mitochondrial inner membrane peptidase complex, pval = --', 'TIM22 mitochondrial import inner membrane insertion complex, pval = 0.43974', 'mitochondrial alpha-ketoglutarate dehydrogenase complex, pval = --', 'mitochondrial isocitrate dehydrogenase complex (NAD+), pval = --', 'succinate-CoA ligase complex (GDP-forming), pval = --', 'mitochondrial outer membrane translocase complex, pval = --', 'SAM complex, pval = --', 'Cdc48p-Npl4p-Vms1p AAA ATPase complex, pval = --', 'ERMES complex, pval = --', 'pole plasm mitochondrial srRNA localization, pval = --', 'pole plasm mitochondrial lrRNA localization, pval = --', 'negative regulation of mitochondrial electron transport, NADH to ubiquinone, pval = --', 'regulation of mitochondrial electron transport, NADH to ubiquinone, pval = --', 'positive regulation of mitochondrial electron transport, NADH to ubiquinone, pval = --', 'negative regulation of protein processing involved in protein targeting to mitochondrion, pval = --', 'regulation of protein processing involved in protein targeting to mitochondrion, pval = --', 'mitochondrial chromosome, pval = --', &quot;mitochondrial tRNA 3'-trailer cleavage, endonucleolytic, pval = --&quot;, 'mitochondrial mRNA processing, pval = --', 'mitochondrial tRNA processing, pval = --', &quot;mitochondrial RNA 5'-end processing, pval = --&quot;, &quot;mitochondrial RNA 3'-end processing, pval = --&quot;, 'mitochondrial single subunit type RNA polymerase activity, pval = --', 'regulation of mitochondrial transcription, pval = --', 'positive regulation of mitochondrial transcription, pval = --', 'mitochondrial rRNA transcription, pval = --', 'mitochondrial sequence-specific DNA-binding transcription factor activity, pval = --', 'contact site, pval = --', 'bridge contact site, pval = --', 'mitochondrial membrane scission site, pval = --', 'mitochondrial outer membrane, pval = 0.32699', 'mitochondrial inner membrane, pval = 0.49914', 'intrinsic component of mitochondrial membrane, pval = 0.40992', 'mitochondrial RNA polymerase termination site sequence-specific DNA binding, pval = --', 'mitochondrial RNA surveillance, pval = --', 'regulation of mitochondrial RNA catabolic process, pval = --', 'positive regulation of mitochondrial RNA catabolic process, pval = --', 'negative regulation of mitochondrial RNA catabolic process, pval = --', 'mitochondrial mRNA catabolic process, pval = --', 'mitochondrial tRNA modification, pval = --', 'mitochondrial mRNA modification, pval = --', 'negative regulation of mitochondrial membrane permeability, pval = --', 'positive regulation of mitochondrial membrane permeability, pval = --', 'regulation of mitochondrial membrane permeability involved in programmed necrotic cell death, pval = --', 'regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway, pval = --', 'regulation of mitochondrial membrane permeability involved in apoptotic process, pval = --', 'protein insertion into mitochondrial inner membrane, pval = --', 'protein insertion into mitochondrial outer membrane, pval = --', 'establishment of protein localization to mitochondrial membrane involved in mitochondrial fission, pval = --', 'mitochondrial inner membrane fusion, pval = --', 'mitochondrial outer membrane fusion, pval = --', 'protein insertion into mitochondrial membrane involved in apoptotic signaling pathway, pval = --', 'mitochondrial outer membrane translocase complex assembly, pval = --', 'cristae formation, pval = --']
level 4: ['mitochondrial oxoglutarate dehydrogenase complex, pval = --', 'mitochondrial polycistronic RNA processing, pval = --', &quot;mitochondrial mRNA 5'-end processing, pval = --&quot;, &quot;mitochondrial mRNA 3'-end processing, pval = --&quot;, &quot;mitochondrial tRNA 3'-end processing, pval = --&quot;, &quot;mitochondrial tRNA 5'-end processing, pval = --&quot;, 'cytoplasmic side of mitochondrial outer membrane, pval = --', 'intrinsic component of mitochondrial outer membrane, pval = 0.43974', 'extrinsic component of mitochondrial outer membrane, pval = --', 'mitochondrial crista, pval = --', 'matrix side of mitochondrial inner membrane, pval = --', 'mitochondrial inner boundary membrane, pval = --', 'mitochondrial crista junction, pval = --', 'mitochondrial respirasome, pval = --', 'intrinsic component of mitochondrial inner membrane, pval = 0.43974', 'extrinsic component of mitochondrial inner membrane, pval = --', 'integral component of mitochondrial membrane, pval = 0.40992', 'mitochondrial mRNA surveillance, pval = --', 'mitochondrial ncRNA surveillance, pval = --', 'regulation of mitochondrial mRNA catabolic process, pval = --', 'positive regulation of mitochondrial mRNA catabolic process, pval = --', 'negative regulation of mitochondrial mRNA catabolic process, pval = --', 'mitochondrial tRNA thio-modification, pval = --', 'mitochondrial tRNA pseudouridine synthesis, pval = --', 'mitochondrial tRNA methylation, pval = --', 'mitochondrial tRNA threonylcarbamoyladenosine modification, pval = --', 'mitochondrial tRNA wobble uridine modification, pval = --', 'negative regulation of mitochondrial membrane permeability involved in apoptotic process, pval = --', 'mitochondrial outer membrane permeabilization involved in programmed cell death, pval = --', 'regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway, pval = --', 'positive regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway, pval = --', 'negative regulation of mitochondrial outer membrane permeabilization involved in apoptotic signaling pathway, pval = --', 'positive regulation of mitochondrial membrane permeability involved in apoptotic process, pval = --', 'protein insertion into mitochondrial inner membrane from matrix, pval = --', 'negative regulation of protein import into mitochondrial outer membrane, pval = --', 'positive regulation of protein import into mitochondrial outer membrane, pval = --', 'regulation of protein import into mitochondrial outer membrane, pval = --', 'positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway, pval = --', 'negative regulation of cristae formation, pval = --', 'positive regulation of cristae formation, pval = --', 'regulation of cristae formation, pval = --']
level 5: ['mitochondrial mRNA polyadenylation, pval = --', 'integral component of mitochondrial outer membrane, pval = 0.43974', 'extrinsic component of matrix side of mitochondrial inner membrane, pval = --', 'integral component of mitochondrial inner membrane, pval = 0.43974', 'mitochondrial tRNA wobble position uridine thiolation, pval = --', 'mitochondrial outer membrane permeabilization, pval = --']" id="mitochondria" init-pval="0.16438" min-pval="0.0846" min-pval-children="0.0846" title="Mitochondria">
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z" style="fill:#E7C8CD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M281.3,410.3c-2.6,5.5-13.8,2.8-23.1-3
		c-12.2-7.7,0.2-26.3,11-15.6C277.5,399.9,283.6,405.3,281.3,410.3z" style="fill:#E7C8CD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M463.6,106c13.9-0.7,23.1,0.5,32.1,17.7
		c5.9,11.2-7,14.4-10.6,7.1c-5.2-10.5-9.4-13.4-21.9-12.6C454.7,118.8,452.5,106.5,463.6,106z" style="fill:#E7C8CD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495,340.9c-11,17,0.4,13.3-11.7,30.9
		c-6.5,9.5-16.4,0.6-12.6-6.1c10.6-18.6,0.8-16.5,12.6-32.5C488.4,326.1,500,333.3,495,340.9z" style="fill:#E7C8CD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M132.5,351.2c11.1,0.8,14.3,1,22.2-5.3
		c7.1-5.7,12.3,3.3,6.4,9.4c-6.8,7.1-16,11.7-27.4,8.8C123,361.3,122.8,350.5,132.5,351.2z" style="fill:#E7C8CD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M324,73.1c5.8-12,16.3-4.1,19.1,2.7
		c2.8,6.8,0.4,12.7-7.8,13C326,89,318.8,83.8,324,73.1z" style="fill:#E7C8CD;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M483.2,333.2c-11.8,16.1-2,13.9-12.6,32.5
		c-3.8,6.6,6.1,15.6,12.6,6.1c12.1-17.6,0.7-13.9,11.7-30.9C500,333.3,488.4,326.1,483.2,333.2z M478,353.2
		c-0.1-0.7,0.1-1.4,0.3-2.1c0.2-0.8,0.3-1.7,0.5-2.5c0.2-0.8,0.7-1.4,1.7-1.3c0.4,0.1,0.7,0.2,1.1,0.3c0.9,0.2,1.8,0.5,2.8,0.7
		c0.9,0.2,2.8,0.4,2.8-1c0-0.8-0.8-1.2-1.4-1.4c-0.8-0.3-1.5-0.6-2.3-0.9c-1.3-0.5-3.2-1.2-3-2.9c0-0.5,0.3-1.1,0.6-1.5
		c0.5-0.7,1.3-1.2,2.1-1c0.7,0.2,1.3,0.7,2,1c0.5,0.3,1.1,0.1,1.4-0.4c0.3-0.5,0.1-1.2-0.4-1.4c-0.1-0.1-0.3-0.2-0.4-0.2
		c-0.8-0.4-2.4-1-1.9-2.2c0.3-0.7,1-1.1,1.7-0.9c0.7,0.2,1.2,0.8,1.9,1.2c0.5,0.3,1.1,0.1,1.4-0.3c0.3-0.5,0.1-1.1-0.3-1.4
		c-0.4-0.3-0.9-0.6-1.3-0.9c-0.6-0.4-0.6-0.9,0-1.4c0.3-0.2,0.7-0.3,1.1-0.3c1.5-0.2,3.1,0.3,4.3,1.3c0.5,0.4,0.9,0.9,1.2,1.5
		c0.2,0.3,0.3,0.7,0.4,1c0.2,0.8,0.3,1.7-0.3,2.4c-0.6,0.6-1.5,0-2.1-0.2c-0.3-0.1-0.9-0.1-1.1,0.2c-0.5,0.4-0.5,1.1,0,1.5
		c0.4,0.3,1,0.6,1.2,1c0.2,0.4,0.2,0.9-0.1,1.3c-0.2,0.4-0.5,0.8-0.9,1c-0.9,0.5-2-0.3-2.8-0.6c-0.5-0.2-1.2,0-1.4,0.5
		c-0.2,0.4-0.1,0.9,0.3,1.2c0.1,0.1,0.2,0.1,0.2,0.1c0.7,0.3,1.5,0.7,1.8,1.4c0.3,0.7,0.1,1.5-0.1,2.2c-0.2,0.7-0.4,1.5-1.1,1.9
		c-0.5,0.3-1.2,0.4-1.9,0.3c-0.6-0.1-1.2-0.2-1.7-0.3c-0.6-0.1-1.1,0.3-1.2,0.9c-0.1,0.4,0.1,0.8,0.4,1c0.3,0.2,0.7,0.3,1.1,0.3
		c0.5,0.1,0.9,0.1,1.4,0.2c0.5,0,1.1,0,1.5,0.4c0.4,0.4,0.5,1.1,0.3,1.6c-0.6,1.3-2.4,0.1-2.9,1.3c-0.2,0.4-0.1,0.8,0.1,1.2
		c0.2,0.3,0.6,0.4,1,0.5c0.4,0,0.8,0.1,1.2,0.3c1,0.6,0.5,2.3-0.5,2.6c-0.7,0.2-1.4-0.2-2-0.4c-0.5-0.2-1.1,0.1-1.4,0.6
		c-0.2,0.4,0,0.9,0.3,1.2c0.6,0.5,1.7,0.3,2.1,1c0.3,0.5,0.1,1.1-0.3,1.5c-0.6,0.4-1.3,0.3-1.9,0.1c-0.6-0.2-1.4-0.6-1.9,0.1
		c-0.7,1,0.4,1.5,1.1,1.9c0.3,0.2,0.7,0.5,0.8,0.9c0.2,0.6-0.2,1.3-0.5,1.8c-0.4,0.7-1,1.3-1.7,1.7c-0.7,0.4-1.6,0-2.2-0.4
		c-0.3-0.2-0.7-0.5-1-0.7c-0.5-0.3-1.1-0.2-1.5,0.3c-0.2,0.2-0.2,0.6-0.1,0.9c0.3,0.9,2.6,1.1,2.1,2.4c-0.5,1.3-2.5,0.9-3.4,0.4
		c-0.4-0.2-0.8-0.4-1.2-0.7c-0.4-0.4-0.8-0.8-1.1-1.3c-0.3-0.4-0.5-0.9-0.6-1.3c-0.5-1.4-0.3-3.1,1.2-3.9c0.7-0.4,1.4,0.1,2,0.4
		c0.9,0.4,1.7,0.9,2.5,1.5c0.5,0.3,1.1,0.2,1.5-0.3c0.3-0.5,0.2-1.1-0.3-1.5c-0.3-0.2-0.6-0.4-0.9-0.5c-0.7-0.4-1.3-0.7-2-1.1
		c-0.6-0.3-1.2-0.6-1.5-1.2c-0.3-0.5-0.1-1.2,0.3-1.6c0.5-0.6,1.2-0.7,1.9-0.4c0.6,0.2,1.1,0.4,1.7,0.7c0.5,0.3,1.1,0,1.4-0.5
		c0.3-0.5,0-1.2-0.5-1.4c-0.3-0.2-0.7-0.3-1-0.4c-0.6-0.3-1.5-0.5-1.7-1.3c-0.1-0.4,0.2-0.9,0.6-1.1c0.4-0.3,1-0.2,1.4-0.1
		c0.5,0.1,1.1,0.3,1.6,0.5c1.3,0.5,2-1.5,0.8-2c-1-0.4-1.9-0.9-2.5-1.7C478.2,354.1,478.1,353.7,478,353.2z" style="fill:#A55978;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M154.7,345.9c-7.9,6.3-11.1,6.1-22.2,5.3
		c-9.6-0.7-9.4,10.1,1.2,12.9c11.3,2.9,20.6-1.7,27.4-8.8C167,349.1,161.7,340.2,154.7,345.9z M161.3,349.1c-0.4,0.1-1-0.5-1.2-0.8
		c-0.3-0.3-0.7-0.4-1.1-0.2c-0.2,0.1-0.3,0.2-0.4,0.3c-0.3,0.4-0.3,0.9-0.1,1.3c0.2,0.5,0.7,0.8,1,1.3c0.4,0.6,0.6,1.3,0.5,1.9
		c-0.2,1.1-1.4,2.2-2.4,2.7c-0.3,0.2-0.7,0.2-1.1,0.1c-0.7-0.2-1.1-0.8-1.6-1.3c-0.5-0.6-1.2-1.3-2-1c-0.1,0-0.1,0.1-0.2,0.1
		c-1.1,0.7-0.2,1.9,0.3,2.6c0.5,0.7,0.9,1.6,0.6,2.5c-0.2,0.6-0.8,1-1.4,1.3c-0.7,0.4-1.4,0.7-2.1,1c-0.8,0.5-2.2,1.2-2.8,0
		c-0.2-0.4-0.3-0.9-0.4-1.3c-0.2-0.6-0.4-1.5-1.2-1.4c-1.4,0.1-1.2,1.8-0.8,2.7c0.2,0.5,0.3,1.2-0.2,1.5c-0.4,0.2-1,0.2-1.4,0
		c-0.9-0.5-0.6-2-0.7-2.9c-0.1-0.6-0.7-1-1.3-0.8c-0.1,0-0.1,0-0.2,0.1c-1,0.4-0.6,1.3-0.6,2.1c0.1,1.1-0.7,2.1-1.8,2.2
		c-0.7,0-1.4-0.1-2.1-0.3c-1.6-0.4-2.3-2.3-1.7-3.8c0,0,0.1-0.6,0.1-0.6c0.2-0.6-0.2-1.2-0.8-1.3c-0.2-0.1-0.5,0-0.7,0
		c-0.3,0.1-0.6,0.4-0.7,0.7l-0.1,0.6c-0.2,0.8-0.2,2-1.1,2.3c-0.5,0.2-1.1,0-1.6-0.3c-0.7-0.5-1.4-1.3-1.9-2c-0.4-0.6-0.6-1.3-0.7-2
		c0-0.6,0.1-1.2,0.4-1.6c0.3-0.4,0.8-0.7,1.3-0.4c0.5,0.4,0.5,1.1,0.5,1.6c0,0,0,1,1,1.2c1.2,0.2,1.4-1.5,1.7-2.2
		c0.2-0.4,0.5-0.9,0.8-1.2c0.4-0.4,1-0.5,1.6-0.5c0.7,0,1.3,0.4,1.6,1c0.4,0.7,0.3,1.5,0.3,2.3c0,0.7,0,1.6,0.7,2.1
		c0.5,0.3,1.3,0.3,1.7-0.1c0.4-0.4,0.5-1,0.6-1.6c0.1-0.9,0.2-1.8,1-2.4c0.7-0.5,1.5-0.7,2.3-0.8c1-0.2,2.4-0.3,3.3,0.4
		c0.5,0.4,0.9,1,1.2,1.6c0.4,0.7,0.8,1.5,1.1,2.3c0.2,0.6,0.3,1.5,1.1,1.7c0.6,0.2,1.2-0.3,1.3-0.9c0.2-0.7-0.2-1.4-0.4-2
		c-0.3-0.7-0.6-1.4-0.9-2.1c-0.4-1-0.6-2,0.2-2.8c0.4-0.5,1-0.9,1.6-1.1c0.3-0.1,0.6-0.2,1-0.3c1.3-0.2,2.2,0.8,2.9,1.7
		c0.4,0.5,1.1,0.5,1.5,0.2c0.5-0.4,0.5-1.1,0.2-1.5c-0.4-0.5-1.3-1.2-1.4-1.9c-0.1-1.3,1.2-2.2,2.2-2.6c1.7-0.7,2.7,0.1,3.2,0.8
		C161.2,347.5,162,348.9,161.3,349.1z" style="fill:#A55978;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M335.3,88.7c8.2-0.3,10.6-6.2,7.8-13
		c-2.8-6.8-13.3-14.6-19.1-2.7C318.8,83.8,326,89,335.3,88.7z M324.5,80c-0.1-1.8,0.6-3.6,1.2-5.3c0.3-0.8,0.7-1.6,1.2-2.4
		c0.5-0.7,1.1-1.6,1.8-2.1c0.4-0.3,1.3-0.7,1.7-0.1c0.3,0.4,0.3,1,0.1,1.5c-0.2,0.7-0.6,1.2-0.9,1.9c-0.4,1.1-1,2.2-1.4,3.4
		c-0.2,0.6,0.1,1.3,0.8,1.5c0.1,0,0.3,0.1,0.4,0.1c0.5,0,1-0.3,1.1-0.8c0.2-0.7,0.5-1.4,0.9-2.1c0.6-1.1,1.2-2.3,1.9-3.4
		c0.3-0.4,0.6-0.8,1.1-0.9c0.8-0.3,1.7,0,2.2,0.7c0.8,1.2-0.2,2.6-0.8,3.6c-0.6,0.9-1.1,1.8-1.4,2.3c-0.3,0.6-0.1,1.3,0.4,1.6
		c0.2,0.1,0.4,0.2,0.6,0.2c0.4,0,0.8-0.2,1-0.6c0.5-0.8,1-1.6,1.4-2.4c0.3-0.6,0.6-1.4,1.5-1.4c0.8,0,1.3,0.7,1.6,1.4
		c1.2,2.9,1.2,5.6,0.1,7.3c-0.3,0.5-0.9,1.5-1.6,1.4c-1.2-0.2-0.3-2-0.4-2.7c-0.1-0.4-0.4-0.7-0.8-0.9c-0.6-0.2-1.3,0.1-1.5,0.8
		c-0.2,0.7-0.5,1.4-0.8,2c-0.4,0.8-1,1.4-1.9,1.4c-0.4,0-0.9-0.1-1.2-0.4c-0.2-0.2-0.5-0.8-0.5-1.2c0.1-1.3,1.1-2.8-0.2-3.8
		c-0.2-0.2-0.6-0.3-0.9-0.3c-1,0.2-1.4,1.7-2,2.5c-0.9,1.3-2.6,1-3.6,0c0,0-0.1-0.1-0.1-0.1C324.8,81.9,324.5,80.9,324.5,80z" style="fill:#A55978;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M258.2,407.3c9.3,5.9,20.6,8.6,23.1,3
		c2.3-5-3.7-10.4-12.1-18.6C258.4,381,246,399.6,258.2,407.3z M258,404.6c-0.7-0.3-0.3-1.2,0-1.7c0.6-1,1.3-1.9,1.9-2.9
		c0.3-0.5,0.2-1.2-0.4-1.5c-0.5-0.3-1.2-0.2-1.5,0.4c-0.3,0.5-1.5,3.1-2.4,2.1c-0.4-0.4-0.4-1.1-0.4-1.6c0-0.7-0.1-1.4,0.1-2.1
		c0.1-0.3,0.1-0.6,0.2-1c0.3-1.1,0.9-2.2,1.6-3.1c0.8-1,1.9-1.7,3.1-2.2c0.3-0.1,1.4-0.5,1.3,0.2c-0.1,0.5-0.7,1-1,1.4
		c-0.5,0.6-1.3,1.4-1.2,2.2c0.1,0.6,0.8,1.1,1.4,0.8c0.6-0.2,0.9-1,1.2-1.5c0.5-0.8,1.2-1.5,1.8-2.2c0.3-0.4,0.7-0.6,1.1-0.6
		c0.6,0,1.3,0.3,1.7,0.9c0.2,0.3,0.2,0.7,0.1,1c-0.2,0.7-0.8,1.2-1.3,1.7c-0.5,0.5-1.2,1.1-0.8,1.8c0.3,0.7,1.3,0.9,1.8,0.3
		c0.6-0.7,1.2-1.6,2-2c0.7-0.4,1.4,0.1,1.8,0.8c0.8,1.3-0.8,2.3-1.5,3.2c-0.5,0.6-0.9,1.2-1.3,1.8c-0.7,1-1.4,2-1.9,3
		c-0.3,0.6,0,1.2,0.5,1.5c0.2,0.1,0.3,0.1,0.5,0.1c0.4,0,0.8-0.2,1-0.6c0.2-0.5,0.5-0.9,0.7-1.3c1-1.4,1.9-2.9,3.1-4.1
		c0.4-0.4,0.7-0.8,1.3-0.8c0.5,0,0.9,0.3,1.1,0.7c0.5,1-0.2,2-0.7,2.9c-0.3,0.4-0.7,0.9-0.8,1.4c-0.2,0.6,0.1,1.2,0.7,1.4
		c0.1,0,0.2,0.1,0.3,0.1c0.7,0,1.1-0.8,1.4-1.3c0.3-0.5,0.9-1.5,1.6-1.5c0.4,0,0.7,0.3,1,0.6c0.4,0.4,0.5,1.1,0.5,1.6
		c-0.1,0.6-0.6,0.9-0.9,1.4c-0.5,0.7-1,1.4-1.3,2.2c-0.2,0.6,0,1.2,0.6,1.5c0.1,0.1,0.3,0.1,0.4,0.1c0.7,0,0.9-0.6,1.2-1.1
		c0.2-0.5,0.7-1.3,1.3-1.3c0.7-0.1,0.9,1,0.8,1.5c-0.2,2.1-2.9,2.7-4.6,2.7c-0.6,0-1.2-0.1-1.9-0.1c-0.6-0.1-1.2,0-1.7-0.4
		c-0.7-0.4-0.4-1.1-0.5-1.8c-0.2-0.7-0.8-1.2-1.6-0.9c-0.6,0.3-0.9,1.1-1.6,1.3c-0.6,0.2-1.3,0-1.9-0.2c-0.8-0.4-1.6-0.8-2.4-1.1
		c-0.8-0.4-2-0.7-2.2-1.7c-0.2-0.8,0.5-1.5,1-2.1c0.4-0.4,0.4-1.1,0-1.6c-0.4-0.4-1.1-0.4-1.6,0c-0.5,0.5-0.8,1.1-1.2,1.5
		c-0.2,0.2-0.5,0.4-0.9,0.5C258.5,404.7,258.3,404.7,258,404.6z" style="fill:#A55978;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M495.7,123.6c-9.1-17.2-18.3-18.4-32.1-17.7
		c-11,0.5-8.9,12.8-0.4,12.3c12.5-0.8,16.7,2,21.9,12.6C488.7,138,501.6,134.9,495.7,123.6z M491.5,132.8c-0.9,0.1-1.8-0.2-2.6-0.6
		c-0.4-0.2-0.8-0.5-1.1-0.8c-0.5-0.6-0.7-1.2-0.1-1.7c0.4-0.4,1-0.5,1.4-0.8c0.5-0.3,0.7-0.8,0.4-1.4c-0.3-0.5-0.9-0.5-1.4-0.3
		c-0.7,0.3-1.6,1-2.4,0.5c-0.8-0.4-0.3-1.5,0.1-2c0.5-0.6,1.2-0.9,1.7-1.4c0.3-0.2,0.5-0.6,0.4-1c0-0.1-0.1-0.3-0.2-0.4
		c-0.4-0.5-1.1-0.4-1.6,0c-0.6,0.4-1.2,1-1.9,1.3c-0.5,0.2-1-0.1-1.2-0.6c-0.2-0.5,0.1-1.1,0.4-1.6c0.3-0.4,0.7-0.7,0.6-1.3
		c0-0.2-0.1-0.3-0.3-0.4c-0.9-0.8-1.6,0.4-2.4,0.6c-1,0.3-1.1-0.9-1-1.6c0.1-0.5,0.5-1.1,0.4-1.6c-0.2-0.9-1.2-0.8-1.8-0.4
		c-0.5,0.4-1,0.7-1.7,0.7c-0.6,0-1.1-0.3-1.7-0.5c-0.8-0.3-1.6-0.6-2.4-0.7c-0.6-0.1-1.1-0.2-1.7-0.3c-0.5-0.1-1.1-0.1-1.5-0.4
		c-0.4-0.4-0.2-0.9-0.3-1.4c-0.1-0.4-0.5-0.8-1-0.7c-0.3,0-0.7,0.3-0.9,0.6c-0.2,0.4-0.2,1-0.5,1.4c-0.3,0.3-0.8,0.4-1.2,0.4
		c-0.5-0.1-0.9-0.3-1.1-0.7c-0.4-0.8-0.1-3.1-1.6-2.7c-0.5,0.1-0.9,0.7-0.7,1.2c0.2,0.8,0.5,2-0.7,2.1c-1.2,0.1-2.5-0.6-3.2-1.5
		c-0.4-0.5-0.6-1.1-0.8-1.7c-0.2-0.7-0.2-1.5,0-2.2c0.1-0.4,0.5-1.3,1.2-1.1c0.5,0.2,0.5,1.1,0.5,1.6c0,0.5,0.5,0.9,1.1,0.9
		c0.6,0,1-0.5,1-1c0-0.8-0.4-1.7-0.2-2.5c0.2-1,1.9-1.1,2.6-0.7c0.9,0.5,0.6,1.8,1,2.6c0.4,1,2,0.6,1.9-0.5c0-0.6-0.2-1.4,0-2
		c0.1-0.3,0.3-0.5,0.6-0.5c0.4-0.1,0.9,0,1.2,0.3c0.3,0.4,0.1,1,0,1.4c-0.1,0.5,0.4,1,0.9,1c1.1,0.1,1-1.1,1.4-1.7
		c0.2-0.3,0.5-0.5,0.9-0.6c0.6-0.2,1.4-0.2,1.7,0.5c0.3,0.7,0.1,1.5,0,2.3c-0.1,0.5-0.2,1-0.2,1.4c-0.1,0.5,0.3,1,0.8,1.1
		c0.6,0.1,1.1-0.3,1.2-0.8c0.2-0.9,0.2-2,0.6-2.9c0.4-0.7,1.3-0.9,2.1-0.8c1.3,0.2,0.7,1.9,0.4,2.7c-0.3,0.8-0.6,1.5-0.8,2.3
		c-0.1,0.3-0.2,0.5-0.3,0.8c-0.2,0.5,0.1,1.1,0.7,1.2c0.8,0.2,1.2-0.4,1.4-1c0.3-0.8,0.5-1.6,0.7-2.4c0.2-0.6,0.4-1.4,1.1-1.6
		c0.4-0.1,0.9-0.1,1.3,0c0.5,0.2,1.1,0.6,1,1.2c-0.1,1-2.1,2.7-0.7,3.5c0.3,0.2,0.8,0.2,1.1,0c0.6-0.4,0.9-1.3,1.8-1.4
		c0.8-0.1,1.5,0.6,1.7,1.3c0.4,1.2-1.3,2-1.4,3.1c-0.1,1,1.1,1.7,1.8,0.8c0.4-0.5,1-1.2,1.7-1.1c0.6,0.1,1,0.5,1.3,1
		c0.3,0.5,0.4,1,0.2,1.6c-0.3,0.8-1.9,1.4-1.4,2.4c0.2,0.5,0.8,0.7,1.3,0.5c0.6-0.3,1.1-0.6,1.8-0.3c1.1,0.4,1.5,2.2,0.3,2.8
		c-0.6,0.3-1.2,0.4-1.1,1.2c0.1,0.5,0.6,0.9,1.1,0.8c0.7-0.1,1.5-0.4,1.9,0.4c0.2,0.4,0.2,1.1,0.1,1.6c-0.3,1.1-1.2,1.9-2.2,2.4
		C492.5,132.7,492,132.8,491.5,132.8z" style="fill:#A55978;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
<path d="M131.4,243.7c-15.2,16.1-13.2,26.2-7.6,40.3
		c4.5,11.3,16.9,4.2,13.3-4.4c-5.3-12.7-3.7-18.2,5.7-27.7c1.7-1.7,3.3-3.6,4.6-5.7c1-1.6,2.1-3.5,2.2-5.3c0.2-1.9-0.8-3.9-2.4-4.9
		c-4.2-2.7-9.3,1.6-12.4,4.3C133.7,241.4,132.5,242.5,131.4,243.7z M143.7,244.4c-0.1,0-0.3,0-0.5,0c-1.3,0.3-1.5,1.3-0.6,2.2
		c1,1.1-0.4,3.7-1.9,2.9c-0.6-0.3-0.9-0.8-1.4-1.2c-0.5-0.4-1.1-0.5-1.6,0.2c-0.5,0.5-0.2,1.1,0.2,1.6c0.6,0.6,1.6,1.3,1.4,2.3
		c-0.2,1-1.4,0.9-2.2,0.7c-0.8-0.2-1.4-0.8-2.1-1.3c-0.4-0.2-0.9-0.2-1.2,0c-0.1,0.1-0.2,0.2-0.3,0.3c-0.4,0.7,0,1.3,0.5,1.7
		c0.7,0.5,1.5,0.8,2,1.5c0.4,0.5,0.2,1.1-0.2,1.4c-0.5,0.4-1.2,0.4-1.8,0.2c-0.6-0.2-1-0.5-1.6-0.1c-0.2,0.1-0.3,0.3-0.4,0.4
		c-0.6,1.2,1,1.5,1.5,2.3c0.7,1-0.6,1.5-1.4,1.6c-0.6,0.1-1.3-0.1-1.8,0.2c-0.8,0.6-0.5,1.5,0.2,2c0.6,0.4,1.1,0.8,1.3,1.5
		c0.2,0.6,0.1,1.3,0,2c0,0.9,0,1.9,0.1,2.8c0.1,0.6,0.2,1.2,0.2,1.9c0.1,0.5,0.3,1.2,0.1,1.7c-0.2,0.6-0.9,0.6-1.4,0.9
		c-0.4,0.3-0.6,0.9-0.4,1.4c0.1,0.3,0.6,0.6,0.9,0.7c0.5,0,1.1-0.2,1.6,0c0.5,0.2,0.7,0.7,0.8,1.1c0.1,0.5,0,1.1-0.4,1.4
		c-0.7,0.8-3.3,1.3-2.3,2.8c0.3,0.5,1,0.7,1.5,0.3c0.8-0.5,1.9-1.3,2.5-0.1c0.6,1.2,0.2,2.8-0.4,3.9c-0.4,0.6-0.9,1.1-1.5,1.5
		c-0.7,0.4-1.5,0.8-2.4,0.8c-0.5,0-1.6,0-1.6-0.8c0-0.6,1-0.9,1.5-1.1c0.6-0.2,0.8-0.9,0.6-1.5c-0.2-0.6-0.9-0.8-1.5-0.6
		c-0.9,0.4-1.7,1.1-2.6,1.2c-1.2,0.2-1.9-1.5-1.7-2.5c0.2-1.2,1.7-1.3,2.4-2c0.9-0.8-0.1-2.3-1.2-1.8c-0.7,0.3-1.4,0.8-2.1,0.7
		c-0.3,0-0.6-0.1-0.8-0.4c-0.2-0.4-0.4-1-0.1-1.3c0.3-0.5,1-0.4,1.5-0.5c0.6-0.1,0.9-0.8,0.8-1.3c-0.3-1.2-1.5-0.6-2.3-0.8
		c-0.4-0.1-0.7-0.4-1-0.7c-0.5-0.6-0.7-1.4-0.1-1.9c0.7-0.6,1.6-0.7,2.4-0.9c0.6-0.1,1.1-0.2,1.5-0.3c0.6-0.1,1-0.7,0.9-1.3
		c-0.1-0.6-0.7-1-1.3-0.9c-1,0.2-2.2,0.6-3.3,0.5c-0.9-0.1-1.5-1-1.6-1.9c-0.2-1.4,1.8-1.5,2.8-1.5c0.9,0,1.8,0,2.8,0
		c0.3,0,0.6,0,0.9,0c0.6,0,1.1-0.6,1-1.2c-0.1-0.9-0.8-1.1-1.6-1.1c-0.9,0-1.9,0.1-2.8,0.2c-0.7,0-1.7,0.2-2.1-0.5
		c-0.2-0.3-0.4-0.9-0.4-1.3c0-0.6,0.3-1.3,0.9-1.5c1.1-0.3,3.6,1.1,4-0.7c0.1-0.4-0.1-0.9-0.4-1.2c-0.7-0.5-1.7-0.5-2.2-1.3
		c-0.4-0.8,0.1-1.8,0.8-2.3c1.2-0.8,2.5,0.5,3.8,0.3c1.1-0.2,1.4-1.8,0.2-2.2c-0.7-0.2-1.6-0.5-1.8-1.3c-0.1-0.6,0.2-1.3,0.6-1.8
		c0.4-0.5,1-0.9,1.6-0.9c1,0,2.1,1.5,3.1,0.5c0.4-0.4,0.5-1.1,0.1-1.5c-0.5-0.5-1-1-1-1.8c0.1-1.4,1.8-2.4,2.9-1.4
		c0.7,0.7,2.3,1.3,2.6-0.2c0.1-1-1.3-1.7-0.7-2.8c1.2-2.3,5.5,2,5.4-1c0-0.9-1-1.7-0.9-2.6c0.2-1.9,3.3-0.6,4.1,0
		c0.8,0.6,1.4,1.6,1.5,2.7C147.3,243.5,145.6,244.4,143.7,244.4z" style="fill:#A55978;stroke:#000000;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="ton plasma_membrane cell_junctions" descendants="level 0: ['plasma membrane, pval = 0.56389']
level 1: ['intrinsic component of plasma membrane, pval = 0.76092', 'plasmodesmatal plasma membrane, pval = --', 'plasmodesmatal cytoplasmic sleeve, pval = --', 'plasma membrane protein complex, pval = 0.81411', 'cell trailing edge membrane, pval = --', 'leading edge membrane, pval = 0.03464', 'endomembrane system, pval = 0.00075', 'receptor clustering, pval = --', 'growth cone membrane, pval = --', 'sarcolemma, pval = 0.9286', 'extrinsic component of plasma membrane, pval = 0.93038', 'orthogonal array, pval = --', 'plasma membrane respirasome, pval = --', 'cytoproct, pval = --', 'cytostome, pval = --', 'sperm plasma membrane, pval = --', 'plasma membrane region, pval = 0.29057', 'phagocytic cup, pval = --', 'mesosome, pval = --', 'plasma membrane-derived thylakoid lumen, pval = --', 'plasma membrane ATP synthesis coupled electron transport, pval = --', 'immunological synapse, pval = --', 'protein transport within plasma membrane, pval = --', 'tetraspanin-enriched microdomain, pval = --', 'cell body membrane, pval = --', 'medial membrane band, pval = --', 'cell envelope, pval = --', 'lateral plasma membrane, pval = 0.88858', 'cellular bud membrane, pval = --', 'external side of plasma membrane, pval = 0.91', 'cytoplasmic side of plasma membrane, pval = 0.54382', 'receptor diffusion trapping, pval = --', 'porosome, pval = --', 'unencapsulated part of cell, pval = --', 'plasmodesmatal desmotubule, pval = --', 'cornified envelope, pval = --']
level 2: ['intrinsic component of plasma membrane of cell tip, pval = --', 'integral component of plasma membrane, pval = 0.87053', 'anchored component of plasma membrane, pval = 0.1204', 'intrinsic component of synaptic membrane, pval = 0.94537', 'intrinsic component of external side of plasma membrane, pval = 0.19304', 'intrinsic component of the cytoplasmic side of the plasma membrane, pval = 0.43974', 'MHC protein complex, pval = 0.43974', 'alphav-beta3 integrin-CD47 complex, pval = --', 'MalFGK2 complex, pval = --', 'NADPH oxidase complex, pval = --', 'connexin complex, pval = --', 'flotillin complex, pval = --', 'amino acid transport complex, pval = --', 'SC5b-7 complex, pval = --', 'alphav-beta5 integrin-vitronectin complex, pval = --', 'ATP-binding cassette (ABC) transporter complex, pval = --', 'heterotrimeric G-protein complex, pval = --', 'Wnt-Frizzled-LRP5/6 complex, pval = 0.43974', 'voltage-gated potassium channel complex, pval = --', 'auxin efflux carrier complex, pval = --', 'high-affinity iron permease complex, pval = --', 'subapical complex, pval = --', 'MPP7-DLG1-LIN7 complex, pval = --', 'voltage-gated sodium channel complex, pval = --', 'sucrase-isomaltase complex, pval = --', 'alphav-beta3 integrin-thrombospondin complex, pval = --', 'plasma membrane-derived thylakoid ribulose bisphosphate carboxylase complex, pval = --', 'potassium:proton exchanging ATPase complex, pval = --', 'CFTR-NHERF-ezrin complex, pval = --', 'alphav-beta1 integrin-osteopontin complex, pval = --', 'alphav-beta5 integrin-osteopontin complex, pval = --', 'voltage-gated calcium channel complex, pval = 0.96934', 'sodium:potassium-exchanging ATPase complex, pval = --', 'interleukin-12-interleukin-12 receptor complex, pval = --', 'CD19-Vav-PIK3R1 complex, pval = --', 'alphav-beta3 integrin-osteopontin complex, pval = --', 'plasma membrane proton-transporting V-type ATPase, V0 domain, pval = --', 'plasma membrane proton-transporting V-type ATPase, V1 domain, pval = --', 'alphav-beta3 integrin-PKCalpha complex, pval = 0.19304', 'caveolar macromolecular signaling complex, pval = --', 'inhibin-betaglycan-ActRII complex, pval = 0.43974', 'Fc receptor complex, pval = --', 'alpha4-beta1 integrin-CD47 complex, pval = --', 'alphaIIb-beta3 integrin-ICAM-4 complex, pval = --', 'alpha4-beta4 integrin-EMILIN-1 complex, pval = --', 'alpha6-beta1 integrin-CYR61 complex, pval = --', 'alphaV-beta3 integrin-tumstatin complex, pval = --', 'alpha5-beta1 integrin-endostatin complex, pval = --', 'alphaV-beta3 integrin-NOV complex, pval = --', 'alpha5-beta1 integrin-fibronectin-NOV complex, pval = --', 'alpha7-beta1 integrin-nicotinamide riboside kinase complex, pval = --', 'Grb2-SHP-2 complex, pval = --', 'catenin complex, pval = --', 'alphaPDGFR-SHP-2 complex, pval = --', 'alphaPDGFR-PLC-gamma-1-PI3K-SHP-2 complex, pval = --', 'low-density lipoprotein receptor complex, pval = --', 'alphaV-beta8 integrin-MMP14-TGFbeta-1 complex, pval = --', 'alpha4-beta1 integrin-paxillin complex, pval = --', 'alpha4-beta1 integrin-JAM2 complex, pval = --', 'alphav-beta3 integrin-IGF-1-IGF1R complex, pval = 0.40992', 'alphav-beta3 integrin-HMGB1 complex, pval = 0.43974', 'plasma membrane signaling receptor complex, pval = 0.82502', 'alpha7-beta1 integrin-focal adhesion kinase complex, pval = --', 'alpha9-beta1 integrin-thrombospondin-1 complex, pval = --', 'alphaV-beta3 integrin-CD98 complex, pval = --', 'alpha5-beta5-fibronectin-SFRP2 complex, pval = --', 'alpha7-beta1 integrin-laminin alpha-2 complex, pval = --', 'plasma membrane proton-transporting ATP synthase, stator stalk, pval = --', 'plasma membrane proton-transporting ATP synthase complex, coupling factor F(o), pval = --', 'plasma membrane proton-transporting ATP synthase, catalytic core, pval = --', 'alphaV-beta3 integrin-laminin alpha-4 complex, pval = --', 'plasma membrane proton-transporting ATP synthase complex, catalytic core F(1), pval = --', 'alpha5-beta1 integrin-LPP3 complex, pval = --', 'alpha9-beta1 integrin-ADAM8 complex, pval = --', 'plasma membrane proton-transporting ATP synthase complex, pval = --', 'alphaX-beta2 integrin-ICAM-4 complex, pval = --', 'alpha9-beta1 integrin-VEGF-A complex, pval = --', 'alpha9-beta1 integrin-VEGF-D complex, pval = --', 'alpha1-beta1 integrin-tyrosine-protein phosphatase non-receptor type 2 complex, pval = --', 'alpha9-beta1 integrin-VEGF-C complex, pval = --', 'alphaV-beta6 integrin-osteopontin complex, pval = --', 'alpha9-beta1 integrin-osteopontin complex, pval = --', 'alphaV-beta3 integrin-EGFR complex, pval = --', 'alphaV-beta3 integrin-LPP3 complex, pval = --', '1,3-beta-D-glucan synthase complex, pval = --', 'alpha5-beta1 integrin-osteopontin complex, pval = --', 'Ost-alpha/Ost-beta complex, pval = --', 'plasma membrane respiratory chain complex IV, pval = --', 'plasma membrane respiratory chain complex III, pval = --', 'plasma membrane proton-transporting ATP synthase, central stalk, pval = --', 'alphaIIb-beta3 integrin-CD47-FAK complex, pval = --', 'plasma membrane respiratory chain complex II, pval = --', 'alphaIIb-beta3 integrin-CD47-Src complex, pval = --', 'plasma membrane respiratory chain complex I, pval = --', 'Delta1 complex, pval = --', 'gamma-secretase-Delta1 complex, pval = --', 'gamma-secretase complex, pval = --', 'EGFR-Grb2-Sos complex, pval = --', 'EGFR-Shc-Grb2-Sos complex, pval = --', 'TAT protein transport complex, pval = --', 'alphaIIb-beta3 integrin-CIB complex, pval = --', 'Shc-Grb2-Sos complex, pval = --', 'Grb2-Sos complex, pval = --', 'plasma membrane fumarate reductase complex, pval = --', 'plasma membrane succinate dehydrogenase complex, pval = --', 'protocadherin-alpha-protocadherin-gamma complex, pval = --', 'ciliary necklace, pval = --', 'clathrin coat of coated pit, pval = --', 'AP-2 adaptor complex, pval = --', 'membrane attack complex, pval = --', 'alphaE-beta7 integrin-E-cadherin complex, pval = --', 'alpha9-beta1 integrin-vascular cell adhesion molecule-1 complex, pval = --', 'alpha9-beta1 integrin-ADAM12 complex, pval = --', 'alphav-beta3 integrin-ADAM23 complex, pval = --', 'alpha4-beta1 integrin-thrombospondin-1 complex, pval = --', 'alpha4-beta1 integrin-CD81 complex, pval = --', 'alpha4-beta1 integrin-CD82 complex, pval = --', 'alpha7-beta1 integrin-CD151 complex, pval = --', 'alphav-beta3 integrin-vitronectin complex, pval = 0.43974', 'alpha6-beta4 integrin-CD151 complex, pval = --', 'alpha9-beta1 integrin-ADAM2 complex, pval = --', 'plasma membrane proton-transporting V-type ATPase complex, pval = --', 'alpha9-beta1 integrin-ADAM1 complex, pval = --', 'EmrE multidrug transporter complex, pval = --', 'transmembrane collagen trimer, pval = --', 'alpha9-beta1 integrin-ADAM9 complex, pval = --', 'alpha9-beta1 integrin-ADAM3 complex, pval = --', 'NarGHI complex, pval = --', 'alphav-beta3 integrin-ADAM15 complex, pval = --', 'alpha9-beta1 integrin-ADAM15 complex, pval = --', 'alpha6-beta1 integrin-CD151 complex, pval = --', 'alpha3-beta1 integrin-CD151 complex, pval = --', 'MdtBC Complex, pval = --', 'alphaIIb-beta3 integrin-talin complex, pval = --', 'alpha4-beta1 integrin-CD53 complex, pval = --', 'alpha4-beta1 integrin-CD63 complex, pval = --', 'alphaV-beta3 integrin-tissue transglutaminase complex, pval = --', 'alphaM-beta2 integrin-CD63 complex, pval = --', 'inaD signaling complex, pval = --', 'alpha3-beta1 integrin-CD63 complex, pval = --', 'alpha9-beta1 integrin-tenascin complex, pval = --', 'alpha3-beta1 integrin-basigin complex, pval = --', 'alpha2-beta1 integrin-CD47 complex, pval = --', 'alphaV-beta3 integrin-CD47-FCER2 complex, pval = --', 'alphaIIb-beta3 integrin-CD9-CD47-platelet glycoprotein Ib complex, pval = --', 'alphaIIb-beta3 integrin-CD9 complex, pval = --', 'alpha5-beta1 integrin-tissue transglutaminase complex, pval = --', 'alpha11-beta1 integrin-collagen type I complex, pval = --', 'glycine-gated chloride channel complex, pval = --', 'alpha2-beta1 integrin-chondroadherin complex, pval = --', 'fibronectin-tissue transglutaminase complex, pval = --', 'dystroglycan complex, pval = --', 'dystrophin-associated glycoprotein complex, pval = --', 'dystrobrevin complex, pval = --', 'syntrophin complex, pval = --', 'sarcoglycan complex, pval = --', 'alpha4-beta1 integrin-thrombospondin-2 complex, pval = --', 'Shc-EGFR complex, pval = 0.19304', 'Grb2-EGFR complex, pval = --', 'Grb2-Shc complex, pval = --', 'alphaIIb-beta3 integrin-fibronectin-tissue transglutaminase complex, pval = --', 'alpha3-beta1 integrin-tissue transglutaminase complex, pval = --', 'alpha5-beta1 integrin-fibronectin-tissue transglutaminase complex, pval = --', 'alpha1-beta1 integrin-tissue transglutaminase complex, pval = --', 'alpha3-beta1 integrin-thrombospondin complex, pval = --', 'alpha6-beta4 integrin-CD9 complex, pval = --', 'alphaV-beta3 integrin-paxillin-Pyk2 complex, pval = --', 'alphaV-beta3 integrin-gelsolin complex, pval = --', 'alphaV-beta6 integrin-TGFbeta-3 complex, pval = --', 'alpha6-beta4 integrin-Fyn complex, pval = --', 'interleukin4-interleukin-4 receptor complex, pval = --', 'EFF-1 complex, pval = --', 'Cus cation efflux complex, pval = --', 'platelet-derived growth factor receptor-ligand complex, pval = --', 'D-amino-acid dehydrogenase complex, pval = --', 'alpha6-beta4 integrin-Shc-Grb2 complex, pval = --', 'alpha6-beta4 integrin-laminin 5 complex, pval = --', 'inward rectifying potassium channel, pval = --', 'alphav-beta3 integrin-collagen alpha3(VI) complex, pval = --', 'plasma membrane-derived thylakoid photosystem II, pval = --', 'alpha1-beta1 integrin-alpha3(VI) complex, pval = --', 'alpha2-beta1 integrin-alpha3(VI) complex, pval = --', 'contractile vacuole pore, pval = --', 'plasma membrane-derived photosystem I, pval = --', 'death-inducing signaling complex, pval = 0.22736', 'uropod membrane, pval = --', 'lamellipodium membrane, pval = 0.19304', 'ruffle membrane, pval = 0.02922', 'neuron projection membrane, pval = 0.68643', 'endoplasmic reticulum, pval = 0.10307', 'nuclear envelope, pval = 0.10877', 'endosome, pval = 0.00915', 'spitzenkorper, pval = --', 'Golgi apparatus, pval = 0.02368', 'retromer complex, pval = --', 'omegasome, pval = --', 'secretory granule, pval = 0.05448', 'transport vesicle, pval = 0.49914', 'spine apparatus, pval = --', 'nuclear outer membrane-endoplasmic reticulum membrane network, pval = 0.00106', 'clathrin-coated pit, pval = --', 'skeletal muscle acetylcholine-gated channel clustering, pval = --', 'neurotransmitter-gated ion channel clustering, pval = --', 'regulation of receptor clustering, pval = --', 'negative regulation of receptor clustering, pval = --', 'positive regulation of receptor clustering, pval = --', 'ciliary receptor clustering involved in smoothened signaling pathway, pval = --', 'neuroligin clustering involved in postsynaptic membrane assembly, pval = --', 'neurexin clustering involved in presynaptic membrane assembly, pval = --', 'T-tubule, pval = 0.97378', 'extrinsic component of synaptic membrane, pval = 0.43974', 'extrinsic component of external side of plasma membrane, pval = --', 'extrinsic component of cytoplasmic side of plasma membrane, pval = 0.59248', 'sperm head plasma membrane, pval = --', 'plasma membrane bounded cell projection, pval = 0.51847', 'periciliary membrane compartment, pval = --', 'basal plasma membrane, pval = 0.82468', 'caveola neck, pval = --', 'synaptic membrane, pval = 0.96764', 'cleavage furrow, pval = 0.0846', 'hinge region between urothelial plaques of apical plasma membrane, pval = --', 'plasma membrane-derived thylakoid membrane, pval = --', 'plasma membrane of cell tip, pval = --', 'apical plasma membrane, pval = 0.31529', 'basolateral plasma membrane, pval = 0.85139', 'apicolateral plasma membrane, pval = --', 'plasma membrane raft, pval = 0.40337', 'photoreceptor inner segment membrane, pval = --', 'neurotransmitter receptor diffusion trapping, pval = --', 'cell projection membrane, pval = 0.03096', 'phagocytic cup lip, pval = --', 'phagocytic cup base, pval = --', 'plasma membrane electron transport, NADH to quinone, pval = --', 'protein transport out of plasma membrane raft, pval = --', 'neuronal cell body membrane, pval = --', 'glycocalyx, pval = --', 'outer membrane-bounded periplasmic space, pval = --', 'Gram-negative-bacterium-type cell wall, pval = --', 'cell outer membrane, pval = --', 'periplasmic side of plasma membrane, pval = --', 'external side of apical plasma membrane, pval = --', 'cytoplasmic side of apical plasma membrane, pval = --', 'cytoplasmic side of dendritic spine plasma membrane, pval = --', 'cytoplasmic side of plasma membrane, cell tip, pval = --', 'plasmodesmatal endoplasmic reticulum, pval = --']
level 3: ['interleukin-12 receptor complex, pval = --', 'apical plasma membrane urothelial plaque, pval = --', 'histamine-gated chloride channel complex, pval = --', 'high molecular weight kininogen receptor complex, pval = --', 'pre-T cell receptor complex, pval = --', 'interleukin-18 receptor complex, pval = --', 'interleukin-20 receptor complex, pval = --', 'enzyme IIA-maltose transporter complex, pval = --', 'oncostatin-M receptor complex, pval = --', 'integral component of the cytoplasmic side of the plasma membrane, pval = --', 'granulocyte macrophage colony-stimulating factor receptor complex, pval = --', 'ciliary neurotrophic factor receptor complex, pval = --', 'interleukin-4 receptor complex, pval = --', 'maltose transport complex, pval = --', 'integral component of muscle cell projection membrane, pval = --', 'type I interferon receptor complex, pval = --', 'interleukin-1 receptor complex, pval = --', 'interleukin-28 receptor complex, pval = --', 'interleukin-23 receptor complex, pval = --', 'acetylcholine-gated channel complex, pval = --', 'interleukin-3 receptor complex, pval = --', 'glycoprotein Ib-IX-V complex, pval = --', 'interleukin-2 receptor complex, pval = --', 'interleukin-6 receptor complex, pval = --', 'interleukin-5 receptor complex, pval = --', 'interleukin-13 receptor complex, pval = --', 'interleukin-9 receptor complex, pval = --', 'insulin receptor complex, pval = --', 'spanning component of plasma membrane, pval = --', 'BMP receptor complex, pval = 0.43974', 'G protein-coupled receptor complex, pval = --', 'integral component of external side of plasma membrane, pval = --', 'CD40 receptor complex, pval = 0.82468', 'integrin complex, pval = 0.46882', 'bacterial-type flagellum stator complex, pval = --', 'activin receptor complex, pval = 0.68643', 'ionotropic glutamate receptor complex, pval = --', 'integral component of synaptic membrane, pval = 0.90208', 'anchored component of the cytoplasmic side of the plasma membrane, pval = 0.43974', 'anchored component of synaptic membrane, pval = --', 'anchored component of external side of plasma membrane, pval = 0.19304', 'intrinsic component of postsynaptic membrane, pval = --', 'intrinsic component of presynaptic membrane, pval = 0.68643', 'intrinsic component of periplasmic side of plasma membrane, pval = --', 'MHC class II protein complex, pval = --', 'MHC class I protein complex, pval = 0.43974', 'MHC class Ib protein complex, pval = --', 'ATP-dependent methionine-importing complex, pval = --', 'BtuCD complex, pval = --', 'cobalamin transport complex, pval = --', 'ATP-binding cassette (ABC) transporter complex, substrate-binding subunit-containing, pval = --', 'ATP-binding cassette (ABC) transporter complex, integrated substrate binding, pval = --', 'MsbA transporter complex, pval = --', 'glycerol-3-phosphate-transporting ATPase complex, pval = --', 'ATP-binding cassette (ABC) transporter complex, transmembrane substrate-binding subunit-containing, pval = --', 'ProVWX complex, pval = --', 'Wnt signalosome, pval = 0.94537', 'inward rectifier potassium channel complex, pval = --', 'Kv4.3-DPP10 channel complex, pval = --', 'Kv4.3-DPP6 channel complex, pval = --', 'Kv4.2-DPP6 channel complex, pval = --', 'HCN channel complex, pval = --', 'Kv4.1-DPP6 channel complex, pval = --', 'Kv4.2-Kv4.3 channel complex, pval = --', 'Kv4.1-DPP10 channel complex, pval = --', 'Kv4.2-KChIP3 channel complex, pval = --', 'Kv4.2-KChIP2 channel complex, pval = --', 'Kv4.3-KChIP1 channel complex, pval = --', 'Kv4.2-KChIP4 channel complex, pval = --', 'Kv4.2-KChIP1 channel complex, pval = --', 'CatSper complex, pval = --', 'ryanodine receptor complex, pval = --', 'L-type voltage-gated calcium channel complex, pval = 0.93038', 'light-activated voltage-gated calcium channel complex, pval = --', 'Fc-epsilon receptor I complex, pval = --', 'Fc-alpha receptor I complex, pval = --', 'Fc-gamma receptor I complex, pval = --', 'Fc-gamma receptor III complex, pval = --', 'Toll-like receptor 1-Toll-like receptor 2 protein complex, pval = 0.43974', 'Toll-like receptor 2-Toll-like receptor 6 protein complex, pval = --', 'ERBB4-EGFR complex, pval = --', 'ERBB4:ERBB2 complex, pval = --', 'ERBB3:ERBB2 complex, pval = --', 'EGFR:ERBB2 complex, pval = --', 'ERBB4-ERBB4 complex, pval = --', 'ERBB4-ERBB3 complex, pval = --', 'HFE-transferrin receptor complex, pval = 0.19304', 'transforming growth factor beta ligand-receptor complex, pval = --', 'T cell receptor complex, pval = 0.43974', 'B cell receptor complex, pval = 0.43974', 'neurotransmitter receptor complex, pval = --', 'plasma membrane succinate dehydrogenase complex (ubiquinone), pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-a1 complex, pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-a3 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-a1 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-a3 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-b2 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-b4 complex, pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-b2 complex, pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-b4 complex, pval = --', 'collagen type XIII trimer, pval = --', 'collagen type XXIII trimer, pval = --', 'collagen type XXV trimer, pval = --', 'collagen type XVII trimer, pval = --', 'TRAIL death-inducing signaling complex, pval = --', 'CD95 death-inducing signaling complex, pval = 0.40992', 'axolemma, pval = 0.43974', 'dendrite membrane, pval = --', 'dendritic spine membrane, pval = --', 'stereocilium membrane, pval = --', 'oligosaccharyltransferase complex, pval = --', 'MHC class I peptide loading complex, pval = 0.43974', 'rough endoplasmic reticulum, pval = --', 'smooth endoplasmic reticulum, pval = 0.43974', 'maintenance of protein localization in endoplasmic reticulum, pval = --', 'GPI-anchor transamidase complex, pval = --', 'endoplasmic reticulum calcium ion homeostasis, pval = --', 'endoplasmic reticulum quality control compartment, pval = --', 'egasyn-beta-glucuronidase complex, pval = --', 'glucosidase II complex, pval = --', 'extrinsic component of endoplasmic reticulum membrane, pval = 0.43974', 'VCP-NPL4-UFD1 AAA ATPase complex, pval = --', 'protein glycosylation in endoplasmic reticulum, pval = --', 'Dsl1/NZR complex, pval = --', 'fatty acid elongase complex, pval = --', 'perinuclear endoplasmic reticulum, pval = 0.19304', 'SREBP-SCAP-Insig complex, pval = --', 'membrane stack, pval = --', 'endoplasmic reticulum exit site, pval = --', 'endoplasmic reticulum chaperone complex, pval = 0.43974', 'Ig heavy chain-bound endoplasmic reticulum chaperone complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase complex, pval = --', 'GET complex, pval = --', 'glycosylphosphatidylinositol-mannosyltransferase I complex, pval = --', 'endoplasmic reticulum lumen, pval = 0.87374', 'UDP-N-acetylglucosamine transferase complex, pval = --', 'ERMES complex, pval = --', 'procollagen-proline 4-dioxygenase complex, pval = --', 'sarcoplasmic reticulum, pval = 0.22736', 'endoplasmic reticulum subcompartment, pval = 0.0018', 'glycosylphosphatidylinositol-mannosyltransferase II complex, pval = --', 'ER ubiquitin ligase complex, pval = --', 'microtubule organizing center attachment site, pval = --', 'nuclear membrane, pval = 0.42226', 'nuclear envelope lumen, pval = --', 'nuclear pore, pval = --', 'nucleoplasmic reticulum, pval = --', 'nuclear membrane protein complex, pval = --', 'early endosome, pval = 0.1586', 'tubular endosome, pval = --', 'late endosome, pval = 0.23172', 'endolysosome, pval = --', 'proteolysis within endosome associated with antigen processing and presentation, pval = --', 'endosome lumen, pval = 0.66415', 'presynaptic endosome, pval = --', 'endosome membrane, pval = 0.10877', 'postsynaptic endosome, pval = --', 'endosomal signal transduction, pval = --', 'sorting endosome, pval = --', 'ESCRT complex, pval = --', 'TRAPPII protein complex, pval = --', 'recycling endosome, pval = 0.36956', 'pinosome, pval = --', 'CORVET complex, pval = --', 'EGO complex, pval = --', 'Golgi subcompartment, pval = 0.19764', 'Golgi transport complex, pval = 0.43974', 'Golgi calcium ion homeostasis, pval = --', 'exomer complex, pval = --', 'cis-Golgi network, pval = 0.43974', 'RIC1-RGP1 guanyl-nucleotide exchange factor complex, pval = --', 'postsynaptic Golgi apparatus, pval = --', 'TRAPPIII protein complex, pval = --', 'inositol phosphoceramide synthase complex, pval = --', 'protein glycosylation in Golgi, pval = --', 'Golgi-associated vesicle lumen, pval = --', 'Golgi lumen, pval = 0.89614', 'Golgi membrane, pval = 0.14987', 'Golgi apparatus mannose trimming, pval = --', 'GARP complex, pval = --', 'clathrin coat of trans-Golgi network vesicle, pval = --', 'COPI vesicle coat, pval = --', 'acroblast, pval = --', 'retromer, tubulation complex, pval = --', 'retromer, cargo-selective complex, pval = --', 'mucin granule, pval = --', 'dense core granule, pval = --', 'platelet alpha granule, pval = 0.60334', 'acrosomal vesicle, pval = 0.40992', 'zymogen granule, pval = --', 'chromaffin granule, pval = --', 'azurophil granule, pval = 0.68643', 'specific granule, pval = 0.03701', 'lamellar body, pval = --', 'tertiary granule, pval = 0.01521', 'mucocyst, pval = --', 'cortical granule, pval = --', 'platelet dense granule, pval = --', 'secretory granule membrane, pval = 0.00058', 'ficolin-1-rich granule, pval = 0.23999', 'Weibel-Palade body, pval = --', 'insulin-responsive compartment, pval = --', 'secretory granule lumen, pval = 0.37117', 'exocytic vesicle, pval = --', 'clathrin-sculpted monoamine transport vesicle, pval = --', 'clathrin-sculpted acetylcholine transport vesicle, pval = --', 'COPI-coated Golgi to ER transport vesicle, pval = --', 'COPI-coated inter-Golgi transport vesicle, pval = --', 'transport vesicle lumen, pval = --', 'trans-Golgi network transport vesicle, pval = --', 'transport vesicle membrane, pval = 0.68643', 'clathrin-sculpted glutamate transport vesicle, pval = --', 'spine apparatus lumen, pval = --', 'spine apparatus membrane, pval = --', 'annulate lamellae, pval = --', 'endoplasmic reticulum membrane, pval = 0.0018', 'nuclear outer membrane, pval = --', 'perinuclear endoplasmic reticulum membrane, pval = 0.43974', 'Nem1-Spo7 phosphatase complex, pval = --', 'negative regulation of skeletal muscle acetylcholine-gated channel clustering, pval = --', 'regulation of skeletal muscle acetylcholine-gated channel clustering, pval = --', 'positive regulation of skeletal muscle acetylcholine-gated channel clustering, pval = --', 'NMDA glutamate receptor clustering, pval = --', 'gamma-aminobutyric acid receptor clustering, pval = --', 'glycine receptor clustering, pval = --', 'glutamate receptor clustering, pval = --', 'regulation of glutamate receptor clustering, pval = --', 'negative regulation of AMPA glutamate receptor clustering, pval = --', 'positive regulation of AMPA glutamate receptor clustering, pval = --', 'extrinsic component of presynaptic membrane, pval = --', 'extrinsic component of postsynaptic membrane, pval = 0.43974', 'extrinsic component of periplasmic side of plasma membrane, pval = --', 'subplasmalemmal coating, pval = --', 'neuron projection, pval = 0.33088', 'pseudopodium, pval = 0.03701', 'cilium, pval = 0.75262', 'glial cell projection, pval = 0.43974', 'plasma membrane bounded cell projection cytoplasm, pval = 0.46882', 'ruffle, pval = 0.10178', 'cytoneme, pval = --', 'mating projection, pval = --', 'microspike, pval = --', 'invadopodium, pval = 0.43974', 'bleb, pval = --', 'root hair, pval = --', 'pollen tube, pval = --', 'dendritic cell dendrite, pval = --', 'podocyte foot, pval = --', 'actin-based cell projection, pval = 0.1586', 'podocyte primary projection, pval = --', 'muscle cell projection, pval = --', 'macropinocytic cup, pval = --', 'lamellipodium, pval = 0.32699', 'rhabdomere, pval = --', 'uropod, pval = --', 'dinoflagellate peduncle, pval = --', 'cell hair, pval = --', 'postsynaptic specialization membrane, pval = 0.90208', 'postsynaptic membrane, pval = 0.97378', 'presynaptic endocytic zone membrane, pval = --', 'postsynaptic endocytic zone membrane, pval = --', 'presynaptic periactive zone, pval = --', 'presynaptic active zone membrane, pval = 0.43974', 'presynaptic membrane, pval = 0.94537', 'cleavage furrow leading edge, pval = --', 'cleavage furrow rim, pval = --', 'cellularization cleavage furrow, pval = --', 'plasma membrane of growing cell tip, pval = --', 'mating projection tip membrane, pval = --', 'intracellular canaliculus, pval = --', 'eisosome membrane domain/MCC, pval = --', 'caveola, pval = 0.42226', 'postsynaptic neurotransmitter receptor diffusion trapping, pval = --', 'mating projection membrane, pval = --', 'invadopodium membrane, pval = 0.43974', 'rhabdomere membrane, pval = --', 'brush border membrane, pval = --', 'microvillus membrane, pval = 0.40992', 'filopodium membrane, pval = 0.03701', 'muscle cell projection membrane, pval = --', 'macropinocytic cup membrane, pval = --', 'pseudopodium membrane, pval = --', 'ciliary membrane, pval = 0.3878', 'slime layer, pval = --', 'capsule, pval = --', 'mycolate outer membrane, pval = --', 'curli secretion complex, pval = --', 'intrinsic component of cell outer membrane, pval = --', 'extrinsic component of cell outer membrane, pval = --', 'periplasmic side of cell outer membrane, pval = --', 'external side of cell outer membrane, pval = --', 'starch utilization system complex, pval = --']
level 4: ['calcitonin family receptor complex, pval = --', 'G protein-coupled receptor dimeric complex, pval = --', 'integral component of periplasmic side of plasma membrane, pval = --', 'integrin alphaIIb-beta3 complex, pval = --', 'integrin alpha9-beta1 complex, pval = 0.43974', 'integrin alpha8-beta1 complex, pval = --', 'integrin alpha6-beta1 complex, pval = --', 'integrin alpha5-beta1 complex, pval = --', 'integrin alpha7-beta1 complex, pval = --', 'integrin alpha6-beta4 complex, pval = --', 'integrin alpha2-beta1 complex, pval = --', 'integrin alpha1-beta1 complex, pval = --', 'integrin alpha4-beta1 complex, pval = --', 'integrin alpha3-beta1 complex, pval = 0.43974', 'integrin alpha4-beta7 complex, pval = --', 'integrin alphaD-beta2 complex, pval = --', 'integrin alphaE-beta7 complex, pval = --', 'integrin alphav-beta3 complex, pval = 0.43974', 'integrin alphav-beta1 complex, pval = --', 'integrin alphav-beta8 complex, pval = 0.43974', 'integrin alphav-beta6 complex, pval = --', 'integrin alphav-beta5 complex, pval = --', 'integrin alphaM-beta2 complex, pval = --', 'integrin alphaL-beta2 complex, pval = --', 'integrin alphaX-beta2 complex, pval = --', 'integrin alpha11-beta1 complex, pval = --', 'integrin alpha10-beta1 complex, pval = --', 'NMDA selective glutamate receptor complex, pval = --', 'AMPA glutamate receptor complex, pval = --', 'kainate selective glutamate receptor complex, pval = --', 'integral component of presynaptic membrane, pval = 0.43974', 'integral component of postsynaptic membrane, pval = --', 'anchored component of postsynaptic membrane, pval = --', 'anchored component of presynaptic membrane, pval = --', 'intrinsic component of postsynaptic endocytic zone membrane, pval = --', 'intrinsic component of postsynaptic specialization membrane, pval = --', 'intrinsic component of presynaptic active zone membrane, pval = 0.43974', 'intrinsic component of presynaptic endocytic zone membrane, pval = --', 'I(KACh) inward rectifier potassium channel complex, pval = --', 'Kir2 inward rectifier potassium channel complex, pval = --', 'TCR signalosome, pval = --', 'gamma-delta T cell receptor complex, pval = --', 'alpha-beta T cell receptor complex, pval = --', 'IgG B cell receptor complex, pval = --', 'IgM B cell receptor complex, pval = --', 'IgA B cell receptor complex, pval = --', 'IgE B cell receptor complex, pval = --', 'IgD B cell receptor complex, pval = --', 'G protein-coupled GABA receptor complex, pval = --', 'G protein-coupled serotonin receptor complex, pval = --', 'serotonin-activated cation-selective channel complex, pval = --', 'oligosaccharyltransferase I complex, pval = --', 'oligosaccharyltransferase II complex, pval = --', 'oligosaccharyltransferase III complex, pval = --', 'Tapasin-ERp57 complex, pval = --', 'TAP complex, pval = --', 'rough endoplasmic reticulum membrane, pval = --', 'rough endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum membrane, pval = --', 'smooth endoplasmic reticulum calcium ion homeostasis, pval = --', 'smooth endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum cisterna, pval = --', 'protein retention in ER lumen, pval = --', 'SREBP-SCAP complex retention in endoplasmic reticulum, pval = --', 'positive regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'endoplasmic reticulum mannose trimming, pval = --', 'protein galactosylation in endoplasmic reticulum, pval = --', 'perinuclear endoplasmic reticulum lumen, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt4p homodimer complex, pval = --', 'sarcoplasmic reticulum lumen, pval = --', 'cortical endoplasmic reticulum lumen, pval = --', 'luminal surveillance complex, pval = --', 'mannosyl-oligosaccharide 1,2-alpha-mannosidase complex, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(III) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(II) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(I) type, pval = --', 'sarcoplasmic reticulum membrane, pval = 0.43974', 'longitudinal sarcoplasmic reticulum, pval = --', 'terminal cisterna, pval = --', 'subrhabdomeral cisterna, pval = --', 'endoplasmic reticulum tubular network, pval = --', 'endoplasmic reticulum cisternal network, pval = --', 'CUE1-UBC7 ubiquitin-conjugating enzyme complex, pval = --', 'Hrd1p ubiquitin ligase complex, pval = --', 'Doa10p ubiquitin ligase complex, pval = --', 'meiotic nuclear membrane microtubule tethering complex, pval = --', 'nuclear inner membrane, pval = 0.43974', '5-lipoxygenase complex, pval = --', 'nuclear pore outer ring, pval = --', 'nucleoplasmic side of nuclear pore, pval = --', 'cytoplasmic side of nuclear pore, pval = --', 'structural constituent of nuclear pore, pval = --', 'nuclear pore inner ring, pval = --', 'nuclear pore central transport channel, pval = --', 'nuclear pore linkers, pval = --', 'nuclear pore nuclear basket, pval = --', 'nuclear pore cytoplasmic filaments, pval = --', 'nuclear pore transmembrane ring, pval = --', 'nuclear membrane mitotic spindle pole body tethering complex, pval = --', 'nuclear membrane microtubule tethering complex, pval = --', 'early endosome membrane, pval = 0.36956', 'early endosome lumen, pval = 0.43974', 'postsynaptic early endosome, pval = --', 'multivesicular body, pval = 0.19304', 'late endosome lumen, pval = --', 'late endosome membrane, pval = 0.59248', 'Mon1-Ccz1 complex, pval = --', 'endolysosome membrane, pval = --', 'endolysosome lumen, pval = --', 'recycling endosome lumen, pval = --', 'presynaptic endosome membrane, pval = --', 'postsynaptic endosome membrane, pval = --', 'ESCRT IV complex, pval = --', 'recycling endosome membrane, pval = 0.3878', 'Vps55/Vps68 complex, pval = --', 'ESCRT III complex, pval = --', 'ESCRT II complex, pval = --', 'ESCRT I complex, pval = --', 'intrinsic component of endosome membrane, pval = --', 'extrinsic component of endosome membrane, pval = --', 'intralumenal vesicle formation, pval = --', 'cytoplasmic side of endosome membrane, pval = --', 'lumenal side of endosome membrane, pval = --', 'postsynaptic recycling endosome, pval = --', 'ESCRT-0 complex, pval = --', 'basolateral recycling endosome, pval = --', 'apical recycling endosome, pval = --', 'peri-centrosomal recycling endosome, pval = --', 'EARP complex, pval = --', 'micropinosome, pval = --', 'macropinosome, pval = --', 'trans-Golgi network, pval = 0.59248', 'Golgi cisterna, pval = 0.43974', 'Golgi stack, pval = 0.43974', 'cis-Golgi network membrane, pval = --', 'negative regulation of protein glycosylation in Golgi, pval = --', 'positive regulation of protein glycosylation in Golgi, pval = --', 'regulation of protein glycosylation in Golgi, pval = --', 'protein galactosylation in Golgi, pval = --', 'Golgi stack lumen, pval = --', 'trans-Golgi network transport vesicle lumen, pval = --', 'Golgi vesicle budding, pval = --', 'intrinsic component of Golgi membrane, pval = 0.68643', 'extrinsic component of Golgi membrane, pval = --', 'lumenal side of Golgi membrane, pval = --', 'cytoplasmic side of Golgi membrane, pval = --', 'trans-Golgi network membrane, pval = 0.68643', 'Golgi cisterna membrane, pval = 0.43974', 'AP-1 adaptor complex, pval = --', 'dense core granule lumen, pval = --', 'neuronal dense core vesicle, pval = --', 'dense core granule membrane, pval = --', 'platelet alpha granule membrane, pval = 0.43974', 'platelet alpha granule lumen, pval = 0.815', 'acrosomal membrane, pval = 0.43974', 'acrosomal matrix, pval = --', 'acrosomal lumen, pval = --', 'zymogen granule membrane, pval = --', 'chromaffin granule membrane, pval = --', 'chromaffin granule lumen, pval = --', 'azurophil granule membrane, pval = --', 'azurophil granule lumen, pval = 0.68643', 'specific granule lumen, pval = 0.19304', 'specific granule membrane, pval = 0.19304', 'epidermal lamellar body, pval = --', 'alveolar lamellar body, pval = --', 'dendritic lamellar body, pval = --', 'lamellar body membrane, pval = --', 'tertiary granule membrane, pval = 0.0846', 'tertiary granule lumen, pval = 0.40992', 'ficolin-1-rich granule membrane, pval = 0.19304', 'platelet dense granule membrane, pval = --', 'platelet dense granule lumen, pval = --', 'ficolin-1-rich granule lumen, pval = 0.5379', 'Golgi to plasma membrane transport vesicle, pval = --', 'synaptic vesicle, pval = --', 'endosome to plasma membrane transport vesicle, pval = --', 'clathrin-sculpted monoamine transport vesicle membrane, pval = --', 'clathrin-sculpted monoamine transport vesicle lumen, pval = --', 'clathrin-sculpted acetylcholine transport vesicle membrane, pval = --', 'clathrin-sculpted acetylcholine transport vesicle lumen, pval = --', 'Golgi to ER transport vesicle membrane, pval = --', 'inter-Golgi transport vesicle membrane, pval = --', 'trans-Golgi network transport vesicle membrane, pval = --', 'ER to Golgi transport vesicle membrane, pval = 0.43974', 'exocytic vesicle membrane, pval = --', 'clathrin-sculpted glutamate transport vesicle membrane, pval = --', 'lumenal side of transport vesicle membrane, pval = --', 'cytoplasmic side of transport vesicle membrane, pval = --', 'clathrin-sculpted glutamate transport vesicle lumen, pval = --', 'extrinsic component of spine apparatus membrane, pval = --', 'intrinsic component of spine apparatus membrane, pval = --', 'endoplasmic reticulum palmitoyltransferase complex, pval = --', 'intrinsic component of endoplasmic reticulum membrane, pval = 0.43974', 'Derlin-1 retrotranslocation complex, pval = --', 'Derlin-1-VIMP complex, pval = --', 'glycosylphosphatidylinositol-N-acetylglucosaminyltransferase (GPI-GnT) complex, pval = --', 'lumenal side of endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of endoplasmic reticulum membrane, pval = --', 'IRE1-TRAF2-ASK1 complex, pval = 0.43974', 'signal peptidase complex, pval = --', 'Ire1 complex, pval = --', 'intrinsic component of nuclear outer membrane, pval = --', 'extrinsic component of nuclear outer membrane, pval = --', 'AMPA glutamate receptor clustering, pval = --', 'regulation of AMPA glutamate receptor clustering, pval = --', 'extrinsic component of presynaptic endocytic zone membrane, pval = --', 'extrinsic component of presynaptic active zone membrane, pval = --', 'extrinsic component of postsynaptic endocytic zone, pval = --', 'extrinsic component of postsynaptic specialization membrane, pval = 0.43974', 'telodendria, pval = --', 'kinocilium, pval = 0.43974', 'dendrite, pval = 0.30606', 'axon, pval = 0.35954', 'growth cone lamellipodium, pval = --', 'proximal neuron projection, pval = --', 'dendritic filopodium, pval = --', 'photoreceptor cell cilium, pval = 0.59248', 'cell body fiber, pval = --', 'neuron projection terminus, pval = --', 'dendritic tree, pval = 0.30606', 'axo-dendritic transport, pval = --', 'Lewy neurite, pval = --', 'gemmule, pval = --', 'neuron projection cytoplasm, pval = 0.3878', 'stereocilium, pval = 0.43974', 'accessory outer segment, pval = --', 'neuron spine, pval = 0.08911', 'neuron projection branch point, pval = --', 'rod spherule, pval = --', 'growth cone filopodium, pval = 0.43974', 'ciliary basal body, pval = 0.90208', 'motile cilium, pval = 0.43974', 'non-motile cilium, pval = 0.3878', 'intraciliary transport, pval = --', 'ciliary rootlet, pval = --', 'ciliary plasm, pval = 0.72894', 'ring centriole, pval = --', 'ciliary cap, pval = --', 'centriolar subdistal appendage, pval = --', 'ciliary pocket, pval = --', 'ciliary transition zone, pval = --', 'BBSome, pval = --', 'ciliary transition fiber, pval = --', 'ciliary tip, pval = 0.59248', 'ciliary shaft, pval = --', 'ciliary inversin compartment, pval = --', 'ciliary base, pval = 0.76316', 'capitate projection, pval = --', 'astrocyte projection, pval = --', 'spiny bracelet of Nageotte, pval = --', 'terminal loop, pval = --', 'Schwann cell microvillus, pval = --', 'mating projection tip, pval = --', 'mating projection base, pval = --', 'mating projection septin band, pval = --', 'mating projection septin ring, pval = --', 'root hair tip, pval = --', 'pollen tube tip, pval = --', 'microvillus, pval = 0.36956', 'actin filament bundle of actin-based cell projection, pval = --', 'cluster of actin-based cell projections, pval = 0.68643', 'filopodium, pval = 0.23999', 'macropinocytic cup cytoskeleton, pval = --', 'leading edge of lamellipodium, pval = --', 'rhabdomere microvillus, pval = --', 'postsynaptic density membrane, pval = 0.90208', 'postsynaptic specialization membrane of symmetric synapse, pval = --', 'G protein-coupled receptor activity involved in regulation of postsynaptic membrane potential, pval = --', 'ATPase coupled ion transmembrane transporter activity involved in regulation of postsynaptic membrane potential, pval = --', 'ion antiporter activity involved in regulation of postsynaptic membrane potential, pval = --', 'postsynaptic neurotransmitter receptor activity, pval = --', 'voltage-gated ion channel activity involved in regulation of presynaptic membrane potential, pval = --', 'ligand-gated ion channel activity involved in regulation of presynaptic membrane potential, pval = --', 'ATPase coupled ion transmembrane transporter activity involved in regulation of presynaptic membrane potential, pval = --', 'ion antiporter activity involved in regulation of presynaptic membrane potential, pval = --', 'neurotransmitter receptor activity involved in regulation of presynaptic cytosolic calcium ion concentration, pval = --', 'G protein-coupled neurotransmitter receptor activity involved in regulation of presynaptic membrane potential, pval = --', 'cellularization cleavage furrow invagination front, pval = --', 'regulation of postsynaptic neurotransmitter receptor diffusion trapping, pval = --', 'rhabdomere microvillus membrane, pval = --', 'cone photoreceptor disc membrane, pval = --', 'ciliary pocket membrane, pval = --', 'non-motile cilium membrane, pval = --', 'photoreceptor outer segment membrane, pval = 0.43974', 'extrinsic component of mycolate outer membrane, pval = --', 'intrinsic component of mycolate outer membrane, pval = --', 'integral component of cell outer membrane, pval = --', 'intrinsic component of external side of cell outer membrane, pval = --', 'intrinsic component of periplasmic side of cell outer membrane, pval = --', 'anchored component of cell outer membrane, pval = --', 'extrinsic component of external side of cell outer membrane, pval = --', 'extrinsic component of periplasmic side of cell outer membrane, pval = --', 'external side of mycolate outer membrane, pval = --']
level 5: ['amylin receptor complex, pval = --', 'adrenomedullin receptor complex, pval = --', 'CGRP receptor complex, pval = --', 'G protein-coupled receptor heterodimeric complex, pval = --', 'G protein-coupled receptor homodimeric complex, pval = --', 'integral component of presynaptic active zone membrane, pval = 0.43974', 'integral component of presynaptic endocytic zone membrane, pval = --', 'integral component of postsynaptic endocytic zone membrane, pval = --', 'integral component of postsynaptic specialization membrane, pval = --', 'anchored component of postynaptic endocytic zone membrane, pval = --', 'anchored component of postsynaptic specialization membrane, pval = --', 'anchored component of presynaptic active zone membrane, pval = --', 'anchored component of presynaptic endocytic zone membrane, pval = --', 'intrinsic component of postsynaptic density membrane, pval = --', 'intrinsic component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'signal recognition particle receptor complex, pval = --', 'translocon complex, pval = --', 'endoplasmic reticulum Sec complex, pval = --', 'lumenal side of rough endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of rough endoplasmic reticulum membrane, pval = --', 'lumenal side of smooth endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of smooth endoplasmic reticulum membrane, pval = --', 'positive regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'hypolemmal cisterna, pval = --', 'trimming of terminal mannose on B branch, pval = --', 'trimming of second mannose on A branch, pval = --', 'trimming of first mannose on A branch, pval = --', 'trimming of terminal mannose on C branch, pval = --', 'longitudinal sarcoplasmic reticulum lumen, pval = --', 'terminal cisterna lumen, pval = --', 'phospholamban complex, pval = --', 'junctional sarcoplasmic reticulum membrane, pval = --', 'free sarcoplasmic reticulum membrane, pval = --', 'cortical endoplasmic reticulum, pval = --', 'rough endoplasmic reticulum cisterna, pval = --', 'Hrd1p ubiquitin ligase ERAD-M complex, pval = --', 'Hrd1p ubiquitin ligase ERAD-L complex, pval = --', 'Sad1-Kms1 LINC complex, pval = --', 'intrinsic component of nuclear inner membrane, pval = --', 'Asi complex, pval = --', 'cytoplasmic side of early endosome membrane, pval = --', 'lumenal side of early endosome membrane, pval = --', 'postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle, pval = 0.43974', 'multivesicular body lumen, pval = --', 'multivesicular body membrane, pval = --', 'endosomal scaffold complex, pval = --', 'Ragulator complex, pval = --', 'cytoplasmic side of late endosome membrane, pval = --', 'lumenal side of late endosome membrane, pval = --', 'intrinsic component of presynaptic endosome membrane, pval = --', 'extrinsic component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic endosome membrane, pval = --', 'postsynaptic recycling endosome membrane, pval = --', 'integral component of endosome membrane, pval = --', 'positive regulation of intralumenal vesicle formation, pval = --', 'negative regulation of intralumenal vesicle formation, pval = --', 'regulation of intralumenal vesicle formation, pval = --', 'Golgi medial cisterna, pval = --', 'Golgi trans cisterna, pval = --', 'Golgi cis cisterna, pval = --', 'COPII vesicles tethering complex, pval = --', 'Golgi transport vesicle coating, pval = --', 'COPII-coated vesicle budding, pval = --', 'Golgi vesicle bud deformation and release, pval = --', 'COPI-coated vesicle budding, pval = --', 'intrinsic component of Golgi trans cisterna membrane, pval = --', 'intrinsic component of Golgi medial cisterna membrane, pval = --', 'intrinsic component of Golgi cis cisterna membrane, pval = --', 'integral component of Golgi membrane, pval = 0.68643', 'Golgi medial cisterna membrane, pval = --', 'Golgi cis cisterna membrane, pval = --', 'Golgi trans cisterna membrane, pval = --', 'neuronal dense core vesicle lumen, pval = --', 'neuronal dense core vesicle membrane, pval = --', 'extrinsic component of dense core granule membrane, pval = --', 'intrinsic component of dense core granule membrane, pval = --', 'protein C inhibitor-acrosin complex, pval = --', 'outer acrosomal membrane, pval = --', 'inner acrosomal membrane, pval = --', 'epidermal lamellar body membrane, pval = --', 'alveolar lamellar body membrane, pval = --', 'synaptic vesicle lumen, pval = --', 'synaptic vesicle, resting pool, pval = --', 'synaptic vesicle, readily releasable pool, pval = --', 'synaptic vesicle, recycling pool, pval = --', 'synaptic vesicle membrane, pval = --', 'cytoplasmic side of trans-Golgi network transport vesicle membrane, pval = --', 'lumenal side of trans-Golgi network transport vesicle membrane, pval = --', 'COPII vesicle coat, pval = --', 'anchored component of spine apparatus membrane, pval = --', 'integral component of spine apparatus membrane, pval = --', 'serine C-palmitoyltransferase complex, pval = --', 'integral component of endoplasmic reticulum membrane, pval = 0.43974', 'integral component of lumenal side of endoplasmic reticulum membrane, pval = --', 'integral component of cytoplasmic side of endoplasmic reticulum membrane, pval = --', 'integral component of nuclear outer membrane, pval = --', 'extrinsic component of postsynaptic density membrane, pval = 0.43974', 'extrinsic component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'rod teleodendria, pval = --', 'cone teleodendria, pval = --', 'kinociliary basal body, pval = 0.43974', 'dendrite terminus, pval = 0.68643', 'sensory dendrite, pval = --', 'dendritic microtubule, pval = --', 'dendrite cytoplasm, pval = 0.82468', 'dendritic transport, pval = --', 'proximal dendrite, pval = --', 'primary dendrite, pval = --', 'distal dendrite, pval = --', 'apical dendrite, pval = 0.68643', 'CA3 pyramidal cell dendrite, pval = --', 'basal dendrite, pval = 0.43974', 'dendritic shaft, pval = 0.19304', 'dendritic spine, pval = 0.08911', 'dendritic branch, pval = --', 'giant axon, pval = --', 'axon microtubule bundle, pval = --', 'parallel fiber, pval = --', 'axonal transport, pval = --', 'axon cytoplasm, pval = 0.40992', 'distal axon, pval = 0.49131', 'C-fiber, pval = --', 'axonal dopamine secretion, pval = --', 'axon hillock, pval = --', 'axonal spine, pval = --', 'main axon, pval = 0.40992', 'axon collateral, pval = --', 'pericellular basket, pval = --', 'pinceau fiber, pval = --', 'dense core granule cytoskeletal transport, pval = --', 'cerebellar mossy fiber, pval = --', 'climbing fiber, pval = --', 'dentate gyrus mossy fiber, pval = --', 'photoreceptor outer segment, pval = 0.59248', 'photoreceptor connecting cilium, pval = --', 'axon terminus, pval = --', 'dendritic branch point, pval = --', 'slow axonal transport, pval = --', 'axo-dendritic protein transport, pval = --', 'neuronal ribonucleoprotein granule, pval = --', 'actin filament bundle of stereocilium, pval = --', 'stereocilium bundle, pval = 0.43974', 'stereocilium tip, pval = 0.43974', 'stereocilium shaft, pval = --', 'stereocilium base, pval = --', 'spine mat, pval = --', 'somatic spine, pval = --', 'right posteriolateral basal body, pval = --', 'right anterior basal body, pval = --', 'left posteriolateral basal body, pval = --', 'right ventral basal body, pval = --', 'left ventral basal body, pval = --', 'right caudal basal body, pval = --', 'left caudal basal body, pval = --', 'left anterior basal body, pval = --', 'mastigoneme, pval = --', '9+2 motile cilium, pval = 0.43974', '9+0 motile cilium, pval = --', '9+2 non-motile cilium, pval = 0.43974', '9+0 non-motile cilium, pval = 0.59248', 'intraciliary transport involved in cilium assembly, pval = --', 'intraciliary retrograde transport, pval = --', 'intraciliary anterograde transport, pval = --', 'axoneme, pval = 0.72894', 'ciliary pocket collar, pval = --', 'central cylinder, pval = --', 'NPHP complex, pval = --', 'MKS complex, pval = --', 'Y-shaped link, pval = --', 'astrocyte end-foot, pval = --', 'mating projection tip polarisome, pval = --', 'actin fusion focus, pval = --', 'brush border, pval = --', 'microvillar actin bundle, pval = --', 'actin filament bundle of filopodium, pval = --', 'filopodium tip, pval = --', 'G protein-coupled neurotransmitter receptor activity involved in regulation of postsynaptic membrane potential, pval = --', 'acetylcholine receptor activity, pval = --', 'regulation of postsynaptic neurotransmitter receptor activity, pval = --', 'neurotransmitter receptor activity involved in regulation of postsynaptic membrane potential, pval = --', 'neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration, pval = --', 'dopamine neurotransmitter receptor activity, pval = --', 'extrinsic component of external side of mycolate outer membrane, pval = --', 'integral component of mycolate outer membrane, pval = --', 'Bam protein complex, pval = --', 'anchored component of periplasmic side of cell outer membrane, pval = --']
level 6: ['amylin receptor complex 1, pval = --', 'amylin receptor complex 3, pval = --', 'amylin receptor complex 2, pval = --', 'integral component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'integral component of postsynaptic density membrane, pval = --', 'anchored component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'anchored component of postsynaptic density membrane, pval = --', 'Sec61 translocon complex, pval = --', 'Ssh1 translocon complex, pval = --', 'Sec62/Sec63 complex, pval = --', 'sub-surface cisterna, pval = --', 'integral component of nuclear inner membrane, pval = --', 'intrinsic component of postsynaptic early endosome membrane, pval = --', 'extrinsic component of postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle membrane, pval = --', 'multivesicular body, internal vesicle lumen, pval = 0.43974', 'multivesicular body HOPS complex, pval = --', 'anchored component of presynaptic endosome membrane, pval = --', 'integral component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic recycling endosome membrane, pval = --', 'anchored component of postsynaptic endosome membrane, pval = --', 'integral component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic recycling endosome membrane, pval = --', 'alpha-1,6-mannosyltransferase complex, pval = --', 'Golgi membrane priming complex assembly, pval = --', 'clathrin coating of Golgi vesicle, pval = --', 'COPI coating of Golgi vesicle, pval = --', 'Golgi membrane coat protein complex assembly, pval = --', 'COPII-coated vesicle cargo loading, pval = --', 'COPII vesicle coating, pval = --', 'integral component of Golgi trans cisterna membrane, pval = --', 'integral component of Golgi medial cisterna membrane, pval = --', 'integral component of Golgi cis cisterna membrane, pval = --', 'intrinsic component of neuronal dense core vesicle membrane, pval = --', 'extrinsic component of neuronal dense core vesicle membrane, pval = --', 'clathrin coat of synaptic vesicle, pval = --', 'extrinsic component of synaptic vesicle membrane, pval = --', 'intrinsic component of synaptic vesicle membrane, pval = --', 'SPOTS complex, pval = --', 'homodimeric serine palmitoyltransferase complex, pval = --', 'ER membrane protein complex, pval = --', 'dendriole, pval = --', 'dendritic growth cone, pval = 0.68643', 'dendritic tuft, pval = --', 'dendritic knob, pval = --', 'dendritic transport of mitochondrion, pval = --', 'retrograde dendritic transport, pval = --', 'anterograde dendritic transport, pval = --', 'dendritic transport of ribonucleoprotein complex, pval = --', 'dendritic spine cytoplasm, pval = --', 'apical proximal dendrite, pval = --', 'basal proximal dendrite, pval = --', 'apical distal dendrite, pval = --', 'basal distal dendrite, pval = --', 'cytoskeleton of dendritic spine, pval = --', 'dendritic spine head, pval = --', 'dendritic spine neck, pval = --', 'thorny excrescence, pval = --', 'axonal transport of mitochondrion, pval = --', 'axonal transport of messenger ribonucleoprotein complex, pval = --', 'anterograde axonal transport, pval = --', 'retrograde axonal transport, pval = --', 'growth cone, pval = 0.05169', 'dopamine secretion, neurotransmission, pval = --', 'internode region of axon, pval = --', 'node of Ranvier, pval = --', 'paranode region of axon, pval = --', 'juxtaparanode region of axon, pval = --', 'varicosity, pval = --', 'axon initial segment, pval = --', 'Schaffer axon collateral, pval = --', 'recurrent axon collateral, pval = --', 'retrograde neuronal dense core vesicle transport, pval = --', 'anterograde neuronal dense core vesicle transport, pval = --', 'hippocampal mossy fiber, pval = --', 'rod photoreceptor outer segment, pval = --', 'cone photoreceptor outer segment, pval = --', 'photoreceptor disc membrane, pval = --', 'photoreceptor proximal connecting cilium, pval = --', 'photoreceptor distal connecting cilium, pval = --', 'calyx of Held, pval = --', 'cone cell pedicle, pval = --', 'vestibular calyx terminal, pval = --', 'Herring body, pval = --', 'terminal bouton, pval = --', 'retrograde axonal protein transport, pval = --', 'anterograde axonal protein transport, pval = --', 'positive regulation of axo-dendritic protein transport, pval = --', 'regulation of axo-dendritic protein transport, pval = --', 'negative regulation of axo-dendritic protein transport, pval = --', 'anterograde dendritic transport of neurotransmitter receptor complex, pval = --', 'stereocilia coupling link, pval = --', 'right middle basal body pair, pval = --', 'right lateral basal body pair, pval = --', 'left middle basal body pair, pval = --', 'left lateral basal body pair, pval = --', 'paraflagellar rod, pval = --', 'transverse flagellum, pval = --', 'longitudinal flagellum, pval = --', 'sperm flagellum, pval = 0.43974', 'right ventral flagellum, pval = --', 'right posteriolateral flagellum, pval = --', 'left ventral flagellum, pval = --', 'left anterior flagellum, pval = --', 'right anterior flagellum, pval = --', 'left posteriolateral flagellum, pval = --', 'right caudal flagellum, pval = --', 'left caudal flagellum, pval = --', 'cilium assembly, pval = --', 'positive regulation of intraciliary retrograde transport, pval = --', 'negative regulation of intraciliary retrograde transport, pval = --', 'regulation of intraciliary retrograde transport, pval = --', 'regulation of intraciliary anterograde transport, pval = --', 'negative regulation of intraciliary anterograde transport, pval = --', 'positive regulation of intraciliary anterograde transport, pval = --', 'axonemal heterotrimeric kinesin-II complex, pval = --', 'axonemal nexin link, pval = --', 'axonemal central apparatus, pval = --', 'axonemal dynein complex, pval = --', 'axonemal microtubule, pval = --', 'distal portion of axoneme, pval = --', 'proximal portion of axoneme, pval = --', 'radial spoke, pval = --', 'axonemal microtubule depolymerization, pval = --', 'satellite fibril, pval = --', 'axonemal basal plate, pval = --', 'axonemal outer doublet, pval = --', 'glial limiting end-foot, pval = --', 'G protein-coupled acetylcholine receptor activity, pval = --', 'acetylcholine receptor regulator activity, pval = --', 'acetylcholine-induced gastric acid secretion, pval = --', 'regulation of acetylcholine-gated cation channel activity, pval = --', 'transmitter-gated ion channel activity involved in regulation of postsynaptic membrane potential, pval = --', 'G protein-coupled neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration, pval = --', 'dopamine neurotransmitter receptor activity, coupled via Gs, pval = --', 'dopamine neurotransmitter receptor activity, coupled via Gi/Go, pval = --']
level 7: ['nuclear membrane complex Bqt3-Bqt4, pval = --', 'anchored component of postsynaptic early endosome membrane, pval = --', 'integral component of postsynaptic early endosome membrane, pval = --', 'anchored component of postsynaptic recycling endosome membrane, pval = --', 'integral component of postsynaptic recycling endosome membrane, pval = --', 'clathrin coating of Golgi vesicle, plasma membrane to endosome targeting, pval = --', 'clathrin coating of Golgi vesicle, trans-Golgi to endosome targeting, pval = --', 'COPI coating of Golgi vesicle, inter-Golgi cisterna, pval = --', 'COPI coating of Golgi vesicle, cis-Golgi to rough ER, pval = --', 'negative regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of COPII vesicle coating, pval = --', 'anchored component of neuronal dense core vesicle membrane, pval = --', 'integral component of neuronal dense core vesicle membrane, pval = --', 'anchored component of synaptic vesicle membrane, pval = --', 'integral component of synaptic vesicle membrane, pval = --', 'leading edge of dendritic growth cone, pval = --', 'retrograde dendritic transport of mitochondrion, pval = --', 'anterograde dendritic transport of mitochondrion, pval = --', 'anterograde dendritic transport of messenger ribonucleoprotein complex, pval = --', 'dendritic transport of messenger ribonucleoprotein complex, pval = --', 'actin cytoskeleton of dendritic spine, pval = --', 'dendritic spine origin, pval = --', 'hippocampal mossy fiber to CA3 synapse, pval = --', 'retrograde axonal transport of mitochondrion, pval = --', 'anterograde axonal transport of mitochondrion, pval = --', 'anterograde axonal transport of messenger ribonucleoprotein complex, pval = --', 'anterograde synaptic vesicle transport, pval = --', 'regulation of retrograde axon cargo transport, pval = --', 'positive regulation of retrograde axon cargo transport, pval = --', 'negative regulation of retrograde axon cargo transport, pval = --', 'retrograde synaptic vesicle transport, pval = --', 'axonal growth cone, pval = 0.40992', 'peripheral region of growth cone, pval = --', 'central region of growth cone, pval = --', 'positive regulation of retrograde dense core granule transport, pval = --', 'negative regulation of retrograde dense core granule transport, pval = --', 'regulation of retrograde dense core granule transport, pval = --', 'regulation of anterograde dense core granule transport, pval = --', 'positive regulation of anterograde dense core granule transport, pval = --', 'negative regulation of anterograde dense core granule transport, pval = --', 'hippocampal mossy fiber expansion, pval = --', 'rod photoreceptor disc lumen, pval = --', 'rod photoreceptor disc membrane, pval = --', 'type II terminal bouton, pval = --', 'type I terminal bouton, pval = --', 'photoreceptor cell terminal bouton, pval = --', 'rod bipolar cell terminal bouton, pval = --', 'multiple synapse bouton, pval = --', 'type III terminal bouton, pval = --', 'S bouton, pval = --', 'F bouton, pval = --', 'C bouton, pval = --', 'anterograde axonal transport of neurotransmitter receptor complex, pval = --', 'stereocilia tip link, pval = --', 'stereocilia ankle link, pval = --', 'right tetrad, pval = --', 'left tetrad, pval = --', 'outer dense fiber, pval = --', 'sperm fibrous sheath, pval = --', 'sperm connecting piece, pval = --', 'sperm annulus, pval = --', 'sperm principal piece, pval = --', 'sperm end piece, pval = --', 'sperm midpiece, pval = --', 'sperm mitochondrial sheath, pval = --', 'cytosolic ciliogenesis, pval = --', 'ciliary transition zone assembly, pval = --', 'non-motile cilium assembly, pval = --', 'positive regulation of cilium assembly, pval = --', 'negative regulation of cilium assembly, pval = --', 'regulation of cilium assembly, pval = --', 'motile cilium assembly, pval = --', 'vesicle targeting, trans-Golgi to periciliary membrane compartment, pval = --', 'ciliary basal body-plasma membrane docking, pval = --', 'axoneme assembly, pval = --', 'axonemal central pair projection, pval = --', 'axonemal central bridge, pval = --', 'axonemal central pair, pval = --', 'inner dynein arm, pval = --', 'outer dynein arm, pval = --', 'A axonemal microtubule, pval = --', 'B axonemal microtubule, pval = --', 'C1 axonemal microtubule, pval = --', 'C2 axonemal microtubule, pval = --', 'radial spoke stalk, pval = --', 'radial spoke head, pval = --', 'negative regulation of axonemal microtubule depolymerization, pval = --', 'acetylcholine receptor activator activity, pval = --', 'acetylcholine receptor inhibitor activity, pval = --', 'negative regulation of acetylcholine-gated cation channel activity, pval = --', 'acetylcholine-gated cation-selective channel activity, pval = --']
level 8: ['regulation of anterograde axonal transport of mitochondrion, pval = --', 'negative regulation of anterograde axonal transport of mitochondrion, pval = --', 'positive regulation of anterograde axonal transport of mitochondrion, pval = --', 'regulation of anterograde synaptic vesicle transport, pval = --', 'negative regulation of anterograde synaptic vesicle transport, pval = --', 'positive regulation of anterograde synaptic vesicle transport, pval = --', 'leading edge of axonal growth cone, pval = --', 'growth cone leading edge, pval = --', 'type Ib terminal bouton, pval = --', 'type Is terminal bouton, pval = --', 'subsynaptic reticulum, pval = --', 'multiple synapse bouton, contacting multiple dendrites, pval = --', 'multiple synapse bouton, contacting single dendrite, pval = --', 'stereocilia tip-link density, pval = --', 'stereocilia ankle link complex, pval = --', 'regulation of non-motile cilium assembly, pval = --', 'negative regulation of non-motile cilium assembly, pval = --', 'positive regulation of non-motile cilium assembly, pval = --', 'positive regulation of motile cilium assembly, pval = --', 'negative regulation of motile cilium assembly, pval = --', 'regulation of motile cilium assembly, pval = --', 'axonemal dynein complex assembly, pval = --', 'radial spoke assembly, pval = --', 'axonemal central apparatus assembly, pval = --', 'sperm axoneme assembly, pval = --']
level 9: ['upper tip-link density, pval = --', 'lower tip-link density, pval = --', 'inner dynein arm assembly, pval = --', 'outer dynein arm assembly, pval = --']" id="plasma_membrane_ton" init-pval="0.56389" min-pval="0.00058" min-pval-children="0.00058" title="Plasma membrane">
<g>
<path d="M360,483.4c-23.3,0-45.4-3.5-66.8-7
			c-7.1-1.1-13.8-2.2-20.6-3.2c-10.4-1.5-20.7-2.7-30.6-4c-34-4.3-66.2-8.3-99-18.4c-11.6-3.6-23.4-6.6-34.9-9.5
			c-31.1-7.8-57.9-14.6-71-30.1c-2.1-2.5-4.1-4.8-6-7c-15.1-17.4-23.4-27-24.5-60.2c-1.2-37.3,7.2-55.6,26.2-93.4
			c4.5-8.8,8.9-16.8,13.5-24.5c0.7-1.2,1.6-1.4,2.3-1.4c1.3,0,2.6,1,4,3c0.5,0.8,1.4,1.2,2.3,1.2c1.1,0,2.3-0.6,3.1-1.7
			c1-1.4,1.3-2.8,0.9-4c-0.3-0.9-1.1-1.5-2.1-1.8c-2.6-0.7-4.2-1.3-4.7-2.7c-0.5-1.2,0.3-2.3,0.5-2.7c4.2-6.3,8.3-12.1,12.6-18.3
			c6.7-9.5,13.6-19.2,21.8-32.4c7.4-11.8,12.5-20.7,17.1-28.5c8.7-15.1,15.6-27,32.9-49.9c26.2-34.7,45.5-44.2,74.1-50.4
			c8.5-1.8,17.6-4.4,27.3-7.1c22.4-6.3,47.9-13.4,77.1-15c12.8-0.7,24-0.7,36.7,0c1.7,0.1,2.7,0.5,3.3,1.2c0.7,0.9,0.5,1.8,0.4,2.7
			c0,0.2-0.1,0.4-0.1,0.7c-0.2,1.8,0.1,3,2.6,3.5c2.3,0.4,4,0.1,4.8-0.7c0.5-0.6,0.6-1.2,0.6-1.6c-0.1-1.7,0.2-2.9,1-3.7
			c0.4-0.4,1.3-1,2.6-0.9c3.9,0.3,7.8,0.7,12,1.1c15.1,1.4,32.3,3,56,4.1c87.9,3.9,119.3,19.3,124.5,60.8
			c5.2,41.9,6.9,51.4,15.5,76.7c3,8.7,4.8,14.8,6.2,20.2c0.4,1.7,0,2.8-0.5,3.4c-0.6,0.8-1.7,1.3-2.9,1.3c-0.5,0-1.1-0.1-1.7-0.3
			c-1.6-0.5-2.9-0.3-3.7,0.7c-0.9,1-1.2,2.7-0.9,4.7c0.2,1.7,1.3,4.4,3.2,4.4c0.8,0,1.9-0.5,3-1.4c1-0.9,2-1.3,3-1.3
			c1.8,0,3.1,1.4,3.4,3.9c1.1,8.4,1.8,18,2.7,33.9c0.8,13.8,4.1,25.6,7.7,38.1c5.8,20.3,11.8,41.4,8.5,74.7c-0.4,4-0.7,7.8-1.1,11.4
			c-4.3,46.4-6.2,67.5-46.5,79.6c-11,3.3-21.5,6.1-31.1,8.7c-1.6,0.4-3-0.2-3.8-1.6c-0.5-0.9-0.7-2.1-0.7-3.6c0-1.4-1.6-2.1-3.2-2.1
			c-0.7,0-1.3,0.1-2.1,0.3c-1.6,0.4-2.7,1.3-3.2,2.5c-0.3,0.7-0.2,1.5,0.2,2c1.1,1.6,1.4,3,1,4.2c-0.4,1-1.3,1.8-2.6,2.1
			c-24.4,6.5-49.3,13.6-84.1,27.1C405.9,479.5,384.2,483.4,360,483.4z M48.7,227.2c0,0-0.1,0.1-0.2,0.3c-4.6,7.6-9,15.5-13.4,24.3
			c-18.9,37.4-27.1,55.5-26,92.3c1,32.4,8.7,41.2,23.9,58.7c1.9,2.2,3.9,4.5,6,7c12.6,14.9,40.4,21.9,69.8,29.3
			c11.5,2.9,23.3,5.9,35,9.5c32.6,10,64.6,14.1,98.6,18.3c9.9,1.2,20.2,2.5,30.6,4c6.7,0.9,13.5,2,20.6,3.2
			c21.3,3.4,43.3,6.9,66.4,6.9c23.9,0,45.3-3.9,65.6-11.8c34.9-13.6,60.8-20.9,84.3-27.2c0.2-0.1,0.8-0.3,0.9-0.6
			c0.2-0.4-0.1-1.2-0.7-2c-0.8-1.2-1-2.7-0.4-4.2c0.7-1.9,2.5-3.4,4.8-4c0.9-0.2,1.8-0.4,2.7-0.4c3.3,0,5.6,1.8,5.6,4.5
			c0,1.4,0.2,2.1,0.4,2.5c0.2,0.4,0.4,0.5,0.6,0.5c0.1,0,0.3,0,0.4-0.1c9.6-2.6,20-5.4,31-8.7c38.7-11.6,40.6-32.2,44.8-77.5
			c0.3-3.6,0.7-7.5,1.1-11.4c3.2-32.9-2.7-53.7-8.5-73.8c-3.6-12.6-7-24.6-7.8-38.6c-0.9-15.9-1.6-25.4-2.7-33.7
			c-0.1-0.9-0.4-1.8-1-1.8c-0.2,0-0.7,0.1-1.4,0.7c-1.6,1.3-3.1,2-4.5,2c-3.3,0-5.2-3.4-5.6-6.5c-0.4-2.8,0.1-5.1,1.5-6.6
			c1.5-1.7,3.8-2.2,6.3-1.4c0.8,0.3,1.6,0.2,1.9-0.2c0.2-0.2,0.2-0.7,0.1-1.3c-1.3-5.3-3.2-11.3-6.1-20c-8.7-25.5-10.3-35-15.6-77.1
			c-2.6-20.9-11.9-34.1-30.3-42.9c-18.5-8.9-46.8-13.8-91.9-15.8c-23.8-1.1-41-2.7-56.1-4.1c-4.1-0.4-8.1-0.8-12-1.1
			c-0.5,0-0.6,0.1-0.7,0.1c-0.1,0.1-0.4,0.6-0.3,1.9c0.1,1.3-0.4,2.5-1.2,3.4c-0.7,0.8-2.1,1.7-4.6,1.7c-0.7,0-1.5-0.1-2.4-0.2
			c-3.4-0.6-5-2.7-4.6-6.1c0-0.3,0.1-0.5,0.1-0.8c0.1-0.3,0.1-0.7,0.1-0.9c0,0-0.3-0.2-1.6-0.2c-12.7-0.7-23.8-0.7-36.5,0
			c-29,1.6-54.3,8.7-76.6,14.9c-9.7,2.7-18.8,5.3-27.4,7.1c-28,6.1-46.9,15.4-72.7,49.5c-17.2,22.7-24.1,34.6-32.8,49.7
			c-4.5,7.9-9.7,16.8-17.1,28.6c-8.3,13.2-15.2,23-21.9,32.5c-4.3,6.1-8.4,11.9-12.6,18.2c-0.3,0.4-0.3,0.5-0.3,0.5
			c0.1,0,0.5,0.5,3,1.1c1.8,0.5,3.2,1.7,3.7,3.3c0.7,1.9,0.2,4.2-1.3,6.2c-1.3,1.7-3.1,2.7-5,2.7c-1.8,0-3.3-0.8-4.3-2.3
			C49.6,227.6,48.9,227.2,48.7,227.2z" style="fill:#FFFFFF;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<path d="M360,483.7c-23.3,0-45.5-3.5-66.8-7
		c-7.1-1.1-13.8-2.2-20.6-3.2c-10.4-1.5-20.7-2.7-30.6-4c-34.1-4.3-66.2-8.3-99-18.4c-11.6-3.6-23.4-6.6-34.9-9.5
		C77,433.8,50.1,427,37,411.5c-2.1-2.4-4-4.7-5.9-6.8l-0.1-0.1c-15.1-17.5-23.5-27.1-24.5-60.4c-1.2-37.4,7.2-55.7,26.3-93.6
		c4.5-8.8,8.9-16.9,13.5-24.5c0.8-1.3,1.8-1.6,2.5-1.6c1.4,0,2.7,1,4.2,3.1c0.5,0.7,1.2,1.1,2.1,1.1c1.1,0,2.1-0.6,2.9-1.6
		c0.9-1.3,1.3-2.6,0.9-3.7c-0.3-0.8-1-1.4-1.9-1.6c-2.7-0.7-4.3-1.4-4.8-2.8c-0.5-1.3,0.3-2.5,0.5-2.9c4.2-6.3,8.3-12.1,12.6-18.3
		c6.7-9.5,13.6-19.2,21.8-32.4c7.4-11.8,12.5-20.7,17-28.5c8.7-15.1,15.6-27.1,32.9-49.9c26.3-34.7,45.6-44.3,74.3-50.5
		c8.5-1.8,17.6-4.4,27.2-7.1c22.4-6.3,47.9-13.4,77.2-15C322,14.1,328,14,333.8,14c6,0,12.2,0.2,18.7,0.6c1.8,0.1,2.9,0.5,3.5,1.3
		c0.7,0.9,0.6,2,0.4,2.9c0,0.2-0.1,0.4-0.1,0.7c-0.2,1.8,0.1,2.8,2.4,3.2c2.1,0.4,3.8,0.1,4.6-0.7c0.5-0.5,0.5-1.1,0.5-1.4
		c-0.1-1.8,0.3-3.1,1.1-3.9c0.4-0.4,1.3-1.1,2.8-0.9c3.8,0.3,7.8,0.7,11.8,1.1l0.1,0c15.1,1.4,32.3,3,56,4.1
		c88,3.9,119.5,19.3,124.7,61c5.2,41.9,6.9,51.4,15.5,76.6c3,8.8,4.8,14.8,6.2,20.2c0.5,1.8,0,2.9-0.5,3.6c-0.7,0.9-1.8,1.4-3.1,1.4
		c-0.6,0-1.2-0.1-1.7-0.3c-1.5-0.5-2.7-0.3-3.5,0.6c-0.8,0.9-1.1,2.5-0.8,4.5c0.2,1.7,1.3,4.2,3,4.2c0.8,0,1.8-0.5,2.8-1.4
		c1.1-0.9,2.1-1.4,3.2-1.4c1.9,0,3.3,1.5,3.6,4.1c1.1,8.4,1.8,18,2.7,33.9c0.8,13.8,4.1,25.6,7.7,38.1c5.8,20.4,11.8,41.4,8.6,74.8
		c-0.4,3.8-0.7,7.4-1,10.9l0,0.5c-4.3,46.5-6.2,67.7-46.7,79.8c-11,3.3-21.5,6.1-31.1,8.7c-1.7,0.5-3.3-0.2-4.1-1.7
		c-0.5-0.9-0.7-2.2-0.7-3.7c0-1.2-1.5-1.8-3-1.8c-0.6,0-1.3,0.1-2,0.3c-1.5,0.4-2.6,1.2-3,2.4c-0.2,0.7-0.2,1.3,0.1,1.8
		c1.1,1.7,1.5,3.1,1,4.4c-0.4,1.1-1.4,1.9-2.7,2.3c-24.4,6.5-49.2,13.5-84,27.1C406,479.7,384.2,483.7,360,483.7z M48.7,225.1
		c-0.6,0-1.4,0.2-2.1,1.3C42,234,37.6,242,33.2,250.8c-19,37.8-27.4,56-26.2,93.3c1,33.1,9.3,42.7,24.4,60.1l0.1,0.1
		c1.9,2.2,3.8,4.4,5.9,6.8c13,15.4,39.8,22.2,70.9,30c11.4,2.9,23.3,5.9,34.9,9.5c32.8,10.1,64.9,14.1,99,18.4
		c9.9,1.2,20.2,2.5,30.6,4c6.7,0.9,13.4,2,20.6,3.2c21.4,3.4,43.4,7,66.8,7c24.1,0,45.8-3.9,66.4-11.9
		c34.8-13.6,59.7-20.6,84.1-27.2c1.2-0.3,2-1,2.4-2c0.4-1.1,0.1-2.5-0.9-4c-0.4-0.6-0.5-1.4-0.2-2.2c0.5-1.3,1.7-2.2,3.3-2.7
		c0.7-0.2,1.5-0.3,2.1-0.3c1.7,0,3.5,0.7,3.5,2.3c0,1.5,0.2,2.7,0.7,3.5c0.7,1.3,2,1.9,3.5,1.5c9.6-2.6,20.1-5.4,31.1-8.7
		c40.1-12,42.1-33.1,46.3-79.4l0-0.5c0.3-3.5,0.7-7.1,1-10.9c3.2-33.3-2.8-54.3-8.5-74.6c-3.6-12.5-6.9-24.3-7.7-38.2
		c-0.9-15.9-1.6-25.5-2.7-33.9c-0.3-2.3-1.5-3.6-3.1-3.6c-0.9,0-1.9,0.4-2.8,1.2c-1.1,1-2.2,1.5-3.1,1.5c-2.1,0-3.3-2.8-3.5-4.6
		c-0.3-2.1,0.1-3.9,1-4.9c0.9-1,2.3-1.3,4-0.7c0.5,0.2,1.1,0.3,1.6,0.3c1.1,0,2.1-0.4,2.7-1.2c0.4-0.6,0.9-1.6,0.4-3.2
		c-1.4-5.4-3.2-11.4-6.2-20.1c-8.6-25.3-10.2-34.7-15.5-76.7c-5.2-41.4-36.5-56.7-124.2-60.6c-23.7-1.1-40.9-2.7-56-4.1l-0.1,0
		c-4.1-0.4-8-0.8-11.8-1.1c-1.3-0.1-2,0.4-2.4,0.8c-0.7,0.7-1,1.9-0.9,3.5c0,0.4,0,1.2-0.6,1.8c-0.9,0.9-2.7,1.2-5,0.8
		c-2.7-0.5-3.1-1.8-2.8-3.7c0-0.2,0.1-0.5,0.1-0.7c0.1-0.9,0.3-1.8-0.3-2.5c-0.5-0.7-1.5-1-3.1-1.1c-6.5-0.4-12.6-0.6-18.7-0.6
		c-5.8,0-11.7,0.2-18,0.5c-29.2,1.6-54.6,8.7-77.1,15c-9.6,2.7-18.8,5.2-27.3,7.1c-28.5,6.2-47.8,15.6-74,50.3
		c-17.3,22.8-24.2,34.8-32.9,49.9c-4.5,7.8-9.7,16.7-17.1,28.5c-8.2,13.2-15.2,23-21.9,32.4c-4.3,6.1-8.4,11.9-12.6,18.3
		c-0.2,0.3-0.9,1.4-0.5,2.4c0.5,1.2,2,1.8,4.5,2.5c1.1,0.3,1.9,1,2.3,2c0.4,1.2,0.1,2.8-1,4.2c-0.9,1.2-2.1,1.8-3.3,1.8
		c-1,0-2-0.5-2.5-1.3C51.2,226,49.9,225.1,48.7,225.1z M360,481.3c-23.2,0-45.2-3.5-66.5-6.9c-7.1-1.1-13.8-2.2-20.6-3.2
		c-10.4-1.5-20.6-2.7-30.6-4l-0.1,0c-33.9-4.3-66-8.3-98.6-18.3c-11.7-3.6-23.5-6.6-35-9.5c-29.4-7.4-57.2-14.5-69.9-29.4
		c-2.1-2.5-4.2-4.9-6-7c-15.3-17.6-22.9-26.4-23.9-58.9c-1.2-36.8,7.1-54.9,26-92.4c4.4-8.8,8.8-16.7,13.4-24.3
		c0.2-0.3,0.2-0.4,0.4-0.4c0.3,0,1.1,0.5,2.2,2.1c0.9,1.4,2.4,2.1,4.1,2.1c1.8,0,3.6-0.9,4.8-2.6c1.4-1.9,1.9-4.1,1.2-5.9
		c-0.5-1.6-1.8-2.7-3.6-3.2c-1.6-0.4-2.7-0.8-3.1-1.2l-0.1-0.1l0-0.1c0-0.1,0-0.3,0.3-0.7c4.1-6.3,8.2-12.1,12.6-18.2
		c6.7-9.5,13.7-19.3,21.9-32.5c7.4-11.8,12.5-20.7,17.1-28.6c8.7-15.1,15.6-26.9,32.8-49.7c25.8-34.1,44.8-43.5,72.9-49.6
		c8.6-1.9,17.7-4.4,27.4-7.1c22.3-6.2,47.6-13.3,76.7-14.9c6.3-0.4,12.2-0.5,17.9-0.5c6,0,12.1,0.2,18.6,0.6c0.6,0,1.1,0.1,1.5,0.2
		l0.3,0.2c0,0.2,0,0.5-0.1,1c0,0.3-0.1,0.5-0.1,0.8c-0.4,3.3,1,5.2,4.4,5.8c0.8,0.2,1.6,0.2,2.3,0.2c2.4,0,3.7-0.9,4.4-1.6
		c0.8-0.8,1.2-2,1.2-3.2c-0.1-1.3,0.2-1.9,0.4-2.1c0.1-0.1,0.3-0.3,0.9-0.2c3.9,0.3,7.8,0.7,11.9,1.1l0.1,0
		c15.2,1.4,32.3,3,56.1,4.1c45.1,2,73.5,6.9,92,15.8C546,48,555.4,61.3,558,82.2c5.3,42.1,6.9,51.7,15.6,77.1c3,8.7,4.8,14.7,6.1,20
		c0.2,0.7,0.1,1.2-0.1,1.5c-0.2,0.3-0.7,0.4-1.2,0.4c-0.3,0-0.7-0.1-1-0.2c-0.7-0.2-1.5-0.4-2.2-0.4c-1.5,0-2.9,0.6-3.8,1.7
		c-1.3,1.5-1.8,3.7-1.4,6.4c0.4,3,2.3,6.3,5.4,6.3c1.4,0,2.8-0.7,4.4-1.9c0.8-0.7,1.4-0.8,1.6-0.8c0.6,0,1.1,0.7,1.2,2
		c1.1,8.3,1.8,17.9,2.7,33.8c0.8,14,4.2,26,7.8,38.6c5.7,20.1,11.7,41,8.5,73.9c-0.4,3.8-0.7,7.5-1,11l0,0.5
		c-4.2,45.4-6.1,66.1-45,77.7c-11.1,3.3-21.5,6.1-31.1,8.7c-0.2,0-0.3,0.1-0.5,0.1c-0.3,0-0.6-0.1-0.9-0.6c-0.2-0.4-0.5-1.2-0.5-2.6
		c0-2.5-2.2-4.2-5.4-4.2c-0.8,0-1.7,0.1-2.6,0.4c-2.3,0.6-4,2-4.6,3.8c-0.5,1.4-0.4,2.8,0.4,4c0.6,0.9,0.9,1.7,0.7,2.2
		c-0.2,0.5-0.8,0.7-1.1,0.8c-23.5,6.3-49.4,13.6-84.3,27.2C405.4,477.4,383.9,481.3,360,481.3z M48.8,227.5c0,0-0.1,0.1-0.1,0.1
		c-4.6,7.6-9,15.5-13.4,24.3c-18.8,37.4-27.1,55.5-26,92.2c1,32.3,8.6,41.1,23.8,58.6c1.8,2.1,3.9,4.5,6,7
		c12.5,14.8,40.3,21.8,69.6,29.2c11.5,2.9,23.3,5.9,35,9.5c32.6,10,64.6,14.1,98.5,18.3l0.1,0c9.9,1.2,20.2,2.5,30.6,4
		c6.8,1,13.5,2,20.6,3.2c21.3,3.4,43.3,6.9,66.4,6.9c23.8,0,45.3-3.8,65.5-11.8c34.9-13.6,60.8-20.9,84.3-27.2
		c0.3-0.1,0.7-0.3,0.8-0.5c0.1-0.3,0-0.9-0.7-1.8c-0.9-1.3-1-2.9-0.5-4.4c0.7-2,2.6-3.5,5-4.1c0.9-0.2,1.9-0.4,2.7-0.4
		c3.5,0,5.9,1.9,5.9,4.7c0,1.3,0.2,2,0.4,2.4c0.2,0.3,0.3,0.3,0.4,0.3c0.1,0,0.2,0,0.3-0.1c9.6-2.6,20-5.4,31-8.7
		c38.6-11.5,40.5-32.1,44.6-77.3l0-0.5c0.3-3.5,0.7-7.2,1-11c3.2-32.8-2.7-53.6-8.5-73.7c-3.6-12.7-7-24.6-7.8-38.7
		c-0.9-15.8-1.6-25.4-2.7-33.7c-0.1-0.9-0.4-1.6-0.8-1.6c-0.1,0-0.5,0-1.3,0.7c-1.6,1.4-3.2,2.1-4.7,2.1c-3.4,0-5.4-3.5-5.9-6.7
		c-0.4-2.9,0.2-5.2,1.5-6.8c1.5-1.7,4-2.3,6.5-1.4c0.7,0.2,1.4,0.1,1.6-0.1c0.1-0.2,0.1-0.6,0-1.1c-1.3-5.3-3.2-11.3-6.1-20
		c-8.7-25.5-10.3-35-15.6-77.2c-2.6-20.8-11.9-33.9-30.2-42.8c-18.4-8.9-46.7-13.8-91.8-15.8c-23.8-1.1-41-2.7-56.1-4.1l-0.1,0
		c-4.1-0.4-8.1-0.8-11.9-1.1c-0.3,0-0.5,0.1-0.5,0.1c0,0-0.3,0.4-0.3,1.7c0.1,1.3-0.4,2.6-1.3,3.5c-0.8,0.8-2.2,1.7-4.8,1.7
		c-0.7,0-1.6-0.1-2.4-0.2c-3.6-0.7-5.2-2.8-4.8-6.4c0-0.3,0.1-0.5,0.1-0.8c0-0.2,0.1-0.5,0.1-0.6c-0.3-0.1-0.8-0.2-1.3-0.2
		c-6.5-0.4-12.5-0.6-18.5-0.6c-5.8,0-11.6,0.2-17.9,0.5c-29,1.6-54.2,8.7-76.5,14.9c-9.7,2.7-18.8,5.3-27.4,7.1
		c-27.9,6-46.8,15.3-72.6,49.4c-17.2,22.7-24,34.6-32.7,49.6c-4.6,7.9-9.7,16.8-17.1,28.6c-8.3,13.2-15.2,23-21.9,32.5
		c-4.3,6.1-8.4,11.9-12.6,18.2c-0.1,0.1-0.1,0.2-0.2,0.3c0.3,0.2,1,0.5,2.8,1c1.9,0.5,3.3,1.8,3.9,3.5c0.7,2,0.2,4.3-1.3,6.4
		c-1.3,1.8-3.2,2.8-5.2,2.8c-1.8,0-3.5-0.9-4.5-2.4C49.6,228,49,227.6,48.8,227.5z" style="fill:#EDEDED;stroke:#9D9D9C;stroke-width:0.15;stroke-miterlimit:10;"></path>
</g>
<g class="col plasma_membrane cell_junctions" descendants="level 0: ['plasma membrane, pval = 0.56389']
level 1: ['intrinsic component of plasma membrane, pval = 0.76092', 'plasmodesmatal plasma membrane, pval = --', 'plasmodesmatal cytoplasmic sleeve, pval = --', 'plasma membrane protein complex, pval = 0.81411', 'cell trailing edge membrane, pval = --', 'leading edge membrane, pval = 0.03464', 'endomembrane system, pval = 0.00075', 'receptor clustering, pval = --', 'growth cone membrane, pval = --', 'sarcolemma, pval = 0.9286', 'extrinsic component of plasma membrane, pval = 0.93038', 'orthogonal array, pval = --', 'plasma membrane respirasome, pval = --', 'cytoproct, pval = --', 'cytostome, pval = --', 'sperm plasma membrane, pval = --', 'plasma membrane region, pval = 0.29057', 'phagocytic cup, pval = --', 'mesosome, pval = --', 'plasma membrane-derived thylakoid lumen, pval = --', 'plasma membrane ATP synthesis coupled electron transport, pval = --', 'immunological synapse, pval = --', 'protein transport within plasma membrane, pval = --', 'tetraspanin-enriched microdomain, pval = --', 'cell body membrane, pval = --', 'medial membrane band, pval = --', 'cell envelope, pval = --', 'lateral plasma membrane, pval = 0.88858', 'cellular bud membrane, pval = --', 'external side of plasma membrane, pval = 0.91', 'cytoplasmic side of plasma membrane, pval = 0.54382', 'receptor diffusion trapping, pval = --', 'porosome, pval = --', 'unencapsulated part of cell, pval = --', 'plasmodesmatal desmotubule, pval = --', 'cornified envelope, pval = --']
level 2: ['intrinsic component of plasma membrane of cell tip, pval = --', 'integral component of plasma membrane, pval = 0.87053', 'anchored component of plasma membrane, pval = 0.1204', 'intrinsic component of synaptic membrane, pval = 0.94537', 'intrinsic component of external side of plasma membrane, pval = 0.19304', 'intrinsic component of the cytoplasmic side of the plasma membrane, pval = 0.43974', 'MHC protein complex, pval = 0.43974', 'alphav-beta3 integrin-CD47 complex, pval = --', 'MalFGK2 complex, pval = --', 'NADPH oxidase complex, pval = --', 'connexin complex, pval = --', 'flotillin complex, pval = --', 'amino acid transport complex, pval = --', 'SC5b-7 complex, pval = --', 'alphav-beta5 integrin-vitronectin complex, pval = --', 'ATP-binding cassette (ABC) transporter complex, pval = --', 'heterotrimeric G-protein complex, pval = --', 'Wnt-Frizzled-LRP5/6 complex, pval = 0.43974', 'voltage-gated potassium channel complex, pval = --', 'auxin efflux carrier complex, pval = --', 'high-affinity iron permease complex, pval = --', 'subapical complex, pval = --', 'MPP7-DLG1-LIN7 complex, pval = --', 'voltage-gated sodium channel complex, pval = --', 'sucrase-isomaltase complex, pval = --', 'alphav-beta3 integrin-thrombospondin complex, pval = --', 'plasma membrane-derived thylakoid ribulose bisphosphate carboxylase complex, pval = --', 'potassium:proton exchanging ATPase complex, pval = --', 'CFTR-NHERF-ezrin complex, pval = --', 'alphav-beta1 integrin-osteopontin complex, pval = --', 'alphav-beta5 integrin-osteopontin complex, pval = --', 'voltage-gated calcium channel complex, pval = 0.96934', 'sodium:potassium-exchanging ATPase complex, pval = --', 'interleukin-12-interleukin-12 receptor complex, pval = --', 'CD19-Vav-PIK3R1 complex, pval = --', 'alphav-beta3 integrin-osteopontin complex, pval = --', 'plasma membrane proton-transporting V-type ATPase, V0 domain, pval = --', 'plasma membrane proton-transporting V-type ATPase, V1 domain, pval = --', 'alphav-beta3 integrin-PKCalpha complex, pval = 0.19304', 'caveolar macromolecular signaling complex, pval = --', 'inhibin-betaglycan-ActRII complex, pval = 0.43974', 'Fc receptor complex, pval = --', 'alpha4-beta1 integrin-CD47 complex, pval = --', 'alphaIIb-beta3 integrin-ICAM-4 complex, pval = --', 'alpha4-beta4 integrin-EMILIN-1 complex, pval = --', 'alpha6-beta1 integrin-CYR61 complex, pval = --', 'alphaV-beta3 integrin-tumstatin complex, pval = --', 'alpha5-beta1 integrin-endostatin complex, pval = --', 'alphaV-beta3 integrin-NOV complex, pval = --', 'alpha5-beta1 integrin-fibronectin-NOV complex, pval = --', 'alpha7-beta1 integrin-nicotinamide riboside kinase complex, pval = --', 'Grb2-SHP-2 complex, pval = --', 'catenin complex, pval = --', 'alphaPDGFR-SHP-2 complex, pval = --', 'alphaPDGFR-PLC-gamma-1-PI3K-SHP-2 complex, pval = --', 'low-density lipoprotein receptor complex, pval = --', 'alphaV-beta8 integrin-MMP14-TGFbeta-1 complex, pval = --', 'alpha4-beta1 integrin-paxillin complex, pval = --', 'alpha4-beta1 integrin-JAM2 complex, pval = --', 'alphav-beta3 integrin-IGF-1-IGF1R complex, pval = 0.40992', 'alphav-beta3 integrin-HMGB1 complex, pval = 0.43974', 'plasma membrane signaling receptor complex, pval = 0.82502', 'alpha7-beta1 integrin-focal adhesion kinase complex, pval = --', 'alpha9-beta1 integrin-thrombospondin-1 complex, pval = --', 'alphaV-beta3 integrin-CD98 complex, pval = --', 'alpha5-beta5-fibronectin-SFRP2 complex, pval = --', 'alpha7-beta1 integrin-laminin alpha-2 complex, pval = --', 'plasma membrane proton-transporting ATP synthase, stator stalk, pval = --', 'plasma membrane proton-transporting ATP synthase complex, coupling factor F(o), pval = --', 'plasma membrane proton-transporting ATP synthase, catalytic core, pval = --', 'alphaV-beta3 integrin-laminin alpha-4 complex, pval = --', 'plasma membrane proton-transporting ATP synthase complex, catalytic core F(1), pval = --', 'alpha5-beta1 integrin-LPP3 complex, pval = --', 'alpha9-beta1 integrin-ADAM8 complex, pval = --', 'plasma membrane proton-transporting ATP synthase complex, pval = --', 'alphaX-beta2 integrin-ICAM-4 complex, pval = --', 'alpha9-beta1 integrin-VEGF-A complex, pval = --', 'alpha9-beta1 integrin-VEGF-D complex, pval = --', 'alpha1-beta1 integrin-tyrosine-protein phosphatase non-receptor type 2 complex, pval = --', 'alpha9-beta1 integrin-VEGF-C complex, pval = --', 'alphaV-beta6 integrin-osteopontin complex, pval = --', 'alpha9-beta1 integrin-osteopontin complex, pval = --', 'alphaV-beta3 integrin-EGFR complex, pval = --', 'alphaV-beta3 integrin-LPP3 complex, pval = --', '1,3-beta-D-glucan synthase complex, pval = --', 'alpha5-beta1 integrin-osteopontin complex, pval = --', 'Ost-alpha/Ost-beta complex, pval = --', 'plasma membrane respiratory chain complex IV, pval = --', 'plasma membrane respiratory chain complex III, pval = --', 'plasma membrane proton-transporting ATP synthase, central stalk, pval = --', 'alphaIIb-beta3 integrin-CD47-FAK complex, pval = --', 'plasma membrane respiratory chain complex II, pval = --', 'alphaIIb-beta3 integrin-CD47-Src complex, pval = --', 'plasma membrane respiratory chain complex I, pval = --', 'Delta1 complex, pval = --', 'gamma-secretase-Delta1 complex, pval = --', 'gamma-secretase complex, pval = --', 'EGFR-Grb2-Sos complex, pval = --', 'EGFR-Shc-Grb2-Sos complex, pval = --', 'TAT protein transport complex, pval = --', 'alphaIIb-beta3 integrin-CIB complex, pval = --', 'Shc-Grb2-Sos complex, pval = --', 'Grb2-Sos complex, pval = --', 'plasma membrane fumarate reductase complex, pval = --', 'plasma membrane succinate dehydrogenase complex, pval = --', 'protocadherin-alpha-protocadherin-gamma complex, pval = --', 'ciliary necklace, pval = --', 'clathrin coat of coated pit, pval = --', 'AP-2 adaptor complex, pval = --', 'membrane attack complex, pval = --', 'alphaE-beta7 integrin-E-cadherin complex, pval = --', 'alpha9-beta1 integrin-vascular cell adhesion molecule-1 complex, pval = --', 'alpha9-beta1 integrin-ADAM12 complex, pval = --', 'alphav-beta3 integrin-ADAM23 complex, pval = --', 'alpha4-beta1 integrin-thrombospondin-1 complex, pval = --', 'alpha4-beta1 integrin-CD81 complex, pval = --', 'alpha4-beta1 integrin-CD82 complex, pval = --', 'alpha7-beta1 integrin-CD151 complex, pval = --', 'alphav-beta3 integrin-vitronectin complex, pval = 0.43974', 'alpha6-beta4 integrin-CD151 complex, pval = --', 'alpha9-beta1 integrin-ADAM2 complex, pval = --', 'plasma membrane proton-transporting V-type ATPase complex, pval = --', 'alpha9-beta1 integrin-ADAM1 complex, pval = --', 'EmrE multidrug transporter complex, pval = --', 'transmembrane collagen trimer, pval = --', 'alpha9-beta1 integrin-ADAM9 complex, pval = --', 'alpha9-beta1 integrin-ADAM3 complex, pval = --', 'NarGHI complex, pval = --', 'alphav-beta3 integrin-ADAM15 complex, pval = --', 'alpha9-beta1 integrin-ADAM15 complex, pval = --', 'alpha6-beta1 integrin-CD151 complex, pval = --', 'alpha3-beta1 integrin-CD151 complex, pval = --', 'MdtBC Complex, pval = --', 'alphaIIb-beta3 integrin-talin complex, pval = --', 'alpha4-beta1 integrin-CD53 complex, pval = --', 'alpha4-beta1 integrin-CD63 complex, pval = --', 'alphaV-beta3 integrin-tissue transglutaminase complex, pval = --', 'alphaM-beta2 integrin-CD63 complex, pval = --', 'inaD signaling complex, pval = --', 'alpha3-beta1 integrin-CD63 complex, pval = --', 'alpha9-beta1 integrin-tenascin complex, pval = --', 'alpha3-beta1 integrin-basigin complex, pval = --', 'alpha2-beta1 integrin-CD47 complex, pval = --', 'alphaV-beta3 integrin-CD47-FCER2 complex, pval = --', 'alphaIIb-beta3 integrin-CD9-CD47-platelet glycoprotein Ib complex, pval = --', 'alphaIIb-beta3 integrin-CD9 complex, pval = --', 'alpha5-beta1 integrin-tissue transglutaminase complex, pval = --', 'alpha11-beta1 integrin-collagen type I complex, pval = --', 'glycine-gated chloride channel complex, pval = --', 'alpha2-beta1 integrin-chondroadherin complex, pval = --', 'fibronectin-tissue transglutaminase complex, pval = --', 'dystroglycan complex, pval = --', 'dystrophin-associated glycoprotein complex, pval = --', 'dystrobrevin complex, pval = --', 'syntrophin complex, pval = --', 'sarcoglycan complex, pval = --', 'alpha4-beta1 integrin-thrombospondin-2 complex, pval = --', 'Shc-EGFR complex, pval = 0.19304', 'Grb2-EGFR complex, pval = --', 'Grb2-Shc complex, pval = --', 'alphaIIb-beta3 integrin-fibronectin-tissue transglutaminase complex, pval = --', 'alpha3-beta1 integrin-tissue transglutaminase complex, pval = --', 'alpha5-beta1 integrin-fibronectin-tissue transglutaminase complex, pval = --', 'alpha1-beta1 integrin-tissue transglutaminase complex, pval = --', 'alpha3-beta1 integrin-thrombospondin complex, pval = --', 'alpha6-beta4 integrin-CD9 complex, pval = --', 'alphaV-beta3 integrin-paxillin-Pyk2 complex, pval = --', 'alphaV-beta3 integrin-gelsolin complex, pval = --', 'alphaV-beta6 integrin-TGFbeta-3 complex, pval = --', 'alpha6-beta4 integrin-Fyn complex, pval = --', 'interleukin4-interleukin-4 receptor complex, pval = --', 'EFF-1 complex, pval = --', 'Cus cation efflux complex, pval = --', 'platelet-derived growth factor receptor-ligand complex, pval = --', 'D-amino-acid dehydrogenase complex, pval = --', 'alpha6-beta4 integrin-Shc-Grb2 complex, pval = --', 'alpha6-beta4 integrin-laminin 5 complex, pval = --', 'inward rectifying potassium channel, pval = --', 'alphav-beta3 integrin-collagen alpha3(VI) complex, pval = --', 'plasma membrane-derived thylakoid photosystem II, pval = --', 'alpha1-beta1 integrin-alpha3(VI) complex, pval = --', 'alpha2-beta1 integrin-alpha3(VI) complex, pval = --', 'contractile vacuole pore, pval = --', 'plasma membrane-derived photosystem I, pval = --', 'death-inducing signaling complex, pval = 0.22736', 'uropod membrane, pval = --', 'lamellipodium membrane, pval = 0.19304', 'ruffle membrane, pval = 0.02922', 'neuron projection membrane, pval = 0.68643', 'endoplasmic reticulum, pval = 0.10307', 'nuclear envelope, pval = 0.10877', 'endosome, pval = 0.00915', 'spitzenkorper, pval = --', 'Golgi apparatus, pval = 0.02368', 'retromer complex, pval = --', 'omegasome, pval = --', 'secretory granule, pval = 0.05448', 'transport vesicle, pval = 0.49914', 'spine apparatus, pval = --', 'nuclear outer membrane-endoplasmic reticulum membrane network, pval = 0.00106', 'clathrin-coated pit, pval = --', 'skeletal muscle acetylcholine-gated channel clustering, pval = --', 'neurotransmitter-gated ion channel clustering, pval = --', 'regulation of receptor clustering, pval = --', 'negative regulation of receptor clustering, pval = --', 'positive regulation of receptor clustering, pval = --', 'ciliary receptor clustering involved in smoothened signaling pathway, pval = --', 'neuroligin clustering involved in postsynaptic membrane assembly, pval = --', 'neurexin clustering involved in presynaptic membrane assembly, pval = --', 'T-tubule, pval = 0.97378', 'extrinsic component of synaptic membrane, pval = 0.43974', 'extrinsic component of external side of plasma membrane, pval = --', 'extrinsic component of cytoplasmic side of plasma membrane, pval = 0.59248', 'sperm head plasma membrane, pval = --', 'plasma membrane bounded cell projection, pval = 0.51847', 'periciliary membrane compartment, pval = --', 'basal plasma membrane, pval = 0.82468', 'caveola neck, pval = --', 'synaptic membrane, pval = 0.96764', 'cleavage furrow, pval = 0.0846', 'hinge region between urothelial plaques of apical plasma membrane, pval = --', 'plasma membrane-derived thylakoid membrane, pval = --', 'plasma membrane of cell tip, pval = --', 'apical plasma membrane, pval = 0.31529', 'basolateral plasma membrane, pval = 0.85139', 'apicolateral plasma membrane, pval = --', 'plasma membrane raft, pval = 0.40337', 'photoreceptor inner segment membrane, pval = --', 'neurotransmitter receptor diffusion trapping, pval = --', 'cell projection membrane, pval = 0.03096', 'phagocytic cup lip, pval = --', 'phagocytic cup base, pval = --', 'plasma membrane electron transport, NADH to quinone, pval = --', 'protein transport out of plasma membrane raft, pval = --', 'neuronal cell body membrane, pval = --', 'glycocalyx, pval = --', 'outer membrane-bounded periplasmic space, pval = --', 'Gram-negative-bacterium-type cell wall, pval = --', 'cell outer membrane, pval = --', 'periplasmic side of plasma membrane, pval = --', 'external side of apical plasma membrane, pval = --', 'cytoplasmic side of apical plasma membrane, pval = --', 'cytoplasmic side of dendritic spine plasma membrane, pval = --', 'cytoplasmic side of plasma membrane, cell tip, pval = --', 'plasmodesmatal endoplasmic reticulum, pval = --']
level 3: ['interleukin-12 receptor complex, pval = --', 'apical plasma membrane urothelial plaque, pval = --', 'histamine-gated chloride channel complex, pval = --', 'high molecular weight kininogen receptor complex, pval = --', 'pre-T cell receptor complex, pval = --', 'interleukin-18 receptor complex, pval = --', 'interleukin-20 receptor complex, pval = --', 'enzyme IIA-maltose transporter complex, pval = --', 'oncostatin-M receptor complex, pval = --', 'integral component of the cytoplasmic side of the plasma membrane, pval = --', 'granulocyte macrophage colony-stimulating factor receptor complex, pval = --', 'ciliary neurotrophic factor receptor complex, pval = --', 'interleukin-4 receptor complex, pval = --', 'maltose transport complex, pval = --', 'integral component of muscle cell projection membrane, pval = --', 'type I interferon receptor complex, pval = --', 'interleukin-1 receptor complex, pval = --', 'interleukin-28 receptor complex, pval = --', 'interleukin-23 receptor complex, pval = --', 'acetylcholine-gated channel complex, pval = --', 'interleukin-3 receptor complex, pval = --', 'glycoprotein Ib-IX-V complex, pval = --', 'interleukin-2 receptor complex, pval = --', 'interleukin-6 receptor complex, pval = --', 'interleukin-5 receptor complex, pval = --', 'interleukin-13 receptor complex, pval = --', 'interleukin-9 receptor complex, pval = --', 'insulin receptor complex, pval = --', 'spanning component of plasma membrane, pval = --', 'BMP receptor complex, pval = 0.43974', 'G protein-coupled receptor complex, pval = --', 'integral component of external side of plasma membrane, pval = --', 'CD40 receptor complex, pval = 0.82468', 'integrin complex, pval = 0.46882', 'bacterial-type flagellum stator complex, pval = --', 'activin receptor complex, pval = 0.68643', 'ionotropic glutamate receptor complex, pval = --', 'integral component of synaptic membrane, pval = 0.90208', 'anchored component of the cytoplasmic side of the plasma membrane, pval = 0.43974', 'anchored component of synaptic membrane, pval = --', 'anchored component of external side of plasma membrane, pval = 0.19304', 'intrinsic component of postsynaptic membrane, pval = --', 'intrinsic component of presynaptic membrane, pval = 0.68643', 'intrinsic component of periplasmic side of plasma membrane, pval = --', 'MHC class II protein complex, pval = --', 'MHC class I protein complex, pval = 0.43974', 'MHC class Ib protein complex, pval = --', 'ATP-dependent methionine-importing complex, pval = --', 'BtuCD complex, pval = --', 'cobalamin transport complex, pval = --', 'ATP-binding cassette (ABC) transporter complex, substrate-binding subunit-containing, pval = --', 'ATP-binding cassette (ABC) transporter complex, integrated substrate binding, pval = --', 'MsbA transporter complex, pval = --', 'glycerol-3-phosphate-transporting ATPase complex, pval = --', 'ATP-binding cassette (ABC) transporter complex, transmembrane substrate-binding subunit-containing, pval = --', 'ProVWX complex, pval = --', 'Wnt signalosome, pval = 0.94537', 'inward rectifier potassium channel complex, pval = --', 'Kv4.3-DPP10 channel complex, pval = --', 'Kv4.3-DPP6 channel complex, pval = --', 'Kv4.2-DPP6 channel complex, pval = --', 'HCN channel complex, pval = --', 'Kv4.1-DPP6 channel complex, pval = --', 'Kv4.2-Kv4.3 channel complex, pval = --', 'Kv4.1-DPP10 channel complex, pval = --', 'Kv4.2-KChIP3 channel complex, pval = --', 'Kv4.2-KChIP2 channel complex, pval = --', 'Kv4.3-KChIP1 channel complex, pval = --', 'Kv4.2-KChIP4 channel complex, pval = --', 'Kv4.2-KChIP1 channel complex, pval = --', 'CatSper complex, pval = --', 'ryanodine receptor complex, pval = --', 'L-type voltage-gated calcium channel complex, pval = 0.93038', 'light-activated voltage-gated calcium channel complex, pval = --', 'Fc-epsilon receptor I complex, pval = --', 'Fc-alpha receptor I complex, pval = --', 'Fc-gamma receptor I complex, pval = --', 'Fc-gamma receptor III complex, pval = --', 'Toll-like receptor 1-Toll-like receptor 2 protein complex, pval = 0.43974', 'Toll-like receptor 2-Toll-like receptor 6 protein complex, pval = --', 'ERBB4-EGFR complex, pval = --', 'ERBB4:ERBB2 complex, pval = --', 'ERBB3:ERBB2 complex, pval = --', 'EGFR:ERBB2 complex, pval = --', 'ERBB4-ERBB4 complex, pval = --', 'ERBB4-ERBB3 complex, pval = --', 'HFE-transferrin receptor complex, pval = 0.19304', 'transforming growth factor beta ligand-receptor complex, pval = --', 'T cell receptor complex, pval = 0.43974', 'B cell receptor complex, pval = 0.43974', 'neurotransmitter receptor complex, pval = --', 'plasma membrane succinate dehydrogenase complex (ubiquinone), pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-a1 complex, pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-a3 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-a1 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-a3 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-b2 complex, pval = --', 'protocadherin-alpha-v4-protocadherin-gamma-b4 complex, pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-b2 complex, pval = --', 'protocadherin-alpha-v7-protocadherin-gamma-b4 complex, pval = --', 'collagen type XIII trimer, pval = --', 'collagen type XXIII trimer, pval = --', 'collagen type XXV trimer, pval = --', 'collagen type XVII trimer, pval = --', 'TRAIL death-inducing signaling complex, pval = --', 'CD95 death-inducing signaling complex, pval = 0.40992', 'axolemma, pval = 0.43974', 'dendrite membrane, pval = --', 'dendritic spine membrane, pval = --', 'stereocilium membrane, pval = --', 'oligosaccharyltransferase complex, pval = --', 'MHC class I peptide loading complex, pval = 0.43974', 'rough endoplasmic reticulum, pval = --', 'smooth endoplasmic reticulum, pval = 0.43974', 'maintenance of protein localization in endoplasmic reticulum, pval = --', 'GPI-anchor transamidase complex, pval = --', 'endoplasmic reticulum calcium ion homeostasis, pval = --', 'endoplasmic reticulum quality control compartment, pval = --', 'egasyn-beta-glucuronidase complex, pval = --', 'glucosidase II complex, pval = --', 'extrinsic component of endoplasmic reticulum membrane, pval = 0.43974', 'VCP-NPL4-UFD1 AAA ATPase complex, pval = --', 'protein glycosylation in endoplasmic reticulum, pval = --', 'Dsl1/NZR complex, pval = --', 'fatty acid elongase complex, pval = --', 'perinuclear endoplasmic reticulum, pval = 0.19304', 'SREBP-SCAP-Insig complex, pval = --', 'membrane stack, pval = --', 'endoplasmic reticulum exit site, pval = --', 'endoplasmic reticulum chaperone complex, pval = 0.43974', 'Ig heavy chain-bound endoplasmic reticulum chaperone complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase complex, pval = --', 'GET complex, pval = --', 'glycosylphosphatidylinositol-mannosyltransferase I complex, pval = --', 'endoplasmic reticulum lumen, pval = 0.87374', 'UDP-N-acetylglucosamine transferase complex, pval = --', 'ERMES complex, pval = --', 'procollagen-proline 4-dioxygenase complex, pval = --', 'sarcoplasmic reticulum, pval = 0.22736', 'endoplasmic reticulum subcompartment, pval = 0.0018', 'glycosylphosphatidylinositol-mannosyltransferase II complex, pval = --', 'ER ubiquitin ligase complex, pval = --', 'microtubule organizing center attachment site, pval = --', 'nuclear membrane, pval = 0.42226', 'nuclear envelope lumen, pval = --', 'nuclear pore, pval = --', 'nucleoplasmic reticulum, pval = --', 'nuclear membrane protein complex, pval = --', 'early endosome, pval = 0.1586', 'tubular endosome, pval = --', 'late endosome, pval = 0.23172', 'endolysosome, pval = --', 'proteolysis within endosome associated with antigen processing and presentation, pval = --', 'endosome lumen, pval = 0.66415', 'presynaptic endosome, pval = --', 'endosome membrane, pval = 0.10877', 'postsynaptic endosome, pval = --', 'endosomal signal transduction, pval = --', 'sorting endosome, pval = --', 'ESCRT complex, pval = --', 'TRAPPII protein complex, pval = --', 'recycling endosome, pval = 0.36956', 'pinosome, pval = --', 'CORVET complex, pval = --', 'EGO complex, pval = --', 'Golgi subcompartment, pval = 0.19764', 'Golgi transport complex, pval = 0.43974', 'Golgi calcium ion homeostasis, pval = --', 'exomer complex, pval = --', 'cis-Golgi network, pval = 0.43974', 'RIC1-RGP1 guanyl-nucleotide exchange factor complex, pval = --', 'postsynaptic Golgi apparatus, pval = --', 'TRAPPIII protein complex, pval = --', 'inositol phosphoceramide synthase complex, pval = --', 'protein glycosylation in Golgi, pval = --', 'Golgi-associated vesicle lumen, pval = --', 'Golgi lumen, pval = 0.89614', 'Golgi membrane, pval = 0.14987', 'Golgi apparatus mannose trimming, pval = --', 'GARP complex, pval = --', 'clathrin coat of trans-Golgi network vesicle, pval = --', 'COPI vesicle coat, pval = --', 'acroblast, pval = --', 'retromer, tubulation complex, pval = --', 'retromer, cargo-selective complex, pval = --', 'mucin granule, pval = --', 'dense core granule, pval = --', 'platelet alpha granule, pval = 0.60334', 'acrosomal vesicle, pval = 0.40992', 'zymogen granule, pval = --', 'chromaffin granule, pval = --', 'azurophil granule, pval = 0.68643', 'specific granule, pval = 0.03701', 'lamellar body, pval = --', 'tertiary granule, pval = 0.01521', 'mucocyst, pval = --', 'cortical granule, pval = --', 'platelet dense granule, pval = --', 'secretory granule membrane, pval = 0.00058', 'ficolin-1-rich granule, pval = 0.23999', 'Weibel-Palade body, pval = --', 'insulin-responsive compartment, pval = --', 'secretory granule lumen, pval = 0.37117', 'exocytic vesicle, pval = --', 'clathrin-sculpted monoamine transport vesicle, pval = --', 'clathrin-sculpted acetylcholine transport vesicle, pval = --', 'COPI-coated Golgi to ER transport vesicle, pval = --', 'COPI-coated inter-Golgi transport vesicle, pval = --', 'transport vesicle lumen, pval = --', 'trans-Golgi network transport vesicle, pval = --', 'transport vesicle membrane, pval = 0.68643', 'clathrin-sculpted glutamate transport vesicle, pval = --', 'spine apparatus lumen, pval = --', 'spine apparatus membrane, pval = --', 'annulate lamellae, pval = --', 'endoplasmic reticulum membrane, pval = 0.0018', 'nuclear outer membrane, pval = --', 'perinuclear endoplasmic reticulum membrane, pval = 0.43974', 'Nem1-Spo7 phosphatase complex, pval = --', 'negative regulation of skeletal muscle acetylcholine-gated channel clustering, pval = --', 'regulation of skeletal muscle acetylcholine-gated channel clustering, pval = --', 'positive regulation of skeletal muscle acetylcholine-gated channel clustering, pval = --', 'NMDA glutamate receptor clustering, pval = --', 'gamma-aminobutyric acid receptor clustering, pval = --', 'glycine receptor clustering, pval = --', 'glutamate receptor clustering, pval = --', 'regulation of glutamate receptor clustering, pval = --', 'negative regulation of AMPA glutamate receptor clustering, pval = --', 'positive regulation of AMPA glutamate receptor clustering, pval = --', 'extrinsic component of presynaptic membrane, pval = --', 'extrinsic component of postsynaptic membrane, pval = 0.43974', 'extrinsic component of periplasmic side of plasma membrane, pval = --', 'subplasmalemmal coating, pval = --', 'neuron projection, pval = 0.33088', 'pseudopodium, pval = 0.03701', 'cilium, pval = 0.75262', 'glial cell projection, pval = 0.43974', 'plasma membrane bounded cell projection cytoplasm, pval = 0.46882', 'ruffle, pval = 0.10178', 'cytoneme, pval = --', 'mating projection, pval = --', 'microspike, pval = --', 'invadopodium, pval = 0.43974', 'bleb, pval = --', 'root hair, pval = --', 'pollen tube, pval = --', 'dendritic cell dendrite, pval = --', 'podocyte foot, pval = --', 'actin-based cell projection, pval = 0.1586', 'podocyte primary projection, pval = --', 'muscle cell projection, pval = --', 'macropinocytic cup, pval = --', 'lamellipodium, pval = 0.32699', 'rhabdomere, pval = --', 'uropod, pval = --', 'dinoflagellate peduncle, pval = --', 'cell hair, pval = --', 'postsynaptic specialization membrane, pval = 0.90208', 'postsynaptic membrane, pval = 0.97378', 'presynaptic endocytic zone membrane, pval = --', 'postsynaptic endocytic zone membrane, pval = --', 'presynaptic periactive zone, pval = --', 'presynaptic active zone membrane, pval = 0.43974', 'presynaptic membrane, pval = 0.94537', 'cleavage furrow leading edge, pval = --', 'cleavage furrow rim, pval = --', 'cellularization cleavage furrow, pval = --', 'plasma membrane of growing cell tip, pval = --', 'mating projection tip membrane, pval = --', 'intracellular canaliculus, pval = --', 'eisosome membrane domain/MCC, pval = --', 'caveola, pval = 0.42226', 'postsynaptic neurotransmitter receptor diffusion trapping, pval = --', 'mating projection membrane, pval = --', 'invadopodium membrane, pval = 0.43974', 'rhabdomere membrane, pval = --', 'brush border membrane, pval = --', 'microvillus membrane, pval = 0.40992', 'filopodium membrane, pval = 0.03701', 'muscle cell projection membrane, pval = --', 'macropinocytic cup membrane, pval = --', 'pseudopodium membrane, pval = --', 'ciliary membrane, pval = 0.3878', 'slime layer, pval = --', 'capsule, pval = --', 'mycolate outer membrane, pval = --', 'curli secretion complex, pval = --', 'intrinsic component of cell outer membrane, pval = --', 'extrinsic component of cell outer membrane, pval = --', 'periplasmic side of cell outer membrane, pval = --', 'external side of cell outer membrane, pval = --', 'starch utilization system complex, pval = --']
level 4: ['calcitonin family receptor complex, pval = --', 'G protein-coupled receptor dimeric complex, pval = --', 'integral component of periplasmic side of plasma membrane, pval = --', 'integrin alphaIIb-beta3 complex, pval = --', 'integrin alpha9-beta1 complex, pval = 0.43974', 'integrin alpha8-beta1 complex, pval = --', 'integrin alpha6-beta1 complex, pval = --', 'integrin alpha5-beta1 complex, pval = --', 'integrin alpha7-beta1 complex, pval = --', 'integrin alpha6-beta4 complex, pval = --', 'integrin alpha2-beta1 complex, pval = --', 'integrin alpha1-beta1 complex, pval = --', 'integrin alpha4-beta1 complex, pval = --', 'integrin alpha3-beta1 complex, pval = 0.43974', 'integrin alpha4-beta7 complex, pval = --', 'integrin alphaD-beta2 complex, pval = --', 'integrin alphaE-beta7 complex, pval = --', 'integrin alphav-beta3 complex, pval = 0.43974', 'integrin alphav-beta1 complex, pval = --', 'integrin alphav-beta8 complex, pval = 0.43974', 'integrin alphav-beta6 complex, pval = --', 'integrin alphav-beta5 complex, pval = --', 'integrin alphaM-beta2 complex, pval = --', 'integrin alphaL-beta2 complex, pval = --', 'integrin alphaX-beta2 complex, pval = --', 'integrin alpha11-beta1 complex, pval = --', 'integrin alpha10-beta1 complex, pval = --', 'NMDA selective glutamate receptor complex, pval = --', 'AMPA glutamate receptor complex, pval = --', 'kainate selective glutamate receptor complex, pval = --', 'integral component of presynaptic membrane, pval = 0.43974', 'integral component of postsynaptic membrane, pval = --', 'anchored component of postsynaptic membrane, pval = --', 'anchored component of presynaptic membrane, pval = --', 'intrinsic component of postsynaptic endocytic zone membrane, pval = --', 'intrinsic component of postsynaptic specialization membrane, pval = --', 'intrinsic component of presynaptic active zone membrane, pval = 0.43974', 'intrinsic component of presynaptic endocytic zone membrane, pval = --', 'I(KACh) inward rectifier potassium channel complex, pval = --', 'Kir2 inward rectifier potassium channel complex, pval = --', 'TCR signalosome, pval = --', 'gamma-delta T cell receptor complex, pval = --', 'alpha-beta T cell receptor complex, pval = --', 'IgG B cell receptor complex, pval = --', 'IgM B cell receptor complex, pval = --', 'IgA B cell receptor complex, pval = --', 'IgE B cell receptor complex, pval = --', 'IgD B cell receptor complex, pval = --', 'G protein-coupled GABA receptor complex, pval = --', 'G protein-coupled serotonin receptor complex, pval = --', 'serotonin-activated cation-selective channel complex, pval = --', 'oligosaccharyltransferase I complex, pval = --', 'oligosaccharyltransferase II complex, pval = --', 'oligosaccharyltransferase III complex, pval = --', 'Tapasin-ERp57 complex, pval = --', 'TAP complex, pval = --', 'rough endoplasmic reticulum membrane, pval = --', 'rough endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum membrane, pval = --', 'smooth endoplasmic reticulum calcium ion homeostasis, pval = --', 'smooth endoplasmic reticulum lumen, pval = --', 'smooth endoplasmic reticulum cisterna, pval = --', 'protein retention in ER lumen, pval = --', 'SREBP-SCAP complex retention in endoplasmic reticulum, pval = --', 'positive regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of endoplasmic reticulum calcium ion concentration, pval = --', 'endoplasmic reticulum mannose trimming, pval = --', 'protein galactosylation in endoplasmic reticulum, pval = --', 'perinuclear endoplasmic reticulum lumen, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt5p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt3p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt1p-Pmt2p dimer complex, pval = --', 'dolichyl-phosphate-mannose-protein mannosyltransferase Pmt4p homodimer complex, pval = --', 'sarcoplasmic reticulum lumen, pval = --', 'cortical endoplasmic reticulum lumen, pval = --', 'luminal surveillance complex, pval = --', 'mannosyl-oligosaccharide 1,2-alpha-mannosidase complex, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(III) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(II) type, pval = --', 'procollagen-proline 4-dioxygenase complex, alpha(I) type, pval = --', 'sarcoplasmic reticulum membrane, pval = 0.43974', 'longitudinal sarcoplasmic reticulum, pval = --', 'terminal cisterna, pval = --', 'subrhabdomeral cisterna, pval = --', 'endoplasmic reticulum tubular network, pval = --', 'endoplasmic reticulum cisternal network, pval = --', 'CUE1-UBC7 ubiquitin-conjugating enzyme complex, pval = --', 'Hrd1p ubiquitin ligase complex, pval = --', 'Doa10p ubiquitin ligase complex, pval = --', 'meiotic nuclear membrane microtubule tethering complex, pval = --', 'nuclear inner membrane, pval = 0.43974', '5-lipoxygenase complex, pval = --', 'nuclear pore outer ring, pval = --', 'nucleoplasmic side of nuclear pore, pval = --', 'cytoplasmic side of nuclear pore, pval = --', 'structural constituent of nuclear pore, pval = --', 'nuclear pore inner ring, pval = --', 'nuclear pore central transport channel, pval = --', 'nuclear pore linkers, pval = --', 'nuclear pore nuclear basket, pval = --', 'nuclear pore cytoplasmic filaments, pval = --', 'nuclear pore transmembrane ring, pval = --', 'nuclear membrane mitotic spindle pole body tethering complex, pval = --', 'nuclear membrane microtubule tethering complex, pval = --', 'early endosome membrane, pval = 0.36956', 'early endosome lumen, pval = 0.43974', 'postsynaptic early endosome, pval = --', 'multivesicular body, pval = 0.19304', 'late endosome lumen, pval = --', 'late endosome membrane, pval = 0.59248', 'Mon1-Ccz1 complex, pval = --', 'endolysosome membrane, pval = --', 'endolysosome lumen, pval = --', 'recycling endosome lumen, pval = --', 'presynaptic endosome membrane, pval = --', 'postsynaptic endosome membrane, pval = --', 'ESCRT IV complex, pval = --', 'recycling endosome membrane, pval = 0.3878', 'Vps55/Vps68 complex, pval = --', 'ESCRT III complex, pval = --', 'ESCRT II complex, pval = --', 'ESCRT I complex, pval = --', 'intrinsic component of endosome membrane, pval = --', 'extrinsic component of endosome membrane, pval = --', 'intralumenal vesicle formation, pval = --', 'cytoplasmic side of endosome membrane, pval = --', 'lumenal side of endosome membrane, pval = --', 'postsynaptic recycling endosome, pval = --', 'ESCRT-0 complex, pval = --', 'basolateral recycling endosome, pval = --', 'apical recycling endosome, pval = --', 'peri-centrosomal recycling endosome, pval = --', 'EARP complex, pval = --', 'micropinosome, pval = --', 'macropinosome, pval = --', 'trans-Golgi network, pval = 0.59248', 'Golgi cisterna, pval = 0.43974', 'Golgi stack, pval = 0.43974', 'cis-Golgi network membrane, pval = --', 'negative regulation of protein glycosylation in Golgi, pval = --', 'positive regulation of protein glycosylation in Golgi, pval = --', 'regulation of protein glycosylation in Golgi, pval = --', 'protein galactosylation in Golgi, pval = --', 'Golgi stack lumen, pval = --', 'trans-Golgi network transport vesicle lumen, pval = --', 'Golgi vesicle budding, pval = --', 'intrinsic component of Golgi membrane, pval = 0.68643', 'extrinsic component of Golgi membrane, pval = --', 'lumenal side of Golgi membrane, pval = --', 'cytoplasmic side of Golgi membrane, pval = --', 'trans-Golgi network membrane, pval = 0.68643', 'Golgi cisterna membrane, pval = 0.43974', 'AP-1 adaptor complex, pval = --', 'dense core granule lumen, pval = --', 'neuronal dense core vesicle, pval = --', 'dense core granule membrane, pval = --', 'platelet alpha granule membrane, pval = 0.43974', 'platelet alpha granule lumen, pval = 0.815', 'acrosomal membrane, pval = 0.43974', 'acrosomal matrix, pval = --', 'acrosomal lumen, pval = --', 'zymogen granule membrane, pval = --', 'chromaffin granule membrane, pval = --', 'chromaffin granule lumen, pval = --', 'azurophil granule membrane, pval = --', 'azurophil granule lumen, pval = 0.68643', 'specific granule lumen, pval = 0.19304', 'specific granule membrane, pval = 0.19304', 'epidermal lamellar body, pval = --', 'alveolar lamellar body, pval = --', 'dendritic lamellar body, pval = --', 'lamellar body membrane, pval = --', 'tertiary granule membrane, pval = 0.0846', 'tertiary granule lumen, pval = 0.40992', 'ficolin-1-rich granule membrane, pval = 0.19304', 'platelet dense granule membrane, pval = --', 'platelet dense granule lumen, pval = --', 'ficolin-1-rich granule lumen, pval = 0.5379', 'Golgi to plasma membrane transport vesicle, pval = --', 'synaptic vesicle, pval = --', 'endosome to plasma membrane transport vesicle, pval = --', 'clathrin-sculpted monoamine transport vesicle membrane, pval = --', 'clathrin-sculpted monoamine transport vesicle lumen, pval = --', 'clathrin-sculpted acetylcholine transport vesicle membrane, pval = --', 'clathrin-sculpted acetylcholine transport vesicle lumen, pval = --', 'Golgi to ER transport vesicle membrane, pval = --', 'inter-Golgi transport vesicle membrane, pval = --', 'trans-Golgi network transport vesicle membrane, pval = --', 'ER to Golgi transport vesicle membrane, pval = 0.43974', 'exocytic vesicle membrane, pval = --', 'clathrin-sculpted glutamate transport vesicle membrane, pval = --', 'lumenal side of transport vesicle membrane, pval = --', 'cytoplasmic side of transport vesicle membrane, pval = --', 'clathrin-sculpted glutamate transport vesicle lumen, pval = --', 'extrinsic component of spine apparatus membrane, pval = --', 'intrinsic component of spine apparatus membrane, pval = --', 'endoplasmic reticulum palmitoyltransferase complex, pval = --', 'intrinsic component of endoplasmic reticulum membrane, pval = 0.43974', 'Derlin-1 retrotranslocation complex, pval = --', 'Derlin-1-VIMP complex, pval = --', 'glycosylphosphatidylinositol-N-acetylglucosaminyltransferase (GPI-GnT) complex, pval = --', 'lumenal side of endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of endoplasmic reticulum membrane, pval = --', 'IRE1-TRAF2-ASK1 complex, pval = 0.43974', 'signal peptidase complex, pval = --', 'Ire1 complex, pval = --', 'intrinsic component of nuclear outer membrane, pval = --', 'extrinsic component of nuclear outer membrane, pval = --', 'AMPA glutamate receptor clustering, pval = --', 'regulation of AMPA glutamate receptor clustering, pval = --', 'extrinsic component of presynaptic endocytic zone membrane, pval = --', 'extrinsic component of presynaptic active zone membrane, pval = --', 'extrinsic component of postsynaptic endocytic zone, pval = --', 'extrinsic component of postsynaptic specialization membrane, pval = 0.43974', 'telodendria, pval = --', 'kinocilium, pval = 0.43974', 'dendrite, pval = 0.30606', 'axon, pval = 0.35954', 'growth cone lamellipodium, pval = --', 'proximal neuron projection, pval = --', 'dendritic filopodium, pval = --', 'photoreceptor cell cilium, pval = 0.59248', 'cell body fiber, pval = --', 'neuron projection terminus, pval = --', 'dendritic tree, pval = 0.30606', 'axo-dendritic transport, pval = --', 'Lewy neurite, pval = --', 'gemmule, pval = --', 'neuron projection cytoplasm, pval = 0.3878', 'stereocilium, pval = 0.43974', 'accessory outer segment, pval = --', 'neuron spine, pval = 0.08911', 'neuron projection branch point, pval = --', 'rod spherule, pval = --', 'growth cone filopodium, pval = 0.43974', 'ciliary basal body, pval = 0.90208', 'motile cilium, pval = 0.43974', 'non-motile cilium, pval = 0.3878', 'intraciliary transport, pval = --', 'ciliary rootlet, pval = --', 'ciliary plasm, pval = 0.72894', 'ring centriole, pval = --', 'ciliary cap, pval = --', 'centriolar subdistal appendage, pval = --', 'ciliary pocket, pval = --', 'ciliary transition zone, pval = --', 'BBSome, pval = --', 'ciliary transition fiber, pval = --', 'ciliary tip, pval = 0.59248', 'ciliary shaft, pval = --', 'ciliary inversin compartment, pval = --', 'ciliary base, pval = 0.76316', 'capitate projection, pval = --', 'astrocyte projection, pval = --', 'spiny bracelet of Nageotte, pval = --', 'terminal loop, pval = --', 'Schwann cell microvillus, pval = --', 'mating projection tip, pval = --', 'mating projection base, pval = --', 'mating projection septin band, pval = --', 'mating projection septin ring, pval = --', 'root hair tip, pval = --', 'pollen tube tip, pval = --', 'microvillus, pval = 0.36956', 'actin filament bundle of actin-based cell projection, pval = --', 'cluster of actin-based cell projections, pval = 0.68643', 'filopodium, pval = 0.23999', 'macropinocytic cup cytoskeleton, pval = --', 'leading edge of lamellipodium, pval = --', 'rhabdomere microvillus, pval = --', 'postsynaptic density membrane, pval = 0.90208', 'postsynaptic specialization membrane of symmetric synapse, pval = --', 'G protein-coupled receptor activity involved in regulation of postsynaptic membrane potential, pval = --', 'ATPase coupled ion transmembrane transporter activity involved in regulation of postsynaptic membrane potential, pval = --', 'ion antiporter activity involved in regulation of postsynaptic membrane potential, pval = --', 'postsynaptic neurotransmitter receptor activity, pval = --', 'voltage-gated ion channel activity involved in regulation of presynaptic membrane potential, pval = --', 'ligand-gated ion channel activity involved in regulation of presynaptic membrane potential, pval = --', 'ATPase coupled ion transmembrane transporter activity involved in regulation of presynaptic membrane potential, pval = --', 'ion antiporter activity involved in regulation of presynaptic membrane potential, pval = --', 'neurotransmitter receptor activity involved in regulation of presynaptic cytosolic calcium ion concentration, pval = --', 'G protein-coupled neurotransmitter receptor activity involved in regulation of presynaptic membrane potential, pval = --', 'cellularization cleavage furrow invagination front, pval = --', 'regulation of postsynaptic neurotransmitter receptor diffusion trapping, pval = --', 'rhabdomere microvillus membrane, pval = --', 'cone photoreceptor disc membrane, pval = --', 'ciliary pocket membrane, pval = --', 'non-motile cilium membrane, pval = --', 'photoreceptor outer segment membrane, pval = 0.43974', 'extrinsic component of mycolate outer membrane, pval = --', 'intrinsic component of mycolate outer membrane, pval = --', 'integral component of cell outer membrane, pval = --', 'intrinsic component of external side of cell outer membrane, pval = --', 'intrinsic component of periplasmic side of cell outer membrane, pval = --', 'anchored component of cell outer membrane, pval = --', 'extrinsic component of external side of cell outer membrane, pval = --', 'extrinsic component of periplasmic side of cell outer membrane, pval = --', 'external side of mycolate outer membrane, pval = --']
level 5: ['amylin receptor complex, pval = --', 'adrenomedullin receptor complex, pval = --', 'CGRP receptor complex, pval = --', 'G protein-coupled receptor heterodimeric complex, pval = --', 'G protein-coupled receptor homodimeric complex, pval = --', 'integral component of presynaptic active zone membrane, pval = 0.43974', 'integral component of presynaptic endocytic zone membrane, pval = --', 'integral component of postsynaptic endocytic zone membrane, pval = --', 'integral component of postsynaptic specialization membrane, pval = --', 'anchored component of postynaptic endocytic zone membrane, pval = --', 'anchored component of postsynaptic specialization membrane, pval = --', 'anchored component of presynaptic active zone membrane, pval = --', 'anchored component of presynaptic endocytic zone membrane, pval = --', 'intrinsic component of postsynaptic density membrane, pval = --', 'intrinsic component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'signal recognition particle receptor complex, pval = --', 'translocon complex, pval = --', 'endoplasmic reticulum Sec complex, pval = --', 'lumenal side of rough endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of rough endoplasmic reticulum membrane, pval = --', 'lumenal side of smooth endoplasmic reticulum membrane, pval = --', 'cytoplasmic side of smooth endoplasmic reticulum membrane, pval = --', 'positive regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'negative regulation of smooth endoplasmic reticulum calcium ion concentration, pval = --', 'hypolemmal cisterna, pval = --', 'trimming of terminal mannose on B branch, pval = --', 'trimming of second mannose on A branch, pval = --', 'trimming of first mannose on A branch, pval = --', 'trimming of terminal mannose on C branch, pval = --', 'longitudinal sarcoplasmic reticulum lumen, pval = --', 'terminal cisterna lumen, pval = --', 'phospholamban complex, pval = --', 'junctional sarcoplasmic reticulum membrane, pval = --', 'free sarcoplasmic reticulum membrane, pval = --', 'cortical endoplasmic reticulum, pval = --', 'rough endoplasmic reticulum cisterna, pval = --', 'Hrd1p ubiquitin ligase ERAD-M complex, pval = --', 'Hrd1p ubiquitin ligase ERAD-L complex, pval = --', 'Sad1-Kms1 LINC complex, pval = --', 'intrinsic component of nuclear inner membrane, pval = --', 'Asi complex, pval = --', 'cytoplasmic side of early endosome membrane, pval = --', 'lumenal side of early endosome membrane, pval = --', 'postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle, pval = 0.43974', 'multivesicular body lumen, pval = --', 'multivesicular body membrane, pval = --', 'endosomal scaffold complex, pval = --', 'Ragulator complex, pval = --', 'cytoplasmic side of late endosome membrane, pval = --', 'lumenal side of late endosome membrane, pval = --', 'intrinsic component of presynaptic endosome membrane, pval = --', 'extrinsic component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic endosome membrane, pval = --', 'postsynaptic recycling endosome membrane, pval = --', 'integral component of endosome membrane, pval = --', 'positive regulation of intralumenal vesicle formation, pval = --', 'negative regulation of intralumenal vesicle formation, pval = --', 'regulation of intralumenal vesicle formation, pval = --', 'Golgi medial cisterna, pval = --', 'Golgi trans cisterna, pval = --', 'Golgi cis cisterna, pval = --', 'COPII vesicles tethering complex, pval = --', 'Golgi transport vesicle coating, pval = --', 'COPII-coated vesicle budding, pval = --', 'Golgi vesicle bud deformation and release, pval = --', 'COPI-coated vesicle budding, pval = --', 'intrinsic component of Golgi trans cisterna membrane, pval = --', 'intrinsic component of Golgi medial cisterna membrane, pval = --', 'intrinsic component of Golgi cis cisterna membrane, pval = --', 'integral component of Golgi membrane, pval = 0.68643', 'Golgi medial cisterna membrane, pval = --', 'Golgi cis cisterna membrane, pval = --', 'Golgi trans cisterna membrane, pval = --', 'neuronal dense core vesicle lumen, pval = --', 'neuronal dense core vesicle membrane, pval = --', 'extrinsic component of dense core granule membrane, pval = --', 'intrinsic component of dense core granule membrane, pval = --', 'protein C inhibitor-acrosin complex, pval = --', 'outer acrosomal membrane, pval = --', 'inner acrosomal membrane, pval = --', 'epidermal lamellar body membrane, pval = --', 'alveolar lamellar body membrane, pval = --', 'synaptic vesicle lumen, pval = --', 'synaptic vesicle, resting pool, pval = --', 'synaptic vesicle, readily releasable pool, pval = --', 'synaptic vesicle, recycling pool, pval = --', 'synaptic vesicle membrane, pval = --', 'cytoplasmic side of trans-Golgi network transport vesicle membrane, pval = --', 'lumenal side of trans-Golgi network transport vesicle membrane, pval = --', 'COPII vesicle coat, pval = --', 'anchored component of spine apparatus membrane, pval = --', 'integral component of spine apparatus membrane, pval = --', 'serine C-palmitoyltransferase complex, pval = --', 'integral component of endoplasmic reticulum membrane, pval = 0.43974', 'integral component of lumenal side of endoplasmic reticulum membrane, pval = --', 'integral component of cytoplasmic side of endoplasmic reticulum membrane, pval = --', 'integral component of nuclear outer membrane, pval = --', 'extrinsic component of postsynaptic density membrane, pval = 0.43974', 'extrinsic component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'rod teleodendria, pval = --', 'cone teleodendria, pval = --', 'kinociliary basal body, pval = 0.43974', 'dendrite terminus, pval = 0.68643', 'sensory dendrite, pval = --', 'dendritic microtubule, pval = --', 'dendrite cytoplasm, pval = 0.82468', 'dendritic transport, pval = --', 'proximal dendrite, pval = --', 'primary dendrite, pval = --', 'distal dendrite, pval = --', 'apical dendrite, pval = 0.68643', 'CA3 pyramidal cell dendrite, pval = --', 'basal dendrite, pval = 0.43974', 'dendritic shaft, pval = 0.19304', 'dendritic spine, pval = 0.08911', 'dendritic branch, pval = --', 'giant axon, pval = --', 'axon microtubule bundle, pval = --', 'parallel fiber, pval = --', 'axonal transport, pval = --', 'axon cytoplasm, pval = 0.40992', 'distal axon, pval = 0.49131', 'C-fiber, pval = --', 'axonal dopamine secretion, pval = --', 'axon hillock, pval = --', 'axonal spine, pval = --', 'main axon, pval = 0.40992', 'axon collateral, pval = --', 'pericellular basket, pval = --', 'pinceau fiber, pval = --', 'dense core granule cytoskeletal transport, pval = --', 'cerebellar mossy fiber, pval = --', 'climbing fiber, pval = --', 'dentate gyrus mossy fiber, pval = --', 'photoreceptor outer segment, pval = 0.59248', 'photoreceptor connecting cilium, pval = --', 'axon terminus, pval = --', 'dendritic branch point, pval = --', 'slow axonal transport, pval = --', 'axo-dendritic protein transport, pval = --', 'neuronal ribonucleoprotein granule, pval = --', 'actin filament bundle of stereocilium, pval = --', 'stereocilium bundle, pval = 0.43974', 'stereocilium tip, pval = 0.43974', 'stereocilium shaft, pval = --', 'stereocilium base, pval = --', 'spine mat, pval = --', 'somatic spine, pval = --', 'right posteriolateral basal body, pval = --', 'right anterior basal body, pval = --', 'left posteriolateral basal body, pval = --', 'right ventral basal body, pval = --', 'left ventral basal body, pval = --', 'right caudal basal body, pval = --', 'left caudal basal body, pval = --', 'left anterior basal body, pval = --', 'mastigoneme, pval = --', '9+2 motile cilium, pval = 0.43974', '9+0 motile cilium, pval = --', '9+2 non-motile cilium, pval = 0.43974', '9+0 non-motile cilium, pval = 0.59248', 'intraciliary transport involved in cilium assembly, pval = --', 'intraciliary retrograde transport, pval = --', 'intraciliary anterograde transport, pval = --', 'axoneme, pval = 0.72894', 'ciliary pocket collar, pval = --', 'central cylinder, pval = --', 'NPHP complex, pval = --', 'MKS complex, pval = --', 'Y-shaped link, pval = --', 'astrocyte end-foot, pval = --', 'mating projection tip polarisome, pval = --', 'actin fusion focus, pval = --', 'brush border, pval = --', 'microvillar actin bundle, pval = --', 'actin filament bundle of filopodium, pval = --', 'filopodium tip, pval = --', 'G protein-coupled neurotransmitter receptor activity involved in regulation of postsynaptic membrane potential, pval = --', 'acetylcholine receptor activity, pval = --', 'regulation of postsynaptic neurotransmitter receptor activity, pval = --', 'neurotransmitter receptor activity involved in regulation of postsynaptic membrane potential, pval = --', 'neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration, pval = --', 'dopamine neurotransmitter receptor activity, pval = --', 'extrinsic component of external side of mycolate outer membrane, pval = --', 'integral component of mycolate outer membrane, pval = --', 'Bam protein complex, pval = --', 'anchored component of periplasmic side of cell outer membrane, pval = --']
level 6: ['amylin receptor complex 1, pval = --', 'amylin receptor complex 3, pval = --', 'amylin receptor complex 2, pval = --', 'integral component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'integral component of postsynaptic density membrane, pval = --', 'anchored component of postsynaptic specialization membrane of symmetric synapse, pval = --', 'anchored component of postsynaptic density membrane, pval = --', 'Sec61 translocon complex, pval = --', 'Ssh1 translocon complex, pval = --', 'Sec62/Sec63 complex, pval = --', 'sub-surface cisterna, pval = --', 'integral component of nuclear inner membrane, pval = --', 'intrinsic component of postsynaptic early endosome membrane, pval = --', 'extrinsic component of postsynaptic early endosome membrane, pval = --', 'multivesicular body, internal vesicle membrane, pval = --', 'multivesicular body, internal vesicle lumen, pval = 0.43974', 'multivesicular body HOPS complex, pval = --', 'anchored component of presynaptic endosome membrane, pval = --', 'integral component of presynaptic endosome membrane, pval = --', 'intrinsic component of postsynaptic recycling endosome membrane, pval = --', 'anchored component of postsynaptic endosome membrane, pval = --', 'integral component of postsynaptic endosome membrane, pval = --', 'extrinsic component of postsynaptic recycling endosome membrane, pval = --', 'alpha-1,6-mannosyltransferase complex, pval = --', 'Golgi membrane priming complex assembly, pval = --', 'clathrin coating of Golgi vesicle, pval = --', 'COPI coating of Golgi vesicle, pval = --', 'Golgi membrane coat protein complex assembly, pval = --', 'COPII-coated vesicle cargo loading, pval = --', 'COPII vesicle coating, pval = --', 'integral component of Golgi trans cisterna membrane, pval = --', 'integral component of Golgi medial cisterna membrane, pval = --', 'integral component of Golgi cis cisterna membrane, pval = --', 'intrinsic component of neuronal dense core vesicle membrane, pval = --', 'extrinsic component of neuronal dense core vesicle membrane, pval = --', 'clathrin coat of synaptic vesicle, pval = --', 'extrinsic component of synaptic vesicle membrane, pval = --', 'intrinsic component of synaptic vesicle membrane, pval = --', 'SPOTS complex, pval = --', 'homodimeric serine palmitoyltransferase complex, pval = --', 'ER membrane protein complex, pval = --', 'dendriole, pval = --', 'dendritic growth cone, pval = 0.68643', 'dendritic tuft, pval = --', 'dendritic knob, pval = --', 'dendritic transport of mitochondrion, pval = --', 'retrograde dendritic transport, pval = --', 'anterograde dendritic transport, pval = --', 'dendritic transport of ribonucleoprotein complex, pval = --', 'dendritic spine cytoplasm, pval = --', 'apical proximal dendrite, pval = --', 'basal proximal dendrite, pval = --', 'apical distal dendrite, pval = --', 'basal distal dendrite, pval = --', 'cytoskeleton of dendritic spine, pval = --', 'dendritic spine head, pval = --', 'dendritic spine neck, pval = --', 'thorny excrescence, pval = --', 'axonal transport of mitochondrion, pval = --', 'axonal transport of messenger ribonucleoprotein complex, pval = --', 'anterograde axonal transport, pval = --', 'retrograde axonal transport, pval = --', 'growth cone, pval = 0.05169', 'dopamine secretion, neurotransmission, pval = --', 'internode region of axon, pval = --', 'node of Ranvier, pval = --', 'paranode region of axon, pval = --', 'juxtaparanode region of axon, pval = --', 'varicosity, pval = --', 'axon initial segment, pval = --', 'Schaffer axon collateral, pval = --', 'recurrent axon collateral, pval = --', 'retrograde neuronal dense core vesicle transport, pval = --', 'anterograde neuronal dense core vesicle transport, pval = --', 'hippocampal mossy fiber, pval = --', 'rod photoreceptor outer segment, pval = --', 'cone photoreceptor outer segment, pval = --', 'photoreceptor disc membrane, pval = --', 'photoreceptor proximal connecting cilium, pval = --', 'photoreceptor distal connecting cilium, pval = --', 'calyx of Held, pval = --', 'cone cell pedicle, pval = --', 'vestibular calyx terminal, pval = --', 'Herring body, pval = --', 'terminal bouton, pval = --', 'retrograde axonal protein transport, pval = --', 'anterograde axonal protein transport, pval = --', 'positive regulation of axo-dendritic protein transport, pval = --', 'regulation of axo-dendritic protein transport, pval = --', 'negative regulation of axo-dendritic protein transport, pval = --', 'anterograde dendritic transport of neurotransmitter receptor complex, pval = --', 'stereocilia coupling link, pval = --', 'right middle basal body pair, pval = --', 'right lateral basal body pair, pval = --', 'left middle basal body pair, pval = --', 'left lateral basal body pair, pval = --', 'paraflagellar rod, pval = --', 'transverse flagellum, pval = --', 'longitudinal flagellum, pval = --', 'sperm flagellum, pval = 0.43974', 'right ventral flagellum, pval = --', 'right posteriolateral flagellum, pval = --', 'left ventral flagellum, pval = --', 'left anterior flagellum, pval = --', 'right anterior flagellum, pval = --', 'left posteriolateral flagellum, pval = --', 'right caudal flagellum, pval = --', 'left caudal flagellum, pval = --', 'cilium assembly, pval = --', 'positive regulation of intraciliary retrograde transport, pval = --', 'negative regulation of intraciliary retrograde transport, pval = --', 'regulation of intraciliary retrograde transport, pval = --', 'regulation of intraciliary anterograde transport, pval = --', 'negative regulation of intraciliary anterograde transport, pval = --', 'positive regulation of intraciliary anterograde transport, pval = --', 'axonemal heterotrimeric kinesin-II complex, pval = --', 'axonemal nexin link, pval = --', 'axonemal central apparatus, pval = --', 'axonemal dynein complex, pval = --', 'axonemal microtubule, pval = --', 'distal portion of axoneme, pval = --', 'proximal portion of axoneme, pval = --', 'radial spoke, pval = --', 'axonemal microtubule depolymerization, pval = --', 'satellite fibril, pval = --', 'axonemal basal plate, pval = --', 'axonemal outer doublet, pval = --', 'glial limiting end-foot, pval = --', 'G protein-coupled acetylcholine receptor activity, pval = --', 'acetylcholine receptor regulator activity, pval = --', 'acetylcholine-induced gastric acid secretion, pval = --', 'regulation of acetylcholine-gated cation channel activity, pval = --', 'transmitter-gated ion channel activity involved in regulation of postsynaptic membrane potential, pval = --', 'G protein-coupled neurotransmitter receptor activity involved in regulation of postsynaptic cytosolic calcium ion concentration, pval = --', 'dopamine neurotransmitter receptor activity, coupled via Gs, pval = --', 'dopamine neurotransmitter receptor activity, coupled via Gi/Go, pval = --']
level 7: ['nuclear membrane complex Bqt3-Bqt4, pval = --', 'anchored component of postsynaptic early endosome membrane, pval = --', 'integral component of postsynaptic early endosome membrane, pval = --', 'anchored component of postsynaptic recycling endosome membrane, pval = --', 'integral component of postsynaptic recycling endosome membrane, pval = --', 'clathrin coating of Golgi vesicle, plasma membrane to endosome targeting, pval = --', 'clathrin coating of Golgi vesicle, trans-Golgi to endosome targeting, pval = --', 'COPI coating of Golgi vesicle, inter-Golgi cisterna, pval = --', 'COPI coating of Golgi vesicle, cis-Golgi to rough ER, pval = --', 'negative regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of cargo loading into COPII-coated vesicle, pval = --', 'regulation of COPII vesicle coating, pval = --', 'anchored component of neuronal dense core vesicle membrane, pval = --', 'integral component of neuronal dense core vesicle membrane, pval = --', 'anchored component of synaptic vesicle membrane, pval = --', 'integral component of synaptic vesicle membrane, pval = --', 'leading edge of dendritic growth cone, pval = --', 'retrograde dendritic transport of mitochondrion, pval = --', 'anterograde dendritic transport of mitochondrion, pval = --', 'anterograde dendritic transport of messenger ribonucleoprotein complex, pval = --', 'dendritic transport of messenger ribonucleoprotein complex, pval = --', 'actin cytoskeleton of dendritic spine, pval = --', 'dendritic spine origin, pval = --', 'hippocampal mossy fiber to CA3 synapse, pval = --', 'retrograde axonal transport of mitochondrion, pval = --', 'anterograde axonal transport of mitochondrion, pval = --', 'anterograde axonal transport of messenger ribonucleoprotein complex, pval = --', 'anterograde synaptic vesicle transport, pval = --', 'regulation of retrograde axon cargo transport, pval = --', 'positive regulation of retrograde axon cargo transport, pval = --', 'negative regulation of retrograde axon cargo transport, pval = --', 'retrograde synaptic vesicle transport, pval = --', 'axonal growth cone, pval = 0.40992', 'peripheral region of growth cone, pval = --', 'central region of growth cone, pval = --', 'positive regulation of retrograde dense core granule transport, pval = --', 'negative regulation of retrograde dense core granule transport, pval = --', 'regulation of retrograde dense core granule transport, pval = --', 'regulation of anterograde dense core granule transport, pval = --', 'positive regulation of anterograde dense core granule transport, pval = --', 'negative regulation of anterograde dense core granule transport, pval = --', 'hippocampal mossy fiber expansion, pval = --', 'rod photoreceptor disc lumen, pval = --', 'rod photoreceptor disc membrane, pval = --', 'type II terminal bouton, pval = --', 'type I terminal bouton, pval = --', 'photoreceptor cell terminal bouton, pval = --', 'rod bipolar cell terminal bouton, pval = --', 'multiple synapse bouton, pval = --', 'type III terminal bouton, pval = --', 'S bouton, pval = --', 'F bouton, pval = --', 'C bouton, pval = --', 'anterograde axonal transport of neurotransmitter receptor complex, pval = --', 'stereocilia tip link, pval = --', 'stereocilia ankle link, pval = --', 'right tetrad, pval = --', 'left tetrad, pval = --', 'outer dense fiber, pval = --', 'sperm fibrous sheath, pval = --', 'sperm connecting piece, pval = --', 'sperm annulus, pval = --', 'sperm principal piece, pval = --', 'sperm end piece, pval = --', 'sperm midpiece, pval = --', 'sperm mitochondrial sheath, pval = --', 'cytosolic ciliogenesis, pval = --', 'ciliary transition zone assembly, pval = --', 'non-motile cilium assembly, pval = --', 'positive regulation of cilium assembly, pval = --', 'negative regulation of cilium assembly, pval = --', 'regulation of cilium assembly, pval = --', 'motile cilium assembly, pval = --', 'vesicle targeting, trans-Golgi to periciliary membrane compartment, pval = --', 'ciliary basal body-plasma membrane docking, pval = --', 'axoneme assembly, pval = --', 'axonemal central pair projection, pval = --', 'axonemal central bridge, pval = --', 'axonemal central pair, pval = --', 'inner dynein arm, pval = --', 'outer dynein arm, pval = --', 'A axonemal microtubule, pval = --', 'B axonemal microtubule, pval = --', 'C1 axonemal microtubule, pval = --', 'C2 axonemal microtubule, pval = --', 'radial spoke stalk, pval = --', 'radial spoke head, pval = --', 'negative regulation of axonemal microtubule depolymerization, pval = --', 'acetylcholine receptor activator activity, pval = --', 'acetylcholine receptor inhibitor activity, pval = --', 'negative regulation of acetylcholine-gated cation channel activity, pval = --', 'acetylcholine-gated cation-selective channel activity, pval = --']
level 8: ['regulation of anterograde axonal transport of mitochondrion, pval = --', 'negative regulation of anterograde axonal transport of mitochondrion, pval = --', 'positive regulation of anterograde axonal transport of mitochondrion, pval = --', 'regulation of anterograde synaptic vesicle transport, pval = --', 'negative regulation of anterograde synaptic vesicle transport, pval = --', 'positive regulation of anterograde synaptic vesicle transport, pval = --', 'leading edge of axonal growth cone, pval = --', 'growth cone leading edge, pval = --', 'type Ib terminal bouton, pval = --', 'type Is terminal bouton, pval = --', 'subsynaptic reticulum, pval = --', 'multiple synapse bouton, contacting multiple dendrites, pval = --', 'multiple synapse bouton, contacting single dendrite, pval = --', 'stereocilia tip-link density, pval = --', 'stereocilia ankle link complex, pval = --', 'regulation of non-motile cilium assembly, pval = --', 'negative regulation of non-motile cilium assembly, pval = --', 'positive regulation of non-motile cilium assembly, pval = --', 'positive regulation of motile cilium assembly, pval = --', 'negative regulation of motile cilium assembly, pval = --', 'regulation of motile cilium assembly, pval = --', 'axonemal dynein complex assembly, pval = --', 'radial spoke assembly, pval = --', 'axonemal central apparatus assembly, pval = --', 'sperm axoneme assembly, pval = --']
level 9: ['upper tip-link density, pval = --', 'lower tip-link density, pval = --', 'inner dynein arm assembly, pval = --', 'outer dynein arm assembly, pval = --']" id="plasma_membrane" init-pval="0.56389" min-pval="0.00058" min-pval-children="0.00058" title="Plasma membrane">
<g>
<path d="M360,483.4c-23.3,0-45.4-3.5-66.8-7c-7.1-1.1-13.8-2.2-20.6-3.2c-10.4-1.5-20.7-2.7-30.6-4
			c-34-4.3-66.2-8.3-99-18.4c-11.6-3.6-23.4-6.6-34.9-9.5c-31.1-7.8-57.9-14.6-71-30.1c-2.1-2.5-4.1-4.8-6-7
			c-15.1-17.4-23.4-27-24.5-60.2c-1.2-37.3,7.2-55.6,26.2-93.4c4.5-8.8,8.9-16.8,13.5-24.5c0.7-1.2,1.6-1.4,2.3-1.4c1.3,0,2.6,1,4,3
			c0.5,0.8,1.4,1.2,2.3,1.2c1.1,0,2.3-0.6,3.1-1.7c1-1.4,1.3-2.8,0.9-4c-0.3-0.9-1.1-1.5-2.1-1.8c-2.6-0.7-4.2-1.3-4.7-2.7
			c-0.5-1.2,0.3-2.3,0.5-2.7c4.2-6.3,8.3-12.1,12.6-18.3c6.7-9.5,13.6-19.2,21.8-32.4c7.4-11.8,12.5-20.7,17.1-28.5
			c8.7-15.1,15.6-27,32.9-49.9c26.2-34.7,45.5-44.2,74.1-50.4c8.5-1.8,17.6-4.4,27.3-7.1c22.4-6.3,47.9-13.4,77.1-15
			c12.8-0.7,24-0.7,36.7,0c1.7,0.1,2.7,0.5,3.3,1.2c0.7,0.9,0.5,1.8,0.4,2.7c0,0.2-0.1,0.4-0.1,0.7c-0.2,1.8,0.1,3,2.6,3.5
			c2.3,0.4,4,0.1,4.8-0.7c0.5-0.6,0.6-1.2,0.6-1.6c-0.1-1.7,0.2-2.9,1-3.7c0.4-0.4,1.3-1,2.6-0.9c3.9,0.3,7.8,0.7,12,1.1
			c15.1,1.4,32.3,3,56,4.1c87.9,3.9,119.3,19.3,124.5,60.8c5.2,41.9,6.9,51.4,15.5,76.7c3,8.7,4.8,14.8,6.2,20.2
			c0.4,1.7,0,2.8-0.5,3.4c-0.6,0.8-1.7,1.3-2.9,1.3c-0.5,0-1.1-0.1-1.7-0.3c-1.6-0.5-2.9-0.3-3.7,0.7c-0.9,1-1.2,2.7-0.9,4.7
			c0.2,1.7,1.3,4.4,3.2,4.4c0.8,0,1.9-0.5,3-1.4c1-0.9,2-1.3,3-1.3c1.8,0,3.1,1.4,3.4,3.9c1.1,8.4,1.8,18,2.7,33.9
			c0.8,13.8,4.1,25.6,7.7,38.1c5.8,20.3,11.8,41.4,8.5,74.7c-0.4,4-0.7,7.8-1.1,11.4c-4.3,46.4-6.2,67.5-46.5,79.6
			c-11,3.3-21.5,6.1-31.1,8.7c-1.6,0.4-3-0.2-3.8-1.6c-0.5-0.9-0.7-2.1-0.7-3.6c0-1.4-1.6-2.1-3.2-2.1c-0.7,0-1.3,0.1-2.1,0.3
			c-1.6,0.4-2.7,1.3-3.2,2.5c-0.3,0.7-0.2,1.5,0.2,2c1.1,1.6,1.4,3,1,4.2c-0.4,1-1.3,1.8-2.6,2.1c-24.4,6.5-49.3,13.6-84.1,27.1
			C405.9,479.5,384.2,483.4,360,483.4z M48.7,227.2c0,0-0.1,0.1-0.2,0.3c-4.6,7.6-9,15.5-13.4,24.3c-18.9,37.4-27.1,55.5-26,92.3
			c1,32.4,8.7,41.2,23.9,58.7c1.9,2.2,3.9,4.5,6,7c12.6,14.9,40.4,21.9,69.8,29.3c11.5,2.9,23.3,5.9,35,9.5
			c32.6,10,64.6,14.1,98.6,18.3c9.9,1.2,20.2,2.5,30.6,4c6.7,0.9,13.5,2,20.6,3.2c21.3,3.4,43.3,6.9,66.4,6.9
			c23.9,0,45.3-3.9,65.6-11.8c34.9-13.6,60.8-20.9,84.3-27.2c0.2-0.1,0.8-0.3,0.9-0.6c0.2-0.4-0.1-1.2-0.7-2c-0.8-1.2-1-2.7-0.4-4.2
			c0.7-1.9,2.5-3.4,4.8-4c0.9-0.2,1.8-0.4,2.7-0.4c3.3,0,5.6,1.8,5.6,4.5c0,1.4,0.2,2.1,0.4,2.5c0.2,0.4,0.4,0.5,0.6,0.5
			c0.1,0,0.3,0,0.4-0.1c9.6-2.6,20-5.4,31-8.7c38.7-11.6,40.6-32.2,44.8-77.5c0.3-3.6,0.7-7.5,1.1-11.4c3.2-32.9-2.7-53.7-8.5-73.8
			c-3.6-12.6-7-24.6-7.8-38.6c-0.9-15.9-1.6-25.4-2.7-33.7c-0.1-0.9-0.4-1.8-1-1.8c-0.2,0-0.7,0.1-1.4,0.7c-1.6,1.3-3.1,2-4.5,2
			c-3.3,0-5.2-3.4-5.6-6.5c-0.4-2.8,0.1-5.1,1.5-6.6c1.5-1.7,3.8-2.2,6.3-1.4c0.8,0.3,1.6,0.2,1.9-0.2c0.2-0.2,0.2-0.7,0.1-1.3
			c-1.3-5.3-3.2-11.3-6.1-20c-8.7-25.5-10.3-35-15.6-77.1c-2.6-20.9-11.9-34.1-30.3-42.9c-18.5-8.9-46.8-13.8-91.9-15.8
			c-23.8-1.1-41-2.7-56.1-4.1c-4.1-0.4-8.1-0.8-12-1.1c-0.5,0-0.6,0.1-0.7,0.1c-0.1,0.1-0.4,0.6-0.3,1.9c0.1,1.3-0.4,2.5-1.2,3.4
			c-0.7,0.8-2.1,1.7-4.6,1.7c-0.7,0-1.5-0.1-2.4-0.2c-3.4-0.6-5-2.7-4.6-6.1c0-0.3,0.1-0.5,0.1-0.8c0.1-0.3,0.1-0.7,0.1-0.9
			c0,0-0.3-0.2-1.6-0.2c-12.7-0.7-23.8-0.7-36.5,0c-29,1.6-54.3,8.7-76.6,14.9c-9.7,2.7-18.8,5.3-27.4,7.1
			c-28,6.1-46.9,15.4-72.7,49.5c-17.2,22.7-24.1,34.6-32.8,49.7c-4.5,7.9-9.7,16.8-17.1,28.6c-8.3,13.2-15.2,23-21.9,32.5
			c-4.3,6.1-8.4,11.9-12.6,18.2c-0.3,0.4-0.3,0.5-0.3,0.5c0.1,0,0.5,0.5,3,1.1c1.8,0.5,3.2,1.7,3.7,3.3c0.7,1.9,0.2,4.2-1.3,6.2
			c-1.3,1.7-3.1,2.7-5,2.7c-1.8,0-3.3-0.8-4.3-2.3C49.6,227.6,48.9,227.2,48.7,227.2z" style="fill:#FAE4EC;"></path>
</g>
<path d="M360,483.7
		c-23.3,0-45.5-3.5-66.8-7c-7.1-1.1-13.8-2.2-20.6-3.2c-10.4-1.5-20.7-2.7-30.6-4c-34.1-4.3-66.2-8.3-99-18.4
		c-11.6-3.6-23.4-6.6-34.9-9.5C77,433.8,50.1,427,37,411.5c-2.1-2.4-4-4.7-5.9-6.8l-0.1-0.1c-15.1-17.5-23.5-27.1-24.5-60.4
		c-1.2-37.4,7.2-55.7,26.3-93.6c4.5-8.8,8.9-16.9,13.5-24.5c0.8-1.3,1.8-1.6,2.5-1.6c1.4,0,2.7,1,4.2,3.1c0.5,0.7,1.2,1.1,2.1,1.1
		c1.1,0,2.1-0.6,2.9-1.6c0.9-1.3,1.3-2.6,0.9-3.7c-0.3-0.8-1-1.4-1.9-1.6c-2.7-0.7-4.3-1.4-4.8-2.8c-0.5-1.3,0.3-2.5,0.5-2.9
		c4.2-6.3,8.3-12.1,12.6-18.3c6.7-9.5,13.6-19.2,21.8-32.4c7.4-11.8,12.5-20.7,17-28.5c8.7-15.1,15.6-27.1,32.9-49.9
		c26.3-34.7,45.6-44.3,74.3-50.5c8.5-1.8,17.6-4.4,27.2-7.1c22.4-6.3,47.9-13.4,77.2-15C322,14.1,328,14,333.8,14
		c6,0,12.2,0.2,18.7,0.6c1.8,0.1,2.9,0.5,3.5,1.3c0.7,0.9,0.6,2,0.4,2.9c0,0.2-0.1,0.4-0.1,0.7c-0.2,1.8,0.1,2.8,2.4,3.2
		c2.1,0.4,3.8,0.1,4.6-0.7c0.5-0.5,0.5-1.1,0.5-1.4c-0.1-1.8,0.3-3.1,1.1-3.9c0.4-0.4,1.3-1.1,2.8-0.9c3.8,0.3,7.8,0.7,11.8,1.1
		l0.1,0c15.1,1.4,32.3,3,56,4.1c88,3.9,119.5,19.3,124.7,61c5.2,41.9,6.9,51.4,15.5,76.6c3,8.8,4.8,14.8,6.2,20.2
		c0.5,1.8,0,2.9-0.5,3.6c-0.7,0.9-1.8,1.4-3.1,1.4c-0.6,0-1.2-0.1-1.7-0.3c-1.5-0.5-2.7-0.3-3.5,0.6c-0.8,0.9-1.1,2.5-0.8,4.5
		c0.2,1.7,1.3,4.2,3,4.2c0.8,0,1.8-0.5,2.8-1.4c1.1-0.9,2.1-1.4,3.2-1.4c1.9,0,3.3,1.5,3.6,4.1c1.1,8.4,1.8,18,2.7,33.9
		c0.8,13.8,4.1,25.6,7.7,38.1c5.8,20.4,11.8,41.4,8.6,74.8c-0.4,3.8-0.7,7.4-1,10.9l0,0.5c-4.3,46.5-6.2,67.7-46.7,79.8
		c-11,3.3-21.5,6.1-31.1,8.7c-1.7,0.5-3.3-0.2-4.1-1.7c-0.5-0.9-0.7-2.2-0.7-3.7c0-1.2-1.5-1.8-3-1.8c-0.6,0-1.3,0.1-2,0.3
		c-1.5,0.4-2.6,1.2-3,2.4c-0.2,0.7-0.2,1.3,0.1,1.8c1.1,1.7,1.5,3.1,1,4.4c-0.4,1.1-1.4,1.9-2.7,2.3c-24.4,6.5-49.2,13.5-84,27.1
		C406,479.7,384.2,483.7,360,483.7z M48.7,225.1c-0.6,0-1.4,0.2-2.1,1.3C42,234,37.6,242,33.2,250.8c-19,37.8-27.4,56-26.2,93.3
		c1,33.1,9.3,42.7,24.4,60.1l0.1,0.1c1.9,2.2,3.8,4.4,5.9,6.8c13,15.4,39.8,22.2,70.9,30c11.4,2.9,23.3,5.9,34.9,9.5
		c32.8,10.1,64.9,14.1,99,18.4c9.9,1.2,20.2,2.5,30.6,4c6.7,0.9,13.4,2,20.6,3.2c21.4,3.4,43.4,7,66.8,7c24.1,0,45.8-3.9,66.4-11.9
		c34.8-13.6,59.7-20.6,84.1-27.2c1.2-0.3,2-1,2.4-2c0.4-1.1,0.1-2.5-0.9-4c-0.4-0.6-0.5-1.4-0.2-2.2c0.5-1.3,1.7-2.2,3.3-2.7
		c0.7-0.2,1.5-0.3,2.1-0.3c1.7,0,3.5,0.7,3.5,2.3c0,1.5,0.2,2.7,0.7,3.5c0.7,1.3,2,1.9,3.5,1.5c9.6-2.6,20.1-5.4,31.1-8.7
		c40.1-12,42.1-33.1,46.3-79.4l0-0.5c0.3-3.5,0.7-7.1,1-10.9c3.2-33.3-2.8-54.3-8.5-74.6c-3.6-12.5-6.9-24.3-7.7-38.2
		c-0.9-15.9-1.6-25.5-2.7-33.9c-0.3-2.3-1.5-3.6-3.1-3.6c-0.9,0-1.9,0.4-2.8,1.2c-1.1,1-2.2,1.5-3.1,1.5c-2.1,0-3.3-2.8-3.5-4.6
		c-0.3-2.1,0.1-3.9,1-4.9c0.9-1,2.3-1.3,4-0.7c0.5,0.2,1.1,0.3,1.6,0.3c1.1,0,2.1-0.4,2.7-1.2c0.4-0.6,0.9-1.6,0.4-3.2
		c-1.4-5.4-3.2-11.4-6.2-20.1c-8.6-25.3-10.2-34.7-15.5-76.7c-5.2-41.4-36.5-56.7-124.2-60.6c-23.7-1.1-40.9-2.7-56-4.1l-0.1,0
		c-4.1-0.4-8-0.8-11.8-1.1c-1.3-0.1-2,0.4-2.4,0.8c-0.7,0.7-1,1.9-0.9,3.5c0,0.4,0,1.2-0.6,1.8c-0.9,0.9-2.7,1.2-5,0.8
		c-2.7-0.5-3.1-1.8-2.8-3.7c0-0.2,0.1-0.5,0.1-0.7c0.1-0.9,0.3-1.8-0.3-2.5c-0.5-0.7-1.5-1-3.1-1.1c-6.5-0.4-12.6-0.6-18.7-0.6
		c-5.8,0-11.7,0.2-18,0.5c-29.2,1.6-54.6,8.7-77.1,15c-9.6,2.7-18.8,5.2-27.3,7.1c-28.5,6.2-47.8,15.6-74,50.3
		c-17.3,22.8-24.2,34.8-32.9,49.9c-4.5,7.8-9.7,16.7-17.1,28.5c-8.2,13.2-15.2,23-21.9,32.4c-4.3,6.1-8.4,11.9-12.6,18.3
		c-0.2,0.3-0.9,1.4-0.5,2.4c0.5,1.2,2,1.8,4.5,2.5c1.1,0.3,1.9,1,2.3,2c0.4,1.2,0.1,2.8-1,4.2c-0.9,1.2-2.1,1.8-3.3,1.8
		c-1,0-2-0.5-2.5-1.3C51.2,226,49.9,225.1,48.7,225.1z M360,481.3c-23.2,0-45.2-3.5-66.5-6.9c-7.1-1.1-13.8-2.2-20.6-3.2
		c-10.4-1.5-20.6-2.7-30.6-4l-0.1,0c-33.9-4.3-66-8.3-98.6-18.3c-11.7-3.6-23.5-6.6-35-9.5c-29.4-7.4-57.2-14.5-69.9-29.4
		c-2.1-2.5-4.2-4.9-6-7c-15.3-17.6-22.9-26.4-23.9-58.9c-1.2-36.8,7.1-54.9,26-92.4c4.4-8.8,8.8-16.7,13.4-24.3
		c0.2-0.3,0.2-0.4,0.4-0.4c0.3,0,1.1,0.5,2.2,2.1c0.9,1.4,2.4,2.1,4.1,2.1c1.8,0,3.6-0.9,4.8-2.6c1.4-1.9,1.9-4.1,1.2-5.9
		c-0.5-1.6-1.8-2.7-3.6-3.2c-1.6-0.4-2.7-0.8-3.1-1.2l-0.1-0.1l0-0.1c0-0.1,0-0.3,0.3-0.7c4.1-6.3,8.2-12.1,12.6-18.2
		c6.7-9.5,13.7-19.3,21.9-32.5c7.4-11.8,12.5-20.7,17.1-28.6c8.7-15.1,15.6-26.9,32.8-49.7c25.8-34.1,44.8-43.5,72.9-49.6
		c8.6-1.9,17.7-4.4,27.4-7.1c22.3-6.2,47.6-13.3,76.7-14.9c6.3-0.4,12.2-0.5,17.9-0.5c6,0,12.1,0.2,18.6,0.6c0.6,0,1.1,0.1,1.5,0.2
		l0.3,0.2c0,0.2,0,0.5-0.1,1c0,0.3-0.1,0.5-0.1,0.8c-0.4,3.3,1,5.2,4.4,5.8c0.8,0.2,1.6,0.2,2.3,0.2c2.4,0,3.7-0.9,4.4-1.6
		c0.8-0.8,1.2-2,1.2-3.2c-0.1-1.3,0.2-1.9,0.4-2.1c0.1-0.1,0.3-0.3,0.9-0.2c3.9,0.3,7.8,0.7,11.9,1.1l0.1,0
		c15.2,1.4,32.3,3,56.1,4.1c45.1,2,73.5,6.9,92,15.8C546,48,555.4,61.3,558,82.2c5.3,42.1,6.9,51.7,15.6,77.1c3,8.7,4.8,14.7,6.1,20
		c0.2,0.7,0.1,1.2-0.1,1.5c-0.2,0.3-0.7,0.4-1.2,0.4c-0.3,0-0.7-0.1-1-0.2c-0.7-0.2-1.5-0.4-2.2-0.4c-1.5,0-2.9,0.6-3.8,1.7
		c-1.3,1.5-1.8,3.7-1.4,6.4c0.4,3,2.3,6.3,5.4,6.3c1.4,0,2.8-0.7,4.4-1.9c0.8-0.7,1.4-0.8,1.6-0.8c0.6,0,1.1,0.7,1.2,2
		c1.1,8.3,1.8,17.9,2.7,33.8c0.8,14,4.2,26,7.8,38.6c5.7,20.1,11.7,41,8.5,73.9c-0.4,3.8-0.7,7.5-1,11l0,0.5
		c-4.2,45.4-6.1,66.1-45,77.7c-11.1,3.3-21.5,6.1-31.1,8.7c-0.2,0-0.3,0.1-0.5,0.1c-0.3,0-0.6-0.1-0.9-0.6c-0.2-0.4-0.5-1.2-0.5-2.6
		c0-2.5-2.2-4.2-5.4-4.2c-0.8,0-1.7,0.1-2.6,0.4c-2.3,0.6-4,2-4.6,3.8c-0.5,1.4-0.4,2.8,0.4,4c0.6,0.9,0.9,1.7,0.7,2.2
		c-0.2,0.5-0.8,0.7-1.1,0.8c-23.5,6.3-49.4,13.6-84.3,27.2C405.4,477.4,383.9,481.3,360,481.3z M48.8,227.5c0,0-0.1,0.1-0.1,0.1
		c-4.6,7.6-9,15.5-13.4,24.3c-18.8,37.4-27.1,55.5-26,92.2c1,32.3,8.6,41.1,23.8,58.6c1.8,2.1,3.9,4.5,6,7
		c12.5,14.8,40.3,21.8,69.6,29.2c11.5,2.9,23.3,5.9,35,9.5c32.6,10,64.6,14.1,98.5,18.3l0.1,0c9.9,1.2,20.2,2.5,30.6,4
		c6.8,1,13.5,2,20.6,3.2c21.3,3.4,43.3,6.9,66.4,6.9c23.8,0,45.3-3.8,65.5-11.8c34.9-13.6,60.8-20.9,84.3-27.2
		c0.3-0.1,0.7-0.3,0.8-0.5c0.1-0.3,0-0.9-0.7-1.8c-0.9-1.3-1-2.9-0.5-4.4c0.7-2,2.6-3.5,5-4.1c0.9-0.2,1.9-0.4,2.7-0.4
		c3.5,0,5.9,1.9,5.9,4.7c0,1.3,0.2,2,0.4,2.4c0.2,0.3,0.3,0.3,0.4,0.3c0.1,0,0.2,0,0.3-0.1c9.6-2.6,20-5.4,31-8.7
		c38.6-11.5,40.5-32.1,44.6-77.3l0-0.5c0.3-3.5,0.7-7.2,1-11c3.2-32.8-2.7-53.6-8.5-73.7c-3.6-12.7-7-24.6-7.8-38.7
		c-0.9-15.8-1.6-25.4-2.7-33.7c-0.1-0.9-0.4-1.6-0.8-1.6c-0.1,0-0.5,0-1.3,0.7c-1.6,1.4-3.2,2.1-4.7,2.1c-3.4,0-5.4-3.5-5.9-6.7
		c-0.4-2.9,0.2-5.2,1.5-6.8c1.5-1.7,4-2.3,6.5-1.4c0.7,0.2,1.4,0.1,1.6-0.1c0.1-0.2,0.1-0.6,0-1.1c-1.3-5.3-3.2-11.3-6.1-20
		c-8.7-25.5-10.3-35-15.6-77.2c-2.6-20.8-11.9-33.9-30.2-42.8c-18.4-8.9-46.7-13.8-91.8-15.8c-23.8-1.1-41-2.7-56.1-4.1l-0.1,0
		c-4.1-0.4-8.1-0.8-11.9-1.1c-0.3,0-0.5,0.1-0.5,0.1c0,0-0.3,0.4-0.3,1.7c0.1,1.3-0.4,2.6-1.3,3.5c-0.8,0.8-2.2,1.7-4.8,1.7
		c-0.7,0-1.6-0.1-2.4-0.2c-3.6-0.7-5.2-2.8-4.8-6.4c0-0.3,0.1-0.5,0.1-0.8c0-0.2,0.1-0.5,0.1-0.6c-0.3-0.1-0.8-0.2-1.3-0.2
		c-6.5-0.4-12.5-0.6-18.5-0.6c-5.8,0-11.6,0.2-17.9,0.5c-29,1.6-54.2,8.7-76.5,14.9c-9.7,2.7-18.8,5.3-27.4,7.1
		c-27.9,6-46.8,15.3-72.6,49.4c-17.2,22.7-24,34.6-32.7,49.6c-4.6,7.9-9.7,16.8-17.1,28.6c-8.3,13.2-15.2,23-21.9,32.5
		c-4.3,6.1-8.4,11.9-12.6,18.2c-0.1,0.1-0.1,0.2-0.2,0.3c0.3,0.2,1,0.5,2.8,1c1.9,0.5,3.3,1.8,3.9,3.5c0.7,2,0.2,4.3-1.3,6.4
		c-1.3,1.8-3.2,2.8-5.2,2.8c-1.8,0-3.5-0.9-4.5-2.4C49.6,228,49,227.6,48.8,227.5z" style="fill:#C02E66;stroke:#000000;stroke-width:0.15;stroke-linejoin:round;stroke-miterlimit:10;"></path>
</g>
</svg>
<div>
<button ng-click="allComponents = {}" type="submit">clear selection</button>
</div>
<strong>Single selection:</strong>
<div>title: {{cellComponent.title}}</div>
<div>min-pval (displayed): {{cellComponent.pval_1}}</div>
<div>init-pval: {{cellComponent.pval_2}}</div>
<div>min-pval-descendants: {{cellComponent.pval_3}}</div>
<div>descendants: </div>{{cellComponent.descendants}}
  <br/>
<div class="row">
<strong>Multi selection:</strong>
<span ng-repeat="(k, v) in allComponents">
<span ng-show="v.selected">{{v.title}}, </span>
</span>
</div>
</div>